
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Human-vaginalMISAG/">
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.4.3, mkdocs-material-8.5.10">
    
    
      
        <title>Class: Human-vaginalMISAG - MIxS</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.975780f9.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.2505c338.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="None" data-md-color-accent="None">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#class-human-vaginalmisag" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="MIxS" class="md-header__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MIxS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Class: Human-vaginalMISAG
            
          </span>
        </div>
      </div>
    </div>
    
    
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="MIxS" class="md-nav__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    MIxS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Reference
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Introduction
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Introduction" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Introduction
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/citation/" class="md-nav__link">
        Cite
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/contents/" class="md-nav__link">
        Contents
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#inheritance" class="md-nav__link">
    Inheritance
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#identifier-and-mapping-information" class="md-nav__link">
    Identifier and Mapping Information
  </a>
  
    <nav class="md-nav" aria-label="Identifier and Mapping Information">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#schema-source" class="md-nav__link">
    Schema Source
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mappings" class="md-nav__link">
    Mappings
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#linkml-source" class="md-nav__link">
    LinkML Source
  </a>
  
    <nav class="md-nav" aria-label="LinkML Source">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#direct" class="md-nav__link">
    Direct
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#induced" class="md-nav__link">
    Induced
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Human-vaginalMISAG.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="class-human-vaginalmisag">Class: Human-vaginalMISAG</h1>
<p><em>Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package human-vaginal</em></p>
<p>URI: <a href="https://w3id.org/mixs/Human-vaginalMISAG">MIXS:Human-vaginalMISAG</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Human-vaginal/">Human-vaginal</a><ul>
<li><strong>human-vaginal MISAG</strong> [ <a href="../MISAG/">MISAG</a>]</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0001107/">samp_name</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>A local identifier or name that for the material sample used for extracting n...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0001320/">samp_taxon_id</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>NCBI taxon id of the sample</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000092/">project_name</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Name of the project within which the sequencing was organized</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000008/">experimental_factor</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Experimental factors are essentially the variable aspects of an experiment de...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000009/">lat_lon</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by latitude and longitude</td>
<td><a href="../Human-vaginal/">Human-vaginal</a>, <a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000010/">geo_loc_name</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by the country or sea name f...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a>, <a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000011/">collection_date</a></td>
<td>1..1 <br/> <a href="../Date/">Date</a></td>
<td>The time of sampling, either as an instance (single point in time) or interva...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a>, <a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0001321/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../NegContTypeEnum/">NegContTypeEnum</a></td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0001322/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The substance, mixture, product, or apparatus used to verify that a process w...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000012/">env_broad_scale</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
<td><a href="../Human-vaginal/">Human-vaginal</a>, <a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000013/">env_local_scale</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Report the entity or entities which are in the sample or specimen’s local vic...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a>, <a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000014/">env_medium</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a>, <a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000025/">ref_biomaterial</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Primary publication if isolated before genome publication; otherwise, primary...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000026/">source_mat_id</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A unique identifier assigned to a material sample (as defined by http://rs</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000015/">rel_to_oxygen</a></td>
<td>0..1 <br/> <a href="../RelToOxygenEnum/">RelToOxygenEnum</a></td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic ...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000002/">samp_collec_device</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The device used to collect an environmental sample</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0001225/">samp_collec_method</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The method employed for collecting the sample</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000016/">samp_mat_process</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A brief description of any processing applied to the sample during or after r...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000017/">size_frac</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Filtering pore size used in sample preparation</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000001/">samp_size</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample coll...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000111/">samp_vol_we_dna_ext</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extractio...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000037/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000038/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000039/">lib_size</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones in the library prepared for the project</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000040/">lib_reads_seqd</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones sequenced from the library</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000041/">lib_layout</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../LibLayoutEnum/">LibLayoutEnum</a></td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000042/">lib_vector</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000043/">lib_screen</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Specific enrichment or screening methods applied before and/or after creating...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000047/">mid</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to spe...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000048/">adapters</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Adapters provide priming sequences for both amplification and sequencing of t...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000050/">seq_meth</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Sequencing machine used</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000053/">tax_ident</a></td>
<td>1..1 <br/> <a href="../TaxIdentEnum/">TaxIdentEnum</a></td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000056/">assembly_qual</a></td>
<td>1..1 <br/> <a href="../AssemblyQualEnum/">AssemblyQualEnum</a></td>
<td>The assembly quality category is based on sets of criteria outlined for each ...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000057/">assembly_name</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Name/version of the assembly provided by the submitter that is used in the ge...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000058/">assembly_software</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000059/">annot</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool used for annotation, or for cases where annotation was provided by a com...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000060/">number_contig</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a giv...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000061/">feat_pred</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000062/">ref_db</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>List of database(s) used for ORF annotation, along with version number and re...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000063/">sim_search_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000064/">tax_class</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Method used for taxonomic classification, along with reference database used,...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000065/">x_16s_recover</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Can a 16S gene be recovered from the submitted SAG or MAG?</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000066/">x_16s_recover_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for 16S rRNA gene extraction</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000067/">trnas</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>The total number of tRNAs identified from the SAG or MAG</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000068/">trna_ext_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for tRNA identification</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000069/">compl_score</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Completeness score is typically based on either the fraction of markers found...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000070/">compl_software</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for completion estimate, i</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000071/">compl_appr</a></td>
<td>0..1 <br/> <a href="../ComplApprEnum/">ComplApprEnum</a></td>
<td>The approach used to determine the completeness of a given genomic assembly, ...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000072/">contam_score</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>The contamination score is based on the fraction of single-copy genes that ar...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000005/">contam_screen_input</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The type of sequence data used as input</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000073/">contam_screen_param</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specific parameters used in the decontamination sofware, such as reference da...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000074/">decontam_software</a></td>
<td>0..1 <br/> <a href="../DecontamSoftwareEnum/">DecontamSoftwareEnum</a></td>
<td>Tool(s) used in contamination screening</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000075/">sort_tech</a></td>
<td>1..1 <br/> <a href="../SortTechEnum/">SortTechEnum</a></td>
<td>Method used to sort/isolate cells or particles of interest</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000076/">single_cell_lysis_appr</a></td>
<td>1..1 <br/> <a href="../SingleCellLysisApprEnum/">SingleCellLysisApprEnum</a></td>
<td>Method used to free DNA from interior of the cell(s) or particle(s)</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000054/">single_cell_lysis_prot</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name of the kit or standard protocol used for cell(s) or particle(s) lysis</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000055/">wga_amp_appr</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Method used to amplify genomic DNA in preparation for sequencing</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000006/">wga_amp_kit</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Kit used to amplify genomic DNA in preparation for sequencing</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000091/">associated resource</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A related resource that is referenced, cited, or otherwise associated to the ...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000090/">sop</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, ...</td>
<td><a href="../MISAG/">MISAG</a></td>
</tr>
<tr>
<td><a href="../0000018/">depth</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The vertical distance below local surface, e</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000094/">alt</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Altitude is a term used to identify heights of objects such as airplanes, spa...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000093/">elev</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Elevation of the sampling site is its height above a fixed reference point, m...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000113/">temp</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Temperature of the sample at the time of sampling</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000965/">menarche</a></td>
<td>0..1 <br/> <a href="../Date/">Date</a></td>
<td>Date of most recent menstruation</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000285/">sexual_act</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Current sexual partner and frequency of sex</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000966/">pregnancy</a></td>
<td>0..1 <br/> <a href="../Date/">Date</a></td>
<td>Date due of pregnancy</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000967/">douche</a></td>
<td>0..1 <br/> <a href="../Date/">Date</a></td>
<td>Date of most recent douche</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000286/">birth_control</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of birth control medication used</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000968/">menopause</a></td>
<td>0..1 <br/> <a href="../Date/">Date</a></td>
<td>Date of onset of menopause</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000969/">hrt</a></td>
<td>0..1 <br/> <a href="../Date/">Date</a></td>
<td>Whether subject had hormone replacement theraphy, and if yes start date</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000287/">hysterectomy</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of whether hysterectomy was performed</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000288/">gynecologic_disord</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>History of gynecological disorders; can include multiple disorders</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000289/">urogenit_disord</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>History of urogenital disorders, can include multiple disorders</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000861/">host_subject_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A unique identifier by which each subject can be referred to, de-identified</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000255/">host_age</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Age of host at the time of sampling; relevant scale depends on species and st...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000811/">host_sex</a></td>
<td>0..1 <br/> <a href="../HostSexEnum/">HostSexEnum</a></td>
<td>Gender or physical sex of the host</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000884/">ihmc_medication_code</a></td>
<td>0..* <br/> <a href="../Integer/">Integer</a></td>
<td>Can include multiple medication codes</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000751/">chem_administration</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>List of chemical compounds administered to the host or site where sampling oc...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000867/">host_body_site</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name of body site where the sample was obtained from, such as a specific orga...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000888/">host_body_product</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Substance produced by the body, e</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000263/">host_tot_mass</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000264/">host_height</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The height of subject</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000869/">host_diet</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Type of diet depending on the host, for animals omnivore, herbivore etc</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000870/">host_last_meal</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Content of last meal and time since feeding; can include multiple values</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000872/">host_family_relation</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Familial relationships to other hosts in the same study; can include multiple...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000365/">host_genotype</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Observed genotype</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000874/">host_phenotype</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Phenotype of human or other host</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000274/">host_body_temp</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Core body temperature of the host when sample was collected</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000317/">host_body_mass_index</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Body mass index, calculated as weight/(height)squared</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000895/">ethnicity</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>A category of people who identify with each other, usually on the basis of pr...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000896/">host_occupation</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Most frequent job performed by subject</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000897/">medic_hist_perform</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Whether full medical history was collected</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000333/">host_pulse</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Resting pulse, measured as beats per minute</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000754/">perturbation</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Type of perturbation, e</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000109/">samp_salinity</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Salinity is the total concentration of all dissolved salts in a liquid or sol...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000183/">salinity</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The total concentration of all dissolved salts in a liquid or solid sample</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000753/">oxy_stat_samp</a></td>
<td>0..1 <br/> <a href="../OxyStatSampEnum/">OxyStatSampEnum</a></td>
<td>Oxygenation status of sample</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000103/">organism_count</a></td>
<td>0..* <br/> <a href="../OrganismCountEnum/">OrganismCountEnum</a></td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or ...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000110/">samp_store_temp</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Temperature at which sample was stored, e</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000755/">samp_store_loc</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000116/">samp_store_dur</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Duration for which the sample was stored</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0001298/">host_symbiont</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>The taxonomic name of the organism(s) found living in mutualistic, commensali...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000752/">misc_param</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Any other measurement performed or parameter collected, that is not listed he...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: http://w3id.org/mixs</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>MIXS:Human-vaginalMISAG</td>
</tr>
<tr>
<td>native</td>
<td>MIXS:Human-vaginalMISAG</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<h3 id="direct">Direct</h3>
<details>

<pre><code class="language-yaml">name: human-vaginal MISAG
description: Combinatorial checklist Minimum Information About a Single Amplified
  Genome with environmental package human-vaginal
from_schema: http://w3id.org/mixs
is_a: human-vaginal
mixins:
- MISAG

</code></pre>

</details>

<h3 id="induced">Induced</h3>
<details>

<pre><code class="language-yaml">name: human-vaginal MISAG
description: Combinatorial checklist Minimum Information About a Single Amplified
  Genome with environmental package human-vaginal
from_schema: http://w3id.org/mixs
is_a: human-vaginal
mixins:
- MISAG
attributes:
  samp_name:
    name: samp_name
    description: A local identifier or name that for the material sample used for
      extracting nucleic acids, and subsequent sequencing. It can refer either to
      the original material collected or to any derived sub-samples. It can have any
      format, but we suggest that you make it concise, unique and consistent within
      your lab, and as informative as possible. INSDC requires every sample name from
      a single Submitter to be unique. Use of a globally unique identifier for the
      field source_mat_id is recommended in addition to sample_name.
    title: sample name
    from_schema: http://w3id.org/mixs
    is_a: investigation field
    string_serialization: '{text}'
    slot_uri: MIXS:0001107
    multivalued: false
    alias: samp_name
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  samp_taxon_id:
    name: samp_taxon_id
    description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa
      sample. Use 'synthetic metagenome’ for mock community/positive controls, or
      'blank sample' for negative controls.
    title: Taxonomy ID of DNA sample
    from_schema: http://w3id.org/mixs
    is_a: investigation field
    string_serialization: '{text} [NCBI:txid]'
    slot_uri: MIXS:0001320
    multivalued: false
    alias: samp_taxon_id
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  project_name:
    name: project_name
    description: Name of the project within which the sequencing was organized
    title: project name
    from_schema: http://w3id.org/mixs
    is_a: investigation field
    string_serialization: '{text}'
    slot_uri: MIXS:0000092
    multivalued: false
    alias: project_name
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  experimental_factor:
    name: experimental_factor
    description: Experimental factors are essentially the variable aspects of an experiment
      design which can be used to describe an experiment, or set of experiments, in
      an increasingly detailed manner. This field accepts ontology terms from Experimental
      Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For
      a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO;
      for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI
    title: experimental factor
    from_schema: http://w3id.org/mixs
    is_a: investigation field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000008
    multivalued: false
    alias: experimental_factor
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  lat_lon:
    name: lat_lon
    description: The geographical origin of the sample as defined by latitude and
      longitude. The values should be reported in decimal degrees and in WGS84 system
    title: geographic location (latitude and longitude)
    from_schema: http://w3id.org/mixs
    is_a: environment field
    string_serialization: '{float} {float}'
    slot_uri: MIXS:0000009
    multivalued: false
    alias: lat_lon
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  geo_loc_name:
    name: geo_loc_name
    description: The geographical origin of the sample as defined by the country or
      sea name followed by specific region name. Country or sea names should be chosen
      from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology
      (http://purl.bioontology.org/ontology/GAZ)
    title: geographic location (country and/or sea,region)
    from_schema: http://w3id.org/mixs
    is_a: environment field
    string_serialization: '{term}: {term}, {text}'
    slot_uri: MIXS:0000010
    multivalued: false
    alias: geo_loc_name
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  collection_date:
    name: collection_date
    description: 'The time of sampling, either as an instance (single point in time)
      or interval. In case no exact time is available, the date/time can be right
      truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10;
      2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant'
    title: collection date
    from_schema: http://w3id.org/mixs
    is_a: environment field
    slot_uri: MIXS:0000011
    multivalued: false
    alias: collection_date
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: date
    required: true
  neg_cont_type:
    name: neg_cont_type
    description: The substance or equipment used as a negative control in an investigation
    title: negative control type
    from_schema: http://w3id.org/mixs
    is_a: investigation field
    slot_uri: MIXS:0001321
    multivalued: false
    alias: neg_cont_type
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: neg_cont_type_enum
    recommended: true
  pos_cont_type:
    name: pos_cont_type
    description: The substance, mixture, product, or apparatus used to verify that
      a process which is part of an investigation delivers a true positive.
    title: positive control type
    from_schema: http://w3id.org/mixs
    is_a: investigation field
    string_serialization: '{term} or {text}'
    slot_uri: MIXS:0001322
    multivalued: false
    alias: pos_cont_type
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  env_broad_scale:
    name: env_broad_scale
    description: 'Report the major environmental system the sample or specimen came
      from. The system(s) identified should have a coarse spatial grain, to provide
      the general environmental context of where the sampling was done (e.g. in the
      desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428.
      EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: broad-scale environmental context
    from_schema: http://w3id.org/mixs
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000012
    multivalued: false
    alias: env_broad_scale
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  env_local_scale:
    name: env_local_scale
    description: 'Report the entity or entities which are in the sample or specimen’s
      local vicinity and which you believe have significant causal influences on your
      sample or specimen. We recommend using EnvO terms which are of smaller spatial
      grain than your entry for env_broad_scale. Terms, such as anatomical sites,
      from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON)
      are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.'
    title: local environmental context
    from_schema: http://w3id.org/mixs
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000013
    multivalued: false
    alias: env_local_scale
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  env_medium:
    name: env_medium
    description: 'Report the environmental material(s) immediately surrounding the
      sample or specimen at the time of sampling. We recommend using subclasses of
      ''environmental material'' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO
      documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
      . Terms from other OBO ontologies are permissible as long as they reference
      mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
      (e.g. a tree, a leaf, a table top).'
    title: environmental medium
    from_schema: http://w3id.org/mixs
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000014
    multivalued: false
    alias: env_medium
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  ref_biomaterial:
    name: ref_biomaterial
    description: Primary publication if isolated before genome publication; otherwise,
      primary genome report.
    title: reference for biomaterial
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000025
    multivalued: false
    alias: ref_biomaterial
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  source_mat_id:
    name: source_mat_id
    description: A unique identifier assigned to a material sample (as defined by
      http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular
      digital record of a material sample) used for extracting nucleic acids, and
      subsequent sequencing. The identifier can refer either to the original material
      collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher,
      /bio_material, or /culture_collection may or may not share the same value as
      the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id
      may both contain 'UAM:Herps:14' , referring to both the specimen voucher and
      sampled tissue with the same identifier. However, the /culture_collection qualifier
      may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id
      would refer to an identifier from some derived culture from which the nucleic
      acids were extracted (e.g. xatc123 or ark:/2154/R2).
    title: source material identifiers
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000026
    multivalued: false
    alias: source_mat_id
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  rel_to_oxygen:
    name: rel_to_oxygen
    description: Is this organism an aerobe, anaerobe? Please note that aerobic and
      anaerobic are valid descriptors for microbial environments
    title: relationship to oxygen
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000015
    multivalued: false
    alias: rel_to_oxygen
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS bacteria
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    range: rel_to_oxygen_enum
    required: false
  samp_collec_device:
    name: samp_collec_device
    description: The device used to collect an environmental sample. This field accepts
      terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO).
      This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).
    title: sample collection device
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000002
    multivalued: false
    alias: samp_collec_device
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  samp_collec_method:
    name: samp_collec_method
    description: The method employed for collecting the sample.
    title: sample collection method
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    string_serialization: '{PMID}|{DOI}|{URL}|{text}'
    slot_uri: MIXS:0001225
    multivalued: false
    alias: samp_collec_method
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  samp_mat_process:
    name: samp_mat_process
    description: A brief description of any processing applied to the sample during
      or after retrieving the sample from environment, or a link to the relevant protocol(s)
      performed.
    title: sample material processing
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000016
    multivalued: false
    alias: samp_mat_process
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  size_frac:
    name: size_frac
    description: Filtering pore size used in sample preparation
    title: size fraction selected
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    string_serialization: '{float}-{float} {unit}'
    slot_uri: MIXS:0000017
    multivalued: false
    alias: size_frac
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  samp_size:
    name: samp_size
    description: The total amount or size (volume (ml), mass (g) or area (m2) ) of
      sample collected.
    title: amount or size of sample collected
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000001
    multivalued: false
    alias: samp_size
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: quantity value
    recommended: true
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    description: 'Volume (ml) or mass (g) of total collected sample processed for
      DNA extraction. Note: total sample collected should be entered under the term
      Sample Size (MIXS:0000001).'
    title: sample volume or weight for DNA extraction
    from_schema: http://w3id.org/mixs
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000111
    multivalued: false
    alias: samp_vol_we_dna_ext
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: quantity value
    required: false
  nucl_acid_ext:
    name: nucl_acid_ext
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the material separation to recover
      the nucleic acid fraction from a sample
    title: nucleic acid extraction
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000037
    multivalued: false
    alias: nucl_acid_ext
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  nucl_acid_amp:
    name: nucl_acid_amp
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the enzymatic amplification (PCR,
      TMA, NASBA) of specific nucleic acids
    title: nucleic acid amplification
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000038
    multivalued: false
    alias: nucl_acid_amp
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  lib_size:
    name: lib_size
    description: Total number of clones in the library prepared for the project
    title: library size
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000039
    multivalued: false
    alias: lib_size
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
    recommended: true
  lib_reads_seqd:
    name: lib_reads_seqd
    description: Total number of clones sequenced from the library
    title: library reads sequenced
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000040
    multivalued: false
    alias: lib_reads_seqd
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
    recommended: true
  lib_layout:
    name: lib_layout
    description: Specify whether to expect single, paired, or other configuration
      of reads
    title: library layout
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000041
    multivalued: false
    alias: lib_layout
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: lib_layout_enum
    recommended: true
  lib_vector:
    name: lib_vector
    description: Cloning vector type(s) used in construction of libraries
    title: library vector
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000042
    multivalued: false
    alias: lib_vector
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  lib_screen:
    name: lib_screen
    description: Specific enrichment or screening methods applied before and/or after
      creating libraries
    title: library screening strategy
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000043
    multivalued: false
    alias: lib_screen
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  mid:
    name: mid
    description: Molecular barcodes, called Multiplex Identifiers (MIDs), that are
      used to specifically tag unique samples in a sequencing run. Sequence should
      be reported in uppercase letters
    title: multiplex identifiers
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{dna}'
    slot_uri: MIXS:0000047
    multivalued: false
    alias: mid
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  adapters:
    name: adapters
    description: Adapters provide priming sequences for both amplification and sequencing
      of the sample-library fragments. Both adapters should be reported; in uppercase
      letters
    title: adapters
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{dna};{dna}'
    slot_uri: MIXS:0000048
    multivalued: false
    alias: adapters
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  seq_meth:
    name: seq_meth
    description: Sequencing machine used. Where possible the term should be taken
      from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).
    title: sequencing method
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000050
    multivalued: false
    alias: seq_meth
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  tax_ident:
    name: tax_ident
    description: The phylogenetic marker(s) used to assign an organism name to the
      SAG or MAG
    title: taxonomic identity marker
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000053
    multivalued: false
    alias: tax_ident
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: tax_ident_enum
    required: true
  assembly_qual:
    name: assembly_qual
    description: 'The assembly quality category is based on sets of criteria outlined
      for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated,
      contiguous sequence per replicon without gaps or ambiguities with a consensus
      error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments
      where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes
      and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no
      review of assembly other than reporting of standard assembly statistics. Low
      Quality Draft:Many fragments with little to no review of assembly other than
      reporting of standard assembly statistics. Assembly statistics include, but
      are not limited to total assembly size, number of contigs, contig N50/L50, and
      maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence
      per replicon without gaps or ambiguities, with extensive manual review and editing
      to annotate putative gene functions and transcriptional units. High-quality
      draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome
      or replicon sequence or predicted complete. Genome fragment(s): One or multiple
      fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for
      which no genome size could be estimated'
    title: assembly quality
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000056
    multivalued: false
    alias: assembly_qual
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: assembly_qual_enum
    required: true
  assembly_name:
    name: assembly_name
    description: Name/version of the assembly provided by the submitter that is used
      in the genome browsers and in the community
    title: assembly name
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{text} {text}'
    slot_uri: MIXS:0000057
    multivalued: false
    alias: assembly_name
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  assembly_software:
    name: assembly_software
    description: Tool(s) used for assembly, including version number and parameters
    title: assembly software
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000058
    multivalued: false
    alias: assembly_software
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  annot:
    name: annot
    description: Tool used for annotation, or for cases where annotation was provided
      by a community jamboree or model organism database rather than by a specific
      submitter
    title: annotation
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000059
    multivalued: false
    alias: annot
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  number_contig:
    name: number_contig
    description: Total number of contigs in the cleaned/submitted assembly that makes
      up a given genome, SAG, MAG, or UViG
    title: number of contigs
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000060
    multivalued: false
    alias: number_contig
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
    required: false
  feat_pred:
    name: feat_pred
    description: Method used to predict UViGs features such as ORFs, integration site,
      etc.
    title: feature prediction
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000061
    multivalued: false
    alias: feat_pred
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  ref_db:
    name: ref_db
    description: List of database(s) used for ORF annotation, along with version number
      and reference to website or publication
    title: reference database(s)
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{database};{version};{reference}'
    slot_uri: MIXS:0000062
    multivalued: false
    alias: ref_db
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  sim_search_meth:
    name: sim_search_meth
    description: Tool used to compare ORFs with database, along with version and cutoffs
      used
    title: similarity search method
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000063
    multivalued: false
    alias: sim_search_meth
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  tax_class:
    name: tax_class
    description: Method used for taxonomic classification, along with reference database
      used, classification rank, and thresholds used to classify new genomes
    title: taxonomic classification
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000064
    multivalued: false
    alias: tax_class
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  x_16s_recover:
    name: x_16s_recover
    description: Can a 16S gene be recovered from the submitted SAG or MAG?
    title: 16S recovered
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{boolean}'
    slot_uri: MIXS:0000065
    multivalued: false
    alias: x_16s_recover
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
    required: false
  x_16s_recover_software:
    name: x_16s_recover_software
    description: Tools used for 16S rRNA gene extraction
    title: 16S recovery software
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000066
    multivalued: false
    alias: x_16s_recover_software
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
    required: false
  trnas:
    name: trnas
    description: The total number of tRNAs identified from the SAG or MAG
    title: number of standard tRNAs extracted
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000067
    multivalued: false
    alias: trnas
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
    required: false
  trna_ext_software:
    name: trna_ext_software
    description: Tools used for tRNA identification
    title: tRNA extraction software
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000068
    multivalued: false
    alias: trna_ext_software
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  compl_score:
    name: compl_score
    description: 'Completeness score is typically based on either the fraction of
      markers found as compared to a database or the percent of a genome found as
      compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium
      Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated
      completeness scores'
    title: completeness score
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000069
    multivalued: false
    alias: compl_score
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
    pattern: ^(high|med|low);(0|([0-9]{1,2})|100)%$
  compl_software:
    name: compl_software
    description: Tools used for completion estimate, i.e. checkm, anvi'o, busco
    title: completeness software
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{software};{version}'
    slot_uri: MIXS:0000070
    multivalued: false
    alias: compl_software
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  compl_appr:
    name: compl_appr
    description: The approach used to determine the completeness of a given genomic
      assembly, which would typically make use of a set of conserved marker genes
      or a closely related reference genome. For UViG completeness, include reference
      genome or group used, and contig feature suggesting a complete genome
    title: completeness approach
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000071
    multivalued: false
    alias: compl_appr
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    - MIUVIG
    range: compl_appr_enum
    required: false
  contam_score:
    name: contam_score
    description: 'The contamination score is based on the fraction of single-copy
      genes that are observed more than once in a query genome. The following scores
      are acceptable for; High Quality Draft: &lt; 5%, Medium Quality Draft: &lt; 10%, Low
      Quality Draft: &lt; 10%. Contamination must be below 5% for a SAG or MAG to be
      deposited into any of the public databases'
    title: contamination score
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{float} percentage'
    slot_uri: MIXS:0000072
    multivalued: false
    alias: contam_score
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
    required: true
  contam_screen_input:
    name: contam_screen_input
    description: The type of sequence data used as input
    title: contamination screening input
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '[reads| contigs]'
    slot_uri: MIXS:0000005
    multivalued: false
    alias: contam_screen_input
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
    required: false
  contam_screen_param:
    name: contam_screen_param
    description: Specific parameters used in the decontamination sofware, such as
      reference database, coverage, and kmers. Combinations of these parameters may
      also be used, i.e. kmer and coverage, or reference database and kmer
    title: contamination screening parameters
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000073
    multivalued: false
    alias: contam_screen_param
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
    required: false
    pattern: ^(ref db|kmer|coverage|combination);.+
  decontam_software:
    name: decontam_software
    description: Tool(s) used in contamination screening
    title: decontamination software
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000074
    multivalued: false
    alias: decontam_software
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: decontam_software_enum
    required: false
  sort_tech:
    name: sort_tech
    description: Method used to sort/isolate cells or particles of interest
    title: sorting technology
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000075
    multivalued: false
    alias: sort_tech
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: sort_tech_enum
    required: true
  single_cell_lysis_appr:
    name: single_cell_lysis_appr
    description: Method used to free DNA from interior of the cell(s) or particle(s)
    title: single cell or viral particle lysis approach
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    slot_uri: MIXS:0000076
    multivalued: false
    alias: single_cell_lysis_appr
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: single_cell_lysis_appr_enum
    required: true
  single_cell_lysis_prot:
    name: single_cell_lysis_prot
    description: Name of the kit or standard protocol used for cell(s) or particle(s)
      lysis
    title: single cell or viral particle lysis kit protocol
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000054
    multivalued: false
    alias: single_cell_lysis_prot
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: string
    required: false
  wga_amp_appr:
    name: wga_amp_appr
    description: Method used to amplify genomic DNA in preparation for sequencing
    title: WGA amplification approach
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '[pcr based|mda based]'
    slot_uri: MIXS:0000055
    multivalued: false
    alias: wga_amp_appr
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: string
    required: true
  wga_amp_kit:
    name: wga_amp_kit
    description: Kit used to amplify genomic DNA in preparation for sequencing
    title: WGA amplification kit
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000006
    multivalued: false
    alias: wga_amp_kit
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: string
    required: false
  associated resource:
    name: associated resource
    description: A related resource that is referenced, cited, or otherwise associated
      to the sequence.
    title: relevant electronic resources
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{PMID} | {DOI} | {URL}'
    slot_uri: MIXS:0000091
    multivalued: false
    alias: associated_resource
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  sop:
    name: sop
    description: Standard operating procedures used in assembly and/or annotation
      of genomes, metagenomes or environmental sequences
    title: relevant standard operating procedures
    from_schema: http://w3id.org/mixs
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000090
    multivalued: false
    alias: sop
    owner: human-vaginal MISAG
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  depth:
    name: depth
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: The vertical distance below local surface, e.g. for sediment or soil
      samples depth is measured from sediment or soil surface, respectively. Depth
      can be reported as an interval for subsurface samples.
    title: depth
    examples:
    - value: 10 meter
    from_schema: http://w3id.org/mixs
    aliases:
    - depth
    is_a: environment field
    slot_uri: MIXS:0000018
    multivalued: false
    alias: depth
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  alt:
    name: alt
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: Altitude is a term used to identify heights of objects such as airplanes,
      space shuttles, rockets, atmospheric balloons and heights of places such as
      atmospheric layers and clouds. It is used to measure the height of an object
      which is above the earth's surface. In this context, the altitude measurement
      is the vertical distance between the earth's surface above sea level and the
      sampled position in the air
    title: altitude
    examples:
    - value: 100 meter
    from_schema: http://w3id.org/mixs
    aliases:
    - altitude
    is_a: environment field
    slot_uri: MIXS:0000094
    multivalued: false
    alias: alt
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  elev:
    name: elev
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: Elevation of the sampling site is its height above a fixed reference
      point, most commonly the mean sea level. Elevation is mainly used when referring
      to points on the earth's surface, while altitude is used for points above the
      surface, such as an aircraft in flight or a spacecraft in orbit.
    title: elevation
    examples:
    - value: 100 meter
    from_schema: http://w3id.org/mixs
    aliases:
    - elevation
    is_a: environment field
    slot_uri: MIXS:0000093
    multivalued: false
    alias: elev
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  temp:
    name: temp
    description: Temperature of the sample at the time of sampling.
    title: temperature
    from_schema: http://w3id.org/mixs
    is_a: environment field
    slot_uri: MIXS:0000113
    multivalued: false
    alias: temp
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
    required: false
  menarche:
    name: menarche
    description: Date of most recent menstruation
    title: menarche
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000965
    multivalued: false
    alias: menarche
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: date
    required: false
  sexual_act:
    name: sexual_act
    description: Current sexual partner and frequency of sex
    title: sexual activity
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000285
    multivalued: false
    alias: sexual_act
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: string
    required: false
  pregnancy:
    name: pregnancy
    description: Date due of pregnancy
    title: pregnancy
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000966
    multivalued: false
    alias: pregnancy
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: date
    required: false
  douche:
    name: douche
    description: Date of most recent douche
    title: douche
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000967
    multivalued: false
    alias: douche
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: date
    required: false
  birth_control:
    name: birth_control
    description: Specification of birth control medication used
    title: birth control
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000286
    multivalued: false
    alias: birth_control
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: string
    required: false
  menopause:
    name: menopause
    description: Date of onset of menopause
    title: menopause
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000968
    multivalued: false
    alias: menopause
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: date
    required: false
  hrt:
    name: hrt
    description: Whether subject had hormone replacement theraphy, and if yes start
      date
    title: HRT
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000969
    multivalued: false
    alias: hrt
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: date
    required: false
  hysterectomy:
    name: hysterectomy
    description: Specification of whether hysterectomy was performed
    title: hysterectomy
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{boolean}'
    slot_uri: MIXS:0000287
    multivalued: false
    alias: hysterectomy
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: string
    required: false
  gynecologic_disord:
    name: gynecologic_disord
    description: History of gynecological disorders; can include multiple disorders.
      The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      female reproductive system disease (https://disease-ontology.org/?id=DOID:229).
    title: gynecological disorder
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000288
    multivalued: true
    alias: gynecologic_disord
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: string
    required: false
  urogenit_disord:
    name: urogenit_disord
    description: History of urogenital disorders, can include multiple disorders.
      The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      reproductive system disease (https://disease-ontology.org/?id=DOID:15) or urinary
      system disease (https://disease-ontology.org/?id=DOID:18).
    title: urogenital disorder
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000289
    multivalued: true
    alias: urogenit_disord
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    range: string
    required: false
  host_subject_id:
    name: host_subject_id
    description: A unique identifier by which each subject can be referred to, de-identified.
    title: host subject id
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000861
    multivalued: false
    alias: host_subject_id
    owner: human-vaginal MISAG
    domain_of:
    - symbiont-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_age:
    name: host_age
    description: Age of host at the time of sampling; relevant scale depends on species
      and study, e.g. Could be seconds for amoebae or centuries for trees
    title: host age
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000255
    multivalued: false
    alias: host_age
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: quantity value
    required: false
  host_sex:
    name: host_sex
    description: Gender or physical sex of the host.
    title: host sex
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000811
    multivalued: false
    alias: host_sex
    owner: human-vaginal MISAG
    domain_of:
    - symbiont-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: host_sex_enum
    required: false
  ihmc_medication_code:
    name: ihmc_medication_code
    description: Can include multiple medication codes
    title: IHMC medication code
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000884
    multivalued: true
    alias: ihmc_medication_code
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: integer
    required: false
  chem_administration:
    name: chem_administration
    description: List of chemical compounds administered to the host or site where
      sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can
      include multiple compounds. For chemical entities of biological interest ontology
      (chebi) (v 163), http://purl.bioontology.org/ontology/chebi
    title: chemical administration
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{termLabel} {[termID]};{timestamp}'
    slot_uri: MIXS:0000751
    multivalued: true
    alias: chem_administration
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false
  host_body_site:
    name: host_body_site
    description: Name of body site where the sample was obtained from, such as a specific
      organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology
      (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms,
      please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON
    title: host body site
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000867
    multivalued: false
    alias: host_body_site
    owner: human-vaginal MISAG
    domain_of:
    - symbiont-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_body_product:
    name: host_body_product
    description: Substance produced by the body, e.g. Stool, mucus, where the sample
      was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy
      ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma
      or https://www.ebi.ac.uk/ols/ontologies/uberon
    title: host body product
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000888
    multivalued: false
    alias: host_body_product
    owner: human-vaginal MISAG
    domain_of:
    - symbiont-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_tot_mass:
    name: host_tot_mass
    description: Total mass of the host at collection, the unit depends on host
    title: host total mass
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000263
    multivalued: false
    alias: host_tot_mass
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: quantity value
    required: false
  host_height:
    name: host_height
    description: The height of subject
    title: host height
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000264
    multivalued: false
    alias: host_height
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: quantity value
    required: false
  host_diet:
    name: host_diet
    description: Type of diet depending on the host, for animals omnivore, herbivore
      etc., for humans high-fat, meditteranean etc.; can include multiple diet types
    title: host diet
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000869
    multivalued: true
    alias: host_diet
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_last_meal:
    name: host_last_meal
    description: Content of last meal and time since feeding; can include multiple
      values
    title: host last meal
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text};{duration}'
    slot_uri: MIXS:0000870
    multivalued: true
    alias: host_last_meal
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_family_relation:
    name: host_family_relation
    description: Familial relationships to other hosts in the same study; can include
      multiple relationships
    title: host family relationship
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text};{text}'
    slot_uri: MIXS:0000872
    multivalued: true
    alias: host_family_relation
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_genotype:
    name: host_genotype
    description: Observed genotype
    title: host genotype
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000365
    multivalued: false
    alias: host_genotype
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_phenotype:
    name: host_phenotype
    description: Phenotype of human or other host. For phenotypic quality ontology
      (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato.
      For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP
    title: host phenotype
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000874
    multivalued: false
    alias: host_phenotype
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_body_temp:
    name: host_body_temp
    description: Core body temperature of the host when sample was collected
    title: host body temperature
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000274
    multivalued: false
    alias: host_body_temp
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: quantity value
    required: false
  host_body_mass_index:
    name: host_body_mass_index
    description: Body mass index, calculated as weight/(height)squared
    title: host body-mass index
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000317
    multivalued: false
    alias: host_body_mass_index
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: quantity value
    required: false
  ethnicity:
    name: ethnicity
    description: A category of people who identify with each other, usually on the
      basis of presumed similarities such as a common language, ancestry, history,
      society, culture, nation or social treatment within their residing area. https://en.wikipedia.org/wiki/List_of_contemporary_ethnic_groups
    title: ethnicity
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000895
    multivalued: true
    alias: ethnicity
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: string
    required: false
  host_occupation:
    name: host_occupation
    description: Most frequent job performed by subject
    title: host occupation
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000896
    multivalued: false
    alias: host_occupation
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: integer
    required: false
  medic_hist_perform:
    name: medic_hist_perform
    description: Whether full medical history was collected
    title: medical history performed
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{boolean}'
    slot_uri: MIXS:0000897
    multivalued: false
    alias: medic_hist_perform
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: string
    required: false
  host_pulse:
    name: host_pulse
    description: Resting pulse, measured as beats per minute
    title: host pulse
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000333
    multivalued: false
    alias: host_pulse
    owner: human-vaginal MISAG
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: quantity value
    required: false
  perturbation:
    name: perturbation
    description: Type of perturbation, e.g. chemical administration, physical disturbance,
      etc., coupled with perturbation regimen including how many times the perturbation
      was repeated, how long each perturbation lasted, and the start and end time
      of the entire perturbation period; can include multiple perturbation types
    title: perturbation
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text};{Rn/start_time/end_time/duration}'
    slot_uri: MIXS:0000754
    multivalued: true
    alias: perturbation
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false
  samp_salinity:
    name: samp_salinity
    description: Salinity is the total concentration of all dissolved salts in a liquid
      or solid (in the form of an extract obtained by centrifugation) sample. While
      salinity can be measured by a complete chemical analysis, this method is difficult
      and time consuming. More often, it is instead derived from the conductivity
      measurement. This is known as practical salinity. These derivations compare
      the specific conductance of the sample to a salinity standard such as seawater
    title: sample salinity
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000109
    multivalued: false
    alias: samp_salinity
    owner: human-vaginal MISAG
    domain_of:
    - symbiont-associated
    - human-vaginal
    range: quantity value
    required: false
  salinity:
    name: salinity
    description: The total concentration of all dissolved salts in a liquid or solid
      sample. While salinity can be measured by a complete chemical analysis, this
      method is difficult and time consuming. More often, it is instead derived from
      the conductivity measurement. This is known as practical salinity. These derivations
      compare the specific conductance of the sample to a salinity standard such as
      seawater.
    title: salinity
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000183
    multivalued: false
    alias: salinity
    owner: human-vaginal MISAG
    domain_of:
    - food-farm environment
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: quantity value
    required: false
  oxy_stat_samp:
    name: oxy_stat_samp
    description: Oxygenation status of sample
    title: oxygenation status of sample
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000753
    multivalued: false
    alias: oxy_stat_samp
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: oxy_stat_samp_enum
    required: false
  organism_count:
    name: organism_count
    description: 'Total cell count of any organism (or group of organisms) per gram,
      volume or area of sample, should include name of organism followed by count.
      The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should
      also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)'
    title: organism count
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000103
    multivalued: true
    alias: organism_count
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    range: organism_count_enum
    required: false
  samp_store_temp:
    name: samp_store_temp
    description: Temperature at which sample was stored, e.g. -80 degree Celsius
    title: sample storage temperature
    from_schema: http://w3id.org/mixs
    is_a: core field
    slot_uri: MIXS:0000110
    multivalued: false
    alias: samp_store_temp
    owner: human-vaginal MISAG
    domain_of:
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: quantity value
    required: false
  samp_store_loc:
    name: samp_store_loc
    description: Location at which sample was stored, usually name of a specific freezer/room
    title: sample storage location
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000755
    multivalued: false
    alias: samp_store_loc
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false
  samp_store_dur:
    name: samp_store_dur
    description: Duration for which the sample was stored
    title: sample storage duration
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{duration}'
    slot_uri: MIXS:0000116
    multivalued: false
    alias: samp_store_dur
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false
  host_symbiont:
    name: host_symbiont
    description: The taxonomic name of the organism(s) found living in mutualistic,
      commensalistic, or parasitic symbiosis with the specific host.
    title: observed host symbionts
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0001298
    multivalued: true
    alias: host_symbiont
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  misc_param:
    name: misc_param
    description: Any other measurement performed or parameter collected, that is not
      listed here
    title: miscellaneous parameter
    from_schema: http://w3id.org/mixs
    is_a: core field
    string_serialization: '{text};{float} {unit}'
    slot_uri: MIXS:0000752
    multivalued: true
    alias: misc_param
    owner: human-vaginal MISAG
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false

</code></pre>

</details>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
      
    
    
  </body>
</html>