
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://GenomicsStandardsConsortium.github.io/mixs/MimarksCHumanAssociated/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.6.0, mkdocs-material-9.5.27" name="generator"/>
<title>Class: MimarksC combined with HumanAssociated (MimarksCHumanAssociated) - mixs</title>
<link href="../assets/stylesheets/main.6543a935.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-mimarksc-combined-with-humanassociated-mimarkschumanassociated">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header md-header--shadow" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="mixs" class="md-header__button md-logo" data-md-component="logo" href=".." title="mixs">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            mixs
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: MimarksC combined with HumanAssociated (MimarksCHumanAssociated)
            
          </span>
</div>
</div>
</div>
<script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
<div class="md-search__suggest" data-md-component="search-suggest"></div>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="" tabindex="0">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list" role="presentation"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/GenomicsStandardsConsortium/mixs" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="mixs" class="md-nav__button md-logo" data-md-component="logo" href=".." title="mixs">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    mixs
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/GenomicsStandardsConsortium/mixs" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
<span class="md-ellipsis">
    Index
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../term_list/">
<span class="md-ellipsis">
    Full term table
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../combinations/">
<span class="md-ellipsis">
    Combinations
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../enumerations/">
<span class="md-ellipsis">
    Enumerations
  </span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#inheritance">
<span class="md-ellipsis">
      Inheritance
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
<span class="md-ellipsis">
      Slots
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#usages">
<span class="md-ellipsis">
      Usages
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
<span class="md-ellipsis">
      Identifier and Mapping Information
    </span>
</a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
<span class="md-ellipsis">
      Schema Source
    </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
<span class="md-ellipsis">
      Mappings
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
<span class="md-ellipsis">
      LinkML Source
    </span>
</a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
<span class="md-ellipsis">
      Direct
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
<span class="md-ellipsis">
      Induced
    </span>
</a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<h1 id="class-mimarksc-combined-with-humanassociated-mimarkschumanassociated">Class: MimarksC combined with HumanAssociated (MimarksCHumanAssociated)</h1>
<p><em>MIxS Data that comply with the MimarksC checklist and the HumanAssociated Extension</em></p>
<p>URI: <a href="https://w3id.org/mixs/0010009_0016003">MIXS:0010009_0016003</a></p>
<div class="mermaid"> classDiagram
    class MimarksCHumanAssociated
    click MimarksCHumanAssociated href "../MimarksCHumanAssociated"
      MimarksC &lt;|-- MimarksCHumanAssociated
        click MimarksC href "../MimarksC"
      HumanAssociated &lt;|-- MimarksCHumanAssociated
        click HumanAssociated href "../HumanAssociated"

      MimarksCHumanAssociated : alt

      MimarksCHumanAssociated : amniotic_fluid_color

      MimarksCHumanAssociated : associated_resource

      MimarksCHumanAssociated : biotic_relationship




    MimarksCHumanAssociated --&gt; "0..1 _recommended_" BioticRelationshipEnum : biotic_relationship
    click BioticRelationshipEnum href "../BioticRelationshipEnum"


      MimarksCHumanAssociated : blood_blood_disord

      MimarksCHumanAssociated : chem_administration

      MimarksCHumanAssociated : chimera_check

      MimarksCHumanAssociated : collection_date

      MimarksCHumanAssociated : depth

      MimarksCHumanAssociated : diet_last_six_month

      MimarksCHumanAssociated : drug_usage

      MimarksCHumanAssociated : elev

      MimarksCHumanAssociated : env_broad_scale

      MimarksCHumanAssociated : env_local_scale

      MimarksCHumanAssociated : env_medium

      MimarksCHumanAssociated : ethnicity

      MimarksCHumanAssociated : experimental_factor

      MimarksCHumanAssociated : extrachrom_elements

      MimarksCHumanAssociated : foetal_health_stat

      MimarksCHumanAssociated : geo_loc_name

      MimarksCHumanAssociated : gestation_state

      MimarksCHumanAssociated : host_age

      MimarksCHumanAssociated : host_body_mass_index

      MimarksCHumanAssociated : host_body_product

      MimarksCHumanAssociated : host_body_site

      MimarksCHumanAssociated : host_body_temp

      MimarksCHumanAssociated : host_diet

      MimarksCHumanAssociated : host_disease_stat

      MimarksCHumanAssociated : host_fam_rel

      MimarksCHumanAssociated : host_genotype

      MimarksCHumanAssociated : host_height

      MimarksCHumanAssociated : host_hiv_stat

      MimarksCHumanAssociated : host_last_meal

      MimarksCHumanAssociated : host_occupation

      MimarksCHumanAssociated : host_phenotype

      MimarksCHumanAssociated : host_pulse

      MimarksCHumanAssociated : host_sex

      MimarksCHumanAssociated : host_subject_id

      MimarksCHumanAssociated : host_symbiont

      MimarksCHumanAssociated : host_tot_mass

      MimarksCHumanAssociated : ihmc_medication_code

      MimarksCHumanAssociated : isol_growth_condt

      MimarksCHumanAssociated : kidney_disord

      MimarksCHumanAssociated : lat_lon

      MimarksCHumanAssociated : maternal_health_stat

      MimarksCHumanAssociated : medic_hist_perform

      MimarksCHumanAssociated : misc_param

      MimarksCHumanAssociated : neg_cont_type




    MimarksCHumanAssociated --&gt; "0..1 _recommended_" NegContTypeEnum : neg_cont_type
    click NegContTypeEnum href "../NegContTypeEnum"


      MimarksCHumanAssociated : nose_throat_disord

      MimarksCHumanAssociated : nucl_acid_amp

      MimarksCHumanAssociated : nucl_acid_ext

      MimarksCHumanAssociated : organism_count

      MimarksCHumanAssociated : oxy_stat_samp




    MimarksCHumanAssociated --&gt; "0..1" OxyStatSampEnum : oxy_stat_samp
    click OxyStatSampEnum href "../OxyStatSampEnum"


      MimarksCHumanAssociated : pcr_cond

      MimarksCHumanAssociated : pcr_primers

      MimarksCHumanAssociated : perturbation

      MimarksCHumanAssociated : pet_farm_animal

      MimarksCHumanAssociated : pos_cont_type

      MimarksCHumanAssociated : project_name

      MimarksCHumanAssociated : pulmonary_disord

      MimarksCHumanAssociated : rel_to_oxygen




    MimarksCHumanAssociated --&gt; "0..1 _recommended_" RelToOxygenEnum : rel_to_oxygen
    click RelToOxygenEnum href "../RelToOxygenEnum"


      MimarksCHumanAssociated : salinity

      MimarksCHumanAssociated : samp_collect_device

      MimarksCHumanAssociated : samp_collect_method

      MimarksCHumanAssociated : samp_mat_process

      MimarksCHumanAssociated : samp_name

      MimarksCHumanAssociated : samp_size

      MimarksCHumanAssociated : samp_store_dur

      MimarksCHumanAssociated : samp_store_loc

      MimarksCHumanAssociated : samp_store_temp

      MimarksCHumanAssociated : samp_taxon_id

      MimarksCHumanAssociated : samp_vol_we_dna_ext

      MimarksCHumanAssociated : seq_meth

      MimarksCHumanAssociated : seq_quality_check




    MimarksCHumanAssociated --&gt; "0..1 _recommended_" SeqQualityCheckEnum : seq_quality_check
    click SeqQualityCheckEnum href "../SeqQualityCheckEnum"


      MimarksCHumanAssociated : smoker

      MimarksCHumanAssociated : sop

      MimarksCHumanAssociated : source_mat_id

      MimarksCHumanAssociated : study_complt_stat

      MimarksCHumanAssociated : subspecf_gen_lin

      MimarksCHumanAssociated : target_gene

      MimarksCHumanAssociated : target_subfragment

      MimarksCHumanAssociated : temp

      MimarksCHumanAssociated : travel_out_six_month

      MimarksCHumanAssociated : trophic_level




    MimarksCHumanAssociated --&gt; "0..1 _recommended_" TrophicLevelEnum : trophic_level
    click TrophicLevelEnum href "../TrophicLevelEnum"


      MimarksCHumanAssociated : twin_sibling

      MimarksCHumanAssociated : urine_collect_meth




    MimarksCHumanAssociated --&gt; "0..1" UrineCollectMethEnum : urine_collect_meth
    click UrineCollectMethEnum href "../UrineCollectMethEnum"


      MimarksCHumanAssociated : urogenit_tract_disor

      MimarksCHumanAssociated : weight_loss_3_month


</div>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Extension/">Extension</a><ul>
<li><a href="../HumanAssociated/">HumanAssociated</a><ul>
<li><strong>MimarksCHumanAssociated</strong> [ <a href="../MimarksC/">MimarksC</a>]</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../samp_name/">samp_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>A local identifier or name that for the material sample used for extracting n...</td>
<td><a href="../MimarksC/">MimarksC</a>, <a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../pcr_primers/">pcr_primers</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>PCR primers that were used to amplify the sequence of the targeted gene, locu...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../nucl_acid_amp/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../target_subfragment/">target_subfragment</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Name of subfragment of a gene or locus</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../temp/">temp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Temperature of the sample at the time of sampling</td>
<td><a href="../MimarksC/">MimarksC</a>, <a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../pcr_cond/">pcr_cond</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Description of reaction conditions and components of polymerase chain reactio...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../nucl_acid_ext/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../samp_size/">samp_size</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample coll...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../isol_growth_condt/">isol_growth_condt</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identif...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../alt/">alt</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Heights of objects such as airplanes, space shuttles, rockets, atmospheric ba...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../source_mat_id/">source_mat_id</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A unique identifier assigned to a material sample (as defined by <a href="http://rs">http://rs</a></td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../extrachrom_elements/">extrachrom_elements</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Do plasmids exist of significant phenotypic consequence (e</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extractio...</td>
<td><a href="../MimarksC/">MimarksC</a>, <a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../rel_to_oxygen/">rel_to_oxygen</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../RelToOxygenEnum/">RelToOxygenEnum</a></td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic ...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../samp_collect_device/">samp_collect_device</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The device used to collect an environmental sample</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../biotic_relationship/">biotic_relationship</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../BioticRelationshipEnum/">BioticRelationshipEnum</a></td>
<td>Description of relationship(s) between the subject organism and other organis...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../seq_quality_check/">seq_quality_check</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../SeqQualityCheckEnum/">SeqQualityCheckEnum</a></td>
<td>Indicate if the sequence has been called by automatic systems (none) or under...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../project_name/">project_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Name of the project within which the sequencing was organized</td>
<td><a href="../MimarksC/">MimarksC</a>, <a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../neg_cont_type/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../NegContTypeEnum/">NegContTypeEnum</a></td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../chimera_check/">chimera_check</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for chimera checking, including version number and parameters, t...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../trophic_level/">trophic_level</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../TrophicLevelEnum/">TrophicLevelEnum</a></td>
<td>Trophic levels are the feeding position in a food chain</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../pos_cont_type/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The substance, mixture, product, or apparatus used to verify that a process w...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../subspecf_gen_lin/">subspecf_gen_lin</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Information about the genetic distinctness of the sequenced organism below th...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the entity or entities which are in the sample or specimen s local vic...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../samp_mat_process/">samp_mat_process</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A brief description of any processing applied to the sample during or after r...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../depth/">depth</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The vertical distance below local surface</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../samp_collect_method/">samp_collect_method</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The method employed for collecting the sample</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../env_medium/">env_medium</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../samp_taxon_id/">samp_taxon_id</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>NCBI taxon id of the sample</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by the country or sea name f...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../collection_date/">collection_date</a></td>
<td>1 <br/> <a href="../Datetime/">Datetime</a></td>
<td>The time of sampling, either as an instance (single point in time) or interva...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../seq_meth/">seq_meth</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Sequencing machine used</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by latitude and longitude</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../elev/">elev</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Elevation of the sampling site is its height above a fixed reference point, m...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../experimental_factor/">experimental_factor</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Variable aspects of an experiment design that can be used to describe an expe...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../target_gene/">target_gene</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Targeted gene or locus name for marker gene studies</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../associated_resource/">associated_resource</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A related resource that is referenced, cited, or otherwise associated to the ...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../sop/">sop</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, ...</td>
<td><a href="../MimarksC/">MimarksC</a></td>
</tr>
<tr>
<td><a href="../host_subject_id/">host_subject_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A unique identifier by which each subject can be referred to, de-identified</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_age/">host_age</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Age of host at the time of sampling; relevant scale depends on species and st...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_sex/">host_sex</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Gender or physical sex of the host</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_disease_stat/">host_disease_stat</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>List of diseases with which the host has been diagnosed; can include multiple...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../ihmc_medication_code/">ihmc_medication_code</a></td>
<td>* <br/> <a href="../Integer/">Integer</a></td>
<td>Can include multiple medication codes</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../chem_administration/">chem_administration</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>List of chemical compounds administered to the host or site where sampling oc...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_body_site/">host_body_site</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name of body site where the sample was obtained from, such as a specific orga...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_body_product/">host_body_product</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Substance produced by the body, e</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_tot_mass/">host_tot_mass</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_height/">host_height</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The height of subject</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_diet/">host_diet</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Type of diet depending on the host, for animals omnivore, herbivore etc</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_last_meal/">host_last_meal</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Content of last meal and time since feeding; can include multiple values</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_fam_rel/">host_fam_rel</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Relationships to other hosts in the same study; can include multiple relation...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_genotype/">host_genotype</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Observed genotype</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_phenotype/">host_phenotype</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Phenotype of human or other host</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_body_temp/">host_body_temp</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Core body temperature of the host when sample was collected</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../smoker/">smoker</a></td>
<td>0..1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>Specification of smoking status</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_hiv_stat/">host_hiv_stat</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>HIV status of subject, if yes HAART initiation status should also be indicate...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../drug_usage/">drug_usage</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Any drug used by subject and the frequency of usage; can include multiple dru...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_body_mass_index/">host_body_mass_index</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Body mass index, calculated as weight/(height)squared</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../diet_last_six_month/">diet_last_six_month</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of major diet changes in the last six months, if yes the change...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../weight_loss_3_month/">weight_loss_3_month</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of weight loss in the last three months, if yes should be furth...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../ethnicity/">ethnicity</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>A category of people who identify with each other, usually on the basis of pr...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_occupation/">host_occupation</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Most frequent job performed by subject</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../pet_farm_animal/">pet_farm_animal</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Specification of presence of pets or farm animals in the environment of subje...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../travel_out_six_month/">travel_out_six_month</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Specification of the countries travelled in the last six months; can include ...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../twin_sibling/">twin_sibling</a></td>
<td>0..1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>Specification of twin sibling presence</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../medic_hist_perform/">medic_hist_perform</a></td>
<td>0..1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>Whether full medical history was collected</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../study_complt_stat/">study_complt_stat</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of study completion status, if no the reason should be specifie...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../pulmonary_disord/">pulmonary_disord</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>History of pulmonary disorders; can include multiple disorders</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../nose_throat_disord/">nose_throat_disord</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>History of nose-throat disorders; can include multiple disorders,  The terms ...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../blood_blood_disord/">blood_blood_disord</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>History of blood disorders; can include multiple disorders</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_pulse/">host_pulse</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Resting pulse, measured as beats per minute</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../gestation_state/">gestation_state</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of the gestation state</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../maternal_health_stat/">maternal_health_stat</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of the maternal health status</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../foetal_health_stat/">foetal_health_stat</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of foetal health status, should also include abortion</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../amniotic_fluid_color/">amniotic_fluid_color</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specification of the color of the amniotic fluid sample</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../kidney_disord/">kidney_disord</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>History of kidney disorders; can include multiple disorders</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../urogenit_tract_disor/">urogenit_tract_disor</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>History of urogenital tract disorders; can include multiple disorders</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../urine_collect_meth/">urine_collect_meth</a></td>
<td>0..1 <br/> <a href="../UrineCollectMethEnum/">UrineCollectMethEnum</a></td>
<td>Specification of urine collection method</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../perturbation/">perturbation</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Type of perturbation, e</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../salinity/">salinity</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The total concentration of all dissolved salts in a liquid or solid sample</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../oxy_stat_samp/">oxy_stat_samp</a></td>
<td>0..1 <br/> <a href="../OxyStatSampEnum/">OxyStatSampEnum</a></td>
<td>Oxygenation status of sample</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../organism_count/">organism_count</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or ...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../samp_store_temp/">samp_store_temp</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Temperature at which sample was stored, e</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../samp_store_dur/">samp_store_dur</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Duration for which the sample was stored</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../host_symbiont/">host_symbiont</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>The taxonomic name of the organism(s) found living in mutualistic, commensali...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../samp_store_loc/">samp_store_loc</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
<tr>
<td><a href="../misc_param/">misc_param</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Any other measurement performed or parameter collected, that is not listed he...</td>
<td><a href="../HumanAssociated/">HumanAssociated</a></td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<table>
<thead>
<tr>
<th>used by</th>
<th>used in</th>
<th>type</th>
<th>used</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../MixsCompliantData/">MixsCompliantData</a></td>
<td><a href="../mimarks_c_human_associated_data/">mimarks_c_human_associated_data</a></td>
<td>range</td>
<td><a href="./">MimarksCHumanAssociated</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: <a href="https://w3id.org/mixs">https://w3id.org/mixs</a></li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>MIXS:0010009_0016003</td>
</tr>
<tr>
<td>native</td>
<td>MIXS:MimarksCHumanAssociated</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: MimarksCHumanAssociated
description: MIxS Data that comply with the MimarksC checklist and the HumanAssociated
  Extension
title: MimarksC combined with HumanAssociated
in_subset:
- combination_classes
from_schema: https://w3id.org/mixs
is_a: HumanAssociated
mixins:
- MimarksC
class_uri: MIXS:0010009_0016003

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: MimarksCHumanAssociated
description: MIxS Data that comply with the MimarksC checklist and the HumanAssociated
  Extension
title: MimarksC combined with HumanAssociated
in_subset:
- combination_classes
from_schema: https://w3id.org/mixs
is_a: HumanAssociated
mixins:
- MimarksC
attributes:
  samp_name:
    name: samp_name
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: ''
    description: A local identifier or name that for the material sample used for
      extracting nucleic acids, and subsequent sequencing. It can refer either to
      the original material collected or to any derived sub-samples. It can have any
      format, but we suggest that you make it concise, unique and consistent within
      your lab, and as informative as possible. INSDC requires every sample name from
      a single Submitter to be unique. Use of a globally unique identifier for the
      field source_mat_id is recommended in addition to sample_name
    title: sample name
    examples:
    - value: ISDsoil1
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - sample
    slot_uri: MIXS:0001107
    alias: samp_name
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
  pcr_primers:
    name: pcr_primers
    description: PCR primers that were used to amplify the sequence of the targeted
      gene, locus or subfragment. This field should contain all the primers used for
      a single PCR reaction if multiple forward or reverse primers are present in
      a single PCR reaction. The primer sequence should be reported in uppercase letters
    title: pcr primers
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: FWD:{dna};REV:{dna}
    slot_uri: MIXS:0000046
    alias: pcr_primers
    owner: MimarksCHumanAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    recommended: true
  nucl_acid_amp:
    name: nucl_acid_amp
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the enzymatic amplification (PCR,
      TMA, NASBA) of specific nucleic acids
    title: nucleic acid amplification
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000038
    alias: nucl_acid_amp
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  target_subfragment:
    name: target_subfragment
    description: Name of subfragment of a gene or locus. Important to e.g. identify
      special regions on marker genes like V6 on 16S rRNA
    title: target subfragment
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000045
    alias: target_subfragment
    owner: MimarksCHumanAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    recommended: true
  temp:
    name: temp
    description: Temperature of the sample at the time of sampling
    title: temperature
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000113
    alias: temp
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  pcr_cond:
    name: pcr_cond
    description: Description of reaction conditions and components of polymerase chain
      reaction performed during library preparation.
    title: pcr conditions
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: initial denaturation:degrees_minutes;annealing:degrees_minutes;elongation:degrees_minutes;final
      elongation:degrees_minutes;total cycles
    slot_uri: MIXS:0000049
    alias: pcr_cond
    owner: MimarksCHumanAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    recommended: true
  nucl_acid_ext:
    name: nucl_acid_ext
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the material separation to recover
      the nucleic acid fraction from a sample
    title: nucleic acid extraction
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000037
    alias: nucl_acid_ext
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  samp_size:
    name: samp_size
    description: The total amount or size (volume (ml), mass (g) or area (m2) ) of
      sample collected
    title: amount or size of sample collected
    examples:
    - value: 5 liter
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - sample
    - size
    slot_uri: MIXS:0000001
    alias: samp_size
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  isol_growth_condt:
    name: isol_growth_condt
    description: Publication reference in the form of pubmed ID (pmid), digital object
      identifier (doi) or url for isolation and growth condition specifications of
      the organism/material
    title: isolation and growth condition
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000003
    alias: isol_growth_condt
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - MimarksC
    - Agriculture
    range: string
    required: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  alt:
    name: alt
    description: Heights of objects such as airplanes, space shuttles, rockets, atmospheric
      balloons and heights of places such as atmospheric layers and clouds. It is
      used to measure the height of an object which is above the earth's surface.
      In this context, the altitude measurement is the vertical distance between the
      earth's surface above sea level and the sampled position in the air
    title: altitude
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000094
    alias: alt
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - HostAssociated
    - MiscellaneousNaturalOrArtificialEnvironment
    - SymbiontAssociated
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  source_mat_id:
    name: source_mat_id
    description: A unique identifier assigned to a material sample (as defined by
      http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular
      digital record of a material sample) used for extracting nucleic acids, and
      subsequent sequencing. The identifier can refer either to the original material
      collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher,
      /bio_material, or /culture_collection may or may not share the same value as
      the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id
      may both contain 'UAM:Herps:14' , referring to both the specimen voucher and
      sampled tissue with the same identifier. However, the /culture_collection qualifier
      may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id
      would refer to an identifier from some derived culture from which the nucleic
      acids were extracted (e.g. xatc123 or ark:/2154/R2)
    title: source material identifiers
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000026
    multivalued: true
    alias: source_mat_id
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - SymbiontAssociated
    range: string
    recommended: true
  extrachrom_elements:
    name: extrachrom_elements
    description: Do plasmids exist of significant phenotypic consequence (e.g. ones
      that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids
      (borrelia has 15+ plasmids)
    title: extrachromosomal elements
    examples:
    - value: '5'
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000023
    alias: extrachrom_elements
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MimarksC
    range: integer
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: milliliter, gram, milligram, square centimeter
    description: 'Volume (ml) or mass (g) of total collected sample processed for
      DNA extraction. Note: total sample collected should be entered under the term
      Sample Size (MIXS:0000001)'
    title: sample volume or weight for DNA extraction
    examples:
    - value: 1500 milliliter
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - dna
    - sample
    - volume
    - weight
    slot_uri: MIXS:0000111
    alias: samp_vol_we_dna_ext
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  rel_to_oxygen:
    name: rel_to_oxygen
    description: Is this organism an aerobe, anaerobe? Please note that aerobic and
      anaerobic are valid descriptors for microbial environments
    title: relationship to oxygen
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000015
    alias: rel_to_oxygen
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    range: RelToOxygenEnum
    recommended: true
  samp_collect_device:
    name: samp_collect_device
    description: The device used to collect an environmental sample. This field accepts
      terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO).
      This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)
    title: sample collection device
    examples:
    - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000002
    alias: samp_collect_device
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
  biotic_relationship:
    name: biotic_relationship
    description: Description of relationship(s) between the subject organism and other
      organism(s) it is associated with. E.g., parasite on species X; mutualist with
      species Y. The target organism is the subject of the relationship, and the other
      organism(s) is the object
    title: observed biotic relationship
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000028
    alias: biotic_relationship
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - MimarksC
    - Miuvig
    - Agriculture
    range: BioticRelationshipEnum
    recommended: true
  seq_quality_check:
    name: seq_quality_check
    description: Indicate if the sequence has been called by automatic systems (none)
      or undergone a manual editing procedure (e.g. by inspecting the raw data or
      chromatograms). Applied only for sequences that are not submitted to SRA,ENA
      or DRA
    title: sequence quality check
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000051
    alias: seq_quality_check
    owner: MimarksCHumanAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: SeqQualityCheckEnum
    recommended: true
  project_name:
    name: project_name
    description: Name of the project within which the sequencing was organized
    title: project name
    examples:
    - value: Forest soil metagenome
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - project
    slot_uri: MIXS:0000092
    alias: project_name
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
  neg_cont_type:
    name: neg_cont_type
    annotations:
      Expected_value:
        tag: Expected_value
        value: enumeration or text
    description: The substance or equipment used as a negative control in an investigation
    title: negative control type
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - type
    slot_uri: MIXS:0001321
    alias: neg_cont_type
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: NegContTypeEnum
    recommended: true
  chimera_check:
    name: chimera_check
    description: Tool(s) used for chimera checking, including version number and parameters,
      to discover and remove chimeric sequences. A chimeric sequence is comprised
      of two or more phylogenetically distinct parent sequences
    title: chimera check software
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000052
    alias: chimera_check
    owner: MimarksCHumanAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    recommended: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  trophic_level:
    name: trophic_level
    description: Trophic levels are the feeding position in a food chain. Microbes
      can be a range of producers (e.g. chemolithotroph)
    title: trophic level
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000032
    alias: trophic_level
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MimarksC
    - Agriculture
    range: TrophicLevelEnum
    recommended: true
  pos_cont_type:
    name: pos_cont_type
    description: The substance, mixture, product, or apparatus used to verify that
      a process which is part of an investigation delivers a true positive
    title: positive control type
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - type
    string_serialization: '{term} or {text}'
    slot_uri: MIXS:0001322
    alias: pos_cont_type
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  subspecf_gen_lin:
    name: subspecf_gen_lin
    description: Information about the genetic distinctness of the sequenced organism
      below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any
      relevant genetic typing schemes like Group I plasmid. Subspecies should not
      be recorded in this term, but in the NCBI taxonomy. Supply both the lineage
      name and the lineage rank separated by a colon, e.g., biovar:abc123
    title: subspecific genetic lineage
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{rank name}:{text}'
    slot_uri: MIXS:0000020
    alias: subspecf_gen_lin
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - MimarksC
    - FoodFoodProductionFacility
    range: string
    recommended: true
  env_local_scale:
    name: env_local_scale
    annotations:
      Expected_value:
        tag: Expected_value
        value: Environmental entities having causal influences upon the entity at
          time of sampling
    description: 'Report the entity or entities which are in the sample or specimen
      s local vicinity and which you believe have significant causal influences on
      your sample or specimen. We recommend using EnvO terms which are of smaller
      spatial grain than your entry for env_broad_scale. Terms, such as anatomical
      sites, from other OBO Library ontologies which interoperate with EnvO (e.g.
      UBERON) are accepted in this field. EnvO documentation about how to use the
      field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: local environmental context
    examples:
    - value: hillside [ENVO:01000333]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - context
    - environmental
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000013
    alias: env_local_scale
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
  samp_mat_process:
    name: samp_mat_process
    description: A brief description of any processing applied to the sample during
      or after retrieving the sample from environment, or a link to the relevant protocol(s)
      performed
    title: sample material processing
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000016
    alias: samp_mat_process
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  depth:
    name: depth
    description: The vertical distance below local surface. For sediment or soil samples
      depth is measured from sediment or soil surface, respectively. Depth can be
      reported as an interval for subsurface samples
    title: depth
    examples:
    - value: 10 meter
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000018
    alias: depth
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  samp_collect_method:
    name: samp_collect_method
    description: The method employed for collecting the sample
    title: sample collection method
    examples:
    - value: swabbing
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0001225
    alias: samp_collect_method
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$|([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}|{text}$
      interpolated: true
      partial_match: true
  env_medium:
    name: env_medium
    description: 'Report the environmental material(s) immediately surrounding the
      sample or specimen at the time of sampling. We recommend using subclasses of
      ''environmental material'' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO
      documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
      . Terms from other OBO ontologies are permissible as long as they reference
      mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
      (e.g. a tree, a leaf, a table top)'
    title: environmental medium
    examples:
    - value: bluegrass field soil [ENVO:00005789]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - environmental
    slot_uri: MIXS:0000014
    alias: env_medium
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
    structured_pattern:
      syntax: ^{termLabel} \[{termID}\]$
      interpolated: true
      partial_match: true
  samp_taxon_id:
    name: samp_taxon_id
    description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa
      sample. Use 'synthetic metagenome  for mock community/positive controls, or
      'blank sample' for negative controls
    title: taxonomy ID of DNA sample
    examples:
    - value: Gut Metagenome [NCBITaxon:749906]
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - dna
    - identifier
    - sample
    - taxon
    slot_uri: MIXS:0001320
    alias: samp_taxon_id
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[NCBITaxon:\d+\]$
    structured_pattern:
      syntax: ^{text} \[{NCBItaxon_id}\]$
      interpolated: true
      partial_match: true
  geo_loc_name:
    name: geo_loc_name
    description: The geographical origin of the sample as defined by the country or
      sea name followed by specific region name. Country or sea names should be chosen
      from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology
      (http://purl.bioontology.org/ontology/GAZ)
    title: geographic location (country and/or sea,region)
    examples:
    - value: 'USA: Maryland, Bethesda'
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - geographic
    - location
    slot_uri: MIXS:0000010
    alias: geo_loc_name
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: string
    required: true
    pattern: '^([^\s-]{1,2}|[^\s-]+.+[^\s-]+): ([^\s-]{1,2}|[^\s-]+.+[^\s-]+), ([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$'
    structured_pattern:
      syntax: '^{text}: {text}, {text}$'
      interpolated: true
      partial_match: true
  collection_date:
    name: collection_date
    description: 'The time of sampling, either as an instance (single point in time)
      or interval. In case no exact time is available, the date/time can be right
      truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10;
      2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant'
    title: collection date
    examples:
    - value: '2013-03-25T12:42:31+01:00'
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - date
    slot_uri: MIXS:0000011
    alias: collection_date
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: datetime
    required: true
  seq_meth:
    name: seq_meth
    description: Sequencing machine used. Where possible the term should be taken
      from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)
    title: sequencing method
    examples:
    - value: 454 Genome Sequencer FLX [OBI:0000702]
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - method
    slot_uri: MIXS:0000050
    alias: seq_meth
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+)|(([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\])$
    structured_pattern:
      syntax: ^{text}|({termLabel} \[{termID}\])$
      interpolated: true
      partial_match: true
  lat_lon:
    name: lat_lon
    annotations:
      Expected_value:
        tag: Expected_value
        value: decimal degrees,  limit to 8 decimal points
    description: The geographical origin of the sample as defined by latitude and
      longitude. The values should be reported in decimal degrees and in WGS84 system
    title: geographic location (latitude and longitude)
    examples:
    - value: 50.586825 6.408977
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - geographic
    - location
    string_serialization: '{float} {float}'
    slot_uri: MIXS:0000009
    alias: lat_lon
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: string
    required: true
    pattern: ^(-?((?:[0-8]?[0-9](?:\.\d{0,8})?)|90)) -?[0-9]+(?:\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$
    structured_pattern:
      syntax: ^{lat} {lon}$
      interpolated: true
      partial_match: true
  elev:
    name: elev
    description: Elevation of the sampling site is its height above a fixed reference
      point, most commonly the mean sea level. Elevation is mainly used when referring
      to points on the earth's surface, while altitude is used for points above the
      surface, such as an aircraft in flight or a spacecraft in orbit
    title: elevation
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000093
    alias: elev
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - HostAssociated
    - HydrocarbonResourcesCores
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  env_broad_scale:
    name: env_broad_scale
    description: 'Report the major environmental system the sample or specimen came
      from. The system(s) identified should have a coarse spatial grain, to provide
      the general environmental context of where the sampling was done (e.g. in the
      desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428.
      EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: broad-scale environmental context
    examples:
    - value: rangeland biome [ENVO:01000247]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - context
    - environmental
    slot_uri: MIXS:0000012
    alias: env_broad_scale
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
    structured_pattern:
      syntax: ^{termLabel} \[{termID}\]$
      interpolated: true
      partial_match: true
  experimental_factor:
    name: experimental_factor
    annotations:
      Expected_value:
        tag: Expected_value
        value: text or EFO and/or OBI
    description: Variable aspects of an experiment design that can be used to describe
      an experiment, or set of experiments, in an increasingly detailed manner. This
      field accepts ontology terms from Experimental Factor Ontology (EFO) and/or
      Ontology for Biomedical Investigations (OBI)
    title: experimental factor
    examples:
    - value: time series design [EFO:0001779]
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - experimental
    - factor
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000008
    multivalued: true
    alias: experimental_factor
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    pattern: ^\S+.*\S+ \[[a-zA-Z]{2,}:\d+\]$
  target_gene:
    name: target_gene
    description: Targeted gene or locus name for marker gene studies
    title: target gene
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000044
    alias: target_gene
    owner: MimarksCHumanAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    required: true
  associated_resource:
    name: associated_resource
    annotations:
      Expected_value:
        tag: Expected_value
        value: reference to resource
    description: A related resource that is referenced, cited, or otherwise associated
      to the sequence
    title: relevant electronic resources
    examples:
    - value: http://www.earthmicrobiome.org/
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - resource
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000091
    multivalued: true
    alias: associated_resource
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  sop:
    name: sop
    description: Standard operating procedures used in assembly and/or annotation
      of genomes, metagenomes or environmental sequences
    title: relevant standard operating procedures
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000090
    multivalued: true
    alias: sop
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  host_subject_id:
    name: host_subject_id
    description: A unique identifier by which each subject can be referred to, de-identified
    title: host subject id
    examples:
    - value: MPI123
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000861
    alias: host_subject_id
    owner: MimarksCHumanAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - SymbiontAssociated
    range: string
  host_age:
    name: host_age
    description: Age of host at the time of sampling; relevant scale depends on species
      and study, e.g. Could be seconds for amoebae or centuries for trees
    title: host age
    examples:
    - value: 30 years
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000255
    alias: host_age
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  host_sex:
    name: host_sex
    annotations:
      Expected_value:
        tag: Expected_value
        value: enumeration
    description: Gender or physical sex of the host
    title: host sex
    comments:
    - example of non-binary from Excel sheets does not match any of the enumerated
      values
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    string_serialization: '[female|hermaphrodite|non-binary|male|transgender|transgender
      (female to male)|transgender (male to female)

      |undeclared]'
    slot_uri: MIXS:0000811
    alias: host_sex
    owner: MimarksCHumanAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: string
  host_disease_stat:
    name: host_disease_stat
    description: List of diseases with which the host has been diagnosed; can include
      multiple diagnoses. The value of the field depends on host; for humans the terms
      should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org,
      non-human host diseases are free text
    title: host disease status
    examples:
    - value: measles [DOID:8622]
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000031
    alias: host_disease_stat
    owner: MimarksCHumanAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    range: string
  ihmc_medication_code:
    name: ihmc_medication_code
    description: Can include multiple medication codes
    title: IHMC medication code
    examples:
    - value: '810'
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - code
    slot_uri: MIXS:0000884
    multivalued: true
    alias: ihmc_medication_code
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: integer
  chem_administration:
    name: chem_administration
    annotations:
      Expected_value:
        tag: Expected_value
        value: CHEBI;timestamp
    description: List of chemical compounds administered to the host or site where
      sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can
      include multiple compounds. For chemical entities of biological interest ontology
      (chebi) (v 163), http://purl.bioontology.org/ontology/chebi
    title: chemical administration
    examples:
    - value: agar [CHEBI:2509];2018-05-11T20:00Z
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - administration
    string_serialization: '{termLabel} [{termID}];{timestamp}'
    slot_uri: MIXS:0000751
    multivalued: true
    alias: chem_administration
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
  host_body_site:
    name: host_body_site
    description: Name of body site where the sample was obtained from, such as a specific
      organ or tissue (tongue, lung etc...). Use terms from the foundational model
      of anatomy ontology (fma) or the Uber-anatomy ontology (UBERON)
    title: host body site
    examples:
    - value: Lung parenchyma [fma27360]
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000867
    alias: host_body_site
    owner: MimarksCHumanAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - SymbiontAssociated
    range: string
  host_body_product:
    name: host_body_product
    annotations:
      Expected_value:
        tag: Expected_value
        value: FMA or UBERON
    description: Substance produced by the body, e.g. Stool, mucus, where the sample
      was obtained from. Use terms from the foundational model of anatomy ontology
      (fma) or Uber-anatomy ontology (UBERON)
    title: host body product
    examples:
    - value: mucus [FMA:66938]
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - body
    - host
    - host.
    - product
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000888
    alias: host_body_product
    owner: MimarksCHumanAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - SymbiontAssociated
    range: string
  host_tot_mass:
    name: host_tot_mass
    description: Total mass of the host at collection, the unit depends on host
    title: host total mass
    examples:
    - value: 65 kilogram
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000263
    alias: host_tot_mass
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  host_height:
    name: host_height
    description: The height of subject
    title: host height
    examples:
    - value: 1.75 meter
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000264
    alias: host_height
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  host_diet:
    name: host_diet
    description: Type of diet depending on the host, for animals omnivore, herbivore
      etc., for humans high-fat, meditteranean etc.; can include multiple diet types
    title: host diet
    examples:
    - value: high-fat
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000869
    multivalued: true
    alias: host_diet
    owner: MimarksCHumanAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: string
  host_last_meal:
    name: host_last_meal
    description: Content of last meal and time since feeding; can include multiple
      values
    title: host last meal
    examples:
    - value: french fries;P5H30M
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text};{duration}'
    slot_uri: MIXS:0000870
    multivalued: true
    alias: host_last_meal
    owner: MimarksCHumanAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: string
  host_fam_rel:
    name: host_fam_rel
    description: Relationships to other hosts in the same study; can include multiple
      relationships
    title: host family relationship
    examples:
    - value: mother;ID298
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text};{text}'
    slot_uri: MIXS:0000872
    multivalued: true
    alias: host_fam_rel
    owner: MimarksCHumanAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - SymbiontAssociated
    range: string
  host_genotype:
    name: host_genotype
    description: Observed genotype
    title: host genotype
    examples:
    - value: ST1
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000365
    alias: host_genotype
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
  host_phenotype:
    name: host_phenotype
    description: Phenotype of human or other host. Use terms from the phenotypic quality
      ontology (pato) or the Human Phenotype Ontology (HP)
    title: host phenotype
    examples:
    - value: Tinnitus [HP:0000360]
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000874
    alias: host_phenotype
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
  host_body_temp:
    name: host_body_temp
    description: Core body temperature of the host when sample was collected
    title: host body temperature
    examples:
    - value: 36.5 degree Celsius
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000274
    alias: host_body_temp
    owner: MimarksCHumanAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  smoker:
    name: smoker
    description: Specification of smoking status
    title: smoker
    examples:
    - value: 'yes'
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000262
    alias: smoker
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: boolean
  host_hiv_stat:
    name: host_hiv_stat
    annotations:
      Expected_value:
        tag: Expected_value
        value: HIV status;HAART initiation status
    description: HIV status of subject, if yes HAART initiation status should also
      be indicated as [YES or NO]
    title: host HIV status
    examples:
    - value: yes;yes
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - status
    string_serialization: '{boolean};{boolean}'
    slot_uri: MIXS:0000265
    alias: host_hiv_stat
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  drug_usage:
    name: drug_usage
    annotations:
      Expected_value:
        tag: Expected_value
        value: drug name;frequency
    description: Any drug used by subject and the frequency of usage; can include
      multiple drugs used
    title: drug usage
    examples:
    - value: Lipitor;2/day
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - drug
    - use
    string_serialization: '{text};{integer}/[year|month|week|day|hour]'
    slot_uri: MIXS:0000894
    multivalued: true
    alias: drug_usage
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  host_body_mass_index:
    name: host_body_mass_index
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: kilogram per square meter
    description: Body mass index, calculated as weight/(height)squared
    title: host body-mass index
    examples:
    - value: 22 kilogram per square meter
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000317
    alias: host_body_mass_index
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  diet_last_six_month:
    name: diet_last_six_month
    annotations:
      Expected_value:
        tag: Expected_value
        value: diet change;current diet
    description: Specification of major diet changes in the last six months, if yes
      the change should be specified
    title: major diet change in last six months
    examples:
    - value: yes;vegetarian diet
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - diet
    - months
    string_serialization: '{boolean};{text}'
    slot_uri: MIXS:0000266
    alias: diet_last_six_month
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  weight_loss_3_month:
    name: weight_loss_3_month
    annotations:
      Expected_value:
        tag: Expected_value
        value: weight loss specification;measurement value
      Preferred_unit:
        tag: Preferred_unit
        value: kilogram, gram
    description: Specification of weight loss in the last three months, if yes should
      be further specified to include amount of weight loss
    title: weight loss in last three months
    examples:
    - value: yes;5 kilogram
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - months
    - weight
    string_serialization: '{boolean};{float} {unit}'
    slot_uri: MIXS:0000295
    alias: weight_loss_3_month
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  ethnicity:
    name: ethnicity
    annotations:
      Expected_value:
        tag: Expected_value
        value: text recommend from Wikipedia list
    description: A category of people who identify with each other, usually on the
      basis of presumed similarities such as a common language, ancestry, history,
      society, culture, nation or social treatment within their residing area. https://en.wikipedia.org/wiki/List_of_contemporary_ethnic_groups
    title: ethnicity
    examples:
    - value: native american
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000895
    multivalued: true
    alias: ethnicity
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: string
  host_occupation:
    name: host_occupation
    description: Most frequent job performed by subject
    title: host occupation
    comments:
    - Couldn't convert host_occupation with value veterinary to integer
    - almost all host_occupation values in the NCBI biosample_set are strings, not
      integers
    examples:
    - value: veterinary
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000896
    alias: host_occupation
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: string
  pet_farm_animal:
    name: pet_farm_animal
    annotations:
      Expected_value:
        tag: Expected_value
        value: presence status;type of animal or pet
    description: Specification of presence of pets or farm animals in the environment
      of subject, if yes the animals should be specified; can include multiple animals
      present
    title: presence of pets or farm animals
    examples:
    - value: yes; 5 cats
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - animal
    - farm
    - presence
    string_serialization: '{boolean};{text}'
    slot_uri: MIXS:0000267
    multivalued: true
    alias: pet_farm_animal
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  travel_out_six_month:
    name: travel_out_six_month
    annotations:
      Expected_value:
        tag: Expected_value
        value: country name
    description: Specification of the countries travelled in the last six months;
      can include multiple travels
    title: travel outside the country in last six months
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - months
    string_serialization: '{text}'
    slot_uri: MIXS:0000268
    multivalued: true
    alias: travel_out_six_month
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  twin_sibling:
    name: twin_sibling
    description: Specification of twin sibling presence
    title: twin sibling presence
    examples:
    - value: 'yes'
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - presence
    slot_uri: MIXS:0000326
    alias: twin_sibling
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: boolean
  medic_hist_perform:
    name: medic_hist_perform
    description: Whether full medical history was collected
    title: medical history performed
    examples:
    - value: '1'
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - history
    slot_uri: MIXS:0000897
    alias: medic_hist_perform
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: boolean
  study_complt_stat:
    name: study_complt_stat
    annotations:
      Expected_value:
        tag: Expected_value
        value: YES or NO due to (1)adverse event (2) non-compliance (3) lost to follow
          up (4)other-specify
    description: Specification of study completion status, if no the reason should
      be specified
    title: study completion status
    examples:
    - value: no;non-compliance
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - status
    string_serialization: '{boolean};[adverse event|non-compliance|lost to follow
      up|other-specify]'
    slot_uri: MIXS:0000898
    alias: study_complt_stat
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  pulmonary_disord:
    name: pulmonary_disord
    description: History of pulmonary disorders; can include multiple disorders. The
      terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      lung disease (https://disease-ontology.org/?id=DOID:850)
    title: lung/pulmonary disorder
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - disorder
    slot_uri: MIXS:0000269
    multivalued: true
    alias: pulmonary_disord
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  nose_throat_disord:
    name: nose_throat_disord
    description: History of nose-throat disorders; can include multiple disorders,  The
      terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      lung disease (https://disease-ontology.org/?id=DOID:850), upper respiratory
      tract disease (https://disease-ontology.org/?id=DOID:974)
    title: nose throat disorder
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - disorder
    slot_uri: MIXS:0000270
    multivalued: true
    alias: nose_throat_disord
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  blood_blood_disord:
    name: blood_blood_disord
    description: History of blood disorders; can include multiple disorders.  The
      terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      hematopoietic system disease (https://disease-ontology.org/?id=DOID:74)
    title: blood/blood disorder
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - disorder
    slot_uri: MIXS:0000271
    multivalued: true
    alias: blood_blood_disord
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  host_pulse:
    name: host_pulse
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: beats per minute
    description: Resting pulse, measured as beats per minute
    title: host pulse
    examples:
    - value: 65 beats per minute
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000333
    alias: host_pulse
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  gestation_state:
    name: gestation_state
    description: Specification of the gestation state
    title: amniotic fluid/gestation state
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000272
    alias: gestation_state
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  maternal_health_stat:
    name: maternal_health_stat
    description: Specification of the maternal health status
    title: amniotic fluid/maternal health status
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - status
    slot_uri: MIXS:0000273
    alias: maternal_health_stat
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  foetal_health_stat:
    name: foetal_health_stat
    description: Specification of foetal health status, should also include abortion
    title: amniotic fluid/foetal health status
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - status
    slot_uri: MIXS:0000275
    alias: foetal_health_stat
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  amniotic_fluid_color:
    name: amniotic_fluid_color
    description: Specification of the color of the amniotic fluid sample
    title: amniotic fluid/color
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000276
    alias: amniotic_fluid_color
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  kidney_disord:
    name: kidney_disord
    description: History of kidney disorders; can include multiple disorders. The
      terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      kidney disease (https://disease-ontology.org/?id=DOID:557)
    title: urine/kidney disorder
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - disorder
    slot_uri: MIXS:0000277
    multivalued: true
    alias: kidney_disord
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  urogenit_tract_disor:
    name: urogenit_tract_disor
    description: History of urogenital tract disorders; can include multiple disorders.
      The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      urinary system disease (https://disease-ontology.org/?id=DOID:18)
    title: urine/urogenital tract disorder
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - disorder
    slot_uri: MIXS:0000278
    multivalued: true
    alias: urogenit_tract_disor
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: string
  urine_collect_meth:
    name: urine_collect_meth
    description: Specification of urine collection method
    title: urine/collection method
    examples:
    - value: catheter
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - method
    slot_uri: MIXS:0000899
    alias: urine_collect_meth
    owner: MimarksCHumanAssociated
    domain_of:
    - HumanAssociated
    range: UrineCollectMethEnum
  perturbation:
    name: perturbation
    annotations:
      Expected_value:
        tag: Expected_value
        value: perturbation type name;perturbation interval and duration
    description: Type of perturbation, e.g. chemical administration, physical disturbance,
      etc., coupled with perturbation regimen including how many times the perturbation
      was repeated, how long each perturbation lasted, and the start and end time
      of the entire perturbation period; can include multiple perturbation types
    title: perturbation
    examples:
    - value: antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - perturbation
    string_serialization: '{text};{Rn/start_time/end_time/duration}'
    slot_uri: MIXS:0000754
    multivalued: true
    alias: perturbation
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
  salinity:
    name: salinity
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: practical salinity unit, percentage
    description: The total concentration of all dissolved salts in a liquid or solid
      sample. While salinity can be measured by a complete chemical analysis, this
      method is difficult and time consuming. More often, it is instead derived from
      the conductivity measurement. This is known as practical salinity. These derivations
      compare the specific conductance of the sample to a salinity standard such as
      seawater
    title: salinity
    examples:
    - value: 25 practical salinity unit
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - salinity
    slot_uri: MIXS:0000183
    alias: salinity
    owner: MimarksCHumanAssociated
    domain_of:
    - Air
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  oxy_stat_samp:
    name: oxy_stat_samp
    description: Oxygenation status of sample
    title: oxygenation status of sample
    examples:
    - value: aerobic
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - oxygen
    - sample
    - status
    slot_uri: MIXS:0000753
    alias: oxy_stat_samp
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - Air
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: OxyStatSampEnum
  organism_count:
    name: organism_count
    description: 'Total cell count of any organism (or group of organisms) per gram,
      volume or area of sample, should include name of organism followed by count.
      The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should
      also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)'
    title: organism count
    examples:
    - value: total prokaryotes;3.5e7 cells per milliliter;qPCR
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text};{float} {unit};[ATP|MPN|qPCR|other]'
    slot_uri: MIXS:0000103
    multivalued: true
    alias: organism_count
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
  samp_store_temp:
    name: samp_store_temp
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: degree Celsius
    description: Temperature at which sample was stored, e.g. -80 degree Celsius
    title: sample storage temperature
    examples:
    - value: -80 degree Celsius
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - sample
    - storage
    - temperature
    slot_uri: MIXS:0000110
    alias: samp_store_temp
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  samp_store_dur:
    name: samp_store_dur
    description: Duration for which the sample was stored. Indicate the duration for
      which the sample was stored written in ISO 8601 format
    title: sample storage duration
    examples:
    - value: P1Y6M
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - duration
    - period
    - sample
    - storage
    slot_uri: MIXS:0000116
    alias: samp_store_dur
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^P(?:(?:\d+D|\d+M(?:\d+D)?|\d+Y(?:\d+M(?:\d+D)?)?)(?:T(?:\d+H(?:\d+M(?:\d+S)?)?|\d+M(?:\d+S)?|\d+S))?|T(?:\d+H(?:\d+M(?:\d+S)?)?|\d+M(?:\d+S)?|\d+S)|\d+W)$
    structured_pattern:
      syntax: ^{duration}$
      interpolated: true
      partial_match: true
  host_symbiont:
    name: host_symbiont
    description: The taxonomic name of the organism(s) found living in mutualistic,
      commensalistic, or parasitic symbiosis with the specific host. The sampled symbiont
      can have its own symbionts. For example, parasites may have hyperparasites (=parasites
      of the parasite)
    title: observed host symbionts
    examples:
    - value: flukeworms
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0001298
    multivalued: true
    alias: host_symbiont
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
  samp_store_loc:
    name: samp_store_loc
    annotations:
      Expected_value:
        tag: Expected_value
        value: location name
    description: Location at which sample was stored, usually name of a specific freezer/room
    title: sample storage location
    examples:
    - value: Freezer no:5
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - location
    - sample
    - storage
    string_serialization: '{text}'
    slot_uri: MIXS:0000755
    alias: samp_store_loc
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
  misc_param:
    name: misc_param
    annotations:
      Expected_value:
        tag: Expected_value
        value: parameter name;measurement value
    description: Any other measurement performed or parameter collected, that is not
      listed here
    title: miscellaneous parameter
    examples:
    - value: Bicarbonate ion concentration;2075 micromole per kilogram
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - parameter
    string_serialization: '{text};{float} {unit}'
    slot_uri: MIXS:0000752
    multivalued: true
    alias: misc_param
    owner: MimarksCHumanAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
class_uri: MIXS:0010009_0016003

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link", "search.suggest"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
<script src="../assets/javascripts/bundle.ad660dcc.min.js"></script>
<script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
<script src="../javascripts/tablesort.js"></script>
<script src="https://unpkg.com/mermaid@8.8.0/dist/mermaid.min.js"></script><script>mermaid.initialize({});</script></body>
</html>