
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Host-associated/">
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.3.0, mkdocs-material-8.3.9">
    
    
      
        <title>Host-associated - MIxS</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.1d29e8d0.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.cbb835fc.min.css">
        
      
      
    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#host-associated" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="MIxS" class="md-header__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MIxS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Host-associated
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="MIxS" class="md-nav__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    MIxS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        About
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Introduction
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Introduction" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Introduction
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/citation/" class="md-nav__link">
        Cite
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/contents/" class="md-nav__link">
        Contents
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#combinations" class="md-nav__link">
    Combinations
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#usages" class="md-nav__link">
    Usages
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#identifier-and-mapping-information" class="md-nav__link">
    Identifier and Mapping Information
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                
  <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Host-associated.md" title="Edit this page" class="md-content__button md-icon">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>



<h1 id="host-associated">Host-associated</h1>
<p>host-associated</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Host-associated">mixs.vocab:Host-associated</a></p>
<!-- no inheritance hierarchy -->

<h2 id="combinations">Combinations</h2>
<ul>
<li><strong>Host-associated</strong><ul>
<li><a href="../Host-associatedMIGSEukaryote/">Host-associatedMIGSEukaryote</a> [ MIGS eukaryote]</li>
<li><a href="../Host-associatedMIGSBacteria/">Host-associatedMIGSBacteria</a> [ MIGS bacteria]</li>
<li><a href="../Host-associatedMIGSPlant/">Host-associatedMIGSPlant</a> [ MIGS plant]</li>
<li><a href="../Host-associatedMIGSVirus/">Host-associatedMIGSVirus</a> [ MIGS virus]</li>
<li><a href="../Host-associatedMIGSOrg/">Host-associatedMIGSOrg</a> [ MIGS org]</li>
<li><a href="../Host-associatedMIMS/">Host-associatedMIMS</a> [ MIMS]</li>
<li><a href="../Host-associatedMIMARKSSpecimen/">Host-associatedMIMARKSSpecimen</a> [ MIMARKS specimen]</li>
<li><a href="../Host-associatedMIMARKSSurvey/">Host-associatedMIMARKSSurvey</a> [ MIMARKS survey]</li>
<li><a href="../Host-associatedMISAG/">Host-associatedMISAG</a> [ MISAG]</li>
<li><a href="../Host-associatedMIMAG/">Host-associatedMIMAG</a> [ MIMAG]</li>
<li><a href="../Host-associatedMIUVIG/">Host-associatedMIUVIG</a> [ MIUVIG]</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
<th>Info</th>
</tr>
</thead>
<tbody>
<tr>
<td>MIXS:0000009</td>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>0..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000018</td>
<td><a href="../depth/">depth</a></td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000094</td>
<td><a href="../alt/">alt</a></td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000093</td>
<td><a href="../elev/">elev</a></td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000113</td>
<td><a href="../temp/">temp</a></td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000010</td>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>0..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000011</td>
<td><a href="../collection_date/">collection_date</a></td>
<td>0..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000012</td>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>0..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000013</td>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>0..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000014</td>
<td><a href="../env_medium/">env_medium</a></td>
<td>0..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000860</td>
<td><a href="../samp_capt_status/">samp_capt_status</a></td>
<td>0..1</td>
<td>Reason for the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000753</td>
<td><a href="../oxy_stat_samp/">oxy_stat_samp</a></td>
<td>0..1</td>
<td>Oxygenation status of sample.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000249</td>
<td><a href="../samp_dis_stage/">samp_dis_stage</a></td>
<td>0..1</td>
<td>Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000862</td>
<td><a href="../host_sex/">host_sex</a></td>
<td>0..1</td>
<td>Physical sex of the host.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000858</td>
<td><a href="../biol_stat/">biol_stat</a></td>
<td>0..1</td>
<td>The level of genome modification.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000875</td>
<td><a href="../gravidity/">gravidity</a></td>
<td>0..1</td>
<td>Whether or not subject is gravid, and if yes date due or date post-conception, specifying which is used</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000116</td>
<td><a href="../samp_store_dur/">samp_store_dur</a></td>
<td>0..1</td>
<td>Duration for which the sample was stored</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000258</td>
<td><a href="../blood_press_diast/">blood_press_diast</a></td>
<td>0..1</td>
<td>Resting diastolic blood pressure, measured as mm mercury</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000259</td>
<td><a href="../blood_press_syst/">blood_press_syst</a></td>
<td>0..1</td>
<td>Resting systolic blood pressure, measured as mm mercury</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000255</td>
<td><a href="../host_age/">host_age</a></td>
<td>0..1</td>
<td>Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000274</td>
<td><a href="../host_body_temp/">host_body_temp</a></td>
<td>0..1</td>
<td>Core body temperature of the host when sample was collected</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000257</td>
<td><a href="../host_dry_mass/">host_dry_mass</a></td>
<td>0..1</td>
<td>Measurement of dry mass</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000264</td>
<td><a href="../host_height/">host_height</a></td>
<td>0..1</td>
<td>The height of subject</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000256</td>
<td><a href="../host_length/">host_length</a></td>
<td>0..1</td>
<td>The length of subject</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000263</td>
<td><a href="../host_tot_mass/">host_tot_mass</a></td>
<td>0..1</td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000183</td>
<td><a href="../salinity/">salinity</a></td>
<td>0..1</td>
<td>The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000110</td>
<td><a href="../samp_store_temp/">samp_store_temp</a></td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000250</td>
<td><a href="../host_taxid/">host_taxid</a></td>
<td>0..1</td>
<td>NCBI taxon id of the host.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000859</td>
<td><a href="../genetic_mod/">genetic_mod</a></td>
<td>0..1</td>
<td>Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000871</td>
<td><a href="../host_growth_cond/">host_growth_cond</a></td>
<td>0..1</td>
<td>Literature reference giving growth conditions of the host</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001318</td>
<td><a href="../host_subspecf_genlin/">host_subspecf_genlin</a></td>
<td>0..*</td>
<td>Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000868</td>
<td><a href="../host_body_product/">host_body_product</a></td>
<td>0..1</td>
<td>Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma or https://www.ebi.ac.uk/ols/ontologies/uberon.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000867</td>
<td><a href="../host_body_site/">host_body_site</a></td>
<td>0..1</td>
<td>Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms, please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000874</td>
<td><a href="../host_phenotype/">host_phenotype</a></td>
<td>0..1</td>
<td>Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000751</td>
<td><a href="../chem_administration/">chem_administration</a></td>
<td>0..*</td>
<td>List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000247</td>
<td><a href="../ances_data/">ances_data</a></td>
<td>0..1</td>
<td>Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000866</td>
<td><a href="../host_body_habitat/">host_body_habitat</a></td>
<td>0..1</td>
<td>Original body habitat where the sample was obtained from</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000260</td>
<td><a href="../host_color/">host_color</a></td>
<td>0..1</td>
<td>The color of host</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000248</td>
<td><a href="../host_common_name/">host_common_name</a></td>
<td>0..1</td>
<td>Common name of the host.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000869</td>
<td><a href="../host_diet/">host_diet</a></td>
<td>0..*</td>
<td>Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000365</td>
<td><a href="../host_genotype/">host_genotype</a></td>
<td>0..1</td>
<td>Observed genotype</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000251</td>
<td><a href="../host_life_stage/">host_life_stage</a></td>
<td>0..1</td>
<td>Description of life stage of host</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000261</td>
<td><a href="../host_shape/">host_shape</a></td>
<td>0..1</td>
<td>Morphological shape of host</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000861</td>
<td><a href="../host_subject_id/">host_subject_id</a></td>
<td>0..1</td>
<td>A unique identifier by which each subject can be referred to, de-identified.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000252</td>
<td><a href="../host_substrate/">host_substrate</a></td>
<td>0..1</td>
<td>The growth substrate of the host.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001298</td>
<td><a href="../host_symbiont/">host_symbiont</a></td>
<td>0..*</td>
<td>The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000755</td>
<td><a href="../samp_store_loc/">samp_store_loc</a></td>
<td>0..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000870</td>
<td><a href="../host_last_meal/">host_last_meal</a></td>
<td>0..*</td>
<td>Content of last meal and time since feeding; can include multiple values</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000752</td>
<td><a href="../misc_param/">misc_param</a></td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000103</td>
<td><a href="../organism_count/">organism_count</a></td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000754</td>
<td><a href="../perturbation/">perturbation</a></td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000872</td>
<td><a href="../host_family_relation/">host_family_relation</a></td>
<td>0..*</td>
<td>Familial relationships to other hosts in the same study; can include multiple relationships</td>
<td>None</td>
<td>.</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>

              
            </article>
            
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.b97dbffb.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.6c7ad80a.min.js"></script>
      
    
  </body>
</html>