
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://GenomicsStandardsConsortium.github.io/mixs-6-2-for-merge/Miuvig/">
      
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.4.3">
    
    
      
        <title>Checklist: Miuvig - mixs-6-2-for-merge</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.79e020e9.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    <body dir="ltr">
  
    
    
      <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script>
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#checklist-miuvig" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="mixs-6-2-for-merge" class="md-header__button md-logo" aria-label="mixs-6-2-for-merge" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            mixs-6-2-for-merge
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Checklist: Miuvig
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs-6-2-for-merge" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="mixs-6-2-for-merge" class="md-nav__button md-logo" aria-label="mixs-6-2-for-merge" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    mixs-6-2-for-merge
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs-6-2-for-merge" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Index
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="../term_list/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Full term table
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="../combinations/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Combinations
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#inheritance" class="md-nav__link">
    Inheritance
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#terms" class="md-nav__link">
    Terms
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aliases" class="md-nav__link">
    Aliases
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#identifier-and-mapping-information" class="md-nav__link">
    Identifier and Mapping Information
  </a>
  
    <nav class="md-nav" aria-label="Identifier and Mapping Information">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#schema-source" class="md-nav__link">
    Schema Source
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#linkml-source" class="md-nav__link">
    LinkML Source
  </a>
  
    <nav class="md-nav" aria-label="LinkML Source">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#direct" class="md-nav__link">
    Direct
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#induced" class="md-nav__link">
    Induced
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<h1 id="checklist-miuvig">Checklist: Miuvig</h1>
<p><em>Minimum Information About an Uncultivated Virus Genome</em></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Checklist/">Checklist</a><ul>
<li><strong>Miuvig</strong></li>
</ul>
</li>
</ul>
<h2 id="terms">Terms</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../samp_name/">samp_name</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>A local identifier or name that for the material sample used for extracting n...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../size_frac/">size_frac</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Filtering pore size used in sample preparation</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../lib_screen/">lib_screen</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Specific enrichment or screening methods applied before and/or after creating...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../source_uvig/">source_uvig</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Type of dataset from which the UViG was obtained</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../ref_db/">ref_db</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>List of database(s) used for ORF annotation, along with version number and re...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../nucl_acid_amp/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../lib_size/">lib_size</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones in the library prepared for the project</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../mid/">mid</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to spe...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../assembly_name/">assembly_name</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Name/version of the assembly provided by the submitter that is used in the ge...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../temp/">temp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Temperature of the sample at the time of sampling</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../compl_score/">compl_score</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Completeness score is typically based on either the fraction of markers found...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../trnas/">trnas</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The total number of tRNAs identified from the SAG or MAG</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../nucl_acid_ext/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../samp_size/">samp_size</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample coll...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../alt/">alt</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Heights of objects such as airplanes, space shuttles, rockets, atmospheric ba...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../estimated_size/">estimated_size</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The estimated size of the genome prior to sequencing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../source_mat_id/">source_mat_id</a></td>
<td>0..* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A unique identifier assigned to a material sample (as defined by http://rs</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extractio...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../pathogenicity/">pathogenicity</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>To what is the entity pathogenic</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../lib_reads_seqd/">lib_reads_seqd</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones sequenced from the library</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../samp_collect_device/">samp_collect_device</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The device used to collect an environmental sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../number_contig/">number_contig</a></td>
<td>1..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a giv...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../biotic_relationship/">biotic_relationship</a></td>
<td>0..1 <br/> <a href="../BIOTICRELATIONSHIPENUM/">BIOTICRELATIONSHIPENUM</a></td>
<td>Description of relationship(s) between the subject organism and other organis...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../trna_ext_software/">trna_ext_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for tRNA identification</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../lib_layout/">lib_layout</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../LIBLAYOUTENUM/">LIBLAYOUTENUM</a></td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../assembly_qual/">assembly_qual</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>The assembly quality category is based on sets of criteria outlined for each ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../ref_biomaterial/">ref_biomaterial</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Primary publication if isolated before genome publication; otherwise, primary...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../project_name/">project_name</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Name of the project within which the sequencing was organized</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../lib_vector/">lib_vector</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../host_spec_range/">host_spec_range</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>The range and diversity of host species that an organism is capable of infect...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../neg_cont_type/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../NEGCONTTYPEENUM/">NEGCONTTYPEENUM</a></td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../virus_enrich_appr/">virus_enrich_appr</a></td>
<td>1..1 <br/> <a href="../VIRUSENRICHAPPRENUM/">VIRUSENRICHAPPRENUM</a></td>
<td>List of approaches used to enrich the sample for viruses, if any</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../adapters/">adapters</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Adapters provide priming sequences for both amplification and sequencing of t...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../assembly_software/">assembly_software</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../tax_ident/">tax_ident</a></td>
<td>0..1 <br/> <a href="../TAXIDENTENUM/">TAXIDENTENUM</a></td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../annot/">annot</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool used for annotation, or for cases where annotation was provided by a com...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../pos_cont_type/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The substance, mixture, product, or apparatus used to verify that a process w...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../feat_pred/">feat_pred</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../compl_software/">compl_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for completion estimate, i</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Report the entity or entities which are in the sample or specimen s local vic...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../samp_mat_process/">samp_mat_process</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A brief description of any processing applied to the sample during or after r...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sim_search_meth/">sim_search_meth</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../host_disease_stat/">host_disease_stat</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>List of diseases with which the host has been diagnosed; can include multiple...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../depth/">depth</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The vertical distance below local surface</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../samp_collect_method/">samp_collect_method</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The method employed for collecting the sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../compl_appr/">compl_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../COMPLAPPRENUM/">COMPLAPPRENUM</a></td>
<td>The approach used to determine the completeness of a given genomic assembly, ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../specific_host/">specific_host</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../env_medium/">env_medium</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../samp_taxon_id/">samp_taxon_id</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>NCBI taxon id of the sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by the country or sea name f...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../collection_date/">collection_date</a></td>
<td>1..1 <br/> <a href="../Datetime/">Datetime</a></td>
<td>The time of sampling, either as an instance (single point in time) or interva...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../seq_meth/">seq_meth</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Sequencing machine used</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by latitude and longitude</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../elev/">elev</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Elevation of the sampling site is its height above a fixed reference point, m...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../tax_class/">tax_class</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Method used for taxonomic classification, along with reference database used,...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../experimental_factor/">experimental_factor</a></td>
<td>0..* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Variable aspects of an experiment design that can be used to describe an expe...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sort_tech/">sort_tech</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../SORTTECHENUM/">SORTTECHENUM</a></td>
<td>Method used to sort/isolate cells or particles of interest</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sc_lysis_approach/">sc_lysis_approach</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../SCLYSISAPPROACHENUM/">SCLYSISAPPROACHENUM</a></td>
<td>Method used to free DNA from interior of the cell(s) or particle(s)</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sc_lysis_method/">sc_lysis_method</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Name of the kit or standard protocol used for cell(s) or particle(s) lysis</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../wga_amp_appr/">wga_amp_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../WGAAMPAPPRENUM/">WGAAMPAPPRENUM</a></td>
<td>Method used to amplify genomic DNA in preparation for sequencing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../wga_amp_kit/">wga_amp_kit</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Kit used to amplify genomic DNA in preparation for sequencing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../bin_param/">bin_param</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../BINPARAMENUM/">BINPARAMENUM</a></td>
<td>The parameters that have been applied during the extraction of genomes from m...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../bin_software/">bin_software</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for the extraction of genomes from metagenomic datasets, where p...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../reassembly_bin/">reassembly_bin</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Boolean/">Boolean</a></td>
<td>Has an assembly been performed on a genome bin extracted from a metagenomic a...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../mag_cov_software/">mag_cov_software</a></td>
<td>0..1 <br/> <a href="../MAGCOVSOFTWAREENUM/">MAGCOVSOFTWAREENUM</a></td>
<td>Tool(s) used to determine the genome coverage if coverage is used as a binnin...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../vir_ident_software/">vir_ident_software</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for the identification of UViG as a viral genome, software or pr...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../pred_genome_type/">pred_genome_type</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Type of genome predicted for the UViG</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../pred_genome_struc/">pred_genome_struc</a></td>
<td>1..1 <br/> <a href="../PREDGENOMESTRUCENUM/">PREDGENOMESTRUCENUM</a></td>
<td>Expected structure of the viral genome</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../detec_type/">detec_type</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Type of UViG detection</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../otu_class_appr/">otu_class_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Cutoffs and approach used when clustering  species-level  OTUs</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../otu_seq_comp_appr/">otu_seq_comp_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool and thresholds used to compare sequences when computing "species-level" ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../otu_db/">otu_db</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Reference database (i</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../host_pred_appr/">host_pred_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../HOSTPREDAPPRENUM/">HOSTPREDAPPRENUM</a></td>
<td>Tool or approach used for host prediction</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../host_pred_est_acc/">host_pred_est_acc</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>For each tool or approach used for host prediction, estimated false discovery...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../associated_resource/">associated_resource</a></td>
<td>0..* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A related resource that is referenced, cited, or otherwise associated to the ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sop/">sop</a></td>
<td>0..* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, ...</td>
<td>direct</td>
</tr>
</tbody>
</table>
<h2 id="aliases">Aliases</h2>
<ul>
<li>miuvig</li>
</ul>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://w3id.org/mixs-6-2-rc</li>
</ul>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->

<h3 id="direct">Direct</h3>
<details>

<pre><code class="language-yaml">name: Miuvig
description: Minimum Information About an Uncultivated Virus Genome
title: Minimum Information About an Uncultivated Virus Genome
from_schema: https://w3id.org/mixs-6-2-rc
aliases:
- miuvig
is_a: Checklist
mixin: true
slots:
- samp_name
- size_frac
- lib_screen
- source_uvig
- ref_db
- nucl_acid_amp
- lib_size
- mid
- assembly_name
- temp
- compl_score
- trnas
- nucl_acid_ext
- samp_size
- alt
- estimated_size
- source_mat_id
- samp_vol_we_dna_ext
- pathogenicity
- lib_reads_seqd
- samp_collect_device
- number_contig
- biotic_relationship
- trna_ext_software
- lib_layout
- assembly_qual
- ref_biomaterial
- project_name
- lib_vector
- host_spec_range
- neg_cont_type
- virus_enrich_appr
- adapters
- assembly_software
- tax_ident
- annot
- pos_cont_type
- feat_pred
- compl_software
- env_local_scale
- samp_mat_process
- sim_search_meth
- host_disease_stat
- depth
- samp_collect_method
- compl_appr
- specific_host
- env_medium
- samp_taxon_id
- geo_loc_name
- collection_date
- seq_meth
- lat_lon
- elev
- env_broad_scale
- tax_class
- experimental_factor
- sort_tech
- sc_lysis_approach
- sc_lysis_method
- wga_amp_appr
- wga_amp_kit
- bin_param
- bin_software
- reassembly_bin
- mag_cov_software
- vir_ident_software
- pred_genome_type
- pred_genome_struc
- detec_type
- otu_class_appr
- otu_seq_comp_appr
- otu_db
- host_pred_appr
- host_pred_est_acc
- associated_resource
- sop
slot_usage:
  adapters:
    name: adapters
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  alt:
    name: alt
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - HostAssociated
    - MiscellaneousNaturalOrArtificialEnvironment
    - SymbiontAssociated
    recommended: true
  annot:
    name: annot
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  assembly_name:
    name: assembly_name
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  assembly_qual:
    name: assembly_qual
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    required: true
  assembly_software:
    name: assembly_software
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    required: true
  bin_param:
    name: bin_param
    domain_of:
    - Mimag
    - Miuvig
    recommended: true
  bin_software:
    name: bin_software
    domain_of:
    - Mimag
    - Miuvig
    recommended: true
  biotic_relationship:
    name: biotic_relationship
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - MimarksC
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  compl_appr:
    name: compl_appr
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    recommended: true
  compl_score:
    name: compl_score
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    recommended: true
  compl_software:
    name: compl_software
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    required: false
    recommended: false
  depth:
    name: depth
    examples:
    - value: 10 meter
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    recommended: true
  detec_type:
    name: detec_type
    domain_of:
    - Miuvig
    required: true
  elev:
    name: elev
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - HostAssociated
    - HydrocarbonResourcesCores
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - Water
    recommended: true
  estimated_size:
    name: estimated_size
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Miuvig
    required: false
    recommended: false
  experimental_factor:
    name: experimental_factor
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  feat_pred:
    name: feat_pred
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    recommended: true
  host_disease_stat:
    name: host_disease_stat
    examples:
    - value: rabies [DOID:11260]
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    recommended: true
  host_pred_appr:
    name: host_pred_appr
    domain_of:
    - Miuvig
    recommended: true
  host_pred_est_acc:
    name: host_pred_est_acc
    domain_of:
    - Miuvig
    recommended: true
  host_spec_range:
    name: host_spec_range
    domain_of:
    - MigsBa
    - MigsEu
    - MigsPl
    - MigsVi
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  lib_layout:
    name: lib_layout
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  lib_reads_seqd:
    name: lib_reads_seqd
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  lib_screen:
    name: lib_screen
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  lib_size:
    name: lib_size
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  lib_vector:
    name: lib_vector
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  mag_cov_software:
    name: mag_cov_software
    domain_of:
    - Mimag
    - Miuvig
    required: false
    recommended: false
  mid:
    name: mid
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  nucl_acid_amp:
    name: nucl_acid_amp
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  nucl_acid_ext:
    name: nucl_acid_ext
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  number_contig:
    name: number_contig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    required: true
  otu_class_appr:
    name: otu_class_appr
    domain_of:
    - Miuvig
    recommended: true
  otu_db:
    name: otu_db
    domain_of:
    - Miuvig
    recommended: true
  otu_seq_comp_appr:
    name: otu_seq_comp_appr
    domain_of:
    - Miuvig
    recommended: true
  pathogenicity:
    name: pathogenicity
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  pred_genome_struc:
    name: pred_genome_struc
    domain_of:
    - Miuvig
    required: true
  pred_genome_type:
    name: pred_genome_type
    domain_of:
    - Miuvig
    required: true
  reassembly_bin:
    name: reassembly_bin
    domain_of:
    - Mimag
    - Miuvig
    recommended: true
  ref_biomaterial:
    name: ref_biomaterial
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    required: false
    recommended: false
  ref_db:
    name: ref_db
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    recommended: true
  samp_collect_device:
    name: samp_collect_device
    examples:
    - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  samp_collect_method:
    name: samp_collect_method
    examples:
    - value: swabbing
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  samp_mat_process:
    name: samp_mat_process
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  samp_size:
    name: samp_size
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    required: false
    recommended: false
  sc_lysis_approach:
    name: sc_lysis_approach
    domain_of:
    - Misag
    - Miuvig
    recommended: true
  sc_lysis_method:
    name: sc_lysis_method
    domain_of:
    - Misag
    - Miuvig
    recommended: true
  sim_search_meth:
    name: sim_search_meth
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    recommended: true
  size_frac:
    name: size_frac
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    recommended: true
  sop:
    name: sop
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  sort_tech:
    name: sort_tech
    domain_of:
    - Misag
    - Miuvig
    recommended: true
  source_mat_id:
    name: source_mat_id
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - SymbiontAssociated
    recommended: true
  source_uvig:
    name: source_uvig
    domain_of:
    - Miuvig
    required: true
  specific_host:
    name: specific_host
    domain_of:
    - MigsBa
    - MigsEu
    - MigsPl
    - MigsVi
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  tax_class:
    name: tax_class
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    recommended: true
  tax_ident:
    name: tax_ident
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    required: false
    recommended: false
  temp:
    name: temp
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    recommended: true
  trna_ext_software:
    name: trna_ext_software
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    required: false
    recommended: false
  trnas:
    name: trnas
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    required: false
    recommended: false
  vir_ident_software:
    name: vir_ident_software
    domain_of:
    - Miuvig
    required: true
  virus_enrich_appr:
    name: virus_enrich_appr
    domain_of:
    - MigsVi
    - Miuvig
    required: true
  wga_amp_appr:
    name: wga_amp_appr
    domain_of:
    - Misag
    - Miuvig
    recommended: true
  wga_amp_kit:
    name: wga_amp_kit
    domain_of:
    - Misag
    - Miuvig
    recommended: true
class_uri: MIXS:0010012

</code></pre>

</details>

<h3 id="induced">Induced</h3>
<details>

<pre><code class="language-yaml">name: Miuvig
description: Minimum Information About an Uncultivated Virus Genome
title: Minimum Information About an Uncultivated Virus Genome
from_schema: https://w3id.org/mixs-6-2-rc
aliases:
- miuvig
is_a: Checklist
mixin: true
slot_usage:
  adapters:
    name: adapters
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  alt:
    name: alt
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - HostAssociated
    - MiscellaneousNaturalOrArtificialEnvironment
    - SymbiontAssociated
    recommended: true
  annot:
    name: annot
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  assembly_name:
    name: assembly_name
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  assembly_qual:
    name: assembly_qual
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    required: true
  assembly_software:
    name: assembly_software
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    required: true
  bin_param:
    name: bin_param
    domain_of:
    - Mimag
    - Miuvig
    recommended: true
  bin_software:
    name: bin_software
    domain_of:
    - Mimag
    - Miuvig
    recommended: true
  biotic_relationship:
    name: biotic_relationship
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - MimarksC
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  compl_appr:
    name: compl_appr
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    recommended: true
  compl_score:
    name: compl_score
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    recommended: true
  compl_software:
    name: compl_software
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    required: false
    recommended: false
  depth:
    name: depth
    examples:
    - value: 10 meter
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    recommended: true
  detec_type:
    name: detec_type
    domain_of:
    - Miuvig
    required: true
  elev:
    name: elev
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - HostAssociated
    - HydrocarbonResourcesCores
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - Water
    recommended: true
  estimated_size:
    name: estimated_size
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Miuvig
    required: false
    recommended: false
  experimental_factor:
    name: experimental_factor
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  feat_pred:
    name: feat_pred
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    recommended: true
  host_disease_stat:
    name: host_disease_stat
    examples:
    - value: rabies [DOID:11260]
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    recommended: true
  host_pred_appr:
    name: host_pred_appr
    domain_of:
    - Miuvig
    recommended: true
  host_pred_est_acc:
    name: host_pred_est_acc
    domain_of:
    - Miuvig
    recommended: true
  host_spec_range:
    name: host_spec_range
    domain_of:
    - MigsBa
    - MigsEu
    - MigsPl
    - MigsVi
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  lib_layout:
    name: lib_layout
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  lib_reads_seqd:
    name: lib_reads_seqd
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  lib_screen:
    name: lib_screen
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  lib_size:
    name: lib_size
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  lib_vector:
    name: lib_vector
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  mag_cov_software:
    name: mag_cov_software
    domain_of:
    - Mimag
    - Miuvig
    required: false
    recommended: false
  mid:
    name: mid
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  nucl_acid_amp:
    name: nucl_acid_amp
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  nucl_acid_ext:
    name: nucl_acid_ext
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  number_contig:
    name: number_contig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    required: true
  otu_class_appr:
    name: otu_class_appr
    domain_of:
    - Miuvig
    recommended: true
  otu_db:
    name: otu_db
    domain_of:
    - Miuvig
    recommended: true
  otu_seq_comp_appr:
    name: otu_seq_comp_appr
    domain_of:
    - Miuvig
    recommended: true
  pathogenicity:
    name: pathogenicity
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  pred_genome_struc:
    name: pred_genome_struc
    domain_of:
    - Miuvig
    required: true
  pred_genome_type:
    name: pred_genome_type
    domain_of:
    - Miuvig
    required: true
  reassembly_bin:
    name: reassembly_bin
    domain_of:
    - Mimag
    - Miuvig
    recommended: true
  ref_biomaterial:
    name: ref_biomaterial
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    required: false
    recommended: false
  ref_db:
    name: ref_db
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    recommended: true
  samp_collect_device:
    name: samp_collect_device
    examples:
    - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  samp_collect_method:
    name: samp_collect_method
    examples:
    - value: swabbing
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  samp_mat_process:
    name: samp_mat_process
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  samp_size:
    name: samp_size
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    recommended: true
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    required: false
    recommended: false
  sc_lysis_approach:
    name: sc_lysis_approach
    domain_of:
    - Misag
    - Miuvig
    recommended: true
  sc_lysis_method:
    name: sc_lysis_method
    domain_of:
    - Misag
    - Miuvig
    recommended: true
  sim_search_meth:
    name: sim_search_meth
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    recommended: true
  size_frac:
    name: size_frac
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    recommended: true
  sop:
    name: sop
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    recommended: true
  sort_tech:
    name: sort_tech
    domain_of:
    - Misag
    - Miuvig
    recommended: true
  source_mat_id:
    name: source_mat_id
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - SymbiontAssociated
    recommended: true
  source_uvig:
    name: source_uvig
    domain_of:
    - Miuvig
    required: true
  specific_host:
    name: specific_host
    domain_of:
    - MigsBa
    - MigsEu
    - MigsPl
    - MigsVi
    - Miuvig
    - Agriculture
    required: false
    recommended: false
  tax_class:
    name: tax_class
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    recommended: true
  tax_ident:
    name: tax_ident
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    required: false
    recommended: false
  temp:
    name: temp
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    recommended: true
  trna_ext_software:
    name: trna_ext_software
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    required: false
    recommended: false
  trnas:
    name: trnas
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    required: false
    recommended: false
  vir_ident_software:
    name: vir_ident_software
    domain_of:
    - Miuvig
    required: true
  virus_enrich_appr:
    name: virus_enrich_appr
    domain_of:
    - MigsVi
    - Miuvig
    required: true
  wga_amp_appr:
    name: wga_amp_appr
    domain_of:
    - Misag
    - Miuvig
    recommended: true
  wga_amp_kit:
    name: wga_amp_kit
    domain_of:
    - Misag
    - Miuvig
    recommended: true
attributes:
  samp_name:
    name: samp_name
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: ''
    description: A local identifier or name that for the material sample used for
      extracting nucleic acids, and subsequent sequencing. It can refer either to
      the original material collected or to any derived sub-samples. It can have any
      format, but we suggest that you make it concise, unique and consistent within
      your lab, and as informative as possible. INSDC requires every sample name from
      a single Submitter to be unique. Use of a globally unique identifier for the
      field source_mat_id is recommended in addition to sample_name
    title: sample name
    examples:
    - value: ISDsoil1
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - sample
    slot_uri: MIXS:0001107
    multivalued: false
    alias: samp_name
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
  size_frac:
    name: size_frac
    description: Filtering pore size used in sample preparation
    title: size fraction selected
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{float}-{float} {unit}'
    slot_uri: MIXS:0000017
    multivalued: false
    alias: size_frac
    owner: Miuvig
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  lib_screen:
    name: lib_screen
    description: Specific enrichment or screening methods applied before and/or after
      creating libraries
    title: library screening strategy
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text}'
    slot_uri: MIXS:0000043
    multivalued: false
    alias: lib_screen
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  source_uvig:
    name: source_uvig
    description: Type of dataset from which the UViG was obtained
    title: source of UViGs
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '[metagenome (not viral targeted)|viral fraction metagenome
      (virome)|sequence-targeted metagenome|metatranscriptome (not viral targeted)|viral
      fraction RNA metagenome (RNA virome)|sequence-targeted RNA metagenome|microbial
      single amplified genome (SAG)|viral single amplified genome (vSAG)|isolate microbial
      genome|other]'
    slot_uri: MIXS:0000035
    multivalued: false
    alias: source_uvig
    owner: Miuvig
    domain_of:
    - Miuvig
    range: string
    required: true
  ref_db:
    name: ref_db
    description: List of database(s) used for ORF annotation, along with version number
      and reference to website or publication
    title: reference database(s)
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{database};{version};{reference}'
    slot_uri: MIXS:0000062
    multivalued: false
    alias: ref_db
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  nucl_acid_amp:
    name: nucl_acid_amp
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the enzymatic amplification (PCR,
      TMA, NASBA) of specific nucleic acids
    title: nucleic acid amplification
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000038
    multivalued: false
    alias: nucl_acid_amp
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  lib_size:
    name: lib_size
    description: Total number of clones in the library prepared for the project
    title: library size
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000039
    multivalued: false
    alias: lib_size
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: integer
    recommended: true
  mid:
    name: mid
    description: Molecular barcodes, called Multiplex Identifiers (MIDs), that are
      used to specifically tag unique samples in a sequencing run. Sequence should
      be reported in uppercase letters
    title: multiplex identifiers
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000047
    multivalued: false
    alias: mid
    owner: Miuvig
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
    pattern: ^[ACGTRKSYMWBHDVN]+$
    structured_pattern:
      syntax: ^{ambiguous_nucleotides}$
      interpolated: true
      partial_match: true
  assembly_name:
    name: assembly_name
    description: Name/version of the assembly provided by the submitter that is used
      in the genome browsers and in the community
    title: assembly name
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text} {text}'
    slot_uri: MIXS:0000057
    multivalued: false
    alias: assembly_name
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  temp:
    name: temp
    description: Temperature of the sample at the time of sampling
    title: temperature
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000113
    multivalued: false
    alias: temp
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  compl_score:
    name: compl_score
    description: 'Completeness score is typically based on either the fraction of
      markers found as compared to a database or the percent of a genome found as
      compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium
      Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated
      completeness scores'
    title: completeness score
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '[high|med|low];{percentage}'
    slot_uri: MIXS:0000069
    multivalued: false
    alias: compl_score
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    range: string
    recommended: true
  trnas:
    name: trnas
    description: The total number of tRNAs identified from the SAG or MAG
    title: number of standard tRNAs extracted
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{integer}'
    slot_uri: MIXS:0000067
    multivalued: false
    alias: trnas
    owner: Miuvig
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    range: string
    required: false
    recommended: false
  nucl_acid_ext:
    name: nucl_acid_ext
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the material separation to recover
      the nucleic acid fraction from a sample
    title: nucleic acid extraction
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000037
    multivalued: false
    alias: nucl_acid_ext
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  samp_size:
    name: samp_size
    description: The total amount or size (volume (ml), mass (g) or area (m2) ) of
      sample collected
    title: amount or size of sample collected
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000001
    multivalued: false
    alias: samp_size
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  alt:
    name: alt
    description: Heights of objects such as airplanes, space shuttles, rockets, atmospheric
      balloons and heights of places such as atmospheric layers and clouds. It is
      used to measure the height of an object which is above the earth's surface.
      In this context, the altitude measurement is the vertical distance between the
      earth's surface above sea level and the sampled position in the air
    title: altitude
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000094
    multivalued: false
    alias: alt
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - HostAssociated
    - MiscellaneousNaturalOrArtificialEnvironment
    - SymbiontAssociated
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  estimated_size:
    name: estimated_size
    description: The estimated size of the genome prior to sequencing. Of particular
      importance in the sequencing of (eukaryotic) genome which could remain in draft
      form for a long or unspecified period
    title: estimated size
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{integer} bp'
    slot_uri: MIXS:0000024
    multivalued: false
    alias: estimated_size
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Miuvig
    range: string
    required: false
    recommended: false
  source_mat_id:
    name: source_mat_id
    description: A unique identifier assigned to a material sample (as defined by
      http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular
      digital record of a material sample) used for extracting nucleic acids, and
      subsequent sequencing. The identifier can refer either to the original material
      collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher,
      /bio_material, or /culture_collection may or may not share the same value as
      the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id
      may both contain 'UAM:Herps:14' , referring to both the specimen voucher and
      sampled tissue with the same identifier. However, the /culture_collection qualifier
      may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id
      would refer to an identifier from some derived culture from which the nucleic
      acids were extracted (e.g. xatc123 or ark:/2154/R2)
    title: source material identifiers
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text}'
    slot_uri: MIXS:0000026
    multivalued: true
    alias: source_mat_id
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - SymbiontAssociated
    range: string
    recommended: true
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    description: 'Volume (ml) or mass (g) of total collected sample processed for
      DNA extraction. Note: total sample collected should be entered under the term
      Sample Size (MIXS:0000001)'
    title: sample volume or weight for DNA extraction
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000111
    multivalued: false
    alias: samp_vol_we_dna_ext
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: false
    recommended: false
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  pathogenicity:
    name: pathogenicity
    description: To what is the entity pathogenic
    title: known pathogenicity
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text}'
    slot_uri: MIXS:0000027
    multivalued: false
    alias: pathogenicity
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    range: string
    required: false
    recommended: false
  lib_reads_seqd:
    name: lib_reads_seqd
    description: Total number of clones sequenced from the library
    title: library reads sequenced
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000040
    multivalued: false
    alias: lib_reads_seqd
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: integer
    recommended: true
  samp_collect_device:
    name: samp_collect_device
    description: The device used to collect an environmental sample. This field accepts
      terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO).
      This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)
    title: sample collection device
    examples:
    - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000002
    multivalued: false
    alias: samp_collect_device
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
  number_contig:
    name: number_contig
    description: Total number of contigs in the cleaned/submitted assembly that makes
      up a given genome, SAG, MAG, or UViG
    title: number of contigs
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000060
    multivalued: false
    alias: number_contig
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: integer
    required: true
  biotic_relationship:
    name: biotic_relationship
    description: Description of relationship(s) between the subject organism and other
      organism(s) it is associated with. E.g., parasite on species X; mutualist with
      species Y. The target organism is the subject of the relationship, and the other
      organism(s) is the object
    title: observed biotic relationship
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000028
    multivalued: false
    alias: biotic_relationship
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - MimarksC
    - Miuvig
    - Agriculture
    range: BIOTIC_RELATIONSHIP_ENUM
    required: false
    recommended: false
  trna_ext_software:
    name: trna_ext_software
    description: Tools used for tRNA identification
    title: tRNA extraction software
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000068
    multivalued: false
    alias: trna_ext_software
    owner: Miuvig
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    range: string
    required: false
    recommended: false
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  lib_layout:
    name: lib_layout
    description: Specify whether to expect single, paired, or other configuration
      of reads
    title: library layout
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000041
    multivalued: false
    alias: lib_layout
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: LIB_LAYOUT_ENUM
    recommended: true
  assembly_qual:
    name: assembly_qual
    description: 'The assembly quality category is based on sets of criteria outlined
      for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated,
      contiguous sequence per replicon without gaps or ambiguities with a consensus
      error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments
      where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes
      and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no
      review of assembly other than reporting of standard assembly statistics. Low
      Quality Draft:Many fragments with little to no review of assembly other than
      reporting of standard assembly statistics. Assembly statistics include, but
      are not limited to total assembly size, number of contigs, contig N50/L50, and
      maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence
      per replicon without gaps or ambiguities, with extensive manual review and editing
      to annotate putative gene functions and transcriptional units. High-quality
      draft genome: One or multiple fragments, totaling   90% of the expected genome
      or replicon sequence or predicted complete. Genome fragment(s): One or multiple
      fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for
      which no genome size could be estimated'
    title: assembly quality
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '[Finished genome|High-quality draft genome|Medium-quality
      draft genome|Low-quality draft genome|Genome fragment(s)]'
    slot_uri: MIXS:0000056
    multivalued: false
    alias: assembly_qual
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    required: true
  ref_biomaterial:
    name: ref_biomaterial
    description: Primary publication if isolated before genome publication; otherwise,
      primary genome report
    title: reference for biomaterial
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000025
    multivalued: false
    alias: ref_biomaterial
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    required: false
    recommended: false
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  project_name:
    name: project_name
    description: Name of the project within which the sequencing was organized
    title: project name
    examples:
    - value: Forest soil metagenome
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - project
    slot_uri: MIXS:0000092
    multivalued: false
    alias: project_name
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
  lib_vector:
    name: lib_vector
    description: Cloning vector type(s) used in construction of libraries
    title: library vector
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text}'
    slot_uri: MIXS:0000042
    multivalued: false
    alias: lib_vector
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  host_spec_range:
    name: host_spec_range
    description: The range and diversity of host species that an organism is capable
      of infecting, defined by NCBI taxonomy identifier
    title: host specificity or range
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{integer}'
    slot_uri: MIXS:0000030
    multivalued: true
    alias: host_spec_range
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsPl
    - MigsVi
    - Miuvig
    - Agriculture
    range: string
    required: false
    recommended: false
  neg_cont_type:
    name: neg_cont_type
    annotations:
      Expected_value:
        tag: Expected_value
        value: enumeration or text
    description: The substance or equipment used as a negative control in an investigation
    title: negative control type
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - type
    slot_uri: MIXS:0001321
    multivalued: false
    alias: neg_cont_type
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: NEG_CONT_TYPE_ENUM
    recommended: true
  virus_enrich_appr:
    name: virus_enrich_appr
    description: List of approaches used to enrich the sample for viruses, if any
    title: virus enrichment approach
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000036
    alias: virus_enrich_appr
    owner: Miuvig
    domain_of:
    - MigsVi
    - Miuvig
    range: VIRUS_ENRICH_APPR_ENUM
    required: true
  adapters:
    name: adapters
    description: Adapters provide priming sequences for both amplification and sequencing
      of the sample-library fragments. Both adapters should be reported; in uppercase
      letters
    title: adapters
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{dna};{dna}'
    slot_uri: MIXS:0000048
    multivalued: false
    alias: adapters
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  assembly_software:
    name: assembly_software
    description: Tool(s) used for assembly, including version number and parameters
    title: assembly software
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000058
    multivalued: false
    alias: assembly_software
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  tax_ident:
    name: tax_ident
    description: The phylogenetic marker(s) used to assign an organism name to the
      SAG or MAG
    title: taxonomic identity marker
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000053
    alias: tax_ident
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    range: TAX_IDENT_ENUM
    required: false
    recommended: false
  annot:
    name: annot
    description: Tool used for annotation, or for cases where annotation was provided
      by a community jamboree or model organism database rather than by a specific
      submitter
    title: annotation
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text}'
    slot_uri: MIXS:0000059
    multivalued: false
    alias: annot
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    required: false
    recommended: false
  pos_cont_type:
    name: pos_cont_type
    description: The substance, mixture, product, or apparatus used to verify that
      a process which is part of an investigation delivers a true positive
    title: positive control type
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - type
    string_serialization: '{term} or {text}'
    slot_uri: MIXS:0001322
    multivalued: false
    alias: pos_cont_type
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  feat_pred:
    name: feat_pred
    description: Method used to predict UViGs features such as ORFs, integration site,
      etc
    title: feature prediction
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000061
    multivalued: false
    alias: feat_pred
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  compl_software:
    name: compl_software
    description: Tools used for completion estimate, i.e. checkm, anvi'o, busco
    title: completeness software
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{software};{version}'
    slot_uri: MIXS:0000070
    multivalued: false
    alias: compl_software
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    range: string
    required: false
    recommended: false
  env_local_scale:
    name: env_local_scale
    annotations:
      Expected_value:
        tag: Expected_value
        value: Environmental entities having causal influences upon the entity at
          time of sampling
    description: 'Report the entity or entities which are in the sample or specimen
      s local vicinity and which you believe have significant causal influences on
      your sample or specimen. We recommend using EnvO terms which are of smaller
      spatial grain than your entry for env_broad_scale. Terms, such as anatomical
      sites, from other OBO Library ontologies which interoperate with EnvO (e.g.
      UBERON) are accepted in this field. EnvO documentation about how to use the
      field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: local environmental context
    examples:
    - value: hillside [ENVO:01000333]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - context
    - environmental
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000013
    multivalued: false
    alias: env_local_scale
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
  samp_mat_process:
    name: samp_mat_process
    description: A brief description of any processing applied to the sample during
      or after retrieving the sample from environment, or a link to the relevant protocol(s)
      performed
    title: sample material processing
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000016
    multivalued: false
    alias: samp_mat_process
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  sim_search_meth:
    name: sim_search_meth
    description: Tool used to compare ORFs with database, along with version and cutoffs
      used
    title: similarity search method
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000063
    multivalued: false
    alias: sim_search_meth
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  host_disease_stat:
    name: host_disease_stat
    description: List of diseases with which the host has been diagnosed; can include
      multiple diagnoses. The value of the field depends on host; for humans the terms
      should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org,
      non-human host diseases are free text
    title: host disease status
    examples:
    - value: rabies [DOID:11260]
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000031
    multivalued: false
    alias: host_disease_stat
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    range: string
    recommended: true
  depth:
    name: depth
    description: The vertical distance below local surface. For sediment or soil samples
      depth is measured from sediment or soil surface, respectively. Depth can be
      reported as an interval for subsurface samples
    title: depth
    examples:
    - value: 10 meter
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000018
    multivalued: false
    alias: depth
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  samp_collect_method:
    name: samp_collect_method
    description: The method employed for collecting the sample
    title: sample collection method
    examples:
    - value: swabbing
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0001225
    multivalued: false
    alias: samp_collect_method
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$|([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}|{text}$
      interpolated: true
      partial_match: true
  compl_appr:
    name: compl_appr
    description: The approach used to determine the completeness of a given genomic
      assembly, which would typically make use of a set of conserved marker genes
      or a closely related reference genome. For UViG completeness, include reference
      genome or group used, and contig feature suggesting a complete genome
    title: completeness approach
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000071
    alias: compl_appr
    owner: Miuvig
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    range: COMPL_APPR_ENUM
    recommended: true
  specific_host:
    name: specific_host
    description: Report the host's taxonomic name and/or NCBI taxonomy ID
    title: host scientific name
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text}|{NCBI taxid}'
    slot_uri: MIXS:0000029
    multivalued: false
    alias: specific_host
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsPl
    - MigsVi
    - Miuvig
    - Agriculture
    range: string
    required: false
    recommended: false
  env_medium:
    name: env_medium
    description: 'Report the environmental material(s) immediately surrounding the
      sample or specimen at the time of sampling. We recommend using subclasses of
      ''environmental material'' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO
      documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
      . Terms from other OBO ontologies are permissible as long as they reference
      mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
      (e.g. a tree, a leaf, a table top)'
    title: environmental medium
    examples:
    - value: bluegrass field soil [ENVO:00005789]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - environmental
    slot_uri: MIXS:0000014
    multivalued: false
    alias: env_medium
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
    structured_pattern:
      syntax: ^{termLabel} \[{termID}\]$
      interpolated: true
      partial_match: true
  samp_taxon_id:
    name: samp_taxon_id
    description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa
      sample. Use 'synthetic metagenome  for mock community/positive controls, or
      'blank sample' for negative controls
    title: taxonomy ID of DNA sample
    examples:
    - value: Gut Metagenome [NCBITaxon:749906]
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - dna
    - identifier
    - sample
    - taxon
    slot_uri: MIXS:0001320
    multivalued: false
    alias: samp_taxon_id
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[NCBITaxon:\d+\]$
    structured_pattern:
      syntax: ^{text} \[{NCBItaxon_id}\]$
      interpolated: true
      partial_match: true
  geo_loc_name:
    name: geo_loc_name
    description: The geographical origin of the sample as defined by the country or
      sea name followed by specific region name. Country or sea names should be chosen
      from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology
      (http://purl.bioontology.org/ontology/GAZ)
    title: geographic location (country and/or sea,region)
    examples:
    - value: 'USA: Maryland, Bethesda'
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - geographic
    - location
    slot_uri: MIXS:0000010
    multivalued: false
    alias: geo_loc_name
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: string
    required: true
    pattern: '^([^\s-]{1,2}|[^\s-]+.+[^\s-]+): ([^\s-]{1,2}|[^\s-]+.+[^\s-]+), ([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$'
    structured_pattern:
      syntax: '^{text}: {text}, {text}$'
      interpolated: true
      partial_match: true
  collection_date:
    name: collection_date
    description: 'The time of sampling, either as an instance (single point in time)
      or interval. In case no exact time is available, the date/time can be right
      truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10;
      2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant'
    title: collection date
    examples:
    - value: '2013-03-25T12:42:31+01:00'
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - date
    slot_uri: MIXS:0000011
    multivalued: false
    alias: collection_date
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: datetime
    required: true
  seq_meth:
    name: seq_meth
    description: Sequencing machine used. Where possible the term should be taken
      from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)
    title: sequencing method
    examples:
    - value: 454 Genome Sequencer FLX [OBI:0000702]
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - method
    slot_uri: MIXS:0000050
    multivalued: false
    alias: seq_meth
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+)|(([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\])$
    structured_pattern:
      syntax: ^{text}|({termLabel} \[{termID}\])$
      interpolated: true
      partial_match: true
  lat_lon:
    name: lat_lon
    annotations:
      Expected_value:
        tag: Expected_value
        value: decimal degrees,  limit to 8 decimal points
    description: The geographical origin of the sample as defined by latitude and
      longitude. The values should be reported in decimal degrees and in WGS84 system
    title: geographic location (latitude and longitude)
    examples:
    - value: 50.586825 6.408977
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - geographic
    - location
    string_serialization: '{float} {float}'
    slot_uri: MIXS:0000009
    alias: lat_lon
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: string
    required: true
    pattern: ^(-?((?:[0-8]?[0-9](?:\.\d{0,8})?)|90)) -?[0-9]+(?:\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$
    structured_pattern:
      syntax: ^{lat} {lon}$
      interpolated: true
      partial_match: true
  elev:
    name: elev
    description: Elevation of the sampling site is its height above a fixed reference
      point, most commonly the mean sea level. Elevation is mainly used when referring
      to points on the earth's surface, while altitude is used for points above the
      surface, such as an aircraft in flight or a spacecraft in orbit
    title: elevation
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000093
    multivalued: false
    alias: elev
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - HostAssociated
    - HydrocarbonResourcesCores
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  env_broad_scale:
    name: env_broad_scale
    description: 'Report the major environmental system the sample or specimen came
      from. The system(s) identified should have a coarse spatial grain, to provide
      the general environmental context of where the sampling was done (e.g. in the
      desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428.
      EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: broad-scale environmental context
    examples:
    - value: rangeland biome [ENVO:01000247]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - context
    - environmental
    slot_uri: MIXS:0000012
    multivalued: false
    alias: env_broad_scale
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
    structured_pattern:
      syntax: ^{termLabel} \[{termID}\]$
      interpolated: true
      partial_match: true
  tax_class:
    name: tax_class
    description: Method used for taxonomic classification, along with reference database
      used, classification rank, and thresholds used to classify new genomes
    title: taxonomic classification
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000064
    multivalued: false
    alias: tax_class
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  experimental_factor:
    name: experimental_factor
    description: Variable aspects of an experiment design that can be used to describe
      an experiment, or set of experiments, in an increasingly detailed manner. This
      field accepts ontology terms from Experimental Factor Ontology (EFO) and/or
      Ontology for Biomedical Investigations (OBI)
    title: experimental factor
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000008
    multivalued: true
    alias: experimental_factor
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^\S+.*\S+ \[[a-zA-Z]{2,}:\d+\]$
  sort_tech:
    name: sort_tech
    description: Method used to sort/isolate cells or particles of interest
    title: sorting technology
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000075
    multivalued: false
    alias: sort_tech
    owner: Miuvig
    domain_of:
    - Misag
    - Miuvig
    range: SORT_TECH_ENUM
    recommended: true
  sc_lysis_approach:
    name: sc_lysis_approach
    description: Method used to free DNA from interior of the cell(s) or particle(s)
    title: single cell or viral particle lysis approach
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000076
    multivalued: false
    alias: sc_lysis_approach
    owner: Miuvig
    domain_of:
    - Misag
    - Miuvig
    range: SC_LYSIS_APPROACH_ENUM
    recommended: true
  sc_lysis_method:
    name: sc_lysis_method
    description: Name of the kit or standard protocol used for cell(s) or particle(s)
      lysis
    title: single cell or viral particle lysis kit protocol
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text}'
    slot_uri: MIXS:0000054
    multivalued: false
    alias: sc_lysis_method
    owner: Miuvig
    domain_of:
    - Misag
    - Miuvig
    range: string
    recommended: true
  wga_amp_appr:
    name: wga_amp_appr
    description: Method used to amplify genomic DNA in preparation for sequencing
    title: WGA amplification approach
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000055
    multivalued: false
    alias: wga_amp_appr
    owner: Miuvig
    domain_of:
    - Misag
    - Miuvig
    range: WGA_AMP_APPR_ENUM
    recommended: true
  wga_amp_kit:
    name: wga_amp_kit
    description: Kit used to amplify genomic DNA in preparation for sequencing
    title: WGA amplification kit
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{text}'
    slot_uri: MIXS:0000006
    multivalued: false
    alias: wga_amp_kit
    owner: Miuvig
    domain_of:
    - Misag
    - Miuvig
    range: string
    recommended: true
  bin_param:
    name: bin_param
    description: The parameters that have been applied during the extraction of genomes
      from metagenomic datasets
    title: binning parameters
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000077
    alias: bin_param
    owner: Miuvig
    domain_of:
    - Mimag
    - Miuvig
    range: BIN_PARAM_ENUM
    recommended: true
  bin_software:
    name: bin_software
    description: Tool(s) used for the extraction of genomes from metagenomic datasets,
      where possible include a product ID (PID) of the tool(s) used
    title: binning software
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{software};{version}{PID}'
    slot_uri: MIXS:0000078
    multivalued: false
    alias: bin_software
    owner: Miuvig
    domain_of:
    - Mimag
    - Miuvig
    range: string
    recommended: true
  reassembly_bin:
    name: reassembly_bin
    description: Has an assembly been performed on a genome bin extracted from a metagenomic
      assembly?
    title: reassembly post binning
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000079
    multivalued: false
    alias: reassembly_bin
    owner: Miuvig
    domain_of:
    - Mimag
    - Miuvig
    range: boolean
    recommended: true
  mag_cov_software:
    name: mag_cov_software
    description: Tool(s) used to determine the genome coverage if coverage is used
      as a binning parameter in the extraction of genomes from metagenomic datasets
    title: MAG coverage software
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000080
    multivalued: false
    alias: mag_cov_software
    owner: Miuvig
    domain_of:
    - Mimag
    - Miuvig
    range: MAG_COV_SOFTWARE_ENUM
    required: false
    recommended: false
  vir_ident_software:
    name: vir_ident_software
    description: Tool(s) used for the identification of UViG as a viral genome, software
      or protocol name including version number, parameters, and cutoffs used
    title: viral identification software
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000081
    multivalued: false
    alias: vir_ident_software
    owner: Miuvig
    domain_of:
    - Miuvig
    range: string
    required: true
  pred_genome_type:
    name: pred_genome_type
    description: Type of genome predicted for the UViG
    title: predicted genome type
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '[DNA|dsDNA|ssDNA|RNA|dsRNA|ssRNA|ssRNA (+)|ssRNA (-)|mixed|uncharacterized]'
    slot_uri: MIXS:0000082
    multivalued: false
    alias: pred_genome_type
    owner: Miuvig
    domain_of:
    - Miuvig
    range: string
    required: true
  pred_genome_struc:
    name: pred_genome_struc
    description: Expected structure of the viral genome
    title: predicted genome structure
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000083
    multivalued: false
    alias: pred_genome_struc
    owner: Miuvig
    domain_of:
    - Miuvig
    range: PRED_GENOME_STRUC_ENUM
    required: true
  detec_type:
    name: detec_type
    description: Type of UViG detection
    title: detection type
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '[independent sequence (UViG)|provirus (UpViG)]'
    slot_uri: MIXS:0000084
    multivalued: false
    alias: detec_type
    owner: Miuvig
    domain_of:
    - Miuvig
    range: string
    required: true
  otu_class_appr:
    name: otu_class_appr
    description: Cutoffs and approach used when clustering  species-level  OTUs. Note
      that results from standard 95% ANI / 85% AF clustering should be provided alongside
      OTUS defined from another set of thresholds, even if the latter are the ones
      primarily used during the analysis
    title: OTU classification approach
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{ANI cutoff};{AF cutoff};{clustering method}'
    slot_uri: MIXS:0000085
    multivalued: false
    alias: otu_class_appr
    owner: Miuvig
    domain_of:
    - Miuvig
    range: string
    recommended: true
  otu_seq_comp_appr:
    name: otu_seq_comp_appr
    description: Tool and thresholds used to compare sequences when computing &quot;species-level&quot;
      OTUs
    title: OTU sequence comparison approach
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000086
    multivalued: false
    alias: otu_seq_comp_appr
    owner: Miuvig
    domain_of:
    - Miuvig
    range: string
    recommended: true
  otu_db:
    name: otu_db
    description: Reference database (i.e. sequences not generated as part of the current
      study) used to cluster new genomes in &quot;species-level&quot; OTUs, if any
    title: OTU database
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{database};{version}'
    slot_uri: MIXS:0000087
    multivalued: false
    alias: otu_db
    owner: Miuvig
    domain_of:
    - Miuvig
    range: string
    recommended: true
  host_pred_appr:
    name: host_pred_appr
    description: Tool or approach used for host prediction
    title: host prediction approach
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000088
    multivalued: false
    alias: host_pred_appr
    owner: Miuvig
    domain_of:
    - Miuvig
    range: HOST_PRED_APPR_ENUM
    recommended: true
  host_pred_est_acc:
    name: host_pred_est_acc
    description: For each tool or approach used for host prediction, estimated false
      discovery rates should be included, either computed de novo or from the literature
    title: host prediction estimated accuracy
    from_schema: https://w3id.org/mixs-6-2-rc
    slot_uri: MIXS:0000089
    multivalued: false
    alias: host_pred_est_acc
    owner: Miuvig
    domain_of:
    - Miuvig
    range: string
    recommended: true
  associated_resource:
    name: associated_resource
    annotations:
      Expected_value:
        tag: Expected_value
        value: reference to resource
    description: A related resource that is referenced, cited, or otherwise associated
      to the sequence
    title: relevant electronic resources
    examples:
    - value: http://www.earthmicrobiome.org/
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs-6-2-rc
    keywords:
    - resource
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000091
    multivalued: true
    alias: associated_resource
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  sop:
    name: sop
    description: Standard operating procedures used in assembly and/or annotation
      of genomes, metagenomes or environmental sequences
    title: relevant standard operating procedures
    from_schema: https://w3id.org/mixs-6-2-rc
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000090
    multivalued: true
    alias: sop
    owner: Miuvig
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
class_uri: MIXS:0010012

</code></pre>

</details>





                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.a264c092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.6eac0284.min.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../javascripts/tablesort.js"></script>
      
    
  </body>
</html>