<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Human-vaginalMIUVIG/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>Human-vaginalMIUVIG - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">About</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background.md" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation.md" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents.md" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Human-vaginalMIUVIG.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#human-vaginalmiuvig" class="nav-link">Human-vaginalMIUVIG</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#inheritance" class="nav-link">Inheritance</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#slots" class="nav-link">Slots</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#usages" class="nav-link">Usages</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#identifier-and-mapping-information" class="nav-link">Identifier and Mapping Information</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="human-vaginalmiuvig">Human-vaginalMIUVIG</h1>
<p>Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package human-vaginal</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Human-vaginalMIUVIG">mixs.vocab:Human-vaginalMIUVIG</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Human-vaginal/">Human-vaginal</a><ul>
<li><strong>Human-vaginalMIUVIG</strong> [ MIUVIG]</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
<th>Info</th>
</tr>
</thead>
<tbody>
<tr>
<td>MIXS:0001107</td>
<td><a href="../samp_name/">samp_name</a></td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001320</td>
<td><a href="../samp_taxon_id/">samp_taxon_id</a></td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>{text} [NCBI:txid]</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000092</td>
<td><a href="../project_name/">project_name</a></td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000008</td>
<td><a href="../experimental_factor/">experimental_factor</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000009</td>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000010</td>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000011</td>
<td><a href="../collection_date/">collection_date</a></td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001321</td>
<td><a href="../neg_cont_type/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001322</td>
<td><a href="../pos_cont_type/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>{term} or {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000012</td>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000013</td>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000014</td>
<td><a href="../env_medium/">env_medium</a></td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000024</td>
<td><a href="../estimated_size/">estimated_size</a></td>
<td>0..1</td>
<td>The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</td>
<td>{integer} bp</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000025</td>
<td><a href="../ref_biomaterial/">ref_biomaterial</a></td>
<td>0..1</td>
<td>Primary publication if isolated before genome publication; otherwise, primary genome report.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000026</td>
<td><a href="../source_mat_id/">source_mat_id</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000027</td>
<td><a href="../pathogenicity/">pathogenicity</a></td>
<td>0..1</td>
<td>To what is the entity pathogenic</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000028</td>
<td><a href="../biotic_relationship/">biotic_relationship</a></td>
<td>0..1</td>
<td>Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000029</td>
<td><a href="../specific_host/">specific_host</a></td>
<td>0..1</td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID.</td>
<td>{text}</td>
<td>{NCBI taxid}</td>
</tr>
<tr>
<td>MIXS:0000030</td>
<td><a href="../host_spec_range/">host_spec_range</a></td>
<td>0..1</td>
<td>The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000031</td>
<td><a href="../host_disease_stat/">host_disease_stat</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000002</td>
<td><a href="../samp_collec_device/">samp_collec_device</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0001225</td>
<td><a href="../samp_collec_method/">samp_collec_method</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000016</td>
<td><a href="../samp_mat_process/">samp_mat_process</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000017</td>
<td><a href="../size_frac/">size_frac</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Filtering pore size used in sample preparation</td>
<td>{float}-{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000001</td>
<td><a href="../samp_size/">samp_size</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000111</td>
<td><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000035</td>
<td><a href="../source_uvig/">source_uvig</a></td>
<td>1..1</td>
<td>Type of dataset from which the UViG was obtained</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000036</td>
<td><a href="../virus_enrich_appr/">virus_enrich_appr</a></td>
<td>1..1</td>
<td>List of approaches used to enrich the sample for viruses, if any</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000037</td>
<td><a href="../nucl_acid_ext/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000038</td>
<td><a href="../nucl_acid_amp/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000039</td>
<td><a href="../lib_size/">lib_size</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Total number of clones in the library prepared for the project</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000040</td>
<td><a href="../lib_reads_seqd/">lib_reads_seqd</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Total number of clones sequenced from the library</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000041</td>
<td><a href="../lib_layout/">lib_layout</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000042</td>
<td><a href="../lib_vector/">lib_vector</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000043</td>
<td><a href="../lib_screen/">lib_screen</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Specific enrichment or screening methods applied before and/or after creating libraries</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000047</td>
<td><a href="../mid/">mid</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters</td>
<td>{dna}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000048</td>
<td><a href="../adapters/">adapters</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</td>
<td>{dna};{dna}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000050</td>
<td><a href="../seq_meth/">seq_meth</a></td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000053</td>
<td><a href="../tax_ident/">tax_ident</a></td>
<td>0..1</td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000056</td>
<td><a href="../assembly_qual/">assembly_qual</a></td>
<td>1..1</td>
<td>The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000057</td>
<td><a href="../assembly_name/">assembly_name</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</td>
<td>{text} {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000058</td>
<td><a href="../assembly_software/">assembly_software</a></td>
<td>1..1</td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000059</td>
<td><a href="../annot/">annot</a></td>
<td>0..1</td>
<td>Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000060</td>
<td><a href="../number_contig/">number_contig</a></td>
<td>1..1</td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000061</td>
<td><a href="../feat_pred/">feat_pred</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc.</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000062</td>
<td><a href="../ref_db/">ref_db</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>List of database(s) used for ORF annotation, along with version number and reference to website or publication</td>
<td>{database};{version};{reference}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000063</td>
<td><a href="../sim_search_meth/">sim_search_meth</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000064</td>
<td><a href="../tax_class/">tax_class</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000067</td>
<td><a href="../trnas/">trnas</a></td>
<td>0..1</td>
<td>The total number of tRNAs identified from the SAG or MAG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000068</td>
<td><a href="../trna_ext_software/">trna_ext_software</a></td>
<td>0..1</td>
<td>Tools used for tRNA identification</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000069</td>
<td><a href="../compl_score/">compl_score</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000070</td>
<td><a href="../compl_software/">compl_software</a></td>
<td>0..1</td>
<td>Tools used for completion estimate, i.e. checkm, anvi'o, busco</td>
<td>{software};{version}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000071</td>
<td><a href="../compl_appr/">compl_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The approach used to determine the completeness of a given genomic assembly, which would typically make use of a set of conserved marker genes or a closely related reference genome. For UViG completeness, include reference genome or group used, and contig feature suggesting a complete genome</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000075</td>
<td><a href="../sort_tech/">sort_tech</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to sort/isolate cells or particles of interest</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000076</td>
<td><a href="../single_cell_lysis_appr/">single_cell_lysis_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to free DNA from interior of the cell(s) or particle(s)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000054</td>
<td><a href="../single_cell_lysis_prot/">single_cell_lysis_prot</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name of the kit or standard protocol used for cell(s) or particle(s) lysis</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000055</td>
<td><a href="../wga_amp_appr/">wga_amp_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to amplify genomic DNA in preparation for sequencing</td>
<td>[pcr based</td>
<td>mda based]</td>
</tr>
<tr>
<td>MIXS:0000006</td>
<td><a href="../wga_amp_kit/">wga_amp_kit</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Kit used to amplify genomic DNA in preparation for sequencing</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000077</td>
<td><a href="../bin_param/">bin_param</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The parameters that have been applied during the extraction of genomes from metagenomic datasets</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000078</td>
<td><a href="../bin_software/">bin_software</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool(s) used for the extraction of genomes from metagenomic datasets, where possible include a product ID (PID) of the tool(s) used.</td>
<td>{software};{version}{PID}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000079</td>
<td><a href="../reassembly_bin/">reassembly_bin</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Has an assembly been performed on a genome bin extracted from a metagenomic assembly?</td>
<td>{boolean}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000080</td>
<td><a href="../mag_cov_software/">mag_cov_software</a></td>
<td>0..1</td>
<td>Tool(s) used to determine the genome coverage if coverage is used as a binning parameter in the extraction of genomes from metagenomic datasets</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000081</td>
<td><a href="../vir_ident_software/">vir_ident_software</a></td>
<td>1..1</td>
<td>Tool(s) used for the identification of UViG as a viral genome, software or protocol name including version number, parameters, and cutoffs used</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000082</td>
<td><a href="../pred_genome_type/">pred_genome_type</a></td>
<td>1..1</td>
<td>Type of genome predicted for the UViG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000083</td>
<td><a href="../pred_genome_struc/">pred_genome_struc</a></td>
<td>1..1</td>
<td>Expected structure of the viral genome</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000084</td>
<td><a href="../detec_type/">detec_type</a></td>
<td>1..1</td>
<td>Type of UViG detection</td>
<td>[independent sequence (UViG)</td>
<td>provirus (UpViG)]</td>
</tr>
<tr>
<td>MIXS:0000085</td>
<td><a href="../otu_class_appr/">otu_class_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Cutoffs and approach used when clustering “species-level” OTUs. Note that results from standard 95% ANI / 85% AF clustering should be provided alongside OTUS defined from another set of thresholds, even if the latter are the ones primarily used during the analysis</td>
<td>{ANI cutoff};{AF cutoff};{clustering method}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000086</td>
<td><a href="../otu_seq_comp_appr/">otu_seq_comp_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool and thresholds used to compare sequences when computing "species-level" OTUs</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000087</td>
<td><a href="../otu_db/">otu_db</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Reference database (i.e. sequences not generated as part of the current study) used to cluster new genomes in "species-level" OTUs, if any</td>
<td>{database};{version}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000088</td>
<td><a href="../host_pred_appr/">host_pred_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool or approach used for host prediction</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000089</td>
<td><a href="../host_pred_est_acc/">host_pred_est_acc</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>For each tool or approach used for host prediction, estimated false discovery rates should be included, either computed de novo or from the literature</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000091</td>
<td><a href="../associated_resource/">associated_resource</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000090</td>
<td><a href="../sop/">sop</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000018</td>
<td><a href="../depth/">depth</a></td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000094</td>
<td><a href="../alt/">alt</a></td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000093</td>
<td><a href="../elev/">elev</a></td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000113</td>
<td><a href="../temp/">temp</a></td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000965</td>
<td><a href="../menarche/">menarche</a></td>
<td>0..1</td>
<td>Date of most recent menstruation</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000285</td>
<td><a href="../sexual_act/">sexual_act</a></td>
<td>0..1</td>
<td>Current sexual partner and frequency of sex</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000966</td>
<td><a href="../pregnancy/">pregnancy</a></td>
<td>0..1</td>
<td>Date due of pregnancy</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000967</td>
<td><a href="../douche/">douche</a></td>
<td>0..1</td>
<td>Date of most recent douche</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000286</td>
<td><a href="../birth_control/">birth_control</a></td>
<td>0..1</td>
<td>Specification of birth control medication used</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000968</td>
<td><a href="../menopause/">menopause</a></td>
<td>0..1</td>
<td>Date of onset of menopause</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000969</td>
<td><a href="../hrt/">hrt</a></td>
<td>0..1</td>
<td>Whether subject had hormone replacement theraphy, and if yes start date</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000287</td>
<td><a href="../hysterectomy/">hysterectomy</a></td>
<td>0..1</td>
<td>Specification of whether hysterectomy was performed</td>
<td>{boolean}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000288</td>
<td><a href="../gynecologic_disord/">gynecologic_disord</a></td>
<td>0..*</td>
<td>History of gynecological disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, female reproductive system disease (https://disease-ontology.org/?id=DOID:229).</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000289</td>
<td><a href="../urogenit_disord/">urogenit_disord</a></td>
<td>0..*</td>
<td>History of urogenital disorders, can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, reproductive system disease (https://disease-ontology.org/?id=DOID:15) or urinary system disease (https://disease-ontology.org/?id=DOID:18).</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000861</td>
<td><a href="../host_subject_id/">host_subject_id</a></td>
<td>0..1</td>
<td>A unique identifier by which each subject can be referred to, de-identified.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000255</td>
<td><a href="../host_age/">host_age</a></td>
<td>0..1</td>
<td>Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000811</td>
<td><a href="../host_sex/">host_sex</a></td>
<td>0..1</td>
<td>Gender or physical sex of the host.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000884</td>
<td><a href="../ihmc_medication_code/">ihmc_medication_code</a></td>
<td>0..*</td>
<td>Can include multiple medication codes</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000751</td>
<td><a href="../chem_administration/">chem_administration</a></td>
<td>0..*</td>
<td>List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</td>
<td>{termLabel} {[termID]};{timestamp}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000867</td>
<td><a href="../host_body_site/">host_body_site</a></td>
<td>0..1</td>
<td>Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms, please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000888</td>
<td><a href="../host_body_product/">host_body_product</a></td>
<td>0..1</td>
<td>Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma or https://www.ebi.ac.uk/ols/ontologies/uberon</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000263</td>
<td><a href="../host_tot_mass/">host_tot_mass</a></td>
<td>0..1</td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000264</td>
<td><a href="../host_height/">host_height</a></td>
<td>0..1</td>
<td>The height of subject</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000869</td>
<td><a href="../host_diet/">host_diet</a></td>
<td>0..*</td>
<td>Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000870</td>
<td><a href="../host_last_meal/">host_last_meal</a></td>
<td>0..*</td>
<td>Content of last meal and time since feeding; can include multiple values</td>
<td>{text};{duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000872</td>
<td><a href="../host_family_relation/">host_family_relation</a></td>
<td>0..*</td>
<td>Familial relationships to other hosts in the same study; can include multiple relationships</td>
<td>{text};{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000365</td>
<td><a href="../host_genotype/">host_genotype</a></td>
<td>0..1</td>
<td>Observed genotype</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000874</td>
<td><a href="../host_phenotype/">host_phenotype</a></td>
<td>0..1</td>
<td>Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000274</td>
<td><a href="../host_body_temp/">host_body_temp</a></td>
<td>0..1</td>
<td>Core body temperature of the host when sample was collected</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000317</td>
<td><a href="../host_body_mass_index/">host_body_mass_index</a></td>
<td>0..1</td>
<td>Body mass index, calculated as weight/(height)squared</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000895</td>
<td><a href="../ethnicity/">ethnicity</a></td>
<td>0..*</td>
<td>A category of people who identify with each other, usually on the basis of presumed similarities such as a common language, ancestry, history, society, culture, nation or social treatment within their residing area. https://en.wikipedia.org/wiki/List_of_contemporary_ethnic_groups</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000896</td>
<td><a href="../host_occupation/">host_occupation</a></td>
<td>0..1</td>
<td>Most frequent job performed by subject</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000897</td>
<td><a href="../medic_hist_perform/">medic_hist_perform</a></td>
<td>0..1</td>
<td>Whether full medical history was collected</td>
<td>{boolean}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000333</td>
<td><a href="../host_pulse/">host_pulse</a></td>
<td>0..1</td>
<td>Resting pulse, measured as beats per minute</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000754</td>
<td><a href="../perturbation/">perturbation</a></td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000109</td>
<td><a href="../samp_salinity/">samp_salinity</a></td>
<td>0..1</td>
<td>Salinity is the total concentration of all dissolved salts in a liquid or solid (in the form of an extract obtained by centrifugation) sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000183</td>
<td><a href="../salinity/">salinity</a></td>
<td>0..1</td>
<td>The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000753</td>
<td><a href="../oxy_stat_samp/">oxy_stat_samp</a></td>
<td>0..1</td>
<td>Oxygenation status of sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000103</td>
<td><a href="../organism_count/">organism_count</a></td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000110</td>
<td><a href="../samp_store_temp/">samp_store_temp</a></td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000755</td>
<td><a href="../samp_store_loc/">samp_store_loc</a></td>
<td>0..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000116</td>
<td><a href="../samp_store_dur/">samp_store_dur</a></td>
<td>0..1</td>
<td>Duration for which the sample was stored</td>
<td>{duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001298</td>
<td><a href="../host_symbiont/">host_symbiont</a></td>
<td>0..*</td>
<td>The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000752</td>
<td><a href="../misc_param/">misc_param</a></td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
