
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Human-vaginalMIGSEukaryote/">
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.4.2, mkdocs-material-8.5.10">
    
    
      
        <title>Class: Human-vaginalMIGSEukaryote - MIxS</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.975780f9.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.2505c338.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="None" data-md-color-accent="None">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#class-human-vaginalmigseukaryote" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="MIxS" class="md-header__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MIxS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Class: Human-vaginalMIGSEukaryote
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="MIxS" class="md-nav__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    MIxS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Reference
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Introduction
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Introduction" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Introduction
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/citation/" class="md-nav__link">
        Cite
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/contents/" class="md-nav__link">
        Contents
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#inheritance" class="md-nav__link">
    Inheritance
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#identifier-and-mapping-information" class="md-nav__link">
    Identifier and Mapping Information
  </a>
  
    <nav class="md-nav" aria-label="Identifier and Mapping Information">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#schema-source" class="md-nav__link">
    Schema Source
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mappings" class="md-nav__link">
    Mappings
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#linkml-specification" class="md-nav__link">
    LinkML Specification
  </a>
  
    <nav class="md-nav" aria-label="LinkML Specification">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#direct" class="md-nav__link">
    Direct
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#induced" class="md-nav__link">
    Induced
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Human-vaginalMIGSEukaryote.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="class-human-vaginalmigseukaryote">Class: Human-vaginalMIGSEukaryote</h1>
<p><em>Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package human-vaginal</em></p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Human-vaginalMIGSEukaryote">mixs.vocab:Human-vaginalMIGSEukaryote</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Human-vaginal/">Human-vaginal</a><ul>
<li><strong>human-vaginal MIGS eukaryote</strong> [ <a href="../MIGSEukaryote/">MIGSEukaryote</a>]</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0000289/">urogenit_disord</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>History of urogenital disorders, can include multiple disorders</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000895/">ethnicity</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>A category of people who identify with each other, usually on the basis of pr...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000070/">compl_software</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Tools used for completion estimate, i</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000870/">host_last_meal</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>Content of last meal and time since feeding; can include multiple values</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000012/">env_broad_scale</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a>, <a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000255/">host_age</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Age of host at the time of sampling; relevant scale depends on species and st...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000811/">host_sex</a></td>
<td>0..1 <br/> <a href="../HostSexEnum/">HostSexEnum</a></td>
<td>Gender or physical sex of the host</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000024/">estimated_size</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>The estimated size of the genome prior to sequencing</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000013/">env_local_scale</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Report the entity or entities which are in the sample or specimen’s local vic...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a>, <a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000041/">lib_layout</a></td>
<td>0..1 <br/> <a href="../LibLayoutEnum/">LibLayoutEnum</a></td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000028/">biotic_relationship</a></td>
<td>0..1 <br/> <a href="../BioticRelationshipEnum/">BioticRelationshipEnum</a></td>
<td>Description of relationship(s) between the subject organism and other organis...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000026/">source_mat_id</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>A unique identifier assigned to a material sample (as defined by http://rs</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000064/">tax_class</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Method used for taxonomic classification, along with reference database used,...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0001107/">samp_name</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>A local identifier or name that for the material sample used for extracting n...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000027/">pathogenicity</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>To what is the entity pathogenic</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000874/">host_phenotype</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Phenotype of human or other host</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000288/">gynecologic_disord</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>History of gynecological disorders; can include multiple disorders</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000032/">trophic_level</a></td>
<td>0..1 <br/> <a href="../TrophicLevelEnum/">TrophicLevelEnum</a></td>
<td>Trophic levels are the feeding position in a food chain</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000110/">samp_store_temp</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Temperature at which sample was stored, e</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000050/">seq_meth</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Sequencing machine used</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000043/">lib_screen</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Specific enrichment or screening methods applied before and/or after creating...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000060/">number_contig</a></td>
<td>0..1 <br/> <a href="xsd:integer">xsd:integer</a></td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a giv...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000061/">feat_pred</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000111/">samp_vol_we_dna_ext</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extractio...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000030/">host_spec_range</a></td>
<td>0..1 <br/> <a href="xsd:integer">xsd:integer</a></td>
<td>The range and diversity of host species that an organism is capable of infect...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000039/">lib_size</a></td>
<td>0..1 <br/> <a href="xsd:integer">xsd:integer</a></td>
<td>Total number of clones in the library prepared for the project</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000025/">ref_biomaterial</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Primary publication if isolated before genome publication; otherwise, primary...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000016/">samp_mat_process</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>A brief description of any processing applied to the sample during or after r...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000365/">host_genotype</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Observed genotype</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000037/">nucl_acid_ext</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000092/">project_name</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Name of the project within which the sequencing was organized</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000038/">nucl_acid_amp</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000023/">extrachrom_elements</a></td>
<td>0..1 <br/> <a href="xsd:integer">xsd:integer</a></td>
<td>Do plasmids exist of significant phenotypic consequence (e</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000109/">samp_salinity</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Salinity is the total concentration of all dissolved salts in a liquid or sol...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0001322/">pos_cont_type</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>The substance, mixture, product, or apparatus used to verify that a process w...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000053/">tax_ident</a></td>
<td>0..1 <br/> <a href="../TaxIdentEnum/">TaxIdentEnum</a></td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000872/">host_family_relation</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>Familial relationships to other hosts in the same study; can include multiple...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000018/">depth</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The vertical distance below local surface, e</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000968/">menopause</a></td>
<td>0..1 <br/> <a href="xsd:date">xsd:date</a></td>
<td>Date of onset of menopause</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0001298/">host_symbiont</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>The taxonomic name of the organism(s) found living in mutualistic, commensali...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000090/">sop</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, ...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000062/">ref_db</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>List of database(s) used for ORF annotation, along with version number and re...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000967/">douche</a></td>
<td>0..1 <br/> <a href="xsd:date">xsd:date</a></td>
<td>Date of most recent douche</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000753/">oxy_stat_samp</a></td>
<td>0..1 <br/> <a href="../OxyStatSampEnum/">OxyStatSampEnum</a></td>
<td>Oxygenation status of sample</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000317/">host_body_mass_index</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Body mass index, calculated as weight/(height)squared</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000002/">samp_collec_device</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>The device used to collect an environmental sample</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000011/">collection_date</a></td>
<td>0..1 <br/> <a href="xsd:date">xsd:date</a></td>
<td>The time of sampling, either as an instance (single point in time) or interva...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a>, <a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000020/">subspecf_gen_lin</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Information about the genetic distinctness of the sequenced organism below th...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000751/">chem_administration</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>List of chemical compounds administered to the host or site where sampling oc...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000021/">ploidy</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>The ploidy level of the genome (e</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000040/">lib_reads_seqd</a></td>
<td>0..1 <br/> <a href="xsd:integer">xsd:integer</a></td>
<td>Total number of clones sequenced from the library</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000897/">medic_hist_perform</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Whether full medical history was collected</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000286/">birth_control</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Specification of birth control medication used</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000103/">organism_count</a></td>
<td>0..* <br/> <a href="../OrganismCountEnum/">OrganismCountEnum</a></td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or ...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000033/">propagation</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>The type of reproduction from the parent stock</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000059/">annot</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Tool used for annotation, or for cases where annotation was provided by a com...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000116/">samp_store_dur</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Duration for which the sample was stored</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000287/">hysterectomy</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Specification of whether hysterectomy was performed</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000048/">adapters</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Adapters provide priming sequences for both amplification and sequencing of t...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000001/">samp_size</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample coll...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000014/">env_medium</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a>, <a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000755/">samp_store_loc</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000274/">host_body_temp</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Core body temperature of the host when sample was collected</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000056/">assembly_qual</a></td>
<td>0..1 <br/> <a href="../AssemblyQualEnum/">AssemblyQualEnum</a></td>
<td>The assembly quality category is based on sets of criteria outlined for each ...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000969/">hrt</a></td>
<td>0..1 <br/> <a href="xsd:date">xsd:date</a></td>
<td>Whether subject had hormone replacement theraphy, and if yes start date</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000094/">alt</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Altitude is a term used to identify heights of objects such as airplanes, spa...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000093/">elev</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Elevation of the sampling site is its height above a fixed reference point, m...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000285/">sexual_act</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Current sexual partner and frequency of sex</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000091/">associated resource</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>A related resource that is referenced, cited, or otherwise associated to the ...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000263/">host_tot_mass</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000264/">host_height</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The height of subject</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000003/">isol_growth_condt</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identif...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000008/">experimental_factor</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Experimental factors are essentially the variable aspects of an experiment de...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000896/">host_occupation</a></td>
<td>0..1 <br/> <a href="xsd:integer">xsd:integer</a></td>
<td>Most frequent job performed by subject</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000752/">misc_param</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>Any other measurement performed or parameter collected, that is not listed he...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000022/">num_replicons</a></td>
<td>0..1 <br/> <a href="xsd:integer">xsd:integer</a></td>
<td>Reports the number of replicons in a nuclear genome of eukaryotes, in the gen...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000884/">ihmc_medication_code</a></td>
<td>0..* <br/> <a href="xsd:integer">xsd:integer</a></td>
<td>Can include multiple medication codes</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000057/">assembly_name</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Name/version of the assembly provided by the submitter that is used in the ge...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000063/">sim_search_meth</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000113/">temp</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Temperature of the sample at the time of sampling</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000888/">host_body_product</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Substance produced by the body, e</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0001320/">samp_taxon_id</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>NCBI taxon id of the sample</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000042/">lib_vector</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000058/">assembly_software</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0001321/">neg_cont_type</a></td>
<td>0..1 <br/> <a href="../NegContTypeEnum/">NegContTypeEnum</a></td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000029/">specific_host</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000867/">host_body_site</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Name of body site where the sample was obtained from, such as a specific orga...</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000333/">host_pulse</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Resting pulse, measured as beats per minute</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000861/">host_subject_id</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>A unique identifier by which each subject can be referred to, de-identified</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000031/">host_disease_stat</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>List of diseases with which the host has been diagnosed; can include multiple...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000966/">pregnancy</a></td>
<td>0..1 <br/> <a href="xsd:date">xsd:date</a></td>
<td>Date due of pregnancy</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000009/">lat_lon</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>The geographical origin of the sample as defined by latitude and longitude</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a>, <a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000965/">menarche</a></td>
<td>0..1 <br/> <a href="xsd:date">xsd:date</a></td>
<td>Date of most recent menstruation</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000069/">compl_score</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>Completeness score is typically based on either the fraction of markers found...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0001225/">samp_collec_method</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>The method employed for collecting the sample</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a></td>
</tr>
<tr>
<td><a href="../0000010/">geo_loc_name</a></td>
<td>0..1 <br/> <a href="xsd:string">xsd:string</a></td>
<td>The geographical origin of the sample as defined by the country or sea name f...</td>
<td><a href="../MIGSEukaryote/">MIGSEukaryote</a>, <a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000754/">perturbation</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>Type of perturbation, e</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000869/">host_diet</a></td>
<td>0..* <br/> <a href="xsd:string">xsd:string</a></td>
<td>Type of diet depending on the host, for animals omnivore, herbivore etc</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
<tr>
<td><a href="../0000183/">salinity</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The total concentration of all dissolved salts in a liquid or solid sample</td>
<td><a href="../Human-vaginal/">Human-vaginal</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: http://w3id.org/mixs/checklists</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>['mixs.vocab:Human-vaginalMIGSEukaryote']</td>
</tr>
<tr>
<td>native</td>
<td>['mixs.vocab:Human-vaginalMIGSEukaryote']</td>
</tr>
</tbody>
</table>
<h2 id="linkml-specification">LinkML Specification</h2>
<h3 id="direct">Direct</h3>
<details>

<pre><code class="language-yaml">name: human-vaginal MIGS eukaryote
description: 'Combinatorial checklist Minimal Information about a Genome Sequence:
  eukaryote with environmental package human-vaginal'
from_schema: http://w3id.org/mixs/checklists
is_a: human-vaginal
mixins:
- MIGS eukaryote

</code></pre>

</details>

<h3 id="induced">Induced</h3>
<details>

<pre><code class="language-yaml">name: human-vaginal MIGS eukaryote
description: 'Combinatorial checklist Minimal Information about a Genome Sequence:
  eukaryote with environmental package human-vaginal'
from_schema: http://w3id.org/mixs/checklists
is_a: human-vaginal
mixins:
- MIGS eukaryote
attributes:
  samp_name:
    name: samp_name
    annotations:
      expected_value:
        tag: expected_value
        value: text
    description: A local identifier or name that for the material sample used for
      extracting nucleic acids, and subsequent sequencing. It can refer either to
      the original material collected or to any derived sub-samples. It can have any
      format, but we suggest that you make it concise, unique and consistent within
      your lab, and as informative as possible. INSDC requires every sample name from
      a single Submitter to be unique. Use of a globally unique identifier for the
      field source_mat_id is recommended in addition to sample_name.
    title: sample name
    examples:
    - value: ISDsoil1
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample name
    is_a: investigation field
    string_serialization: '{text}'
    slot_uri: MIXS:0001107
    multivalued: false
    alias: samp_name
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  samp_taxon_id:
    name: samp_taxon_id
    annotations:
      expected_value:
        tag: expected_value
        value: Taxonomy ID
    description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa
      sample. Use 'synthetic metagenome’ for mock community/positive controls, or
      'blank sample' for negative controls.
    title: Taxonomy ID of DNA sample
    examples:
    - value: Gut Metagenome [NCBI:txid749906]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - Taxonomy ID of DNA sample
    is_a: investigation field
    string_serialization: '{text} [NCBI:txid]'
    slot_uri: MIXS:0001320
    multivalued: false
    alias: samp_taxon_id
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  project_name:
    name: project_name
    description: Name of the project within which the sequencing was organized
    title: project name
    examples:
    - value: Forest soil metagenome
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - project name
    is_a: investigation field
    string_serialization: '{text}'
    slot_uri: MIXS:0000092
    multivalued: false
    alias: project_name
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  experimental_factor:
    name: experimental_factor
    annotations:
      expected_value:
        tag: expected_value
        value: text or EFO and/or OBI
    description: Experimental factors are essentially the variable aspects of an experiment
      design which can be used to describe an experiment, or set of experiments, in
      an increasingly detailed manner. This field accepts ontology terms from Experimental
      Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For
      a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO;
      for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI
    title: experimental factor
    examples:
    - value: time series design [EFO:EFO_0001779]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - experimental factor
    is_a: investigation field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000008
    multivalued: false
    alias: experimental_factor
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  lat_lon:
    name: lat_lon
    annotations:
      expected_value:
        tag: expected_value
        value: decimal degrees,  limit to 8 decimal points
    description: The geographical origin of the sample as defined by latitude and
      longitude. The values should be reported in decimal degrees and in WGS84 system
    title: geographic location (latitude and longitude)
    examples:
    - value: 50.586825 6.408977
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - geographic location (latitude and longitude)
    is_a: environment field
    string_serialization: '{float} {float}'
    slot_uri: MIXS:0000009
    multivalued: false
    alias: lat_lon
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  geo_loc_name:
    name: geo_loc_name
    annotations:
      expected_value:
        tag: expected_value
        value: 'country or sea name (INSDC or GAZ): region(GAZ), specific location
          name'
    description: The geographical origin of the sample as defined by the country or
      sea name followed by specific region name. Country or sea names should be chosen
      from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology
      (http://purl.bioontology.org/ontology/GAZ)
    title: geographic location (country and/or sea,region)
    examples:
    - value: 'USA: Maryland, Bethesda'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - geographic location (country and/or sea,region)
    is_a: environment field
    string_serialization: '{term}: {term}, {text}'
    slot_uri: MIXS:0000010
    multivalued: false
    alias: geo_loc_name
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  collection_date:
    name: collection_date
    annotations:
      expected_value:
        tag: expected_value
        value: date and time
    description: 'The time of sampling, either as an instance (single point in time)
      or interval. In case no exact time is available, the date/time can be right
      truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10;
      2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant'
    title: collection date
    examples:
    - value: 2018-05-11T10:00:00+01:00; 2018-05-11
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - collection date
    is_a: environment field
    slot_uri: MIXS:0000011
    multivalued: false
    alias: collection_date
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: date
    required: true
  neg_cont_type:
    name: neg_cont_type
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration or text
    description: The substance or equipment used as a negative control in an investigation
    title: negative control type
    examples:
    - value: ''
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - negative control type
    is_a: investigation field
    slot_uri: MIXS:0001321
    multivalued: false
    alias: neg_cont_type
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: neg_cont_type_enum
    recommended: true
  pos_cont_type:
    name: pos_cont_type
    description: The substance, mixture, product, or apparatus used to verify that
      a process which is part of an investigation delivers a true positive.
    title: positive control type
    examples:
    - value: ''
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - positive control type
    is_a: investigation field
    string_serialization: '{term} or {text}'
    slot_uri: MIXS:0001322
    multivalued: false
    alias: pos_cont_type
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  env_broad_scale:
    name: env_broad_scale
    annotations:
      expected_value:
        tag: expected_value
        value: The major environment type(s) where the sample was collected. Recommend
          subclasses of biome [ENVO:00000428]. Multiple terms can be separated by
          one or more pipes.
    description: 'Report the major environmental system the sample or specimen came
      from. The system(s) identified should have a coarse spatial grain, to provide
      the general environmental context of where the sampling was done (e.g. in the
      desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428.
      EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: broad-scale environmental context
    examples:
    - value: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water
        sample from the photic zone in middle of the Atlantic Ocean
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - broad-scale environmental context
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000012
    multivalued: false
    alias: env_broad_scale
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  env_local_scale:
    name: env_local_scale
    annotations:
      expected_value:
        tag: expected_value
        value: Environmental entities having causal influences upon the entity at
          time of sampling.
    description: 'Report the entity or entities which are in the sample or specimen’s
      local vicinity and which you believe have significant causal influences on your
      sample or specimen. We recommend using EnvO terms which are of smaller spatial
      grain than your entry for env_broad_scale. Terms, such as anatomical sites,
      from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON)
      are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.'
    title: local environmental context
    examples:
    - value: 'litter layer [ENVO:01000338]; Annotating a pooled sample taken from
        various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb
        and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub
        layer [ENVO:01000336].'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - local environmental context
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000013
    multivalued: false
    alias: env_local_scale
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  env_medium:
    name: env_medium
    annotations:
      expected_value:
        tag: expected_value
        value: The material displaced by the entity at time of sampling. Recommend
          subclasses of environmental material [ENVO:00010483].
    description: 'Report the environmental material(s) immediately surrounding the
      sample or specimen at the time of sampling. We recommend using subclasses of
      ''environmental material'' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO
      documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
      . Terms from other OBO ontologies are permissible as long as they reference
      mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
      (e.g. a tree, a leaf, a table top).'
    title: environmental medium
    examples:
    - value: 'soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m
        of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating
        a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005]'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - environmental medium
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000014
    multivalued: false
    alias: env_medium
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  subspecf_gen_lin:
    name: subspecf_gen_lin
    annotations:
      expected_value:
        tag: expected_value
        value: Genetic lineage below lowest rank of NCBI taxonomy, which is subspecies,
          e.g. serovar, biotype, ecotype, variety, cultivar.
    description: Information about the genetic distinctness of the sequenced organism
      below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any
      relevant genetic typing schemes like Group I plasmid. Subspecies should not
      be recorded in this term, but in the NCBI taxonomy. Supply both the lineage
      name and the lineage rank separated by a colon, e.g., biovar:abc123.
    title: subspecific genetic lineage
    examples:
    - value: serovar:Newport
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - subspecific genetic lineage
    is_a: nucleic acid sequence source field
    string_serialization: '{rank name}:{text}'
    slot_uri: MIXS:0000020
    multivalued: false
    alias: subspecf_gen_lin
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMARKS survey
    range: string
    recommended: true
  ploidy:
    name: ploidy
    annotations:
      expected_value:
        tag: expected_value
        value: PATO
    description: The ploidy level of the genome (e.g. allopolyploid, haploid, diploid,
      triploid, tetraploid). It has implications for the downstream study of duplicated
      gene and regions of the genomes (and perhaps for difficulties in assembly).
      For terms, please select terms listed under class ploidy (PATO:001374) of Phenotypic
      Quality Ontology (PATO), and for a browser of PATO (v 2018-03-27) please refer
      to http://purl.bioontology.org/ontology/PATO
    title: ploidy
    examples:
    - value: allopolyploidy [PATO:0001379]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - ploidy
    is_a: nucleic acid sequence source field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000021
    multivalued: false
    alias: ploidy
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    range: string
    required: false
  num_replicons:
    name: num_replicons
    annotations:
      expected_value:
        tag: expected_value
        value: 'for eukaryotes and bacteria: chromosomes (haploid count); for viruses:
          segments'
    description: Reports the number of replicons in a nuclear genome of eukaryotes,
      in the genome of a bacterium or archaea or the number of segments in a segmented
      virus. Always applied to the haploid chromosome count of a eukaryote
    title: number of replicons
    examples:
    - value: '2'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - number of replicons
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000022
    multivalued: false
    alias: num_replicons
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS virus
    range: integer
    required: false
  extrachrom_elements:
    name: extrachrom_elements
    annotations:
      expected_value:
        tag: expected_value
        value: number of extrachromosmal elements
    description: Do plasmids exist of significant phenotypic consequence (e.g. ones
      that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids
      (borrelia has 15+ plasmids)
    title: extrachromosomal elements
    examples:
    - value: '5'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - extrachromosomal elements
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000023
    multivalued: false
    alias: extrachrom_elements
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS org
    - MIMARKS survey
    range: integer
    required: false
  estimated_size:
    name: estimated_size
    annotations:
      expected_value:
        tag: expected_value
        value: number of base pairs
    description: The estimated size of the genome prior to sequencing. Of particular
      importance in the sequencing of (eukaryotic) genome which could remain in draft
      form for a long or unspecified period.
    title: estimated size
    examples:
    - value: 300000 bp
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - estimated size
    is_a: nucleic acid sequence source field
    string_serialization: '{integer} bp'
    slot_uri: MIXS:0000024
    multivalued: false
    alias: estimated_size
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIUVIG
    range: string
    required: false
  ref_biomaterial:
    name: ref_biomaterial
    annotations:
      expected_value:
        tag: expected_value
        value: PMID, DOI or URL
    description: Primary publication if isolated before genome publication; otherwise,
      primary genome report.
    title: reference for biomaterial
    examples:
    - value: doi:10.1016/j.syapm.2018.01.009
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - reference for biomaterial
    is_a: nucleic acid sequence source field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000025
    multivalued: false
    alias: ref_biomaterial
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  source_mat_id:
    name: source_mat_id
    annotations:
      expected_value:
        tag: expected_value
        value: 'for cultures of microorganisms: identifiers for two culture collections;
          for other material a unique arbitrary identifer'
    description: A unique identifier assigned to a material sample (as defined by
      http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular
      digital record of a material sample) used for extracting nucleic acids, and
      subsequent sequencing. The identifier can refer either to the original material
      collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher,
      /bio_material, or /culture_collection may or may not share the same value as
      the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id
      may both contain 'UAM:Herps:14' , referring to both the specimen voucher and
      sampled tissue with the same identifier. However, the /culture_collection qualifier
      may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id
      would refer to an identifier from some derived culture from which the nucleic
      acids were extracted (e.g. xatc123 or ark:/2154/R2).
    title: source material identifiers
    examples:
    - value: MPI012345
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - source material identifiers
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000026
    multivalued: false
    alias: source_mat_id
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  pathogenicity:
    name: pathogenicity
    annotations:
      expected_value:
        tag: expected_value
        value: names of organisms that the entity is pathogenic to
    description: To what is the entity pathogenic
    title: known pathogenicity
    examples:
    - value: human, animal, plant, fungi, bacteria
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - known pathogenicity
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000027
    multivalued: false
    alias: pathogenicity
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS virus
    - MIUVIG
    range: string
    recommended: true
  biotic_relationship:
    name: biotic_relationship
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Description of relationship(s) between the subject organism and other
      organism(s) it is associated with. E.g., parasite on species X; mutualist with
      species Y. The target organism is the subject of the relationship, and the other
      organism(s) is the object
    title: observed biotic relationship
    examples:
    - value: free living
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - observed biotic relationship
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000028
    multivalued: false
    alias: biotic_relationship
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS virus
    - MIMARKS survey
    - MIUVIG
    range: biotic_relationship_enum
    required: false
  specific_host:
    name: specific_host
    annotations:
      expected_value:
        tag: expected_value
        value: host scientific name, taxonomy ID
    description: Report the host's taxonomic name and/or NCBI taxonomy ID.
    title: host scientific name
    examples:
    - value: Homo sapiens and/or 9606
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host scientific name
    is_a: nucleic acid sequence source field
    string_serialization: '{text}|{NCBI taxid}'
    slot_uri: MIXS:0000029
    multivalued: false
    alias: specific_host
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIUVIG
    range: string
    required: false
  host_spec_range:
    name: host_spec_range
    annotations:
      expected_value:
        tag: expected_value
        value: NCBI taxid
    description: The range and diversity of host species that an organism is capable
      of infecting, defined by NCBI taxonomy identifier.
    title: host specificity or range
    examples:
    - value: '9606'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host specificity or range
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000030
    multivalued: false
    alias: host_spec_range
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIUVIG
    range: integer
    required: false
  host_disease_stat:
    name: host_disease_stat
    annotations:
      expected_value:
        tag: expected_value
        value: disease name or Disease Ontology term
    description: List of diseases with which the host has been diagnosed; can include
      multiple diagnoses. The value of the field depends on host; for humans the terms
      should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org,
      non-human host diseases are free text
    title: host disease status
    examples:
    - value: rabies [DOID:11260]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host disease status
    is_a: nucleic acid sequence source field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000031
    multivalued: false
    alias: host_disease_stat
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS virus
    - MIUVIG
    range: string
    required: false
  trophic_level:
    name: trophic_level
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Trophic levels are the feeding position in a food chain. Microbes
      can be a range of producers (e.g. chemolithotroph)
    title: trophic level
    examples:
    - value: heterotroph
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - trophic level
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000032
    multivalued: false
    alias: trophic_level
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIMARKS survey
    range: trophic_level_enum
    recommended: true
  propagation:
    name: propagation
    annotations:
      expected_value:
        tag: expected_value
        value: 'for virus: lytic, lysogenic, temperate, obligately lytic; for plasmid:
          incompatibility group; for eukaryote: asexual, sexual; other more specific
          values (e.g., incompatibility group) are allowed'
    description: 'The type of reproduction from the parent stock. Values for this
      field is specific to different taxa. For phage or virus: lytic/lysogenic/temperate/obligately
      lytic. For plasmids: incompatibility group. For eukaryotes: sexual/asexual.'
    title: propagation
    examples:
    - value: lytic
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - propagation
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000033
    multivalued: false
    alias: propagation
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS plant
    - MIGS virus
    range: string
    recommended: true
  isol_growth_condt:
    name: isol_growth_condt
    annotations:
      expected_value:
        tag: expected_value
        value: PMID,DOI or URL
    description: Publication reference in the form of pubmed ID (pmid), digital object
      identifier (doi) or url for isolation and growth condition specifications of
      the organism/material
    title: isolation and growth condition
    examples:
    - value: 'doi: 10.1016/j.syapm.2018.01.009'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - isolation and growth condition
    is_a: nucleic acid sequence source field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000003
    multivalued: false
    alias: isol_growth_condt
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMARKS survey
    range: string
    required: true
  samp_collec_device:
    name: samp_collec_device
    annotations:
      expected_value:
        tag: expected_value
        value: device name
    description: The device used to collect an environmental sample. This field accepts
      terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO).
      This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).
    title: sample collection device
    examples:
    - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample collection device
    is_a: nucleic acid sequence source field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000002
    multivalued: false
    alias: samp_collec_device
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  samp_collec_method:
    name: samp_collec_method
    annotations:
      expected_value:
        tag: expected_value
        value: PMID,DOI,url , or text
    description: The method employed for collecting the sample.
    title: sample collection method
    examples:
    - value: swabbing
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample collection method
    is_a: nucleic acid sequence source field
    string_serialization: '{PMID}|{DOI}|{URL}|{text}'
    slot_uri: MIXS:0001225
    multivalued: false
    alias: samp_collec_method
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  samp_mat_process:
    name: samp_mat_process
    annotations:
      expected_value:
        tag: expected_value
        value: text
    description: A brief description of any processing applied to the sample during
      or after retrieving the sample from environment, or a link to the relevant protocol(s)
      performed.
    title: sample material processing
    examples:
    - value: filtering of seawater, storing samples in ethanol
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample material processing
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000016
    multivalued: false
    alias: samp_mat_process
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  samp_size:
    name: samp_size
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: millliter, gram, milligram, liter
    description: The total amount or size (volume (ml), mass (g) or area (m2) ) of
      sample collected.
    title: amount or size of sample collected
    examples:
    - value: 5 liter
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - amount or size of sample collected
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000001
    multivalued: false
    alias: samp_size
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: quantity value
    required: false
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: millliter, gram, milligram, square centimeter
    description: 'Volume (ml) or mass (g) of total collected sample processed for
      DNA extraction. Note: total sample collected should be entered under the term
      Sample Size (MIXS:0000001).'
    title: sample volume or weight for DNA extraction
    examples:
    - value: 1500 milliliter
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample volume or weight for DNA extraction
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000111
    multivalued: false
    alias: samp_vol_we_dna_ext
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: quantity value
    required: false
  nucl_acid_ext:
    name: nucl_acid_ext
    annotations:
      expected_value:
        tag: expected_value
        value: PMID, DOI or URL
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the material separation to recover
      the nucleic acid fraction from a sample
    title: nucleic acid extraction
    examples:
    - value: https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - nucleic acid extraction
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000037
    multivalued: false
    alias: nucl_acid_ext
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  nucl_acid_amp:
    name: nucl_acid_amp
    annotations:
      expected_value:
        tag: expected_value
        value: PMID, DOI or URL
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the enzymatic amplification (PCR,
      TMA, NASBA) of specific nucleic acids
    title: nucleic acid amplification
    examples:
    - value: https://phylogenomics.me/protocols/16s-pcr-protocol/
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - nucleic acid amplification
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000038
    multivalued: false
    alias: nucl_acid_amp
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  lib_size:
    name: lib_size
    annotations:
      expected_value:
        tag: expected_value
        value: number of clones
    description: Total number of clones in the library prepared for the project
    title: library size
    examples:
    - value: '50'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - library size
    is_a: sequencing field
    slot_uri: MIXS:0000039
    multivalued: false
    alias: lib_size
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
    required: false
  lib_reads_seqd:
    name: lib_reads_seqd
    annotations:
      expected_value:
        tag: expected_value
        value: number of reads sequenced
    description: Total number of clones sequenced from the library
    title: library reads sequenced
    examples:
    - value: '20'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - library reads sequenced
    is_a: sequencing field
    slot_uri: MIXS:0000040
    multivalued: false
    alias: lib_reads_seqd
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
    required: false
  lib_layout:
    name: lib_layout
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Specify whether to expect single, paired, or other configuration
      of reads
    title: library layout
    examples:
    - value: paired
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - library layout
    is_a: sequencing field
    slot_uri: MIXS:0000041
    multivalued: false
    alias: lib_layout
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: lib_layout_enum
    required: false
  lib_vector:
    name: lib_vector
    annotations:
      expected_value:
        tag: expected_value
        value: vector
    description: Cloning vector type(s) used in construction of libraries
    title: library vector
    examples:
    - value: Bacteriophage P1
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - library vector
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000042
    multivalued: false
    alias: lib_vector
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  lib_screen:
    name: lib_screen
    annotations:
      expected_value:
        tag: expected_value
        value: screening strategy name
    description: Specific enrichment or screening methods applied before and/or after
      creating libraries
    title: library screening strategy
    examples:
    - value: enriched, screened, normalized
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - library screening strategy
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000043
    multivalued: false
    alias: lib_screen
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  adapters:
    name: adapters
    annotations:
      expected_value:
        tag: expected_value
        value: adapter A and B sequence
    description: Adapters provide priming sequences for both amplification and sequencing
      of the sample-library fragments. Both adapters should be reported; in uppercase
      letters
    title: adapters
    examples:
    - value: AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - adapters
    is_a: sequencing field
    string_serialization: '{dna};{dna}'
    slot_uri: MIXS:0000048
    multivalued: false
    alias: adapters
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  seq_meth:
    name: seq_meth
    annotations:
      expected_value:
        tag: expected_value
        value: Text or OBI
    description: Sequencing machine used. Where possible the term should be taken
      from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).
    title: sequencing method
    examples:
    - value: 454 Genome Sequencer FLX [OBI:0000702]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sequencing method
    is_a: sequencing field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000050
    multivalued: false
    alias: seq_meth
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  tax_ident:
    name: tax_ident
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: The phylogenetic marker(s) used to assign an organism name to the
      SAG or MAG
    title: taxonomic identity marker
    examples:
    - value: 'other: rpoB gene'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - taxonomic identity marker
    is_a: sequencing field
    slot_uri: MIXS:0000053
    multivalued: false
    alias: tax_ident
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: tax_ident_enum
    recommended: true
  assembly_qual:
    name: assembly_qual
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: 'The assembly quality category is based on sets of criteria outlined
      for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated,
      contiguous sequence per replicon without gaps or ambiguities with a consensus
      error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments
      where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes
      and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no
      review of assembly other than reporting of standard assembly statistics. Low
      Quality Draft:Many fragments with little to no review of assembly other than
      reporting of standard assembly statistics. Assembly statistics include, but
      are not limited to total assembly size, number of contigs, contig N50/L50, and
      maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence
      per replicon without gaps or ambiguities, with extensive manual review and editing
      to annotate putative gene functions and transcriptional units. High-quality
      draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome
      or replicon sequence or predicted complete. Genome fragment(s): One or multiple
      fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for
      which no genome size could be estimated'
    title: assembly quality
    examples:
    - value: High-quality draft genome
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - assembly quality
    is_a: sequencing field
    slot_uri: MIXS:0000056
    multivalued: false
    alias: assembly_qual
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: assembly_qual_enum
    required: true
  assembly_name:
    name: assembly_name
    annotations:
      expected_value:
        tag: expected_value
        value: name and version of assembly
    description: Name/version of the assembly provided by the submitter that is used
      in the genome browsers and in the community
    title: assembly name
    examples:
    - value: HuRef, JCVI_ISG_i3_1.0
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - assembly name
    is_a: sequencing field
    string_serialization: '{text} {text}'
    slot_uri: MIXS:0000057
    multivalued: false
    alias: assembly_name
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  assembly_software:
    name: assembly_software
    annotations:
      expected_value:
        tag: expected_value
        value: name and version of software, parameters used
    description: Tool(s) used for assembly, including version number and parameters
    title: assembly software
    examples:
    - value: metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - assembly software
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000058
    multivalued: false
    alias: assembly_software
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: true
  annot:
    name: annot
    annotations:
      expected_value:
        tag: expected_value
        value: name of tool or pipeline used, or annotation source description
    description: Tool used for annotation, or for cases where annotation was provided
      by a community jamboree or model organism database rather than by a specific
      submitter
    title: annotation
    examples:
    - value: prokka
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - annotation
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000059
    multivalued: false
    alias: annot
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  number_contig:
    name: number_contig
    annotations:
      expected_value:
        tag: expected_value
        value: value
    description: Total number of contigs in the cleaned/submitted assembly that makes
      up a given genome, SAG, MAG, or UViG
    title: number of contigs
    examples:
    - value: '40'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - number of contigs
    is_a: sequencing field
    slot_uri: MIXS:0000060
    multivalued: false
    alias: number_contig
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
    required: true
  feat_pred:
    name: feat_pred
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s), parameters used
    description: Method used to predict UViGs features such as ORFs, integration site,
      etc.
    title: feature prediction
    examples:
    - value: Prodigal;2.6.3;default parameters
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - feature prediction
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000061
    multivalued: false
    alias: feat_pred
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  ref_db:
    name: ref_db
    annotations:
      expected_value:
        tag: expected_value
        value: names, versions, and references of databases
    description: List of database(s) used for ORF annotation, along with version number
      and reference to website or publication
    title: reference database(s)
    examples:
    - value: pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017
        doi:10.1093/nar/gkw975
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - reference database(s)
    is_a: sequencing field
    string_serialization: '{database};{version};{reference}'
    slot_uri: MIXS:0000062
    multivalued: false
    alias: ref_db
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  sim_search_meth:
    name: sim_search_meth
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s), parameters used
    description: Tool used to compare ORFs with database, along with version and cutoffs
      used
    title: similarity search method
    examples:
    - value: HMMER3;3.1b2;hmmsearch, cutoff of 50 on score
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - similarity search method
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000063
    multivalued: false
    alias: sim_search_meth
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  tax_class:
    name: tax_class
    annotations:
      expected_value:
        tag: expected_value
        value: classification method, database name, and other parameters
    description: Method used for taxonomic classification, along with reference database
      used, classification rank, and thresholds used to classify new genomes
    title: taxonomic classification
    examples:
    - value: vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification,
        default parameters)
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - taxonomic classification
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000064
    multivalued: false
    alias: tax_class
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  compl_score:
    name: compl_score
    annotations:
      expected_value:
        tag: expected_value
        value: quality;percent completeness
    description: 'Completeness score is typically based on either the fraction of
      markers found as compared to a database or the percent of a genome found as
      compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium
      Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated
      completeness scores'
    title: completeness score
    examples:
    - value: med;60%
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - completeness score
    is_a: sequencing field
    slot_uri: MIXS:0000069
    multivalued: false
    alias: compl_score
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
    pattern: ^(high|med|low);(0|([0-9]{1,2})|100)%$
  compl_software:
    name: compl_software
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s) used
    description: Tools used for completion estimate, i.e. checkm, anvi'o, busco
    title: completeness software
    examples:
    - value: checkm
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - completeness software
    is_a: sequencing field
    string_serialization: '{software};{version}'
    slot_uri: MIXS:0000070
    multivalued: false
    alias: compl_software
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    required: false
  associated resource:
    name: associated resource
    annotations:
      expected_value:
        tag: expected_value
        value: reference to resource
    description: A related resource that is referenced, cited, or otherwise associated
      to the sequence.
    title: relevant electronic resources
    examples:
    - value: http://www.earthmicrobiome.org/
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - relevant electronic resources
    is_a: sequencing field
    string_serialization: '{PMID} | {DOI} | {URL}'
    slot_uri: MIXS:0000091
    multivalued: false
    alias: associated_resource
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  sop:
    name: sop
    annotations:
      expected_value:
        tag: expected_value
        value: reference to SOP
    description: Standard operating procedures used in assembly and/or annotation
      of genomes, metagenomes or environmental sequences
    title: relevant standard operating procedures
    examples:
    - value: http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - relevant standard operating procedures
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000090
    multivalued: false
    alias: sop
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    recommended: true
  depth:
    name: depth
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: The vertical distance below local surface, e.g. for sediment or soil
      samples depth is measured from sediment or soil surface, respectively. Depth
      can be reported as an interval for subsurface samples.
    title: depth
    examples:
    - value: 10 meter
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - depth
    is_a: environment field
    slot_uri: MIXS:0000018
    multivalued: false
    alias: depth
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  alt:
    name: alt
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: Altitude is a term used to identify heights of objects such as airplanes,
      space shuttles, rockets, atmospheric balloons and heights of places such as
      atmospheric layers and clouds. It is used to measure the height of an object
      which is above the earth's surface. In this context, the altitude measurement
      is the vertical distance between the earth's surface above sea level and the
      sampled position in the air
    title: altitude
    examples:
    - value: 100 meter
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - altitude
    is_a: environment field
    slot_uri: MIXS:0000094
    multivalued: false
    alias: alt
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  elev:
    name: elev
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: Elevation of the sampling site is its height above a fixed reference
      point, most commonly the mean sea level. Elevation is mainly used when referring
      to points on the earth's surface, while altitude is used for points above the
      surface, such as an aircraft in flight or a spacecraft in orbit.
    title: elevation
    examples:
    - value: 100 meter
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - elevation
    is_a: environment field
    slot_uri: MIXS:0000093
    multivalued: false
    alias: elev
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  temp:
    name: temp
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: degree Celsius
    description: Temperature of the sample at the time of sampling.
    title: temperature
    examples:
    - value: 25 degree Celsius
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - temperature
    is_a: environment field
    slot_uri: MIXS:0000113
    multivalued: false
    alias: temp
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
    required: false
  menarche:
    name: menarche
    annotations:
      expected_value:
        tag: expected_value
        value: timestamp
      occurrence:
        tag: occurrence
        value: '1'
    description: Date of most recent menstruation
    title: menarche
    examples:
    - value: '2018-05-11'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - menarche
    is_a: core field
    slot_uri: MIXS:0000965
    multivalued: false
    alias: menarche
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: date
    required: false
  sexual_act:
    name: sexual_act
    annotations:
      expected_value:
        tag: expected_value
        value: partner sex;frequency
      occurrence:
        tag: occurrence
        value: '1'
    description: Current sexual partner and frequency of sex
    title: sexual activity
    examples:
    - value: ''
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sexual activity
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000285
    multivalued: false
    alias: sexual_act
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: string
    required: false
  pregnancy:
    name: pregnancy
    annotations:
      expected_value:
        tag: expected_value
        value: timestamp
      occurrence:
        tag: occurrence
        value: '1'
    description: Date due of pregnancy
    title: pregnancy
    examples:
    - value: '2018-05-11'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - pregnancy
    is_a: core field
    slot_uri: MIXS:0000966
    multivalued: false
    alias: pregnancy
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: date
    required: false
  douche:
    name: douche
    annotations:
      expected_value:
        tag: expected_value
        value: timestamp
      occurrence:
        tag: occurrence
        value: '1'
    description: Date of most recent douche
    title: douche
    examples:
    - value: '2018-05-11'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - douche
    is_a: core field
    slot_uri: MIXS:0000967
    multivalued: false
    alias: douche
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: date
    required: false
  birth_control:
    name: birth_control
    annotations:
      expected_value:
        tag: expected_value
        value: medication name
      occurrence:
        tag: occurrence
        value: '1'
    description: Specification of birth control medication used
    title: birth control
    examples:
    - value: ''
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - birth control
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000286
    multivalued: false
    alias: birth_control
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: string
    required: false
  menopause:
    name: menopause
    annotations:
      expected_value:
        tag: expected_value
        value: timestamp
      occurrence:
        tag: occurrence
        value: '1'
    description: Date of onset of menopause
    title: menopause
    examples:
    - value: '2018-05-11'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - menopause
    is_a: core field
    slot_uri: MIXS:0000968
    multivalued: false
    alias: menopause
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: date
    required: false
  hrt:
    name: hrt
    annotations:
      expected_value:
        tag: expected_value
        value: timestamp
      occurrence:
        tag: occurrence
        value: '1'
    description: Whether subject had hormone replacement theraphy, and if yes start
      date
    title: HRT
    examples:
    - value: '2018-05-11'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - HRT
    is_a: core field
    slot_uri: MIXS:0000969
    multivalued: false
    alias: hrt
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: date
    required: false
  hysterectomy:
    name: hysterectomy
    annotations:
      expected_value:
        tag: expected_value
        value: hysterectomy status
      occurrence:
        tag: occurrence
        value: '1'
    description: Specification of whether hysterectomy was performed
    title: hysterectomy
    examples:
    - value: 'no'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - hysterectomy
    is_a: core field
    string_serialization: '{boolean}'
    slot_uri: MIXS:0000287
    multivalued: false
    alias: hysterectomy
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: string
    required: false
  gynecologic_disord:
    name: gynecologic_disord
    annotations:
      expected_value:
        tag: expected_value
        value: gynecological disorder
      occurrence:
        tag: occurrence
        value: m
    description: History of gynecological disorders; can include multiple disorders.
      The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      female reproductive system disease (https://disease-ontology.org/?id=DOID:229).
    title: gynecological disorder
    examples:
    - value: ''
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - gynecological disorder
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000288
    multivalued: true
    alias: gynecologic_disord
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: string
    required: false
  urogenit_disord:
    name: urogenit_disord
    annotations:
      expected_value:
        tag: expected_value
        value: disorder name
      occurrence:
        tag: occurrence
        value: m
    description: History of urogenital disorders, can include multiple disorders.
      The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
      reproductive system disease (https://disease-ontology.org/?id=DOID:15) or urinary
      system disease (https://disease-ontology.org/?id=DOID:18).
    title: urogenital disorder
    examples:
    - value: ''
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - urogenital disorder
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000289
    multivalued: true
    alias: urogenit_disord
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    range: string
    required: false
  host_subject_id:
    name: host_subject_id
    annotations:
      expected_value:
        tag: expected_value
        value: unique identifier
      occurrence:
        tag: occurrence
        value: '1'
    description: A unique identifier by which each subject can be referred to, de-identified.
    title: host subject id
    examples:
    - value: MPI123
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host subject id
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000861
    multivalued: false
    alias: host_subject_id
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - symbiont-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_age:
    name: host_age
    annotations:
      expected_value:
        tag: expected_value
        value: value
      preferred_unit:
        tag: preferred_unit
        value: year, day, hour
      occurrence:
        tag: occurrence
        value: '1'
    description: Age of host at the time of sampling; relevant scale depends on species
      and study, e.g. Could be seconds for amoebae or centuries for trees
    title: host age
    examples:
    - value: 10 days
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host age
    is_a: core field
    slot_uri: MIXS:0000255
    multivalued: false
    alias: host_age
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: quantity value
    required: false
  host_sex:
    name: host_sex
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
      occurrence:
        tag: occurrence
        value: '1'
    description: Gender or physical sex of the host.
    title: host sex
    examples:
    - value: non-binary
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host sex
    is_a: core field
    slot_uri: MIXS:0000811
    multivalued: false
    alias: host_sex
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - symbiont-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: host_sex_enum
    required: false
  ihmc_medication_code:
    name: ihmc_medication_code
    annotations:
      expected_value:
        tag: expected_value
        value: IHMC code
      occurrence:
        tag: occurrence
        value: m
    description: Can include multiple medication codes
    title: IHMC medication code
    examples:
    - value: '810'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - IHMC medication code
    is_a: core field
    slot_uri: MIXS:0000884
    multivalued: true
    alias: ihmc_medication_code
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: integer
    required: false
  chem_administration:
    name: chem_administration
    annotations:
      expected_value:
        tag: expected_value
        value: CHEBI;timestamp
      occurrence:
        tag: occurrence
        value: m
    description: List of chemical compounds administered to the host or site where
      sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can
      include multiple compounds. For chemical entities of biological interest ontology
      (chebi) (v 163), http://purl.bioontology.org/ontology/chebi
    title: chemical administration
    examples:
    - value: agar [CHEBI:2509];2018-05-11T20:00Z
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - chemical administration
    is_a: core field
    string_serialization: '{termLabel} {[termID]};{timestamp}'
    slot_uri: MIXS:0000751
    multivalued: true
    alias: chem_administration
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false
  host_body_site:
    name: host_body_site
    annotations:
      expected_value:
        tag: expected_value
        value: FMA or UBERON
      occurrence:
        tag: occurrence
        value: '1'
    description: Name of body site where the sample was obtained from, such as a specific
      organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology
      (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms,
      please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON
    title: host body site
    examples:
    - value: gill [UBERON:0002535]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host body site
    is_a: core field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000867
    multivalued: false
    alias: host_body_site
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - symbiont-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_body_product:
    name: host_body_product
    annotations:
      expected_value:
        tag: expected_value
        value: FMA or UBERON
      occurrence:
        tag: occurrence
        value: '1'
    description: Substance produced by the body, e.g. Stool, mucus, where the sample
      was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy
      ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma
      or https://www.ebi.ac.uk/ols/ontologies/uberon
    title: host body product
    examples:
    - value: Portion of mucus [fma66938]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host body product
    is_a: core field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000888
    multivalued: false
    alias: host_body_product
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - symbiont-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_tot_mass:
    name: host_tot_mass
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: kilogram, gram
      occurrence:
        tag: occurrence
        value: '1'
    description: Total mass of the host at collection, the unit depends on host
    title: host total mass
    examples:
    - value: 2500 gram
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host total mass
    is_a: core field
    slot_uri: MIXS:0000263
    multivalued: false
    alias: host_tot_mass
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: quantity value
    required: false
  host_height:
    name: host_height
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: centimeter, millimeter, meter
      occurrence:
        tag: occurrence
        value: '1'
    description: The height of subject
    title: host height
    examples:
    - value: 0.1 meter
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host height
    is_a: core field
    slot_uri: MIXS:0000264
    multivalued: false
    alias: host_height
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: quantity value
    required: false
  host_diet:
    name: host_diet
    annotations:
      expected_value:
        tag: expected_value
        value: diet type
      occurrence:
        tag: occurrence
        value: m
    description: Type of diet depending on the host, for animals omnivore, herbivore
      etc., for humans high-fat, meditteranean etc.; can include multiple diet types
    title: host diet
    examples:
    - value: herbivore
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host diet
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000869
    multivalued: true
    alias: host_diet
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_last_meal:
    name: host_last_meal
    annotations:
      expected_value:
        tag: expected_value
        value: content;duration
      occurrence:
        tag: occurrence
        value: m
    description: Content of last meal and time since feeding; can include multiple
      values
    title: host last meal
    examples:
    - value: corn feed;P2H
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host last meal
    is_a: core field
    string_serialization: '{text};{duration}'
    slot_uri: MIXS:0000870
    multivalued: true
    alias: host_last_meal
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_family_relation:
    name: host_family_relation
    annotations:
      expected_value:
        tag: expected_value
        value: relationship type;arbitrary identifier
      occurrence:
        tag: occurrence
        value: m
    description: Familial relationships to other hosts in the same study; can include
      multiple relationships
    title: host family relationship
    examples:
    - value: offspring;Mussel25
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host family relationship
    is_a: core field
    string_serialization: '{text};{text}'
    slot_uri: MIXS:0000872
    multivalued: true
    alias: host_family_relation
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_genotype:
    name: host_genotype
    annotations:
      expected_value:
        tag: expected_value
        value: genotype
      occurrence:
        tag: occurrence
        value: '1'
    description: Observed genotype
    title: host genotype
    examples:
    - value: C57BL/6
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host genotype
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000365
    multivalued: false
    alias: host_genotype
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_phenotype:
    name: host_phenotype
    annotations:
      expected_value:
        tag: expected_value
        value: PATO or HP
      occurrence:
        tag: occurrence
        value: '1'
    description: Phenotype of human or other host. For phenotypic quality ontology
      (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato.
      For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP
    title: host phenotype
    examples:
    - value: elongated [PATO:0001154]
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host phenotype
    is_a: core field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000874
    multivalued: false
    alias: host_phenotype
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  host_body_temp:
    name: host_body_temp
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: degree Celsius
      occurrence:
        tag: occurrence
        value: '1'
    description: Core body temperature of the host when sample was collected
    title: host body temperature
    examples:
    - value: 15 degree Celsius
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host body temperature
    is_a: core field
    slot_uri: MIXS:0000274
    multivalued: false
    alias: host_body_temp
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: quantity value
    required: false
  host_body_mass_index:
    name: host_body_mass_index
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: kilogram per square meter
      occurrence:
        tag: occurrence
        value: '1'
    description: Body mass index, calculated as weight/(height)squared
    title: host body-mass index
    examples:
    - value: 22 kilogram per square meter
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host body-mass index
    is_a: core field
    slot_uri: MIXS:0000317
    multivalued: false
    alias: host_body_mass_index
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: quantity value
    required: false
  ethnicity:
    name: ethnicity
    annotations:
      expected_value:
        tag: expected_value
        value: text recommend from Wikipedia list
      occurrence:
        tag: occurrence
        value: m
    description: A category of people who identify with each other, usually on the
      basis of presumed similarities such as a common language, ancestry, history,
      society, culture, nation or social treatment within their residing area. https://en.wikipedia.org/wiki/List_of_contemporary_ethnic_groups
    title: ethnicity
    examples:
    - value: native american
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - ethnicity
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000895
    multivalued: true
    alias: ethnicity
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: string
    required: false
  host_occupation:
    name: host_occupation
    annotations:
      expected_value:
        tag: expected_value
        value: IHMC code
      occurrence:
        tag: occurrence
        value: '1'
    description: Most frequent job performed by subject
    title: host occupation
    examples:
    - value: veterinary
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host occupation
    is_a: core field
    slot_uri: MIXS:0000896
    multivalued: false
    alias: host_occupation
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: integer
    required: false
  medic_hist_perform:
    name: medic_hist_perform
    annotations:
      expected_value:
        tag: expected_value
        value: true or false
      occurrence:
        tag: occurrence
        value: '1'
    description: Whether full medical history was collected
    title: medical history performed
    examples:
    - value: 'TRUE'
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - medical history performed
    is_a: core field
    string_serialization: '{boolean}'
    slot_uri: MIXS:0000897
    multivalued: false
    alias: medic_hist_perform
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: string
    required: false
  host_pulse:
    name: host_pulse
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: beats per minute
      occurrence:
        tag: occurrence
        value: '1'
    description: Resting pulse, measured as beats per minute
    title: host pulse
    examples:
    - value: 65 beats per minute
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - host pulse
    is_a: core field
    slot_uri: MIXS:0000333
    multivalued: false
    alias: host_pulse
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    range: quantity value
    required: false
  perturbation:
    name: perturbation
    annotations:
      expected_value:
        tag: expected_value
        value: perturbation type name;perturbation interval and duration
      occurrence:
        tag: occurrence
        value: m
    description: Type of perturbation, e.g. chemical administration, physical disturbance,
      etc., coupled with perturbation regimen including how many times the perturbation
      was repeated, how long each perturbation lasted, and the start and end time
      of the entire perturbation period; can include multiple perturbation types
    title: perturbation
    examples:
    - value: antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - perturbation
    is_a: core field
    string_serialization: '{text};{Rn/start_time/end_time/duration}'
    slot_uri: MIXS:0000754
    multivalued: true
    alias: perturbation
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false
  samp_salinity:
    name: samp_salinity
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: milligram per liter, practical salinity unit, percentage
      occurrence:
        tag: occurrence
        value: '1'
    description: Salinity is the total concentration of all dissolved salts in a liquid
      or solid (in the form of an extract obtained by centrifugation) sample. While
      salinity can be measured by a complete chemical analysis, this method is difficult
      and time consuming. More often, it is instead derived from the conductivity
      measurement. This is known as practical salinity. These derivations compare
      the specific conductance of the sample to a salinity standard such as seawater
    title: sample salinity
    examples:
    - value: 1 milligram per liter
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample salinity
    is_a: core field
    slot_uri: MIXS:0000109
    multivalued: false
    alias: samp_salinity
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - symbiont-associated
    - human-vaginal
    range: quantity value
    required: false
  salinity:
    name: salinity
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: practical salinity unit, percentage
      occurrence:
        tag: occurrence
        value: '1'
    description: The total concentration of all dissolved salts in a liquid or solid
      sample. While salinity can be measured by a complete chemical analysis, this
      method is difficult and time consuming. More often, it is instead derived from
      the conductivity measurement. This is known as practical salinity. These derivations
      compare the specific conductance of the sample to a salinity standard such as
      seawater.
    title: salinity
    examples:
    - value: 25 practical salinity unit
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - salinity
    is_a: core field
    slot_uri: MIXS:0000183
    multivalued: false
    alias: salinity
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - food-farm environment
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: quantity value
    required: false
  oxy_stat_samp:
    name: oxy_stat_samp
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
      occurrence:
        tag: occurrence
        value: '1'
    description: Oxygenation status of sample
    title: oxygenation status of sample
    examples:
    - value: aerobic
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - oxygenation status of sample
    is_a: core field
    slot_uri: MIXS:0000753
    multivalued: false
    alias: oxy_stat_samp
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: oxy_stat_samp_enum
    required: false
  organism_count:
    name: organism_count
    annotations:
      expected_value:
        tag: expected_value
        value: organism name;measurement value;enumeration
      preferred_unit:
        tag: preferred_unit
        value: number of cells per cubic meter, number of cells per milliliter, number
          of cells per cubic centimeter
      occurrence:
        tag: occurrence
        value: m
    description: 'Total cell count of any organism (or group of organisms) per gram,
      volume or area of sample, should include name of organism followed by count.
      The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should
      also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)'
    title: organism count
    examples:
    - value: total prokaryotes;3.5e7 cells per milliliter;qPCR
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - organism count
    is_a: core field
    slot_uri: MIXS:0000103
    multivalued: true
    alias: organism_count
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    range: organism_count_enum
    required: false
  samp_store_temp:
    name: samp_store_temp
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: degree Celsius
      occurrence:
        tag: occurrence
        value: '1'
    description: Temperature at which sample was stored, e.g. -80 degree Celsius
    title: sample storage temperature
    examples:
    - value: -80 degree Celsius
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample storage temperature
    is_a: core field
    slot_uri: MIXS:0000110
    multivalued: false
    alias: samp_store_temp
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: quantity value
    required: false
  samp_store_loc:
    name: samp_store_loc
    annotations:
      expected_value:
        tag: expected_value
        value: location name
      occurrence:
        tag: occurrence
        value: '1'
    description: Location at which sample was stored, usually name of a specific freezer/room
    title: sample storage location
    examples:
    - value: Freezer no:5
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample storage location
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0000755
    multivalued: false
    alias: samp_store_loc
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false
  samp_store_dur:
    name: samp_store_dur
    annotations:
      expected_value:
        tag: expected_value
        value: duration
      occurrence:
        tag: occurrence
        value: '1'
    description: Duration for which the sample was stored
    title: sample storage duration
    examples:
    - value: P1Y6M
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - sample storage duration
    is_a: core field
    string_serialization: '{duration}'
    slot_uri: MIXS:0000116
    multivalued: false
    alias: samp_store_dur
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - symbiont-associated
    - water
    - wastewater_sludge
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false
  host_symbiont:
    name: host_symbiont
    annotations:
      expected_value:
        tag: expected_value
        value: species name or common name
      occurrence:
        tag: occurrence
        value: m
    description: The taxonomic name of the organism(s) found living in mutualistic,
      commensalistic, or parasitic symbiosis with the specific host.
    title: observed host symbionts
    examples:
    - value: flukeworms
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - observed host symbionts
    is_a: core field
    string_serialization: '{text}'
    slot_uri: MIXS:0001298
    multivalued: true
    alias: host_symbiont
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - symbiont-associated
    - plant-associated
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    range: string
    required: false
  misc_param:
    name: misc_param
    annotations:
      expected_value:
        tag: expected_value
        value: parameter name;measurement value
      occurrence:
        tag: occurrence
        value: m
    description: Any other measurement performed or parameter collected, that is not
      listed here
    title: miscellaneous parameter
    examples:
    - value: Bicarbonate ion concentration;2075 micromole per kilogram
    from_schema: http://w3id.org/mixs/terms
    aliases:
    - miscellaneous parameter
    is_a: core field
    string_serialization: '{text};{float} {unit}'
    slot_uri: MIXS:0000752
    multivalued: true
    alias: misc_param
    owner: human-vaginal MIGS eukaryote
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - air
    range: string
    required: false

</code></pre>

</details>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
      
    
    
  </body>
</html>