
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Core/">
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.4.3, mkdocs-material-8.5.10">
    
    
      
        <title>Class: Core - MIxS</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.975780f9.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.2505c338.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="None" data-md-color-accent="None">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#class-core" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="MIxS" class="md-header__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MIxS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Class: Core
            
          </span>
        </div>
      </div>
    </div>
    
    
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="MIxS" class="md-nav__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    MIxS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Reference
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Introduction
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Introduction" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Introduction
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/citation/" class="md-nav__link">
        Cite
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/contents/" class="md-nav__link">
        Contents
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#identifier-and-mapping-information" class="md-nav__link">
    Identifier and Mapping Information
  </a>
  
    <nav class="md-nav" aria-label="Identifier and Mapping Information">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#schema-source" class="md-nav__link">
    Schema Source
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mappings" class="md-nav__link">
    Mappings
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#linkml-source" class="md-nav__link">
    LinkML Source
  </a>
  
    <nav class="md-nav" aria-label="LinkML Source">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#direct" class="md-nav__link">
    Direct
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#induced" class="md-nav__link">
    Induced
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Core.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="class-core">Class: Core</h1>
<p><em>Core package. Do not use this directly, this is used to build other packages</em></p>
<p>URI: <a href="https://w3id.org/mixs/Core">MIXS:Core</a></p>
<!-- no inheritance hierarchy -->

<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0001107/">samp_name</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A local identifier or name that for the material sample used for extracting n...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0001320/">samp_taxon_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>NCBI taxon id of the sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000092/">project_name</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name of the project within which the sequencing was organized</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000008/">experimental_factor</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Experimental factors are essentially the variable aspects of an experiment de...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000009/">lat_lon</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by latitude and longitude</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000018/">depth</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The vertical distance below local surface, e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000094/">alt</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Altitude is a term used to identify heights of objects such as airplanes, spa...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000093/">elev</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Elevation of the sampling site is its height above a fixed reference point, m...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000113/">temp</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Temperature of the sample at the time of sampling</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000010/">geo_loc_name</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by the country or sea name f...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000011/">collection_date</a></td>
<td>0..1 <br/> <a href="../Date/">Date</a></td>
<td>The time of sampling, either as an instance (single point in time) or interva...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0001321/">neg_cont_type</a></td>
<td>0..1 <br/> <a href="../NegContTypeEnum/">NegContTypeEnum</a></td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0001322/">pos_cont_type</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The substance, mixture, product, or apparatus used to verify that a process w...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000012/">env_broad_scale</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000013/">env_local_scale</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Report the entity or entities which are in the sample or specimen’s local vic...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000014/">env_medium</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000020/">subspecf_gen_lin</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Information about the genetic distinctness of the sequenced organism below th...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000021/">ploidy</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The ploidy level of the genome (e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000022/">num_replicons</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Reports the number of replicons in a nuclear genome of eukaryotes, in the gen...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000023/">extrachrom_elements</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Do plasmids exist of significant phenotypic consequence (e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000024/">estimated_size</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The estimated size of the genome prior to sequencing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000025/">ref_biomaterial</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Primary publication if isolated before genome publication; otherwise, primary...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000026/">source_mat_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A unique identifier assigned to a material sample (as defined by http://rs</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000027/">pathogenicity</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>To what is the entity pathogenic</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000028/">biotic_relationship</a></td>
<td>0..1 <br/> <a href="../BioticRelationshipEnum/">BioticRelationshipEnum</a></td>
<td>Description of relationship(s) between the subject organism and other organis...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000029/">specific_host</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000030/">host_spec_range</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>The range and diversity of host species that an organism is capable of infect...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000031/">host_disease_stat</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>List of diseases with which the host has been diagnosed; can include multiple...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000032/">trophic_level</a></td>
<td>0..1 <br/> <a href="../TrophicLevelEnum/">TrophicLevelEnum</a></td>
<td>Trophic levels are the feeding position in a food chain</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000033/">propagation</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The type of reproduction from the parent stock</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000034/">encoded_traits</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Should include key traits like antibiotic resistance or xenobiotic degradatio...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000015/">rel_to_oxygen</a></td>
<td>0..1 <br/> <a href="../RelToOxygenEnum/">RelToOxygenEnum</a></td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000003/">isol_growth_condt</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identif...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000002/">samp_collec_device</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The device used to collect an environmental sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0001225/">samp_collec_method</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The method employed for collecting the sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000016/">samp_mat_process</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A brief description of any processing applied to the sample during or after r...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000017/">size_frac</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Filtering pore size used in sample preparation</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000001/">samp_size</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample coll...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000111/">samp_vol_we_dna_ext</a></td>
<td>0..1 <br/> <a href="../QuantityValue/">QuantityValue</a></td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extractio...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000035/">source_uvig</a></td>
<td>0..1 <br/> <a href="../SourceUvigEnum/">SourceUvigEnum</a></td>
<td>Type of dataset from which the UViG was obtained</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000036/">virus_enrich_appr</a></td>
<td>0..1 <br/> <a href="../VirusEnrichApprEnum/">VirusEnrichApprEnum</a></td>
<td>List of approaches used to enrich the sample for viruses, if any</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000037/">nucl_acid_ext</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000038/">nucl_acid_amp</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000039/">lib_size</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones in the library prepared for the project</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000040/">lib_reads_seqd</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones sequenced from the library</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000041/">lib_layout</a></td>
<td>0..1 <br/> <a href="../LibLayoutEnum/">LibLayoutEnum</a></td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000042/">lib_vector</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000043/">lib_screen</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specific enrichment or screening methods applied before and/or after creating...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000044/">target_gene</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Targeted gene or locus name for marker gene studies</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000045/">target_subfragment</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name of subfragment of a gene or locus</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000046/">pcr_primers</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>PCR primers that were used to amplify the sequence of the targeted gene, locu...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000047/">mid</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to spe...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000048/">adapters</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Adapters provide priming sequences for both amplification and sequencing of t...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000049/">pcr_cond</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Description of reaction conditions and components of PCR in the form of 'init...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000050/">seq_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Sequencing machine used</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000051/">seq_quality_check</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Indicate if the sequence has been called by automatic systems (none) or under...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000052/">chimera_check</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for chimera checking, including version number and parameters, t...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000053/">tax_ident</a></td>
<td>0..1 <br/> <a href="../TaxIdentEnum/">TaxIdentEnum</a></td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000056/">assembly_qual</a></td>
<td>0..1 <br/> <a href="../AssemblyQualEnum/">AssemblyQualEnum</a></td>
<td>The assembly quality category is based on sets of criteria outlined for each ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000057/">assembly_name</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name/version of the assembly provided by the submitter that is used in the ge...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000058/">assembly_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000059/">annot</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool used for annotation, or for cases where annotation was provided by a com...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000060/">number_contig</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a giv...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000061/">feat_pred</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000062/">ref_db</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>List of database(s) used for ORF annotation, along with version number and re...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000063/">sim_search_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000064/">tax_class</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Method used for taxonomic classification, along with reference database used,...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000065/">x_16s_recover</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Can a 16S gene be recovered from the submitted SAG or MAG?</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000066/">x_16s_recover_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for 16S rRNA gene extraction</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000067/">trnas</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>The total number of tRNAs identified from the SAG or MAG</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000068/">trna_ext_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for tRNA identification</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000069/">compl_score</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Completeness score is typically based on either the fraction of markers found...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000070/">compl_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for completion estimate, i</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000071/">compl_appr</a></td>
<td>0..1 <br/> <a href="../ComplApprEnum/">ComplApprEnum</a></td>
<td>The approach used to determine the completeness of a given genomic assembly, ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000072/">contam_score</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The contamination score is based on the fraction of single-copy genes that ar...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000005/">contam_screen_input</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The type of sequence data used as input</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000073/">contam_screen_param</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Specific parameters used in the decontamination sofware, such as reference da...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000074/">decontam_software</a></td>
<td>0..1 <br/> <a href="../DecontamSoftwareEnum/">DecontamSoftwareEnum</a></td>
<td>Tool(s) used in contamination screening</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000075/">sort_tech</a></td>
<td>0..1 <br/> <a href="../SortTechEnum/">SortTechEnum</a></td>
<td>Method used to sort/isolate cells or particles of interest</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000076/">single_cell_lysis_appr</a></td>
<td>0..1 <br/> <a href="../SingleCellLysisApprEnum/">SingleCellLysisApprEnum</a></td>
<td>Method used to free DNA from interior of the cell(s) or particle(s)</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000054/">single_cell_lysis_prot</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name of the kit or standard protocol used for cell(s) or particle(s) lysis</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000055/">wga_amp_appr</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Method used to amplify genomic DNA in preparation for sequencing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000006/">wga_amp_kit</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Kit used to amplify genomic DNA in preparation for sequencing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000077/">bin_param</a></td>
<td>0..1 <br/> <a href="../BinParamEnum/">BinParamEnum</a></td>
<td>The parameters that have been applied during the extraction of genomes from m...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000078/">bin_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for the extraction of genomes from metagenomic datasets, where p...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000079/">reassembly_bin</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Has an assembly been performed on a genome bin extracted from a metagenomic a...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000080/">mag_cov_software</a></td>
<td>0..1 <br/> <a href="../MagCovSoftwareEnum/">MagCovSoftwareEnum</a></td>
<td>Tool(s) used to determine the genome coverage if coverage is used as a binnin...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000081/">vir_ident_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for the identification of UViG as a viral genome, software or pr...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000082/">pred_genome_type</a></td>
<td>0..1 <br/> <a href="../PredGenomeTypeEnum/">PredGenomeTypeEnum</a></td>
<td>Type of genome predicted for the UViG</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000083/">pred_genome_struc</a></td>
<td>0..1 <br/> <a href="../PredGenomeStrucEnum/">PredGenomeStrucEnum</a></td>
<td>Expected structure of the viral genome</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000084/">detec_type</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Type of UViG detection</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000085/">otu_class_appr</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Cutoffs and approach used when clustering “species-level” OTUs</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000086/">otu_seq_comp_appr</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool and thresholds used to compare sequences when computing "species-level" ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000087/">otu_db</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference database (i</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000088/">host_pred_appr</a></td>
<td>0..1 <br/> <a href="../HostPredApprEnum/">HostPredApprEnum</a></td>
<td>Tool or approach used for host prediction</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000089/">host_pred_est_acc</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>For each tool or approach used for host prediction, estimated false discovery...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000091/">associated resource</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A related resource that is referenced, cited, or otherwise associated to the ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../0000090/">sop</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, ...</td>
<td>direct</td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: http://w3id.org/mixs</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>MIXS:Core</td>
</tr>
<tr>
<td>native</td>
<td>MIXS:Core</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<h3 id="direct">Direct</h3>
<details>

<pre><code class="language-yaml">name: core
description: Core package. Do not use this directly, this is used to build other packages
from_schema: http://w3id.org/mixs
slots:
- samp_name
- samp_taxon_id
- project_name
- experimental_factor
- lat_lon
- depth
- alt
- elev
- temp
- geo_loc_name
- collection_date
- neg_cont_type
- pos_cont_type
- env_broad_scale
- env_local_scale
- env_medium
- subspecf_gen_lin
- ploidy
- num_replicons
- extrachrom_elements
- estimated_size
- ref_biomaterial
- source_mat_id
- pathogenicity
- biotic_relationship
- specific_host
- host_spec_range
- host_disease_stat
- trophic_level
- propagation
- encoded_traits
- rel_to_oxygen
- isol_growth_condt
- samp_collec_device
- samp_collec_method
- samp_mat_process
- size_frac
- samp_size
- samp_vol_we_dna_ext
- source_uvig
- virus_enrich_appr
- nucl_acid_ext
- nucl_acid_amp
- lib_size
- lib_reads_seqd
- lib_layout
- lib_vector
- lib_screen
- target_gene
- target_subfragment
- pcr_primers
- mid
- adapters
- pcr_cond
- seq_meth
- seq_quality_check
- chimera_check
- tax_ident
- assembly_qual
- assembly_name
- assembly_software
- annot
- number_contig
- feat_pred
- ref_db
- sim_search_meth
- tax_class
- x_16s_recover
- x_16s_recover_software
- trnas
- trna_ext_software
- compl_score
- compl_software
- compl_appr
- contam_score
- contam_screen_input
- contam_screen_param
- decontam_software
- sort_tech
- single_cell_lysis_appr
- single_cell_lysis_prot
- wga_amp_appr
- wga_amp_kit
- bin_param
- bin_software
- reassembly_bin
- mag_cov_software
- vir_ident_software
- pred_genome_type
- pred_genome_struc
- detec_type
- otu_class_appr
- otu_seq_comp_appr
- otu_db
- host_pred_appr
- host_pred_est_acc
- associated resource
- sop

</code></pre>

</details>

<h3 id="induced">Induced</h3>
<details>

<pre><code class="language-yaml">name: core
description: Core package. Do not use this directly, this is used to build other packages
from_schema: http://w3id.org/mixs
attributes:
  samp_name:
    name: samp_name
    annotations:
      expected_value:
        tag: expected_value
        value: text
    description: A local identifier or name that for the material sample used for
      extracting nucleic acids, and subsequent sequencing. It can refer either to
      the original material collected or to any derived sub-samples. It can have any
      format, but we suggest that you make it concise, unique and consistent within
      your lab, and as informative as possible. INSDC requires every sample name from
      a single Submitter to be unique. Use of a globally unique identifier for the
      field source_mat_id is recommended in addition to sample_name.
    title: sample name
    examples:
    - value: ISDsoil1
    from_schema: http://w3id.org/mixs
    aliases:
    - sample name
    is_a: investigation field
    string_serialization: '{text}'
    slot_uri: MIXS:0001107
    multivalued: false
    alias: samp_name
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  samp_taxon_id:
    name: samp_taxon_id
    annotations:
      expected_value:
        tag: expected_value
        value: Taxonomy ID
    description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa
      sample. Use 'synthetic metagenome’ for mock community/positive controls, or
      'blank sample' for negative controls.
    title: Taxonomy ID of DNA sample
    examples:
    - value: Gut Metagenome [NCBI:txid749906]
    from_schema: http://w3id.org/mixs
    aliases:
    - Taxonomy ID of DNA sample
    is_a: investigation field
    string_serialization: '{text} [NCBI:txid]'
    slot_uri: MIXS:0001320
    multivalued: false
    alias: samp_taxon_id
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  project_name:
    name: project_name
    description: Name of the project within which the sequencing was organized
    title: project name
    examples:
    - value: Forest soil metagenome
    from_schema: http://w3id.org/mixs
    aliases:
    - project name
    is_a: investigation field
    string_serialization: '{text}'
    slot_uri: MIXS:0000092
    multivalued: false
    alias: project_name
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  experimental_factor:
    name: experimental_factor
    annotations:
      expected_value:
        tag: expected_value
        value: text or EFO and/or OBI
    description: Experimental factors are essentially the variable aspects of an experiment
      design which can be used to describe an experiment, or set of experiments, in
      an increasingly detailed manner. This field accepts ontology terms from Experimental
      Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For
      a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO;
      for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI
    title: experimental factor
    examples:
    - value: time series design [EFO:EFO_0001779]
    from_schema: http://w3id.org/mixs
    aliases:
    - experimental factor
    is_a: investigation field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000008
    multivalued: false
    alias: experimental_factor
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  lat_lon:
    name: lat_lon
    annotations:
      expected_value:
        tag: expected_value
        value: decimal degrees,  limit to 8 decimal points
    description: The geographical origin of the sample as defined by latitude and
      longitude. The values should be reported in decimal degrees and in WGS84 system
    title: geographic location (latitude and longitude)
    examples:
    - value: 50.586825 6.408977
    from_schema: http://w3id.org/mixs
    aliases:
    - geographic location (latitude and longitude)
    is_a: environment field
    string_serialization: '{float} {float}'
    slot_uri: MIXS:0000009
    multivalued: false
    alias: lat_lon
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  depth:
    name: depth
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: The vertical distance below local surface, e.g. for sediment or soil
      samples depth is measured from sediment or soil surface, respectively. Depth
      can be reported as an interval for subsurface samples.
    title: depth
    examples:
    - value: 10 meter
    from_schema: http://w3id.org/mixs
    aliases:
    - depth
    is_a: environment field
    slot_uri: MIXS:0000018
    multivalued: false
    alias: depth
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  alt:
    name: alt
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: Altitude is a term used to identify heights of objects such as airplanes,
      space shuttles, rockets, atmospheric balloons and heights of places such as
      atmospheric layers and clouds. It is used to measure the height of an object
      which is above the earth's surface. In this context, the altitude measurement
      is the vertical distance between the earth's surface above sea level and the
      sampled position in the air
    title: altitude
    examples:
    - value: 100 meter
    from_schema: http://w3id.org/mixs
    aliases:
    - altitude
    is_a: environment field
    slot_uri: MIXS:0000094
    multivalued: false
    alias: alt
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  elev:
    name: elev
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
    description: Elevation of the sampling site is its height above a fixed reference
      point, most commonly the mean sea level. Elevation is mainly used when referring
      to points on the earth's surface, while altitude is used for points above the
      surface, such as an aircraft in flight or a spacecraft in orbit.
    title: elevation
    examples:
    - value: 100 meter
    from_schema: http://w3id.org/mixs
    aliases:
    - elevation
    is_a: environment field
    slot_uri: MIXS:0000093
    multivalued: false
    alias: elev
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  temp:
    name: temp
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: degree Celsius
    description: Temperature of the sample at the time of sampling.
    title: temperature
    examples:
    - value: 25 degree Celsius
    from_schema: http://w3id.org/mixs
    aliases:
    - temperature
    is_a: environment field
    slot_uri: MIXS:0000113
    multivalued: false
    alias: temp
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    range: quantity value
  geo_loc_name:
    name: geo_loc_name
    annotations:
      expected_value:
        tag: expected_value
        value: 'country or sea name (INSDC or GAZ): region(GAZ), specific location
          name'
    description: The geographical origin of the sample as defined by the country or
      sea name followed by specific region name. Country or sea names should be chosen
      from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology
      (http://purl.bioontology.org/ontology/GAZ)
    title: geographic location (country and/or sea,region)
    examples:
    - value: 'USA: Maryland, Bethesda'
    from_schema: http://w3id.org/mixs
    aliases:
    - geographic location (country and/or sea,region)
    is_a: environment field
    string_serialization: '{term}: {term}, {text}'
    slot_uri: MIXS:0000010
    multivalued: false
    alias: geo_loc_name
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  collection_date:
    name: collection_date
    annotations:
      expected_value:
        tag: expected_value
        value: date and time
    description: 'The time of sampling, either as an instance (single point in time)
      or interval. In case no exact time is available, the date/time can be right
      truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10;
      2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant'
    title: collection date
    examples:
    - value: 2018-05-11T10:00:00+01:00; 2018-05-11
    from_schema: http://w3id.org/mixs
    aliases:
    - collection date
    is_a: environment field
    slot_uri: MIXS:0000011
    multivalued: false
    alias: collection_date
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: date
  neg_cont_type:
    name: neg_cont_type
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration or text
    description: The substance or equipment used as a negative control in an investigation
    title: negative control type
    examples:
    - value: ''
    from_schema: http://w3id.org/mixs
    aliases:
    - negative control type
    is_a: investigation field
    slot_uri: MIXS:0001321
    multivalued: false
    alias: neg_cont_type
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: neg_cont_type_enum
  pos_cont_type:
    name: pos_cont_type
    description: The substance, mixture, product, or apparatus used to verify that
      a process which is part of an investigation delivers a true positive.
    title: positive control type
    examples:
    - value: ''
    from_schema: http://w3id.org/mixs
    aliases:
    - positive control type
    is_a: investigation field
    string_serialization: '{term} or {text}'
    slot_uri: MIXS:0001322
    multivalued: false
    alias: pos_cont_type
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  env_broad_scale:
    name: env_broad_scale
    annotations:
      expected_value:
        tag: expected_value
        value: The major environment type(s) where the sample was collected. Recommend
          subclasses of biome [ENVO:00000428]. Multiple terms can be separated by
          one or more pipes.
    description: 'Report the major environmental system the sample or specimen came
      from. The system(s) identified should have a coarse spatial grain, to provide
      the general environmental context of where the sampling was done (e.g. in the
      desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428.
      EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: broad-scale environmental context
    examples:
    - value: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water
        sample from the photic zone in middle of the Atlantic Ocean
    from_schema: http://w3id.org/mixs
    aliases:
    - broad-scale environmental context
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000012
    multivalued: false
    alias: env_broad_scale
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  env_local_scale:
    name: env_local_scale
    annotations:
      expected_value:
        tag: expected_value
        value: Environmental entities having causal influences upon the entity at
          time of sampling.
    description: 'Report the entity or entities which are in the sample or specimen’s
      local vicinity and which you believe have significant causal influences on your
      sample or specimen. We recommend using EnvO terms which are of smaller spatial
      grain than your entry for env_broad_scale. Terms, such as anatomical sites,
      from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON)
      are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.'
    title: local environmental context
    examples:
    - value: 'litter layer [ENVO:01000338]; Annotating a pooled sample taken from
        various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb
        and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub
        layer [ENVO:01000336].'
    from_schema: http://w3id.org/mixs
    aliases:
    - local environmental context
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000013
    multivalued: false
    alias: env_local_scale
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  env_medium:
    name: env_medium
    annotations:
      expected_value:
        tag: expected_value
        value: The material displaced by the entity at time of sampling. Recommend
          subclasses of environmental material [ENVO:00010483].
    description: 'Report the environmental material(s) immediately surrounding the
      sample or specimen at the time of sampling. We recommend using subclasses of
      ''environmental material'' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO
      documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
      . Terms from other OBO ontologies are permissible as long as they reference
      mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
      (e.g. a tree, a leaf, a table top).'
    title: environmental medium
    examples:
    - value: 'soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m
        of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating
        a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005]'
    from_schema: http://w3id.org/mixs
    aliases:
    - environmental medium
    is_a: environment field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000014
    multivalued: false
    alias: env_medium
    owner: core
    domain_of:
    - agriculture
    - food-farm environment
    - food-food production facility
    - food-animal and animal feed
    - food-human foods
    - symbiont-associated
    - water
    - wastewater_sludge
    - soil
    - sediment
    - plant-associated
    - miscellaneous natural or artificial environment
    - microbial mat_biofilm
    - hydrocarbon resources-fluids_swabs
    - hydrocarbon resources-cores
    - human-vaginal
    - human-skin
    - human-oral
    - human-gut
    - human-associated
    - host-associated
    - built environment
    - air
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  subspecf_gen_lin:
    name: subspecf_gen_lin
    annotations:
      expected_value:
        tag: expected_value
        value: Genetic lineage below lowest rank of NCBI taxonomy, which is subspecies,
          e.g. serovar, biotype, ecotype, variety, cultivar.
    description: Information about the genetic distinctness of the sequenced organism
      below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any
      relevant genetic typing schemes like Group I plasmid. Subspecies should not
      be recorded in this term, but in the NCBI taxonomy. Supply both the lineage
      name and the lineage rank separated by a colon, e.g., biovar:abc123.
    title: subspecific genetic lineage
    examples:
    - value: serovar:Newport
    from_schema: http://w3id.org/mixs
    aliases:
    - subspecific genetic lineage
    is_a: nucleic acid sequence source field
    string_serialization: '{rank name}:{text}'
    slot_uri: MIXS:0000020
    multivalued: false
    alias: subspecf_gen_lin
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMARKS survey
    range: string
  ploidy:
    name: ploidy
    annotations:
      expected_value:
        tag: expected_value
        value: PATO
    description: The ploidy level of the genome (e.g. allopolyploid, haploid, diploid,
      triploid, tetraploid). It has implications for the downstream study of duplicated
      gene and regions of the genomes (and perhaps for difficulties in assembly).
      For terms, please select terms listed under class ploidy (PATO:001374) of Phenotypic
      Quality Ontology (PATO), and for a browser of PATO (v 2018-03-27) please refer
      to http://purl.bioontology.org/ontology/PATO
    title: ploidy
    examples:
    - value: allopolyploidy [PATO:0001379]
    from_schema: http://w3id.org/mixs
    aliases:
    - ploidy
    is_a: nucleic acid sequence source field
    string_serialization: '{termLabel} {[termID]}'
    slot_uri: MIXS:0000021
    multivalued: false
    alias: ploidy
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    range: string
  num_replicons:
    name: num_replicons
    annotations:
      expected_value:
        tag: expected_value
        value: 'for eukaryotes and bacteria: chromosomes (haploid count); for viruses:
          segments'
    description: Reports the number of replicons in a nuclear genome of eukaryotes,
      in the genome of a bacterium or archaea or the number of segments in a segmented
      virus. Always applied to the haploid chromosome count of a eukaryote
    title: number of replicons
    examples:
    - value: '2'
    from_schema: http://w3id.org/mixs
    aliases:
    - number of replicons
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000022
    multivalued: false
    alias: num_replicons
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS virus
    range: integer
  extrachrom_elements:
    name: extrachrom_elements
    annotations:
      expected_value:
        tag: expected_value
        value: number of extrachromosmal elements
    description: Do plasmids exist of significant phenotypic consequence (e.g. ones
      that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids
      (borrelia has 15+ plasmids)
    title: extrachromosomal elements
    examples:
    - value: '5'
    from_schema: http://w3id.org/mixs
    aliases:
    - extrachromosomal elements
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000023
    multivalued: false
    alias: extrachrom_elements
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS org
    - MIMARKS survey
    range: integer
  estimated_size:
    name: estimated_size
    annotations:
      expected_value:
        tag: expected_value
        value: number of base pairs
    description: The estimated size of the genome prior to sequencing. Of particular
      importance in the sequencing of (eukaryotic) genome which could remain in draft
      form for a long or unspecified period.
    title: estimated size
    examples:
    - value: 300000 bp
    from_schema: http://w3id.org/mixs
    aliases:
    - estimated size
    is_a: nucleic acid sequence source field
    string_serialization: '{integer} bp'
    slot_uri: MIXS:0000024
    multivalued: false
    alias: estimated_size
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIUVIG
    range: string
  ref_biomaterial:
    name: ref_biomaterial
    annotations:
      expected_value:
        tag: expected_value
        value: PMID, DOI or URL
    description: Primary publication if isolated before genome publication; otherwise,
      primary genome report.
    title: reference for biomaterial
    examples:
    - value: doi:10.1016/j.syapm.2018.01.009
    from_schema: http://w3id.org/mixs
    aliases:
    - reference for biomaterial
    is_a: nucleic acid sequence source field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000025
    multivalued: false
    alias: ref_biomaterial
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  source_mat_id:
    name: source_mat_id
    annotations:
      expected_value:
        tag: expected_value
        value: 'for cultures of microorganisms: identifiers for two culture collections;
          for other material a unique arbitrary identifer'
    description: A unique identifier assigned to a material sample (as defined by
      http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular
      digital record of a material sample) used for extracting nucleic acids, and
      subsequent sequencing. The identifier can refer either to the original material
      collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher,
      /bio_material, or /culture_collection may or may not share the same value as
      the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id
      may both contain 'UAM:Herps:14' , referring to both the specimen voucher and
      sampled tissue with the same identifier. However, the /culture_collection qualifier
      may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id
      would refer to an identifier from some derived culture from which the nucleic
      acids were extracted (e.g. xatc123 or ark:/2154/R2).
    title: source material identifiers
    examples:
    - value: MPI012345
    from_schema: http://w3id.org/mixs
    aliases:
    - source material identifiers
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000026
    multivalued: false
    alias: source_mat_id
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  pathogenicity:
    name: pathogenicity
    annotations:
      expected_value:
        tag: expected_value
        value: names of organisms that the entity is pathogenic to
    description: To what is the entity pathogenic
    title: known pathogenicity
    examples:
    - value: human, animal, plant, fungi, bacteria
    from_schema: http://w3id.org/mixs
    aliases:
    - known pathogenicity
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000027
    multivalued: false
    alias: pathogenicity
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS virus
    - MIUVIG
    range: string
  biotic_relationship:
    name: biotic_relationship
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Description of relationship(s) between the subject organism and other
      organism(s) it is associated with. E.g., parasite on species X; mutualist with
      species Y. The target organism is the subject of the relationship, and the other
      organism(s) is the object
    title: observed biotic relationship
    examples:
    - value: free living
    from_schema: http://w3id.org/mixs
    aliases:
    - observed biotic relationship
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000028
    multivalued: false
    alias: biotic_relationship
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS virus
    - MIMARKS survey
    - MIUVIG
    range: biotic_relationship_enum
  specific_host:
    name: specific_host
    annotations:
      expected_value:
        tag: expected_value
        value: host scientific name, taxonomy ID
    description: Report the host's taxonomic name and/or NCBI taxonomy ID.
    title: host scientific name
    examples:
    - value: Homo sapiens and/or 9606
    from_schema: http://w3id.org/mixs
    aliases:
    - host scientific name
    is_a: nucleic acid sequence source field
    string_serialization: '{text}|{NCBI taxid}'
    slot_uri: MIXS:0000029
    multivalued: false
    alias: specific_host
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIUVIG
    range: string
  host_spec_range:
    name: host_spec_range
    annotations:
      expected_value:
        tag: expected_value
        value: NCBI taxid
    description: The range and diversity of host species that an organism is capable
      of infecting, defined by NCBI taxonomy identifier.
    title: host specificity or range
    examples:
    - value: '9606'
    from_schema: http://w3id.org/mixs
    aliases:
    - host specificity or range
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000030
    multivalued: false
    alias: host_spec_range
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIUVIG
    range: integer
  host_disease_stat:
    name: host_disease_stat
    annotations:
      expected_value:
        tag: expected_value
        value: disease name or Disease Ontology term
    description: List of diseases with which the host has been diagnosed; can include
      multiple diagnoses. The value of the field depends on host; for humans the terms
      should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org,
      non-human host diseases are free text
    title: host disease status
    examples:
    - value: rabies [DOID:11260]
    from_schema: http://w3id.org/mixs
    aliases:
    - host disease status
    is_a: nucleic acid sequence source field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000031
    multivalued: false
    alias: host_disease_stat
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS virus
    - MIUVIG
    range: string
  trophic_level:
    name: trophic_level
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Trophic levels are the feeding position in a food chain. Microbes
      can be a range of producers (e.g. chemolithotroph)
    title: trophic level
    examples:
    - value: heterotroph
    from_schema: http://w3id.org/mixs
    aliases:
    - trophic level
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000032
    multivalued: false
    alias: trophic_level
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIMARKS survey
    range: trophic_level_enum
  propagation:
    name: propagation
    annotations:
      expected_value:
        tag: expected_value
        value: 'for virus: lytic, lysogenic, temperate, obligately lytic; for plasmid:
          incompatibility group; for eukaryote: asexual, sexual; other more specific
          values (e.g., incompatibility group) are allowed'
    description: 'The type of reproduction from the parent stock. Values for this
      field is specific to different taxa. For phage or virus: lytic/lysogenic/temperate/obligately
      lytic. For plasmids: incompatibility group. For eukaryotes: sexual/asexual.'
    title: propagation
    examples:
    - value: lytic
    from_schema: http://w3id.org/mixs
    aliases:
    - propagation
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000033
    multivalued: false
    alias: propagation
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS plant
    - MIGS virus
    range: string
  encoded_traits:
    name: encoded_traits
    annotations:
      expected_value:
        tag: expected_value
        value: 'for plasmid: antibiotic resistance; for phage: converting genes'
    description: Should include key traits like antibiotic resistance or xenobiotic
      degradation phenotypes for plasmids, converting genes for phage
    title: encoded traits
    examples:
    - value: beta-lactamase class A
    from_schema: http://w3id.org/mixs
    aliases:
    - encoded traits
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000034
    multivalued: false
    alias: encoded_traits
    owner: core
    domain_of:
    - core
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    range: string
  rel_to_oxygen:
    name: rel_to_oxygen
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Is this organism an aerobe, anaerobe? Please note that aerobic and
      anaerobic are valid descriptors for microbial environments
    title: relationship to oxygen
    examples:
    - value: aerobe
    from_schema: http://w3id.org/mixs
    aliases:
    - relationship to oxygen
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000015
    multivalued: false
    alias: rel_to_oxygen
    owner: core
    domain_of:
    - core
    - MIGS bacteria
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    range: rel_to_oxygen_enum
  isol_growth_condt:
    name: isol_growth_condt
    annotations:
      expected_value:
        tag: expected_value
        value: PMID,DOI or URL
    description: Publication reference in the form of pubmed ID (pmid), digital object
      identifier (doi) or url for isolation and growth condition specifications of
      the organism/material
    title: isolation and growth condition
    examples:
    - value: 'doi: 10.1016/j.syapm.2018.01.009'
    from_schema: http://w3id.org/mixs
    aliases:
    - isolation and growth condition
    is_a: nucleic acid sequence source field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000003
    multivalued: false
    alias: isol_growth_condt
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMARKS survey
    range: string
  samp_collec_device:
    name: samp_collec_device
    annotations:
      expected_value:
        tag: expected_value
        value: device name
    description: The device used to collect an environmental sample. This field accepts
      terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO).
      This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).
    title: sample collection device
    examples:
    - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
    from_schema: http://w3id.org/mixs
    aliases:
    - sample collection device
    is_a: nucleic acid sequence source field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000002
    multivalued: false
    alias: samp_collec_device
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  samp_collec_method:
    name: samp_collec_method
    annotations:
      expected_value:
        tag: expected_value
        value: PMID,DOI,url , or text
    description: The method employed for collecting the sample.
    title: sample collection method
    examples:
    - value: swabbing
    from_schema: http://w3id.org/mixs
    aliases:
    - sample collection method
    is_a: nucleic acid sequence source field
    string_serialization: '{PMID}|{DOI}|{URL}|{text}'
    slot_uri: MIXS:0001225
    multivalued: false
    alias: samp_collec_method
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  samp_mat_process:
    name: samp_mat_process
    annotations:
      expected_value:
        tag: expected_value
        value: text
    description: A brief description of any processing applied to the sample during
      or after retrieving the sample from environment, or a link to the relevant protocol(s)
      performed.
    title: sample material processing
    examples:
    - value: filtering of seawater, storing samples in ethanol
    from_schema: http://w3id.org/mixs
    aliases:
    - sample material processing
    is_a: nucleic acid sequence source field
    string_serialization: '{text}'
    slot_uri: MIXS:0000016
    multivalued: false
    alias: samp_mat_process
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  size_frac:
    name: size_frac
    annotations:
      expected_value:
        tag: expected_value
        value: filter size value range
    description: Filtering pore size used in sample preparation
    title: size fraction selected
    examples:
    - value: 0-0.22 micrometer
    from_schema: http://w3id.org/mixs
    aliases:
    - size fraction selected
    is_a: nucleic acid sequence source field
    string_serialization: '{float}-{float} {unit}'
    slot_uri: MIXS:0000017
    multivalued: false
    alias: size_frac
    owner: core
    domain_of:
    - core
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  samp_size:
    name: samp_size
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: millliter, gram, milligram, liter
    description: The total amount or size (volume (ml), mass (g) or area (m2) ) of
      sample collected.
    title: amount or size of sample collected
    examples:
    - value: 5 liter
    from_schema: http://w3id.org/mixs
    aliases:
    - amount or size of sample collected
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000001
    multivalued: false
    alias: samp_size
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: quantity value
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    annotations:
      expected_value:
        tag: expected_value
        value: measurement value
      preferred_unit:
        tag: preferred_unit
        value: millliter, gram, milligram, square centimeter
    description: 'Volume (ml) or mass (g) of total collected sample processed for
      DNA extraction. Note: total sample collected should be entered under the term
      Sample Size (MIXS:0000001).'
    title: sample volume or weight for DNA extraction
    examples:
    - value: 1500 milliliter
    from_schema: http://w3id.org/mixs
    aliases:
    - sample volume or weight for DNA extraction
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000111
    multivalued: false
    alias: samp_vol_we_dna_ext
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: quantity value
  source_uvig:
    name: source_uvig
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Type of dataset from which the UViG was obtained
    title: source of UViGs
    examples:
    - value: viral fraction metagenome (virome)
    from_schema: http://w3id.org/mixs
    aliases:
    - source of UViGs
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000035
    multivalued: false
    alias: source_uvig
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: source_uvig_enum
  virus_enrich_appr:
    name: virus_enrich_appr
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: List of approaches used to enrich the sample for viruses, if any
    title: virus enrichment approach
    examples:
    - value: filtration + FeCl Precipitation + ultracentrifugation + DNAse
    from_schema: http://w3id.org/mixs
    aliases:
    - virus enrichment approach
    is_a: nucleic acid sequence source field
    slot_uri: MIXS:0000036
    multivalued: false
    alias: virus_enrich_appr
    owner: core
    domain_of:
    - core
    - MIGS virus
    - MIUVIG
    range: virus_enrich_appr_enum
  nucl_acid_ext:
    name: nucl_acid_ext
    annotations:
      expected_value:
        tag: expected_value
        value: PMID, DOI or URL
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the material separation to recover
      the nucleic acid fraction from a sample
    title: nucleic acid extraction
    examples:
    - value: https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf
    from_schema: http://w3id.org/mixs
    aliases:
    - nucleic acid extraction
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000037
    multivalued: false
    alias: nucl_acid_ext
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  nucl_acid_amp:
    name: nucl_acid_amp
    annotations:
      expected_value:
        tag: expected_value
        value: PMID, DOI or URL
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the enzymatic amplification (PCR,
      TMA, NASBA) of specific nucleic acids
    title: nucleic acid amplification
    examples:
    - value: https://phylogenomics.me/protocols/16s-pcr-protocol/
    from_schema: http://w3id.org/mixs
    aliases:
    - nucleic acid amplification
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000038
    multivalued: false
    alias: nucl_acid_amp
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  lib_size:
    name: lib_size
    annotations:
      expected_value:
        tag: expected_value
        value: number of clones
    description: Total number of clones in the library prepared for the project
    title: library size
    examples:
    - value: '50'
    from_schema: http://w3id.org/mixs
    aliases:
    - library size
    is_a: sequencing field
    slot_uri: MIXS:0000039
    multivalued: false
    alias: lib_size
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
  lib_reads_seqd:
    name: lib_reads_seqd
    annotations:
      expected_value:
        tag: expected_value
        value: number of reads sequenced
    description: Total number of clones sequenced from the library
    title: library reads sequenced
    examples:
    - value: '20'
    from_schema: http://w3id.org/mixs
    aliases:
    - library reads sequenced
    is_a: sequencing field
    slot_uri: MIXS:0000040
    multivalued: false
    alias: lib_reads_seqd
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
  lib_layout:
    name: lib_layout
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Specify whether to expect single, paired, or other configuration
      of reads
    title: library layout
    examples:
    - value: paired
    from_schema: http://w3id.org/mixs
    aliases:
    - library layout
    is_a: sequencing field
    slot_uri: MIXS:0000041
    multivalued: false
    alias: lib_layout
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: lib_layout_enum
  lib_vector:
    name: lib_vector
    annotations:
      expected_value:
        tag: expected_value
        value: vector
    description: Cloning vector type(s) used in construction of libraries
    title: library vector
    examples:
    - value: Bacteriophage P1
    from_schema: http://w3id.org/mixs
    aliases:
    - library vector
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000042
    multivalued: false
    alias: lib_vector
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  lib_screen:
    name: lib_screen
    annotations:
      expected_value:
        tag: expected_value
        value: screening strategy name
    description: Specific enrichment or screening methods applied before and/or after
      creating libraries
    title: library screening strategy
    examples:
    - value: enriched, screened, normalized
    from_schema: http://w3id.org/mixs
    aliases:
    - library screening strategy
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000043
    multivalued: false
    alias: lib_screen
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  target_gene:
    name: target_gene
    annotations:
      expected_value:
        tag: expected_value
        value: gene name
    description: Targeted gene or locus name for marker gene studies
    title: target gene
    examples:
    - value: 16S rRNA, 18S rRNA, nif, amoA, rpo
    from_schema: http://w3id.org/mixs
    aliases:
    - target gene
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000044
    multivalued: false
    alias: target_gene
    owner: core
    domain_of:
    - core
    - MIMARKS specimen
    - MIMARKS survey
    range: string
  target_subfragment:
    name: target_subfragment
    annotations:
      expected_value:
        tag: expected_value
        value: gene fragment name
    description: Name of subfragment of a gene or locus. Important to e.g. identify
      special regions on marker genes like V6 on 16S rRNA
    title: target subfragment
    examples:
    - value: V6, V9, ITS
    from_schema: http://w3id.org/mixs
    aliases:
    - target subfragment
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000045
    multivalued: false
    alias: target_subfragment
    owner: core
    domain_of:
    - core
    - MIMARKS specimen
    - MIMARKS survey
    range: string
  pcr_primers:
    name: pcr_primers
    annotations:
      expected_value:
        tag: expected_value
        value: 'FWD: forward primer sequence;REV:reverse primer sequence'
    description: PCR primers that were used to amplify the sequence of the targeted
      gene, locus or subfragment. This field should contain all the primers used for
      a single PCR reaction if multiple forward or reverse primers are present in
      a single PCR reaction. The primer sequence should be reported in uppercase letters
    title: pcr primers
    examples:
    - value: FWD:GTGCCAGCMGCCGCGGTAA;REV:GGACTACHVGGGTWTCTAAT
    from_schema: http://w3id.org/mixs
    aliases:
    - pcr primers
    is_a: sequencing field
    string_serialization: FWD:{dna};REV:{dna}
    slot_uri: MIXS:0000046
    multivalued: false
    alias: pcr_primers
    owner: core
    domain_of:
    - core
    - MIMARKS specimen
    - MIMARKS survey
    range: string
  mid:
    name: mid
    annotations:
      expected_value:
        tag: expected_value
        value: multiplex identifier sequence
    description: Molecular barcodes, called Multiplex Identifiers (MIDs), that are
      used to specifically tag unique samples in a sequencing run. Sequence should
      be reported in uppercase letters
    title: multiplex identifiers
    examples:
    - value: GTGAATAT
    from_schema: http://w3id.org/mixs
    aliases:
    - multiplex identifiers
    is_a: sequencing field
    string_serialization: '{dna}'
    slot_uri: MIXS:0000047
    multivalued: false
    alias: mid
    owner: core
    domain_of:
    - core
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  adapters:
    name: adapters
    annotations:
      expected_value:
        tag: expected_value
        value: adapter A and B sequence
    description: Adapters provide priming sequences for both amplification and sequencing
      of the sample-library fragments. Both adapters should be reported; in uppercase
      letters
    title: adapters
    examples:
    - value: AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT
    from_schema: http://w3id.org/mixs
    aliases:
    - adapters
    is_a: sequencing field
    string_serialization: '{dna};{dna}'
    slot_uri: MIXS:0000048
    multivalued: false
    alias: adapters
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  pcr_cond:
    name: pcr_cond
    annotations:
      expected_value:
        tag: expected_value
        value: initial denaturation:degrees_minutes;annealing:degrees_minutes;elongation:degrees_minutes;final
          elongation:degrees_minutes;total cycles
    description: Description of reaction conditions and components of PCR in the form
      of 'initial denaturation:94degC_1.5min; annealing=...'
    title: pcr conditions
    examples:
    - value: initial denaturation:94_3;annealing:50_1;elongation:72_1.5;final elongation:72_10;35
    from_schema: http://w3id.org/mixs
    aliases:
    - pcr conditions
    is_a: sequencing field
    string_serialization: initial denaturation:degrees_minutes;annealing:degrees_minutes;elongation:degrees_minutes;final
      elongation:degrees_minutes;total cycles
    slot_uri: MIXS:0000049
    multivalued: false
    alias: pcr_cond
    owner: core
    domain_of:
    - core
    - MIMARKS specimen
    - MIMARKS survey
    range: string
  seq_meth:
    name: seq_meth
    annotations:
      expected_value:
        tag: expected_value
        value: Text or OBI
    description: Sequencing machine used. Where possible the term should be taken
      from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).
    title: sequencing method
    examples:
    - value: 454 Genome Sequencer FLX [OBI:0000702]
    from_schema: http://w3id.org/mixs
    aliases:
    - sequencing method
    is_a: sequencing field
    string_serialization: '{termLabel} {[termID]}|{text}'
    slot_uri: MIXS:0000050
    multivalued: false
    alias: seq_meth
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  seq_quality_check:
    name: seq_quality_check
    annotations:
      expected_value:
        tag: expected_value
        value: none or manually edited
    description: Indicate if the sequence has been called by automatic systems (none)
      or undergone a manual editing procedure (e.g. by inspecting the raw data or
      chromatograms). Applied only for sequences that are not submitted to SRA,ENA
      or DRA
    title: sequence quality check
    examples:
    - value: none
    from_schema: http://w3id.org/mixs
    aliases:
    - sequence quality check
    is_a: sequencing field
    string_serialization: '[none|manually edited]'
    slot_uri: MIXS:0000051
    multivalued: false
    alias: seq_quality_check
    owner: core
    domain_of:
    - core
    - MIMARKS specimen
    - MIMARKS survey
    range: string
  chimera_check:
    name: chimera_check
    annotations:
      expected_value:
        tag: expected_value
        value: name and version of software, parameters used
    description: Tool(s) used for chimera checking, including version number and parameters,
      to discover and remove chimeric sequences. A chimeric sequence is comprised
      of two or more phylogenetically distinct parent sequences.
    title: chimera check software
    examples:
    - value: uchime;v4.1;default parameters
    from_schema: http://w3id.org/mixs
    aliases:
    - chimera check software
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000052
    multivalued: false
    alias: chimera_check
    owner: core
    domain_of:
    - core
    - MIMARKS specimen
    - MIMARKS survey
    range: string
  tax_ident:
    name: tax_ident
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: The phylogenetic marker(s) used to assign an organism name to the
      SAG or MAG
    title: taxonomic identity marker
    examples:
    - value: 'other: rpoB gene'
    from_schema: http://w3id.org/mixs
    aliases:
    - taxonomic identity marker
    is_a: sequencing field
    slot_uri: MIXS:0000053
    multivalued: false
    alias: tax_ident
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: tax_ident_enum
  assembly_qual:
    name: assembly_qual
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: 'The assembly quality category is based on sets of criteria outlined
      for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated,
      contiguous sequence per replicon without gaps or ambiguities with a consensus
      error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments
      where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes
      and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no
      review of assembly other than reporting of standard assembly statistics. Low
      Quality Draft:Many fragments with little to no review of assembly other than
      reporting of standard assembly statistics. Assembly statistics include, but
      are not limited to total assembly size, number of contigs, contig N50/L50, and
      maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence
      per replicon without gaps or ambiguities, with extensive manual review and editing
      to annotate putative gene functions and transcriptional units. High-quality
      draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome
      or replicon sequence or predicted complete. Genome fragment(s): One or multiple
      fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for
      which no genome size could be estimated'
    title: assembly quality
    examples:
    - value: High-quality draft genome
    from_schema: http://w3id.org/mixs
    aliases:
    - assembly quality
    is_a: sequencing field
    slot_uri: MIXS:0000056
    multivalued: false
    alias: assembly_qual
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: assembly_qual_enum
  assembly_name:
    name: assembly_name
    annotations:
      expected_value:
        tag: expected_value
        value: name and version of assembly
    description: Name/version of the assembly provided by the submitter that is used
      in the genome browsers and in the community
    title: assembly name
    examples:
    - value: HuRef, JCVI_ISG_i3_1.0
    from_schema: http://w3id.org/mixs
    aliases:
    - assembly name
    is_a: sequencing field
    string_serialization: '{text} {text}'
    slot_uri: MIXS:0000057
    multivalued: false
    alias: assembly_name
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  assembly_software:
    name: assembly_software
    annotations:
      expected_value:
        tag: expected_value
        value: name and version of software, parameters used
    description: Tool(s) used for assembly, including version number and parameters
    title: assembly software
    examples:
    - value: metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise
    from_schema: http://w3id.org/mixs
    aliases:
    - assembly software
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000058
    multivalued: false
    alias: assembly_software
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  annot:
    name: annot
    annotations:
      expected_value:
        tag: expected_value
        value: name of tool or pipeline used, or annotation source description
    description: Tool used for annotation, or for cases where annotation was provided
      by a community jamboree or model organism database rather than by a specific
      submitter
    title: annotation
    examples:
    - value: prokka
    from_schema: http://w3id.org/mixs
    aliases:
    - annotation
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000059
    multivalued: false
    alias: annot
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  number_contig:
    name: number_contig
    annotations:
      expected_value:
        tag: expected_value
        value: value
    description: Total number of contigs in the cleaned/submitted assembly that makes
      up a given genome, SAG, MAG, or UViG
    title: number of contigs
    examples:
    - value: '40'
    from_schema: http://w3id.org/mixs
    aliases:
    - number of contigs
    is_a: sequencing field
    slot_uri: MIXS:0000060
    multivalued: false
    alias: number_contig
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
  feat_pred:
    name: feat_pred
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s), parameters used
    description: Method used to predict UViGs features such as ORFs, integration site,
      etc.
    title: feature prediction
    examples:
    - value: Prodigal;2.6.3;default parameters
    from_schema: http://w3id.org/mixs
    aliases:
    - feature prediction
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000061
    multivalued: false
    alias: feat_pred
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  ref_db:
    name: ref_db
    annotations:
      expected_value:
        tag: expected_value
        value: names, versions, and references of databases
    description: List of database(s) used for ORF annotation, along with version number
      and reference to website or publication
    title: reference database(s)
    examples:
    - value: pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017
        doi:10.1093/nar/gkw975
    from_schema: http://w3id.org/mixs
    aliases:
    - reference database(s)
    is_a: sequencing field
    string_serialization: '{database};{version};{reference}'
    slot_uri: MIXS:0000062
    multivalued: false
    alias: ref_db
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  sim_search_meth:
    name: sim_search_meth
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s), parameters used
    description: Tool used to compare ORFs with database, along with version and cutoffs
      used
    title: similarity search method
    examples:
    - value: HMMER3;3.1b2;hmmsearch, cutoff of 50 on score
    from_schema: http://w3id.org/mixs
    aliases:
    - similarity search method
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000063
    multivalued: false
    alias: sim_search_meth
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  tax_class:
    name: tax_class
    annotations:
      expected_value:
        tag: expected_value
        value: classification method, database name, and other parameters
    description: Method used for taxonomic classification, along with reference database
      used, classification rank, and thresholds used to classify new genomes
    title: taxonomic classification
    examples:
    - value: vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification,
        default parameters)
    from_schema: http://w3id.org/mixs
    aliases:
    - taxonomic classification
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000064
    multivalued: false
    alias: tax_class
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  x_16s_recover:
    name: x_16s_recover
    annotations:
      expected_value:
        tag: expected_value
        value: boolean
    description: Can a 16S gene be recovered from the submitted SAG or MAG?
    title: 16S recovered
    examples:
    - value: 'yes'
    from_schema: http://w3id.org/mixs
    aliases:
    - 16S recovered
    is_a: sequencing field
    string_serialization: '{boolean}'
    slot_uri: MIXS:0000065
    multivalued: false
    alias: x_16s_recover
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
  x_16s_recover_software:
    name: x_16s_recover_software
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s), parameters used
    description: Tools used for 16S rRNA gene extraction
    title: 16S recovery software
    examples:
    - value: rambl;v2;default parameters
    from_schema: http://w3id.org/mixs
    aliases:
    - 16S recovery software
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000066
    multivalued: false
    alias: x_16s_recover_software
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
  trnas:
    name: trnas
    annotations:
      expected_value:
        tag: expected_value
        value: value from 0-21
    description: The total number of tRNAs identified from the SAG or MAG
    title: number of standard tRNAs extracted
    examples:
    - value: '18'
    from_schema: http://w3id.org/mixs
    aliases:
    - number of standard tRNAs extracted
    is_a: sequencing field
    slot_uri: MIXS:0000067
    multivalued: false
    alias: trnas
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    - MIUVIG
    range: integer
  trna_ext_software:
    name: trna_ext_software
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s), parameters used
    description: Tools used for tRNA identification
    title: tRNA extraction software
    examples:
    - value: infernal;v2;default parameters
    from_schema: http://w3id.org/mixs
    aliases:
    - tRNA extraction software
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000068
    multivalued: false
    alias: trna_ext_software
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  compl_score:
    name: compl_score
    annotations:
      expected_value:
        tag: expected_value
        value: quality;percent completeness
    description: 'Completeness score is typically based on either the fraction of
      markers found as compared to a database or the percent of a genome found as
      compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium
      Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated
      completeness scores'
    title: completeness score
    examples:
    - value: med;60%
    from_schema: http://w3id.org/mixs
    aliases:
    - completeness score
    is_a: sequencing field
    slot_uri: MIXS:0000069
    multivalued: false
    alias: compl_score
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
    pattern: ^(high|med|low);(0|([0-9]{1,2})|100)%$
  compl_software:
    name: compl_software
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s) used
    description: Tools used for completion estimate, i.e. checkm, anvi'o, busco
    title: completeness software
    examples:
    - value: checkm
    from_schema: http://w3id.org/mixs
    aliases:
    - completeness software
    is_a: sequencing field
    string_serialization: '{software};{version}'
    slot_uri: MIXS:0000070
    multivalued: false
    alias: compl_software
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  compl_appr:
    name: compl_appr
    annotations:
      expected_value:
        tag: expected_value
        value: text
    description: The approach used to determine the completeness of a given genomic
      assembly, which would typically make use of a set of conserved marker genes
      or a closely related reference genome. For UViG completeness, include reference
      genome or group used, and contig feature suggesting a complete genome
    title: completeness approach
    examples:
    - value: 'other: UViG length compared to the average length of reference genomes
        from the P22virus genus (NCBI RefSeq v83)'
    from_schema: http://w3id.org/mixs
    aliases:
    - completeness approach
    is_a: sequencing field
    slot_uri: MIXS:0000071
    multivalued: false
    alias: compl_appr
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    - MIUVIG
    range: compl_appr_enum
  contam_score:
    name: contam_score
    annotations:
      expected_value:
        tag: expected_value
        value: value
    description: 'The contamination score is based on the fraction of single-copy
      genes that are observed more than once in a query genome. The following scores
      are acceptable for; High Quality Draft: &lt; 5%, Medium Quality Draft: &lt; 10%, Low
      Quality Draft: &lt; 10%. Contamination must be below 5% for a SAG or MAG to be
      deposited into any of the public databases'
    title: contamination score
    examples:
    - value: 1%
    from_schema: http://w3id.org/mixs
    aliases:
    - contamination score
    is_a: sequencing field
    string_serialization: '{float} percentage'
    slot_uri: MIXS:0000072
    multivalued: false
    alias: contam_score
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
  contam_screen_input:
    name: contam_screen_input
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: The type of sequence data used as input
    title: contamination screening input
    examples:
    - value: contigs
    from_schema: http://w3id.org/mixs
    aliases:
    - contamination screening input
    is_a: sequencing field
    string_serialization: '[reads| contigs]'
    slot_uri: MIXS:0000005
    multivalued: false
    alias: contam_screen_input
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
  contam_screen_param:
    name: contam_screen_param
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration;value or name
    description: Specific parameters used in the decontamination sofware, such as
      reference database, coverage, and kmers. Combinations of these parameters may
      also be used, i.e. kmer and coverage, or reference database and kmer
    title: contamination screening parameters
    examples:
    - value: kmer
    from_schema: http://w3id.org/mixs
    aliases:
    - contamination screening parameters
    is_a: sequencing field
    slot_uri: MIXS:0000073
    multivalued: false
    alias: contam_screen_param
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: string
    pattern: ^(ref db|kmer|coverage|combination);.+
  decontam_software:
    name: decontam_software
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Tool(s) used in contamination screening
    title: decontamination software
    examples:
    - value: anvi'o
    from_schema: http://w3id.org/mixs
    aliases:
    - decontamination software
    is_a: sequencing field
    slot_uri: MIXS:0000074
    multivalued: false
    alias: decontam_software
    owner: core
    domain_of:
    - core
    - MISAG
    - MIMAG
    range: decontam_software_enum
  sort_tech:
    name: sort_tech
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Method used to sort/isolate cells or particles of interest
    title: sorting technology
    examples:
    - value: optical manipulation
    from_schema: http://w3id.org/mixs
    aliases:
    - sorting technology
    is_a: sequencing field
    slot_uri: MIXS:0000075
    multivalued: false
    alias: sort_tech
    owner: core
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: sort_tech_enum
  single_cell_lysis_appr:
    name: single_cell_lysis_appr
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Method used to free DNA from interior of the cell(s) or particle(s)
    title: single cell or viral particle lysis approach
    examples:
    - value: enzymatic
    from_schema: http://w3id.org/mixs
    aliases:
    - single cell or viral particle lysis approach
    is_a: sequencing field
    slot_uri: MIXS:0000076
    multivalued: false
    alias: single_cell_lysis_appr
    owner: core
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: single_cell_lysis_appr_enum
  single_cell_lysis_prot:
    name: single_cell_lysis_prot
    annotations:
      expected_value:
        tag: expected_value
        value: kit, protocol name
    description: Name of the kit or standard protocol used for cell(s) or particle(s)
      lysis
    title: single cell or viral particle lysis kit protocol
    examples:
    - value: ambion single cell lysis kit
    from_schema: http://w3id.org/mixs
    aliases:
    - single cell or viral particle lysis kit protocol
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000054
    multivalued: false
    alias: single_cell_lysis_prot
    owner: core
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: string
  wga_amp_appr:
    name: wga_amp_appr
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Method used to amplify genomic DNA in preparation for sequencing
    title: WGA amplification approach
    examples:
    - value: mda based
    from_schema: http://w3id.org/mixs
    aliases:
    - WGA amplification approach
    is_a: sequencing field
    string_serialization: '[pcr based|mda based]'
    slot_uri: MIXS:0000055
    multivalued: false
    alias: wga_amp_appr
    owner: core
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: string
  wga_amp_kit:
    name: wga_amp_kit
    annotations:
      expected_value:
        tag: expected_value
        value: kit name
    description: Kit used to amplify genomic DNA in preparation for sequencing
    title: WGA amplification kit
    examples:
    - value: qiagen repli-g
    from_schema: http://w3id.org/mixs
    aliases:
    - WGA amplification kit
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000006
    multivalued: false
    alias: wga_amp_kit
    owner: core
    domain_of:
    - core
    - MISAG
    - MIUVIG
    range: string
  bin_param:
    name: bin_param
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: The parameters that have been applied during the extraction of genomes
      from metagenomic datasets
    title: binning parameters
    examples:
    - value: coverage and kmer
    from_schema: http://w3id.org/mixs
    aliases:
    - binning parameters
    is_a: sequencing field
    slot_uri: MIXS:0000077
    multivalued: false
    alias: bin_param
    owner: core
    domain_of:
    - core
    - MIMAG
    - MIUVIG
    range: bin_param_enum
  bin_software:
    name: bin_software
    annotations:
      expected_value:
        tag: expected_value
        value: names and versions of software(s) used
    description: Tool(s) used for the extraction of genomes from metagenomic datasets,
      where possible include a product ID (PID) of the tool(s) used.
    title: binning software
    examples:
    - value: MetaCluster-TA (RRID:SCR_004599), MaxBin (biotools:maxbin)
    from_schema: http://w3id.org/mixs
    aliases:
    - binning software
    is_a: sequencing field
    string_serialization: '{software};{version}{PID}'
    slot_uri: MIXS:0000078
    multivalued: false
    alias: bin_software
    owner: core
    domain_of:
    - core
    - MIMAG
    - MIUVIG
    range: string
  reassembly_bin:
    name: reassembly_bin
    annotations:
      expected_value:
        tag: expected_value
        value: boolean
    description: Has an assembly been performed on a genome bin extracted from a metagenomic
      assembly?
    title: reassembly post binning
    examples:
    - value: 'no'
    from_schema: http://w3id.org/mixs
    aliases:
    - reassembly post binning
    is_a: sequencing field
    string_serialization: '{boolean}'
    slot_uri: MIXS:0000079
    multivalued: false
    alias: reassembly_bin
    owner: core
    domain_of:
    - core
    - MIMAG
    - MIUVIG
    range: string
  mag_cov_software:
    name: mag_cov_software
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Tool(s) used to determine the genome coverage if coverage is used
      as a binning parameter in the extraction of genomes from metagenomic datasets
    title: MAG coverage software
    examples:
    - value: bbmap
    from_schema: http://w3id.org/mixs
    aliases:
    - MAG coverage software
    is_a: sequencing field
    slot_uri: MIXS:0000080
    multivalued: false
    alias: mag_cov_software
    owner: core
    domain_of:
    - core
    - MIMAG
    - MIUVIG
    range: mag_cov_software_enum
  vir_ident_software:
    name: vir_ident_software
    annotations:
      expected_value:
        tag: expected_value
        value: software name, version and relevant parameters
    description: Tool(s) used for the identification of UViG as a viral genome, software
      or protocol name including version number, parameters, and cutoffs used
    title: viral identification software
    examples:
    - value: VirSorter; 1.0.4; Virome database, category 2
    from_schema: http://w3id.org/mixs
    aliases:
    - viral identification software
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000081
    multivalued: false
    alias: vir_ident_software
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: string
  pred_genome_type:
    name: pred_genome_type
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Type of genome predicted for the UViG
    title: predicted genome type
    examples:
    - value: dsDNA
    from_schema: http://w3id.org/mixs
    aliases:
    - predicted genome type
    is_a: sequencing field
    slot_uri: MIXS:0000082
    multivalued: false
    alias: pred_genome_type
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: pred_genome_type_enum
  pred_genome_struc:
    name: pred_genome_struc
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Expected structure of the viral genome
    title: predicted genome structure
    examples:
    - value: non-segmented
    from_schema: http://w3id.org/mixs
    aliases:
    - predicted genome structure
    is_a: sequencing field
    slot_uri: MIXS:0000083
    multivalued: false
    alias: pred_genome_struc
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: pred_genome_struc_enum
  detec_type:
    name: detec_type
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Type of UViG detection
    title: detection type
    examples:
    - value: independent sequence (UViG)
    from_schema: http://w3id.org/mixs
    aliases:
    - detection type
    is_a: sequencing field
    string_serialization: '[independent sequence (UViG)|provirus (UpViG)]'
    slot_uri: MIXS:0000084
    multivalued: false
    alias: detec_type
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: string
  otu_class_appr:
    name: otu_class_appr
    annotations:
      expected_value:
        tag: expected_value
        value: cutoffs and method used
    description: Cutoffs and approach used when clustering “species-level” OTUs. Note
      that results from standard 95% ANI / 85% AF clustering should be provided alongside
      OTUS defined from another set of thresholds, even if the latter are the ones
      primarily used during the analysis
    title: OTU classification approach
    examples:
    - value: 95% ANI;85% AF; greedy incremental clustering
    from_schema: http://w3id.org/mixs
    aliases:
    - OTU classification approach
    is_a: sequencing field
    string_serialization: '{ANI cutoff};{AF cutoff};{clustering method}'
    slot_uri: MIXS:0000085
    multivalued: false
    alias: otu_class_appr
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: string
  otu_seq_comp_appr:
    name: otu_seq_comp_appr
    annotations:
      expected_value:
        tag: expected_value
        value: software name, version and relevant parameters
    description: Tool and thresholds used to compare sequences when computing &quot;species-level&quot;
      OTUs
    title: OTU sequence comparison approach
    examples:
    - value: 'blastn;2.6.0+;e-value cutoff: 0.001'
    from_schema: http://w3id.org/mixs
    aliases:
    - OTU sequence comparison approach
    is_a: sequencing field
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000086
    multivalued: false
    alias: otu_seq_comp_appr
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: string
  otu_db:
    name: otu_db
    annotations:
      expected_value:
        tag: expected_value
        value: database and version
    description: Reference database (i.e. sequences not generated as part of the current
      study) used to cluster new genomes in &quot;species-level&quot; OTUs, if any
    title: OTU database
    examples:
    - value: NCBI Viral RefSeq;83
    from_schema: http://w3id.org/mixs
    aliases:
    - OTU database
    is_a: sequencing field
    string_serialization: '{database};{version}'
    slot_uri: MIXS:0000087
    multivalued: false
    alias: otu_db
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: string
  host_pred_appr:
    name: host_pred_appr
    annotations:
      expected_value:
        tag: expected_value
        value: enumeration
    description: Tool or approach used for host prediction
    title: host prediction approach
    examples:
    - value: CRISPR spacer match
    from_schema: http://w3id.org/mixs
    aliases:
    - host prediction approach
    is_a: sequencing field
    slot_uri: MIXS:0000088
    multivalued: false
    alias: host_pred_appr
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: host_pred_appr_enum
  host_pred_est_acc:
    name: host_pred_est_acc
    annotations:
      expected_value:
        tag: expected_value
        value: false discovery rate
    description: For each tool or approach used for host prediction, estimated false
      discovery rates should be included, either computed de novo or from the literature
    title: host prediction estimated accuracy
    examples:
    - value: 'CRISPR spacer match: 0 or 1 mismatches, estimated 8% FDR at the host
        genus rank (Edwards et al. 2016 doi:10.1093/femsre/fuv048)'
    from_schema: http://w3id.org/mixs
    aliases:
    - host prediction estimated accuracy
    is_a: sequencing field
    string_serialization: '{text}'
    slot_uri: MIXS:0000089
    multivalued: false
    alias: host_pred_est_acc
    owner: core
    domain_of:
    - core
    - MIUVIG
    range: string
  associated resource:
    name: associated resource
    annotations:
      expected_value:
        tag: expected_value
        value: reference to resource
    description: A related resource that is referenced, cited, or otherwise associated
      to the sequence.
    title: relevant electronic resources
    examples:
    - value: http://www.earthmicrobiome.org/
    from_schema: http://w3id.org/mixs
    aliases:
    - relevant electronic resources
    is_a: sequencing field
    string_serialization: '{PMID} | {DOI} | {URL}'
    slot_uri: MIXS:0000091
    multivalued: false
    alias: associated_resource
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string
  sop:
    name: sop
    annotations:
      expected_value:
        tag: expected_value
        value: reference to SOP
    description: Standard operating procedures used in assembly and/or annotation
      of genomes, metagenomes or environmental sequences
    title: relevant standard operating procedures
    examples:
    - value: http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/
    from_schema: http://w3id.org/mixs
    aliases:
    - relevant standard operating procedures
    is_a: sequencing field
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000090
    multivalued: false
    alias: sop
    owner: core
    domain_of:
    - core
    - MIGS eukaryote
    - MIGS bacteria
    - MIGS plant
    - MIGS virus
    - MIGS org
    - MIMS
    - MIMARKS specimen
    - MIMARKS survey
    - MISAG
    - MIMAG
    - MIUVIG
    range: string

</code></pre>

</details>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
      
    
    
  </body>
</html>