
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://GenomicsStandardsConsortium.github.io/mixs/MimarksSSymbiontAssociated/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.6.0, mkdocs-material-9.5.27" name="generator"/>
<title>Class: MimarksS combined with SymbiontAssociated (MimarksSSymbiontAssociated) - mixs</title>
<link href="../assets/stylesheets/main.6543a935.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-mimarkss-combined-with-symbiontassociated-mimarksssymbiontassociated">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header md-header--shadow" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="mixs" class="md-header__button md-logo" data-md-component="logo" href=".." title="mixs">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            mixs
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: MimarksS combined with SymbiontAssociated (MimarksSSymbiontAssociated)
            
          </span>
</div>
</div>
</div>
<script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
<div class="md-search__suggest" data-md-component="search-suggest"></div>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="" tabindex="0">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list" role="presentation"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/GenomicsStandardsConsortium/mixs" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="mixs" class="md-nav__button md-logo" data-md-component="logo" href=".." title="mixs">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    mixs
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/GenomicsStandardsConsortium/mixs" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
<span class="md-ellipsis">
    Index
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../term_list/">
<span class="md-ellipsis">
    Full term table
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../combinations/">
<span class="md-ellipsis">
    Combinations
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../enumerations/">
<span class="md-ellipsis">
    Enumerations
  </span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#inheritance">
<span class="md-ellipsis">
      Inheritance
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
<span class="md-ellipsis">
      Slots
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#usages">
<span class="md-ellipsis">
      Usages
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
<span class="md-ellipsis">
      Identifier and Mapping Information
    </span>
</a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
<span class="md-ellipsis">
      Schema Source
    </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
<span class="md-ellipsis">
      Mappings
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
<span class="md-ellipsis">
      LinkML Source
    </span>
</a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
<span class="md-ellipsis">
      Direct
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
<span class="md-ellipsis">
      Induced
    </span>
</a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<h1 id="class-mimarkss-combined-with-symbiontassociated-mimarksssymbiontassociated">Class: MimarksS combined with SymbiontAssociated (MimarksSSymbiontAssociated)</h1>
<p><em>MIxS Data that comply with the MimarksS checklist and the SymbiontAssociated Extension</em></p>
<p>URI: <a href="https://w3id.org/mixs/0010008_0016023">MIXS:0010008_0016023</a></p>
<div class="mermaid"> classDiagram
    class MimarksSSymbiontAssociated
    click MimarksSSymbiontAssociated href "../MimarksSSymbiontAssociated"
      MimarksS &lt;|-- MimarksSSymbiontAssociated
        click MimarksS href "../MimarksS"
      SymbiontAssociated &lt;|-- MimarksSSymbiontAssociated
        click SymbiontAssociated href "../SymbiontAssociated"

      MimarksSSymbiontAssociated : adapters

      MimarksSSymbiontAssociated : alt

      MimarksSSymbiontAssociated : assembly_software

      MimarksSSymbiontAssociated : associated_resource

      MimarksSSymbiontAssociated : association_duration

      MimarksSSymbiontAssociated : chem_administration

      MimarksSSymbiontAssociated : chimera_check

      MimarksSSymbiontAssociated : collection_date

      MimarksSSymbiontAssociated : depth

      MimarksSSymbiontAssociated : elev

      MimarksSSymbiontAssociated : env_broad_scale

      MimarksSSymbiontAssociated : env_local_scale

      MimarksSSymbiontAssociated : env_medium

      MimarksSSymbiontAssociated : experimental_factor

      MimarksSSymbiontAssociated : geo_loc_name

      MimarksSSymbiontAssociated : gravidity

      MimarksSSymbiontAssociated : host_age

      MimarksSSymbiontAssociated : host_body_habitat

      MimarksSSymbiontAssociated : host_body_product

      MimarksSSymbiontAssociated : host_body_site

      MimarksSSymbiontAssociated : host_cellular_loc




    MimarksSSymbiontAssociated --&gt; "0..1 _recommended_" HostCellularLocEnum : host_cellular_loc
    click HostCellularLocEnum href "../HostCellularLocEnum"


      MimarksSSymbiontAssociated : host_color

      MimarksSSymbiontAssociated : host_common_name

      MimarksSSymbiontAssociated : host_dependence




    MimarksSSymbiontAssociated --&gt; "1" HostDependenceEnum : host_dependence
    click HostDependenceEnum href "../HostDependenceEnum"


      MimarksSSymbiontAssociated : host_dry_mass

      MimarksSSymbiontAssociated : host_fam_rel

      MimarksSSymbiontAssociated : host_genotype

      MimarksSSymbiontAssociated : host_growth_cond

      MimarksSSymbiontAssociated : host_height

      MimarksSSymbiontAssociated : host_infra_spec_name

      MimarksSSymbiontAssociated : host_infra_spec_rank

      MimarksSSymbiontAssociated : host_length

      MimarksSSymbiontAssociated : host_life_stage

      MimarksSSymbiontAssociated : host_number

      MimarksSSymbiontAssociated : host_of_host_coinf

      MimarksSSymbiontAssociated : host_of_host_disease

      MimarksSSymbiontAssociated : host_of_host_env_loc

      MimarksSSymbiontAssociated : host_of_host_env_med

      MimarksSSymbiontAssociated : host_of_host_fam_rel

      MimarksSSymbiontAssociated : host_of_host_geno

      MimarksSSymbiontAssociated : host_of_host_gravid

      MimarksSSymbiontAssociated : host_of_host_infname

      MimarksSSymbiontAssociated : host_of_host_infrank

      MimarksSSymbiontAssociated : host_of_host_name

      MimarksSSymbiontAssociated : host_of_host_pheno

      MimarksSSymbiontAssociated : host_of_host_sub_id

      MimarksSSymbiontAssociated : host_of_host_taxid

      MimarksSSymbiontAssociated : host_of_host_totmass

      MimarksSSymbiontAssociated : host_phenotype

      MimarksSSymbiontAssociated : host_shape

      MimarksSSymbiontAssociated : host_specificity




    MimarksSSymbiontAssociated --&gt; "0..1 _recommended_" HostSpecificityEnum : host_specificity
    click HostSpecificityEnum href "../HostSpecificityEnum"


      MimarksSSymbiontAssociated : host_subject_id

      MimarksSSymbiontAssociated : host_substrate

      MimarksSSymbiontAssociated : host_symbiont

      MimarksSSymbiontAssociated : host_taxid

      MimarksSSymbiontAssociated : host_tot_mass

      MimarksSSymbiontAssociated : lat_lon

      MimarksSSymbiontAssociated : lib_layout




    MimarksSSymbiontAssociated --&gt; "0..1 _recommended_" LibLayoutEnum : lib_layout
    click LibLayoutEnum href "../LibLayoutEnum"


      MimarksSSymbiontAssociated : lib_reads_seqd

      MimarksSSymbiontAssociated : lib_screen

      MimarksSSymbiontAssociated : lib_size

      MimarksSSymbiontAssociated : lib_vector

      MimarksSSymbiontAssociated : mid

      MimarksSSymbiontAssociated : misc_param

      MimarksSSymbiontAssociated : mode_transmission




    MimarksSSymbiontAssociated --&gt; "0..1 _recommended_" ModeTransmissionEnum : mode_transmission
    click ModeTransmissionEnum href "../ModeTransmissionEnum"


      MimarksSSymbiontAssociated : neg_cont_type




    MimarksSSymbiontAssociated --&gt; "0..1 _recommended_" NegContTypeEnum : neg_cont_type
    click NegContTypeEnum href "../NegContTypeEnum"


      MimarksSSymbiontAssociated : nucl_acid_amp

      MimarksSSymbiontAssociated : nucl_acid_ext

      MimarksSSymbiontAssociated : organism_count

      MimarksSSymbiontAssociated : oxy_stat_samp




    MimarksSSymbiontAssociated --&gt; "0..1" OxyStatSampEnum : oxy_stat_samp
    click OxyStatSampEnum href "../OxyStatSampEnum"


      MimarksSSymbiontAssociated : pcr_cond

      MimarksSSymbiontAssociated : pcr_primers

      MimarksSSymbiontAssociated : perturbation

      MimarksSSymbiontAssociated : pos_cont_type

      MimarksSSymbiontAssociated : project_name

      MimarksSSymbiontAssociated : rel_to_oxygen




    MimarksSSymbiontAssociated --&gt; "0..1" RelToOxygenEnum : rel_to_oxygen
    click RelToOxygenEnum href "../RelToOxygenEnum"


      MimarksSSymbiontAssociated : route_transmission




    MimarksSSymbiontAssociated --&gt; "0..1" RouteTransmissionEnum : route_transmission
    click RouteTransmissionEnum href "../RouteTransmissionEnum"


      MimarksSSymbiontAssociated : salinity

      MimarksSSymbiontAssociated : samp_collect_device

      MimarksSSymbiontAssociated : samp_collect_method

      MimarksSSymbiontAssociated : samp_mat_process

      MimarksSSymbiontAssociated : samp_name

      MimarksSSymbiontAssociated : samp_size

      MimarksSSymbiontAssociated : samp_store_dur

      MimarksSSymbiontAssociated : samp_store_loc

      MimarksSSymbiontAssociated : samp_store_sol

      MimarksSSymbiontAssociated : samp_store_temp

      MimarksSSymbiontAssociated : samp_taxon_id

      MimarksSSymbiontAssociated : samp_vol_we_dna_ext

      MimarksSSymbiontAssociated : seq_meth

      MimarksSSymbiontAssociated : seq_quality_check




    MimarksSSymbiontAssociated --&gt; "0..1 _recommended_" SeqQualityCheckEnum : seq_quality_check
    click SeqQualityCheckEnum href "../SeqQualityCheckEnum"


      MimarksSSymbiontAssociated : size_frac

      MimarksSSymbiontAssociated : sop

      MimarksSSymbiontAssociated : source_mat_id

      MimarksSSymbiontAssociated : sym_life_cycle_type




    MimarksSSymbiontAssociated --&gt; "1" SymLifeCycleTypeEnum : sym_life_cycle_type
    click SymLifeCycleTypeEnum href "../SymLifeCycleTypeEnum"


      MimarksSSymbiontAssociated : symbiont_host_role




    MimarksSSymbiontAssociated --&gt; "0..1 _recommended_" SymbiontHostRoleEnum : symbiont_host_role
    click SymbiontHostRoleEnum href "../SymbiontHostRoleEnum"


      MimarksSSymbiontAssociated : target_gene

      MimarksSSymbiontAssociated : target_subfragment

      MimarksSSymbiontAssociated : temp

      MimarksSSymbiontAssociated : type_of_symbiosis




    MimarksSSymbiontAssociated --&gt; "0..1 _recommended_" TypeOfSymbiosisEnum : type_of_symbiosis
    click TypeOfSymbiosisEnum href "../TypeOfSymbiosisEnum"


      MimarksSSymbiontAssociated : urobiom_sex




    MimarksSSymbiontAssociated --&gt; "0..1" UrobiomSexEnum : urobiom_sex
    click UrobiomSexEnum href "../UrobiomSexEnum"



</div>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Extension/">Extension</a><ul>
<li><a href="../SymbiontAssociated/">SymbiontAssociated</a><ul>
<li><strong>MimarksSSymbiontAssociated</strong> [ <a href="../MimarksS/">MimarksS</a>]</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../samp_name/">samp_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>A local identifier or name that for the material sample used for extracting n...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../pcr_primers/">pcr_primers</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>PCR primers that were used to amplify the sequence of the targeted gene, locu...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../size_frac/">size_frac</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Filtering pore size used in sample preparation</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../lib_screen/">lib_screen</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Specific enrichment or screening methods applied before and/or after creating...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../nucl_acid_amp/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../lib_size/">lib_size</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones in the library prepared for the project</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../target_subfragment/">target_subfragment</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Name of subfragment of a gene or locus</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../mid/">mid</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to spe...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../temp/">temp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Temperature of the sample at the time of sampling</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../nucl_acid_ext/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../samp_size/">samp_size</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample coll...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../alt/">alt</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Heights of objects such as airplanes, space shuttles, rockets, atmospheric ba...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../source_mat_id/">source_mat_id</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A unique identifier assigned to a material sample (as defined by <a href="http://rs">http://rs</a></td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extractio...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../lib_reads_seqd/">lib_reads_seqd</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones sequenced from the library</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../rel_to_oxygen/">rel_to_oxygen</a></td>
<td>0..1 <br/> <a href="../RelToOxygenEnum/">RelToOxygenEnum</a></td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic ...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../samp_collect_device/">samp_collect_device</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The device used to collect an environmental sample</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../seq_quality_check/">seq_quality_check</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../SeqQualityCheckEnum/">SeqQualityCheckEnum</a></td>
<td>Indicate if the sequence has been called by automatic systems (none) or under...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../lib_layout/">lib_layout</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../LibLayoutEnum/">LibLayoutEnum</a></td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../project_name/">project_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Name of the project within which the sequencing was organized</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../lib_vector/">lib_vector</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../adapters/">adapters</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Adapters provide priming sequences for both amplification and sequencing of t...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../neg_cont_type/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../NegContTypeEnum/">NegContTypeEnum</a></td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../assembly_software/">assembly_software</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../chimera_check/">chimera_check</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for chimera checking, including version number and parameters, t...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../pos_cont_type/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The substance, mixture, product, or apparatus used to verify that a process w...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the entity or entities which are in the sample or specimen s local vic...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../samp_mat_process/">samp_mat_process</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A brief description of any processing applied to the sample during or after r...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../depth/">depth</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The vertical distance below local surface</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../samp_collect_method/">samp_collect_method</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The method employed for collecting the sample</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../env_medium/">env_medium</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../samp_taxon_id/">samp_taxon_id</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>NCBI taxon id of the sample</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by the country or sea name f...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../collection_date/">collection_date</a></td>
<td>1 <br/> <a href="../Datetime/">Datetime</a></td>
<td>The time of sampling, either as an instance (single point in time) or interva...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../seq_meth/">seq_meth</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Sequencing machine used</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by latitude and longitude</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../elev/">elev</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Elevation of the sampling site is its height above a fixed reference point, m...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a>, <a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../pcr_cond/">pcr_cond</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Description of reaction conditions and components of polymerase chain reactio...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../experimental_factor/">experimental_factor</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Variable aspects of an experiment design that can be used to describe an expe...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../target_gene/">target_gene</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Targeted gene or locus name for marker gene studies</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../associated_resource/">associated_resource</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A related resource that is referenced, cited, or otherwise associated to the ...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../sop/">sop</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, ...</td>
<td><a href="../MimarksS/">MimarksS</a></td>
</tr>
<tr>
<td><a href="../host_subject_id/">host_subject_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A unique identifier by which each subject can be referred to, de-identified</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_common_name/">host_common_name</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Common name of the host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_taxid/">host_taxid</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>NCBI taxon id of the host, e</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_dependence/">host_dependence</a></td>
<td>1 <br/> <a href="../HostDependenceEnum/">HostDependenceEnum</a></td>
<td>Type of host dependence for the symbiotic host organism to its host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../type_of_symbiosis/">type_of_symbiosis</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../TypeOfSymbiosisEnum/">TypeOfSymbiosisEnum</a></td>
<td>Type of biological interaction established between the symbiotic host organis...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../sym_life_cycle_type/">sym_life_cycle_type</a></td>
<td>1 <br/> <a href="../SymLifeCycleTypeEnum/">SymLifeCycleTypeEnum</a></td>
<td>Type of life cycle of the symbiotic host species (the thing being sampled)</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_life_stage/">host_life_stage</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Description of life stage of host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_age/">host_age</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Age of host at the time of sampling; relevant scale depends on species and st...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../urobiom_sex/">urobiom_sex</a></td>
<td>0..1 <br/> <a href="../UrobiomSexEnum/">UrobiomSexEnum</a></td>
<td>Physical sex of the host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../mode_transmission/">mode_transmission</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../ModeTransmissionEnum/">ModeTransmissionEnum</a></td>
<td>The process through which the symbiotic host organism entered the host from w...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../route_transmission/">route_transmission</a></td>
<td>0..1 <br/> <a href="../RouteTransmissionEnum/">RouteTransmissionEnum</a></td>
<td>Description of path taken by the symbiotic host organism being sampled in ord...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_body_habitat/">host_body_habitat</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Original body habitat where the sample was obtained from</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_body_site/">host_body_site</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name of body site where the sample was obtained from, such as a specific orga...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_body_product/">host_body_product</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Substance produced by the body, e</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_tot_mass/">host_tot_mass</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_height/">host_height</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The height of subject</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_length/">host_length</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The length of subject</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_growth_cond/">host_growth_cond</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Literature reference giving growth conditions of the host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_substrate/">host_substrate</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The growth substrate of the host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_fam_rel/">host_fam_rel</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Relationships to other hosts in the same study; can include multiple relation...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_infra_spec_name/">host_infra_spec_name</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Taxonomic information about the host below subspecies level</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_infra_spec_rank/">host_infra_spec_rank</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Taxonomic rank information about the host below subspecies level, such as var...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_genotype/">host_genotype</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Observed genotype</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_phenotype/">host_phenotype</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Phenotype of human or other host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_dry_mass/">host_dry_mass</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Measurement of dry mass</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_color/">host_color</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The color of host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_shape/">host_shape</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Morphological shape of host</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../gravidity/">gravidity</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Whether or not subject is gravid, and if yes date due or date post-conception...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_number/">host_number</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Number of symbiotic host individuals pooled at the time of collection</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_symbiont/">host_symbiont</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>The taxonomic name of the organism(s) found living in mutualistic, commensali...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_specificity/">host_specificity</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../HostSpecificityEnum/">HostSpecificityEnum</a></td>
<td>Level of specificity of symbiont-host interaction: e</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../symbiont_host_role/">symbiont_host_role</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../SymbiontHostRoleEnum/">SymbiontHostRoleEnum</a></td>
<td>Role of the host in the life cycle of the symbiotic organism</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_cellular_loc/">host_cellular_loc</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../HostCellularLocEnum/">HostCellularLocEnum</a></td>
<td>The localization of the symbiotic host organism within the host from which it...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../association_duration/">association_duration</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Time spent in host of the symbiotic organism at the time of sampling; relevan...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_coinf/">host_of_host_coinf</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The taxonomic name of any coinfecting organism observed in a symbiotic relati...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_name/">host_of_host_name</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Common name of the host of the symbiotic host organism</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_env_loc/">host_of_host_env_loc</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>For a symbiotic host organism the local anatomical environment within its hos...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_env_med/">host_of_host_env_med</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Report the environmental material(s) immediately surrounding the symbiotic ho...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_taxid/">host_of_host_taxid</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>NCBI taxon id of the host of the symbiotic host organism</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_sub_id/">host_of_host_sub_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>A unique identifier by which each host of the symbiotic host organism subject...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_disease/">host_of_host_disease</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>List of diseases with which the host of the symbiotic host organism has been ...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_fam_rel/">host_of_host_fam_rel</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Familial relationship of the host of the symbiotic host organisms to other ho...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_infname/">host_of_host_infname</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Taxonomic name information of the host of the symbiotic host organism below s...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_infrank/">host_of_host_infrank</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Taxonomic rank information about the host of the symbiotic host organism belo...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_geno/">host_of_host_geno</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Observed genotype of the host of the symbiotic host organism</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_pheno/">host_of_host_pheno</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Phenotype of the host of the symbiotic host organism</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_gravid/">host_of_host_gravid</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Whether or not the host of the symbiotic host organism is gravid, and if yes ...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../host_of_host_totmass/">host_of_host_totmass</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Total mass of the host of the symbiotic host organism at collection, the unit...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../chem_administration/">chem_administration</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>List of chemical compounds administered to the host or site where sampling oc...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../perturbation/">perturbation</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Type of perturbation, e</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../salinity/">salinity</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The total concentration of all dissolved salts in a liquid or solid sample</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../oxy_stat_samp/">oxy_stat_samp</a></td>
<td>0..1 <br/> <a href="../OxyStatSampEnum/">OxyStatSampEnum</a></td>
<td>Oxygenation status of sample</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../organism_count/">organism_count</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or ...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../samp_store_temp/">samp_store_temp</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Temperature at which sample was stored, e</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../samp_store_dur/">samp_store_dur</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Duration for which the sample was stored</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../samp_store_loc/">samp_store_loc</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../samp_store_sol/">samp_store_sol</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Solution within which sample was stored, if any</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
<tr>
<td><a href="../misc_param/">misc_param</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Any other measurement performed or parameter collected, that is not listed he...</td>
<td><a href="../SymbiontAssociated/">SymbiontAssociated</a></td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<table>
<thead>
<tr>
<th>used by</th>
<th>used in</th>
<th>type</th>
<th>used</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../MixsCompliantData/">MixsCompliantData</a></td>
<td><a href="../mimarks_s_symbiont_associated_data/">mimarks_s_symbiont_associated_data</a></td>
<td>range</td>
<td><a href="./">MimarksSSymbiontAssociated</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: <a href="https://w3id.org/mixs">https://w3id.org/mixs</a></li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>MIXS:0010008_0016023</td>
</tr>
<tr>
<td>native</td>
<td>MIXS:MimarksSSymbiontAssociated</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: MimarksSSymbiontAssociated
description: MIxS Data that comply with the MimarksS checklist and the SymbiontAssociated
  Extension
title: MimarksS combined with SymbiontAssociated
in_subset:
- combination_classes
from_schema: https://w3id.org/mixs
is_a: SymbiontAssociated
mixins:
- MimarksS
class_uri: MIXS:0010008_0016023

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: MimarksSSymbiontAssociated
description: MIxS Data that comply with the MimarksS checklist and the SymbiontAssociated
  Extension
title: MimarksS combined with SymbiontAssociated
in_subset:
- combination_classes
from_schema: https://w3id.org/mixs
is_a: SymbiontAssociated
mixins:
- MimarksS
attributes:
  samp_name:
    name: samp_name
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: ''
    description: A local identifier or name that for the material sample used for
      extracting nucleic acids, and subsequent sequencing. It can refer either to
      the original material collected or to any derived sub-samples. It can have any
      format, but we suggest that you make it concise, unique and consistent within
      your lab, and as informative as possible. INSDC requires every sample name from
      a single Submitter to be unique. Use of a globally unique identifier for the
      field source_mat_id is recommended in addition to sample_name
    title: sample name
    examples:
    - value: ISDsoil1
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - sample
    slot_uri: MIXS:0001107
    alias: samp_name
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
  pcr_primers:
    name: pcr_primers
    description: PCR primers that were used to amplify the sequence of the targeted
      gene, locus or subfragment. This field should contain all the primers used for
      a single PCR reaction if multiple forward or reverse primers are present in
      a single PCR reaction. The primer sequence should be reported in uppercase letters
    title: pcr primers
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: FWD:{dna};REV:{dna}
    slot_uri: MIXS:0000046
    alias: pcr_primers
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    recommended: true
  size_frac:
    name: size_frac
    annotations:
      Expected_value:
        tag: Expected_value
        value: filter size value range
    description: Filtering pore size used in sample preparation
    title: size fraction selected
    examples:
    - value: 0-0.22 micrometer
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - fraction
    - size
    string_serialization: '{float}-{float} {unit}'
    slot_uri: MIXS:0000017
    alias: size_frac
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
  lib_screen:
    name: lib_screen
    description: Specific enrichment or screening methods applied before and/or after
      creating libraries
    title: library screening strategy
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000043
    alias: lib_screen
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  nucl_acid_amp:
    name: nucl_acid_amp
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the enzymatic amplification (PCR,
      TMA, NASBA) of specific nucleic acids
    title: nucleic acid amplification
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000038
    alias: nucl_acid_amp
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  lib_size:
    name: lib_size
    description: Total number of clones in the library prepared for the project
    title: library size
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000039
    alias: lib_size
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: integer
    recommended: true
  target_subfragment:
    name: target_subfragment
    description: Name of subfragment of a gene or locus. Important to e.g. identify
      special regions on marker genes like V6 on 16S rRNA
    title: target subfragment
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000045
    alias: target_subfragment
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    recommended: true
  mid:
    name: mid
    description: Molecular barcodes, called Multiplex Identifiers (MIDs), that are
      used to specifically tag unique samples in a sequencing run. Sequence should
      be reported in uppercase letters
    title: multiplex identifiers
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000047
    alias: mid
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
    pattern: ^[ACGTRKSYMWBHDVN]+$
    structured_pattern:
      syntax: ^{ambiguous_nucleotides}$
      interpolated: true
      partial_match: true
  temp:
    name: temp
    description: Temperature of the sample at the time of sampling
    title: temperature
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000113
    alias: temp
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  nucl_acid_ext:
    name: nucl_acid_ext
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the material separation to recover
      the nucleic acid fraction from a sample
    title: nucleic acid extraction
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000037
    alias: nucl_acid_ext
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  samp_size:
    name: samp_size
    description: The total amount or size (volume (ml), mass (g) or area (m2) ) of
      sample collected
    title: amount or size of sample collected
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000001
    alias: samp_size
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  alt:
    name: alt
    description: Heights of objects such as airplanes, space shuttles, rockets, atmospheric
      balloons and heights of places such as atmospheric layers and clouds. It is
      used to measure the height of an object which is above the earth's surface.
      In this context, the altitude measurement is the vertical distance between the
      earth's surface above sea level and the sampled position in the air
    title: altitude
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000094
    alias: alt
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - HostAssociated
    - MiscellaneousNaturalOrArtificialEnvironment
    - SymbiontAssociated
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  source_mat_id:
    name: source_mat_id
    description: A unique identifier assigned to a material sample (as defined by
      http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular
      digital record of a material sample) used for extracting nucleic acids, and
      subsequent sequencing. The identifier can refer either to the original material
      collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher,
      /bio_material, or /culture_collection may or may not share the same value as
      the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id
      may both contain 'UAM:Herps:14' , referring to both the specimen voucher and
      sampled tissue with the same identifier. However, the /culture_collection qualifier
      may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id
      would refer to an identifier from some derived culture from which the nucleic
      acids were extracted (e.g. xatc123 or ark:/2154/R2)
    title: source material identifiers
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000026
    multivalued: true
    alias: source_mat_id
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - SymbiontAssociated
    range: string
    recommended: true
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: milliliter, gram, milligram, square centimeter
    description: 'Volume (ml) or mass (g) of total collected sample processed for
      DNA extraction. Note: total sample collected should be entered under the term
      Sample Size (MIXS:0000001)'
    title: sample volume or weight for DNA extraction
    examples:
    - value: 1500 milliliter
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - dna
    - sample
    - volume
    - weight
    slot_uri: MIXS:0000111
    alias: samp_vol_we_dna_ext
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  lib_reads_seqd:
    name: lib_reads_seqd
    description: Total number of clones sequenced from the library
    title: library reads sequenced
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000040
    alias: lib_reads_seqd
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: integer
    recommended: true
  rel_to_oxygen:
    name: rel_to_oxygen
    description: Is this organism an aerobe, anaerobe? Please note that aerobic and
      anaerobic are valid descriptors for microbial environments
    title: relationship to oxygen
    examples:
    - value: aerobe
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - oxygen
    - relationship
    slot_uri: MIXS:0000015
    alias: rel_to_oxygen
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    range: RelToOxygenEnum
  samp_collect_device:
    name: samp_collect_device
    description: The device used to collect an environmental sample. This field accepts
      terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO).
      This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)
    title: sample collection device
    examples:
    - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000002
    alias: samp_collect_device
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
  seq_quality_check:
    name: seq_quality_check
    description: Indicate if the sequence has been called by automatic systems (none)
      or undergone a manual editing procedure (e.g. by inspecting the raw data or
      chromatograms). Applied only for sequences that are not submitted to SRA,ENA
      or DRA
    title: sequence quality check
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000051
    alias: seq_quality_check
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: SeqQualityCheckEnum
    recommended: true
  lib_layout:
    name: lib_layout
    description: Specify whether to expect single, paired, or other configuration
      of reads
    title: library layout
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000041
    alias: lib_layout
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: LibLayoutEnum
    recommended: true
  env_broad_scale:
    name: env_broad_scale
    description: 'Report the major environmental system the sample or specimen came
      from. The system(s) identified should have a coarse spatial grain, to provide
      the general environmental context of where the sampling was done (e.g. in the
      desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428.
      EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: broad-scale environmental context
    examples:
    - value: rangeland biome [ENVO:01000247]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - context
    - environmental
    slot_uri: MIXS:0000012
    alias: env_broad_scale
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
    structured_pattern:
      syntax: ^{termLabel} \[{termID}\]$
      interpolated: true
      partial_match: true
  project_name:
    name: project_name
    description: Name of the project within which the sequencing was organized
    title: project name
    examples:
    - value: Forest soil metagenome
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - project
    slot_uri: MIXS:0000092
    alias: project_name
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
  lib_vector:
    name: lib_vector
    description: Cloning vector type(s) used in construction of libraries
    title: library vector
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000042
    alias: lib_vector
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  adapters:
    name: adapters
    description: Adapters provide priming sequences for both amplification and sequencing
      of the sample-library fragments. Both adapters should be reported; in uppercase
      letters
    title: adapters
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{dna};{dna}'
    slot_uri: MIXS:0000048
    alias: adapters
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  neg_cont_type:
    name: neg_cont_type
    annotations:
      Expected_value:
        tag: Expected_value
        value: enumeration or text
    description: The substance or equipment used as a negative control in an investigation
    title: negative control type
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - type
    slot_uri: MIXS:0001321
    alias: neg_cont_type
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: NegContTypeEnum
    recommended: true
  assembly_software:
    name: assembly_software
    description: Tool(s) used for assembly, including version number and parameters
    title: assembly software
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000058
    alias: assembly_software
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  chimera_check:
    name: chimera_check
    description: Tool(s) used for chimera checking, including version number and parameters,
      to discover and remove chimeric sequences. A chimeric sequence is comprised
      of two or more phylogenetically distinct parent sequences
    title: chimera check software
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000052
    alias: chimera_check
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    recommended: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  pos_cont_type:
    name: pos_cont_type
    description: The substance, mixture, product, or apparatus used to verify that
      a process which is part of an investigation delivers a true positive
    title: positive control type
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - type
    string_serialization: '{term} or {text}'
    slot_uri: MIXS:0001322
    alias: pos_cont_type
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  env_local_scale:
    name: env_local_scale
    annotations:
      Expected_value:
        tag: Expected_value
        value: Environmental entities having causal influences upon the entity at
          time of sampling
    description: 'Report the entity or entities which are in the sample or specimen
      s local vicinity and which you believe have significant causal influences on
      your sample or specimen. We recommend using EnvO terms which are of smaller
      spatial grain than your entry for env_broad_scale. Terms, such as anatomical
      sites, from other OBO Library ontologies which interoperate with EnvO (e.g.
      UBERON) are accepted in this field. EnvO documentation about how to use the
      field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: local environmental context
    examples:
    - value: hillside [ENVO:01000333]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - context
    - environmental
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000013
    alias: env_local_scale
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
  samp_mat_process:
    name: samp_mat_process
    description: A brief description of any processing applied to the sample during
      or after retrieving the sample from environment, or a link to the relevant protocol(s)
      performed
    title: sample material processing
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000016
    alias: samp_mat_process
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  depth:
    name: depth
    description: The vertical distance below local surface. For sediment or soil samples
      depth is measured from sediment or soil surface, respectively. Depth can be
      reported as an interval for subsurface samples
    title: depth
    examples:
    - value: 10 meter
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000018
    alias: depth
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  samp_collect_method:
    name: samp_collect_method
    description: The method employed for collecting the sample
    title: sample collection method
    examples:
    - value: swabbing
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0001225
    alias: samp_collect_method
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$|([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}|{text}$
      interpolated: true
      partial_match: true
  env_medium:
    name: env_medium
    description: 'Report the environmental material(s) immediately surrounding the
      sample or specimen at the time of sampling. We recommend using subclasses of
      ''environmental material'' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO
      documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
      . Terms from other OBO ontologies are permissible as long as they reference
      mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
      (e.g. a tree, a leaf, a table top)'
    title: environmental medium
    examples:
    - value: bluegrass field soil [ENVO:00005789]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - environmental
    slot_uri: MIXS:0000014
    alias: env_medium
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
    structured_pattern:
      syntax: ^{termLabel} \[{termID}\]$
      interpolated: true
      partial_match: true
  samp_taxon_id:
    name: samp_taxon_id
    description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa
      sample. Use 'synthetic metagenome  for mock community/positive controls, or
      'blank sample' for negative controls
    title: taxonomy ID of DNA sample
    examples:
    - value: Gut Metagenome [NCBITaxon:749906]
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - dna
    - identifier
    - sample
    - taxon
    slot_uri: MIXS:0001320
    alias: samp_taxon_id
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[NCBITaxon:\d+\]$
    structured_pattern:
      syntax: ^{text} \[{NCBItaxon_id}\]$
      interpolated: true
      partial_match: true
  geo_loc_name:
    name: geo_loc_name
    description: The geographical origin of the sample as defined by the country or
      sea name followed by specific region name. Country or sea names should be chosen
      from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology
      (http://purl.bioontology.org/ontology/GAZ)
    title: geographic location (country and/or sea,region)
    examples:
    - value: 'USA: Maryland, Bethesda'
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - geographic
    - location
    slot_uri: MIXS:0000010
    alias: geo_loc_name
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: string
    required: true
    pattern: '^([^\s-]{1,2}|[^\s-]+.+[^\s-]+): ([^\s-]{1,2}|[^\s-]+.+[^\s-]+), ([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$'
    structured_pattern:
      syntax: '^{text}: {text}, {text}$'
      interpolated: true
      partial_match: true
  collection_date:
    name: collection_date
    description: 'The time of sampling, either as an instance (single point in time)
      or interval. In case no exact time is available, the date/time can be right
      truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10;
      2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant'
    title: collection date
    examples:
    - value: '2013-03-25T12:42:31+01:00'
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - date
    slot_uri: MIXS:0000011
    alias: collection_date
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: datetime
    required: true
  seq_meth:
    name: seq_meth
    description: Sequencing machine used. Where possible the term should be taken
      from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)
    title: sequencing method
    examples:
    - value: 454 Genome Sequencer FLX [OBI:0000702]
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - method
    slot_uri: MIXS:0000050
    alias: seq_meth
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+)|(([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\])$
    structured_pattern:
      syntax: ^{text}|({termLabel} \[{termID}\])$
      interpolated: true
      partial_match: true
  lat_lon:
    name: lat_lon
    annotations:
      Expected_value:
        tag: Expected_value
        value: decimal degrees,  limit to 8 decimal points
    description: The geographical origin of the sample as defined by latitude and
      longitude. The values should be reported in decimal degrees and in WGS84 system
    title: geographic location (latitude and longitude)
    examples:
    - value: 50.586825 6.408977
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - geographic
    - location
    string_serialization: '{float} {float}'
    slot_uri: MIXS:0000009
    alias: lat_lon
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: string
    required: true
    pattern: ^(-?((?:[0-8]?[0-9](?:\.\d{0,8})?)|90)) -?[0-9]+(?:\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$
    structured_pattern:
      syntax: ^{lat} {lon}$
      interpolated: true
      partial_match: true
  elev:
    name: elev
    description: Elevation of the sampling site is its height above a fixed reference
      point, most commonly the mean sea level. Elevation is mainly used when referring
      to points on the earth's surface, while altitude is used for points above the
      surface, such as an aircraft in flight or a spacecraft in orbit
    title: elevation
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000093
    alias: elev
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - HostAssociated
    - HydrocarbonResourcesCores
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  pcr_cond:
    name: pcr_cond
    description: Description of reaction conditions and components of polymerase chain
      reaction performed during library preparation.
    title: pcr conditions
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: initial denaturation:degrees_minutes;annealing:degrees_minutes;elongation:degrees_minutes;final
      elongation:degrees_minutes;total cycles
    slot_uri: MIXS:0000049
    alias: pcr_cond
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    recommended: true
  experimental_factor:
    name: experimental_factor
    description: Variable aspects of an experiment design that can be used to describe
      an experiment, or set of experiments, in an increasingly detailed manner. This
      field accepts ontology terms from Experimental Factor Ontology (EFO) and/or
      Ontology for Biomedical Investigations (OBI)
    title: experimental factor
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000008
    multivalued: true
    alias: experimental_factor
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^\S+.*\S+ \[[a-zA-Z]{2,}:\d+\]$
  target_gene:
    name: target_gene
    description: Targeted gene or locus name for marker gene studies
    title: target gene
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000044
    alias: target_gene
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MimarksC
    - MimarksS
    - Agriculture
    range: string
    required: true
  associated_resource:
    name: associated_resource
    annotations:
      Expected_value:
        tag: Expected_value
        value: reference to resource
    description: A related resource that is referenced, cited, or otherwise associated
      to the sequence
    title: relevant electronic resources
    examples:
    - value: http://www.earthmicrobiome.org/
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - resource
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000091
    multivalued: true
    alias: associated_resource
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  sop:
    name: sop
    description: Standard operating procedures used in assembly and/or annotation
      of genomes, metagenomes or environmental sequences
    title: relevant standard operating procedures
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000090
    multivalued: true
    alias: sop
    owner: MimarksSSymbiontAssociated
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  host_subject_id:
    name: host_subject_id
    description: A unique identifier by which each subject can be referred to, de-identified
    title: host subject id
    examples:
    - value: P14
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000861
    alias: host_subject_id
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - SymbiontAssociated
    range: string
  host_common_name:
    name: host_common_name
    description: Common name of the host
    title: host common name
    examples:
    - value: trematode
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000248
    alias: host_common_name
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - HostAssociated
    - PlantAssociated
    - SymbiontAssociated
    range: string
  host_taxid:
    name: host_taxid
    description: NCBI taxon id of the host, e.g. 9606
    title: host taxid
    examples:
    - value: '395013'
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{integer}'
    slot_uri: MIXS:0000250
    alias: host_taxid
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - PlantAssociated
    - SymbiontAssociated
    range: string
  host_dependence:
    name: host_dependence
    description: Type of host dependence for the symbiotic host organism to its host
    title: host dependence
    examples:
    - value: obligate
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0001315
    alias: host_dependence
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: HostDependenceEnum
    required: true
  type_of_symbiosis:
    name: type_of_symbiosis
    description: Type of biological interaction established between the symbiotic
      host organism being sampled and its respective host
    title: type of symbiosis
    examples:
    - value: parasitic
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - symbiosis
    - type
    slot_uri: MIXS:0001307
    alias: type_of_symbiosis
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: TypeOfSymbiosisEnum
    recommended: true
  sym_life_cycle_type:
    name: sym_life_cycle_type
    annotations:
      Expected_value:
        tag: Expected_value
        value: type of life cycle of the symbiotic organism (host of the samples)
    description: Type of life cycle of the symbiotic host species (the thing being
      sampled). Simple life cycles occur within a single host, complex ones within
      multiple different hosts over the course of their normal life cycle
    title: symbiotic host organism life cycle type
    examples:
    - value: complex life cycle
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - life
    - organism
    - symbiosis
    - type
    slot_uri: MIXS:0001300
    alias: sym_life_cycle_type
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: SymLifeCycleTypeEnum
    required: true
  host_life_stage:
    name: host_life_stage
    description: Description of life stage of host
    title: host life stage
    examples:
    - value: redia
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000251
    alias: host_life_stage
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - HostAssociated
    - PlantAssociated
    - SymbiontAssociated
    range: string
    required: true
  host_age:
    name: host_age
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: year, day, hour
    description: Age of host at the time of sampling; relevant scale depends on species
      and study, e.g. Could be seconds for amoebae or centuries for trees
    title: host age
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - age
    - host
    - host.
    slot_uri: MIXS:0000255
    alias: host_age
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  urobiom_sex:
    name: urobiom_sex
    annotations:
      Expected_value:
        tag: Expected_value
        value: sex of the symbiotic organism (host of the samples); enumeration
    description: Physical sex of the host
    title: host sex
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000862
    alias: urobiom_sex
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: UrobiomSexEnum
  mode_transmission:
    name: mode_transmission
    description: The process through which the symbiotic host organism entered the
      host from which it was sampled
    title: mode of transmission
    examples:
    - value: horizontal:castrator
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0001312
    alias: mode_transmission
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: ModeTransmissionEnum
    recommended: true
  route_transmission:
    name: route_transmission
    description: Description of path taken by the symbiotic host organism being sampled
      in order to establish a symbiotic relationship with the host (with which it
      was observed at the time of sampling) via a mode of transmission (specified
      in mode_transmission)
    title: route of transmission
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - route
    slot_uri: MIXS:0001316
    alias: route_transmission
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: RouteTransmissionEnum
  host_body_habitat:
    name: host_body_habitat
    description: Original body habitat where the sample was obtained from
    title: host body habitat
    examples:
    - value: anterior end of a tapeworm
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000866
    alias: host_body_habitat
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - SymbiontAssociated
    range: string
  host_body_site:
    name: host_body_site
    description: Name of body site where the sample was obtained from, such as a specific
      organ or tissue (tongue, lung etc...). Use terms from the foundational model
      of anatomy ontology (fma) or the Uber-anatomy ontology (UBERON)
    title: host body site
    examples:
    - value: scolex [UBERON:0015119]
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000867
    alias: host_body_site
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - SymbiontAssociated
    range: string
  host_body_product:
    name: host_body_product
    annotations:
      Expected_value:
        tag: Expected_value
        value: FMA or UBERON
    description: Substance produced by the body, e.g. Stool, mucus, where the sample
      was obtained from. Use terms from the foundational model of anatomy ontology
      (fma) or Uber-anatomy ontology (UBERON)
    title: host body product
    examples:
    - value: mucus [FMA:66938]
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - body
    - host
    - host.
    - product
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000888
    alias: host_body_product
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - SymbiontAssociated
    range: string
  host_tot_mass:
    name: host_tot_mass
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: kilogram, gram
    description: Total mass of the host at collection, the unit depends on host
    title: host total mass
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - mass
    - total
    slot_uri: MIXS:0000263
    alias: host_tot_mass
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  host_height:
    name: host_height
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: centimeter, millimeter, meter
    description: The height of subject
    title: host height
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - height
    - host
    - host.
    slot_uri: MIXS:0000264
    alias: host_height
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  host_length:
    name: host_length
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: centimeter, millimeter, meter
    description: The length of subject
    title: host length
    examples:
    - value: 1 meter
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - length
    slot_uri: MIXS:0000256
    alias: host_length
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - PlantAssociated
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  host_growth_cond:
    name: host_growth_cond
    description: Literature reference giving growth conditions of the host
    title: host growth conditions
    examples:
    - value: https://academic.oup.com/icesjms/article/68/2/349/617247
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - condition
    - growth
    - host
    - host.
    slot_uri: MIXS:0000871
    alias: host_growth_cond
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - SymbiontAssociated
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$|([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}|{text}$
      interpolated: true
      partial_match: true
  host_substrate:
    name: host_substrate
    description: The growth substrate of the host
    title: host substrate
    examples:
    - value: rock
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000252
    alias: host_substrate
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - SymbiontAssociated
    range: string
  host_fam_rel:
    name: host_fam_rel
    description: Relationships to other hosts in the same study; can include multiple
      relationships
    title: host family relationship
    examples:
    - value: clone;P15
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text};{text}'
    slot_uri: MIXS:0000872
    multivalued: true
    alias: host_fam_rel
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - SymbiontAssociated
    range: string
  host_infra_spec_name:
    name: host_infra_spec_name
    description: Taxonomic information about the host below subspecies level
    title: host infra-specific name
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000253
    alias: host_infra_spec_name
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_infra_spec_rank:
    name: host_infra_spec_rank
    description: Taxonomic rank information about the host below subspecies level,
      such as variety, form, rank etc
    title: host infra-specific rank
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - rank
    slot_uri: MIXS:0000254
    alias: host_infra_spec_rank
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_genotype:
    name: host_genotype
    description: Observed genotype
    title: host genotype
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000365
    alias: host_genotype
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
  host_phenotype:
    name: host_phenotype
    description: Phenotype of human or other host. Use terms from the phenotypic quality
      ontology (pato) or the Human Phenotype Ontology (HP)
    title: host phenotype
    examples:
    - value: soldier
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000874
    alias: host_phenotype
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
  host_dry_mass:
    name: host_dry_mass
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: kilogram, gram
    description: Measurement of dry mass
    title: host dry mass
    examples:
    - value: 500 gram
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - dry
    - host
    - host.
    - mass
    slot_uri: MIXS:0000257
    alias: host_dry_mass
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - PlantAssociated
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  host_color:
    name: host_color
    description: The color of host
    title: host color
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000260
    alias: host_color
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - SymbiontAssociated
    range: string
  host_shape:
    name: host_shape
    description: Morphological shape of host
    title: host shape
    examples:
    - value: round
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0000261
    alias: host_shape
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - SymbiontAssociated
    range: string
  gravidity:
    name: gravidity
    description: Whether or not subject is gravid, and if yes date due or date post-conception,
      specifying which is used
    title: gravidity
    examples:
    - value: yes;due date:2018-05-11
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{boolean};{timestamp}'
    slot_uri: MIXS:0000875
    alias: gravidity
    owner: MimarksSSymbiontAssociated
    domain_of:
    - HostAssociated
    - SymbiontAssociated
    range: string
  host_number:
    name: host_number
    annotations:
      Expected_value:
        tag: Expected_value
        value: count
    description: Number of symbiotic host individuals pooled at the time of collection
    title: host number individual
    examples:
    - value: '3'
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - number
    string_serialization: '{float} m'
    slot_uri: MIXS:0001305
    alias: host_number
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_symbiont:
    name: host_symbiont
    description: The taxonomic name of the organism(s) found living in mutualistic,
      commensalistic, or parasitic symbiosis with the specific host. The sampled symbiont
      can have its own symbionts. For example, parasites may have hyperparasites (=parasites
      of the parasite)
    title: observed host symbionts
    examples:
    - value: Paragordius varius
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0001298
    multivalued: true
    alias: host_symbiont
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    - SymbiontAssociated
    range: string
  host_specificity:
    name: host_specificity
    description: 'Level of specificity of symbiont-host interaction: e.g. generalist
      (symbiont able to establish associations with distantly related hosts) or species-specific'
    title: host specificity
    examples:
    - value: species-specific
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    slot_uri: MIXS:0001308
    alias: host_specificity
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: HostSpecificityEnum
    recommended: true
  symbiont_host_role:
    name: symbiont_host_role
    description: Role of the host in the life cycle of the symbiotic organism
    title: host of the symbiont role
    examples:
    - value: intermediate
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - symbiosis
    slot_uri: MIXS:0001303
    alias: symbiont_host_role
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: SymbiontHostRoleEnum
    recommended: true
  host_cellular_loc:
    name: host_cellular_loc
    description: 'The localization of the symbiotic host organism within the host
      from which it was sampled: e.g. intracellular if the symbiotic host organism
      is localized within the cells or extracellular if the symbiotic host organism
      is localized outside of cells'
    title: host cellular location
    examples:
    - value: extracellular
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - location
    slot_uri: MIXS:0001313
    alias: host_cellular_loc
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: HostCellularLocEnum
    recommended: true
  association_duration:
    name: association_duration
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: year, day, hour
    description: Time spent in host of the symbiotic organism at the time of sampling;
      relevant scale depends on symbiotic organism and study
    title: duration of association with the host
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - duration
    - host
    - host.
    - period
    slot_uri: MIXS:0001299
    alias: association_duration
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  host_of_host_coinf:
    name: host_of_host_coinf
    annotations:
      Expected_value:
        tag: Expected_value
        value: species name of coinfecting organism(s)
    description: The taxonomic name of any coinfecting organism observed in a symbiotic
      relationship with the host of the sampled host organism. e.g. where a sample
      collected from a host trematode species (A) which was collected from a host_of_host
      fish (B) that was also infected with a nematode (C), the value here would be
      (C) the nematode {species name} or {common name}. Multiple co-infecting species
      may be added in a comma-separated list. For listing symbiotic organisms associated
      with the host (A) use the term Observed host symbiont
    title: observed coinfecting organisms in host of host
    examples:
    - value: Maritrema novaezealandense
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - observed
    - organism
    string_serialization: '{text}'
    slot_uri: MIXS:0001310
    alias: host_of_host_coinf
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_name:
    name: host_of_host_name
    description: Common name of the host of the symbiotic host organism
    title: host of the symbiotic host common name
    examples:
    - value: snail
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - symbiosis
    slot_uri: MIXS:0001324
    alias: host_of_host_name
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_env_loc:
    name: host_of_host_env_loc
    annotations:
      Expected_value:
        tag: Expected_value
        value: UBERON term(s), multiple values can be separated by pipes
    description: For a symbiotic host organism the local anatomical environment within
      its host may have causal influences. Report the anatomical entity(s) which are
      in the direct environment of the symbiotic host organism being sampled and which
      you believe have significant causal influences on your sample or specimen. For
      example, if the symbiotic host organism being sampled is an intestinal worm,
      its local environmental context will be the term for intestine from UBERON (http://uberon.github.io/)
    title: host of the symbiotic host local environmental context
    examples:
    - value: small intestine[uberon:0002108]
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - context
    - environmental
    - host
    - host.
    - symbiosis
    string_serialization: small intestine [UBERON:0002108]
    slot_uri: MIXS:0001325
    multivalued: true
    alias: host_of_host_env_loc
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_env_med:
    name: host_of_host_env_med
    annotations:
      Expected_value:
        tag: Expected_value
        value: An ontology term for a material such as a tissue type or excreted substance
    description: 'Report the environmental material(s) immediately surrounding the
      symbiotic host organism at the time of sampling. This usually will be a tissue
      or substance type from the host, but may be another material if the symbiont
      is external to the host. We recommend using classes from the UBERON ontology,
      but subclasses of ''environmental material'' (http://purl.obolibrary.org/obo/ENVO_00010483)
      may also be used. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
      . Terms from other OBO ontologies are permissible as long as they reference
      mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
      (e.g. intestines, heart).MIxS . Terms from other OBO ontologies are permissible
      as long as they reference mass/volume nouns (e.g. air, water, blood) and not
      discrete, countable entities (e.g. intestines, heart)'
    title: host of the symbiotic host environemental medium
    examples:
    - value: feces[uberon:0001988]
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - environmental
    - host
    - host.
    - symbiosis
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0001326
    alias: host_of_host_env_med
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_taxid:
    name: host_of_host_taxid
    annotations:
      Expected_value:
        tag: Expected_value
        value: NCBI taxon identifier of the host of the symbiotic taxon organism
    description: NCBI taxon id of the host of the symbiotic host organism
    title: host of the symbiotic host taxon id
    examples:
    - value: '145637'
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - identifier
    - symbiosis
    - taxon
    string_serialization: '{integer}'
    slot_uri: MIXS:0001306
    alias: host_of_host_taxid
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_sub_id:
    name: host_of_host_sub_id
    description: 'A unique identifier by which each host of the symbiotic host organism
      subject can be referred to, de-identified, e.g. #H14'
    title: host of the symbiotic host subject id
    examples:
    - value: H3
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - identifier
    - symbiosis
    slot_uri: MIXS:0001327
    alias: host_of_host_sub_id
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_disease:
    name: host_of_host_disease
    annotations:
      Expected_value:
        tag: Expected_value
        value: disease name or Disease Ontology term
    description: List of diseases with which the host of the symbiotic host organism
      has been diagnosed; can include multiple diagnoses. The value of the field depends
      on host; for humans the terms should be chosen from the DO (Human Disease Ontology)
      at https://www.disease-ontology.org, non-human host diseases are free text
    title: host of the symbiotic host disease status
    examples:
    - value: rabies [DOID:11260]
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - disease
    - host
    - host.
    - status
    - symbiosis
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0001319
    multivalued: true
    alias: host_of_host_disease
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_fam_rel:
    name: host_of_host_fam_rel
    annotations:
      Expected_value:
        tag: Expected_value
        value: relationship type;arbitrary identifier
    description: Familial relationship of the host of the symbiotic host organisms
      to other hosts of symbiotic host organism in the same study; can include multiple
      relationships
    title: host of the symbiotic host family relationship
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - family
    - host
    - host.
    - relationship
    - symbiosis
    string_serialization: '{text};{text}'
    slot_uri: MIXS:0001328
    multivalued: true
    alias: host_of_host_fam_rel
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_infname:
    name: host_of_host_infname
    description: Taxonomic name information of the host of the symbiotic host organism
      below subspecies level
    title: host of the symbiotic host infra-specific name
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - symbiosis
    slot_uri: MIXS:0001329
    alias: host_of_host_infname
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_infrank:
    name: host_of_host_infrank
    description: Taxonomic rank information about the host of the symbiotic host organism
      below subspecies level, such as variety, form, rank etc
    title: host of the symbiotic host infra-specific rank
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - rank
    - symbiosis
    slot_uri: MIXS:0001330
    alias: host_of_host_infrank
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_geno:
    name: host_of_host_geno
    description: Observed genotype of the host of the symbiotic host organism
    title: host of the symbiotic host genotype
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - symbiosis
    slot_uri: MIXS:0001331
    alias: host_of_host_geno
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_pheno:
    name: host_of_host_pheno
    annotations:
      Expected_value:
        tag: Expected_value
        value: phenotype of the host of the symbiotic organism; PATO
    description: Phenotype of the host of the symbiotic host organism. For phenotypic
      quality ontology (PATO) terms, see http://purl.bioontology.org/ontology/pato
    title: host of the symbiotic host phenotype
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - symbiosis
    string_serialization: '{term}'
    slot_uri: MIXS:0001332
    alias: host_of_host_pheno
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_gravid:
    name: host_of_host_gravid
    annotations:
      Expected_value:
        tag: Expected_value
        value: gravidity status;timestamp
    description: Whether or not the host of the symbiotic host organism is gravid,
      and if yes date due or date post-conception, specifying which is used
    title: host of the symbiotic host gravidity
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - symbiosis
    string_serialization: '{boolean};{timestamp}'
    slot_uri: MIXS:0001333
    alias: host_of_host_gravid
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  host_of_host_totmass:
    name: host_of_host_totmass
    description: Total mass of the host of the symbiotic host organism at collection,
      the unit depends on the host
    title: host of the symbiotic host total mass
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - mass
    - symbiosis
    - total
    slot_uri: MIXS:0001334
    alias: host_of_host_totmass
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  chem_administration:
    name: chem_administration
    annotations:
      Expected_value:
        tag: Expected_value
        value: CHEBI;timestamp
    description: List of chemical compounds administered to the host or site where
      sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can
      include multiple compounds. For chemical entities of biological interest ontology
      (chebi) (v 163), http://purl.bioontology.org/ontology/chebi
    title: chemical administration
    examples:
    - value: agar [CHEBI:2509];2018-05-11T20:00Z
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - administration
    string_serialization: '{termLabel} [{termID}];{timestamp}'
    slot_uri: MIXS:0000751
    multivalued: true
    alias: chem_administration
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
  perturbation:
    name: perturbation
    annotations:
      Expected_value:
        tag: Expected_value
        value: perturbation type name;perturbation interval and duration
    description: Type of perturbation, e.g. chemical administration, physical disturbance,
      etc., coupled with perturbation regimen including how many times the perturbation
      was repeated, how long each perturbation lasted, and the start and end time
      of the entire perturbation period; can include multiple perturbation types
    title: perturbation
    examples:
    - value: antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - perturbation
    string_serialization: '{text};{Rn/start_time/end_time/duration}'
    slot_uri: MIXS:0000754
    multivalued: true
    alias: perturbation
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
  salinity:
    name: salinity
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: practical salinity unit, percentage
    description: The total concentration of all dissolved salts in a liquid or solid
      sample. While salinity can be measured by a complete chemical analysis, this
      method is difficult and time consuming. More often, it is instead derived from
      the conductivity measurement. This is known as practical salinity. These derivations
      compare the specific conductance of the sample to a salinity standard such as
      seawater
    title: salinity
    examples:
    - value: 25 practical salinity unit
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - salinity
    slot_uri: MIXS:0000183
    alias: salinity
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Air
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  oxy_stat_samp:
    name: oxy_stat_samp
    description: Oxygenation status of sample
    title: oxygenation status of sample
    examples:
    - value: aerobic
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - oxygen
    - sample
    - status
    slot_uri: MIXS:0000753
    alias: oxy_stat_samp
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - Air
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: OxyStatSampEnum
  organism_count:
    name: organism_count
    description: 'Total cell count of any organism (or group of organisms) per gram,
      volume or area of sample, should include name of organism followed by count.
      The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should
      also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)'
    title: organism count
    examples:
    - value: total prokaryotes;3.5e7 cells per milliliter;qPCR
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text};{float} {unit};[ATP|MPN|qPCR|other]'
    slot_uri: MIXS:0000103
    multivalued: true
    alias: organism_count
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
  samp_store_temp:
    name: samp_store_temp
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: degree Celsius
    description: Temperature at which sample was stored, e.g. -80 degree Celsius
    title: sample storage temperature
    examples:
    - value: -80 degree Celsius
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - sample
    - storage
    - temperature
    slot_uri: MIXS:0000110
    alias: samp_store_temp
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  samp_store_dur:
    name: samp_store_dur
    description: Duration for which the sample was stored. Indicate the duration for
      which the sample was stored written in ISO 8601 format
    title: sample storage duration
    examples:
    - value: P1Y6M
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - duration
    - period
    - sample
    - storage
    slot_uri: MIXS:0000116
    alias: samp_store_dur
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^P(?:(?:\d+D|\d+M(?:\d+D)?|\d+Y(?:\d+M(?:\d+D)?)?)(?:T(?:\d+H(?:\d+M(?:\d+S)?)?|\d+M(?:\d+S)?|\d+S))?|T(?:\d+H(?:\d+M(?:\d+S)?)?|\d+M(?:\d+S)?|\d+S)|\d+W)$
    structured_pattern:
      syntax: ^{duration}$
      interpolated: true
      partial_match: true
  samp_store_loc:
    name: samp_store_loc
    annotations:
      Expected_value:
        tag: Expected_value
        value: location name
    description: Location at which sample was stored, usually name of a specific freezer/room
    title: sample storage location
    examples:
    - value: Freezer no:5
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - location
    - sample
    - storage
    string_serialization: '{text}'
    slot_uri: MIXS:0000755
    alias: samp_store_loc
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
  samp_store_sol:
    name: samp_store_sol
    annotations:
      Expected_value:
        tag: Expected_value
        value: solution name
    description: Solution within which sample was stored, if any
    title: sample storage solution
    examples:
    - value: 5% ethanol
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - sample
    - storage
    string_serialization: '{text}'
    slot_uri: MIXS:0001317
    alias: samp_store_sol
    owner: MimarksSSymbiontAssociated
    domain_of:
    - SymbiontAssociated
    range: string
  misc_param:
    name: misc_param
    annotations:
      Expected_value:
        tag: Expected_value
        value: parameter name;measurement value
    description: Any other measurement performed or parameter collected, that is not
      listed here
    title: miscellaneous parameter
    examples:
    - value: Bicarbonate ion concentration;2075 micromole per kilogram
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - parameter
    string_serialization: '{text};{float} {unit}'
    slot_uri: MIXS:0000752
    multivalued: true
    alias: misc_param
    owner: MimarksSSymbiontAssociated
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
class_uri: MIXS:0010008_0016023

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link", "search.suggest"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
<script src="../assets/javascripts/bundle.ad660dcc.min.js"></script>
<script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
<script src="../javascripts/tablesort.js"></script>
<script src="https://unpkg.com/mermaid@8.8.0/dist/mermaid.min.js"></script><script>mermaid.initialize({});</script></body>
</html>