
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://GenomicsStandardsConsortium.github.io/mixs/MiuvigSoil/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.6.0, mkdocs-material-9.5.27" name="generator"/>
<title>Class: Miuvig combined with Soil (MiuvigSoil) - mixs</title>
<link href="../assets/stylesheets/main.6543a935.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-miuvig-combined-with-soil-miuvigsoil">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header md-header--shadow" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="mixs" class="md-header__button md-logo" data-md-component="logo" href=".." title="mixs">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            mixs
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: Miuvig combined with Soil (MiuvigSoil)
            
          </span>
</div>
</div>
</div>
<script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
<div class="md-search__suggest" data-md-component="search-suggest"></div>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="" tabindex="0">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list" role="presentation"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/GenomicsStandardsConsortium/mixs" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="mixs" class="md-nav__button md-logo" data-md-component="logo" href=".." title="mixs">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    mixs
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/GenomicsStandardsConsortium/mixs" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
<span class="md-ellipsis">
    Index
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../term_list/">
<span class="md-ellipsis">
    Full term table
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../combinations/">
<span class="md-ellipsis">
    Combinations
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../enumerations/">
<span class="md-ellipsis">
    Enumerations
  </span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#inheritance">
<span class="md-ellipsis">
      Inheritance
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
<span class="md-ellipsis">
      Slots
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#usages">
<span class="md-ellipsis">
      Usages
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
<span class="md-ellipsis">
      Identifier and Mapping Information
    </span>
</a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
<span class="md-ellipsis">
      Schema Source
    </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
<span class="md-ellipsis">
      Mappings
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
<span class="md-ellipsis">
      LinkML Source
    </span>
</a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
<span class="md-ellipsis">
      Direct
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
<span class="md-ellipsis">
      Induced
    </span>
</a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<h1 id="class-miuvig-combined-with-soil-miuvigsoil">Class: Miuvig combined with Soil (MiuvigSoil)</h1>
<p><em>MIxS Data that comply with the Miuvig checklist and the Soil Extension</em></p>
<p>URI: <a href="https://w3id.org/mixs/0010012_0016012">MIXS:0010012_0016012</a></p>
<div class="mermaid"> classDiagram
    class MiuvigSoil
    click MiuvigSoil href "../MiuvigSoil"
      Miuvig &lt;|-- MiuvigSoil
        click Miuvig href "../Miuvig"
      Soil &lt;|-- MiuvigSoil
        click Soil href "../Soil"

      MiuvigSoil : adapters

      MiuvigSoil : agrochem_addition

      MiuvigSoil : al_sat

      MiuvigSoil : al_sat_meth

      MiuvigSoil : alt

      MiuvigSoil : annot

      MiuvigSoil : annual_precpt

      MiuvigSoil : annual_temp

      MiuvigSoil : assembly_name

      MiuvigSoil : assembly_qual

      MiuvigSoil : assembly_software

      MiuvigSoil : associated_resource

      MiuvigSoil : bin_param




    MiuvigSoil --&gt; "0..1 _recommended_" BinParamEnum : bin_param
    click BinParamEnum href "../BinParamEnum"


      MiuvigSoil : bin_software

      MiuvigSoil : biotic_relationship




    MiuvigSoil --&gt; "0..1" BioticRelationshipEnum : biotic_relationship
    click BioticRelationshipEnum href "../BioticRelationshipEnum"


      MiuvigSoil : collection_date

      MiuvigSoil : compl_appr




    MiuvigSoil --&gt; "0..1 _recommended_" ComplApprEnum : compl_appr
    click ComplApprEnum href "../ComplApprEnum"


      MiuvigSoil : compl_score

      MiuvigSoil : compl_software

      MiuvigSoil : crop_rotation

      MiuvigSoil : cur_land_use

      MiuvigSoil : cur_vegetation

      MiuvigSoil : cur_vegetation_meth

      MiuvigSoil : depth

      MiuvigSoil : detec_type

      MiuvigSoil : drainage_class




    MiuvigSoil --&gt; "0..1" DrainageClassEnum : drainage_class
    click DrainageClassEnum href "../DrainageClassEnum"


      MiuvigSoil : elev

      MiuvigSoil : env_broad_scale

      MiuvigSoil : env_local_scale

      MiuvigSoil : env_medium

      MiuvigSoil : estimated_size

      MiuvigSoil : experimental_factor

      MiuvigSoil : extreme_event

      MiuvigSoil : fao_class




    MiuvigSoil --&gt; "0..1" FaoClassEnum : fao_class
    click FaoClassEnum href "../FaoClassEnum"


      MiuvigSoil : feat_pred

      MiuvigSoil : fire

      MiuvigSoil : flooding

      MiuvigSoil : geo_loc_name

      MiuvigSoil : heavy_metals

      MiuvigSoil : heavy_metals_meth

      MiuvigSoil : horizon_meth

      MiuvigSoil : host_disease_stat

      MiuvigSoil : host_pred_appr




    MiuvigSoil --&gt; "0..1 _recommended_" HostPredApprEnum : host_pred_appr
    click HostPredApprEnum href "../HostPredApprEnum"


      MiuvigSoil : host_pred_est_acc

      MiuvigSoil : host_spec_range

      MiuvigSoil : lat_lon

      MiuvigSoil : lib_layout




    MiuvigSoil --&gt; "0..1 _recommended_" LibLayoutEnum : lib_layout
    click LibLayoutEnum href "../LibLayoutEnum"


      MiuvigSoil : lib_reads_seqd

      MiuvigSoil : lib_screen

      MiuvigSoil : lib_size

      MiuvigSoil : lib_vector

      MiuvigSoil : link_addit_analys

      MiuvigSoil : link_class_info

      MiuvigSoil : link_climate_info

      MiuvigSoil : local_class

      MiuvigSoil : local_class_meth

      MiuvigSoil : mag_cov_software




    MiuvigSoil --&gt; "0..1" MagCovSoftwareEnum : mag_cov_software
    click MagCovSoftwareEnum href "../MagCovSoftwareEnum"


      MiuvigSoil : micro_biomass_meth

      MiuvigSoil : microbial_biomass

      MiuvigSoil : mid

      MiuvigSoil : misc_param

      MiuvigSoil : neg_cont_type




    MiuvigSoil --&gt; "0..1 _recommended_" NegContTypeEnum : neg_cont_type
    click NegContTypeEnum href "../NegContTypeEnum"


      MiuvigSoil : nucl_acid_amp

      MiuvigSoil : nucl_acid_ext

      MiuvigSoil : number_contig

      MiuvigSoil : org_matter

      MiuvigSoil : org_nitro

      MiuvigSoil : otu_class_appr

      MiuvigSoil : otu_db

      MiuvigSoil : otu_seq_comp_appr

      MiuvigSoil : pathogenicity

      MiuvigSoil : ph

      MiuvigSoil : ph_meth

      MiuvigSoil : pool_dna_extracts

      MiuvigSoil : pos_cont_type

      MiuvigSoil : pred_genome_struc




    MiuvigSoil --&gt; "1" PredGenomeStrucEnum : pred_genome_struc
    click PredGenomeStrucEnum href "../PredGenomeStrucEnum"


      MiuvigSoil : pred_genome_type

      MiuvigSoil : prev_land_use_meth

      MiuvigSoil : previous_land_use

      MiuvigSoil : profile_position




    MiuvigSoil --&gt; "0..1" ProfilePositionEnum : profile_position
    click ProfilePositionEnum href "../ProfilePositionEnum"


      MiuvigSoil : project_name

      MiuvigSoil : reassembly_bin

      MiuvigSoil : ref_biomaterial

      MiuvigSoil : ref_db

      MiuvigSoil : samp_collect_device

      MiuvigSoil : samp_collect_method

      MiuvigSoil : samp_mat_process

      MiuvigSoil : samp_name

      MiuvigSoil : samp_size

      MiuvigSoil : samp_taxon_id

      MiuvigSoil : samp_vol_we_dna_ext

      MiuvigSoil : sc_lysis_approach




    MiuvigSoil --&gt; "0..1 _recommended_" ScLysisApproachEnum : sc_lysis_approach
    click ScLysisApproachEnum href "../ScLysisApproachEnum"


      MiuvigSoil : sc_lysis_method

      MiuvigSoil : season_precpt

      MiuvigSoil : season_temp

      MiuvigSoil : seq_meth

      MiuvigSoil : sieving

      MiuvigSoil : sim_search_meth

      MiuvigSoil : size_frac

      MiuvigSoil : slope_aspect

      MiuvigSoil : slope_gradient

      MiuvigSoil : soil_horizon




    MiuvigSoil --&gt; "0..1" SoilHorizonEnum : soil_horizon
    click SoilHorizonEnum href "../SoilHorizonEnum"


      MiuvigSoil : soil_texture

      MiuvigSoil : soil_texture_meth

      MiuvigSoil : soil_type

      MiuvigSoil : soil_type_meth

      MiuvigSoil : sop

      MiuvigSoil : sort_tech




    MiuvigSoil --&gt; "0..1 _recommended_" SortTechEnum : sort_tech
    click SortTechEnum href "../SortTechEnum"


      MiuvigSoil : source_mat_id

      MiuvigSoil : source_uvig

      MiuvigSoil : specific_host

      MiuvigSoil : store_cond

      MiuvigSoil : tax_class

      MiuvigSoil : tax_ident




    MiuvigSoil --&gt; "0..1" TaxIdentEnum : tax_ident
    click TaxIdentEnum href "../TaxIdentEnum"


      MiuvigSoil : temp

      MiuvigSoil : tillage




    MiuvigSoil --&gt; "*" TillageEnum : tillage
    click TillageEnum href "../TillageEnum"


      MiuvigSoil : tot_nitro_cont_meth

      MiuvigSoil : tot_nitro_content

      MiuvigSoil : tot_org_c_meth

      MiuvigSoil : tot_org_carb

      MiuvigSoil : trna_ext_software

      MiuvigSoil : trnas

      MiuvigSoil : vir_ident_software

      MiuvigSoil : virus_enrich_appr




    MiuvigSoil --&gt; "1" VirusEnrichApprEnum : virus_enrich_appr
    click VirusEnrichApprEnum href "../VirusEnrichApprEnum"


      MiuvigSoil : water_cont_soil_meth

      MiuvigSoil : water_content

      MiuvigSoil : wga_amp_appr




    MiuvigSoil --&gt; "0..1 _recommended_" WgaAmpApprEnum : wga_amp_appr
    click WgaAmpApprEnum href "../WgaAmpApprEnum"


      MiuvigSoil : wga_amp_kit


</div>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Extension/">Extension</a><ul>
<li><a href="../Soil/">Soil</a><ul>
<li><strong>MiuvigSoil</strong> [ <a href="../Miuvig/">Miuvig</a>]</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../samp_name/">samp_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>A local identifier or name that for the material sample used for extracting n...</td>
<td><a href="../Miuvig/">Miuvig</a>, <a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../size_frac/">size_frac</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Filtering pore size used in sample preparation</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../lib_screen/">lib_screen</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Specific enrichment or screening methods applied before and/or after creating...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../source_uvig/">source_uvig</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Type of dataset from which the UViG was obtained</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../ref_db/">ref_db</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>List of database(s) used for ORF annotation, along with version number and re...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../nucl_acid_amp/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../lib_size/">lib_size</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones in the library prepared for the project</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../mid/">mid</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to spe...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../assembly_name/">assembly_name</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Name/version of the assembly provided by the submitter that is used in the ge...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../temp/">temp</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Temperature of the sample at the time of sampling</td>
<td><a href="../Miuvig/">Miuvig</a>, <a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../compl_score/">compl_score</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Completeness score is typically based on either the fraction of markers found...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../trnas/">trnas</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The total number of tRNAs identified from the SAG or MAG</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../nucl_acid_ext/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../samp_size/">samp_size</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample coll...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../alt/">alt</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Heights of objects such as airplanes, space shuttles, rockets, atmospheric ba...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../estimated_size/">estimated_size</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The estimated size of the genome prior to sequencing</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../source_mat_id/">source_mat_id</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A unique identifier assigned to a material sample (as defined by <a href="http://rs">http://rs</a></td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extractio...</td>
<td><a href="../Miuvig/">Miuvig</a>, <a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../pathogenicity/">pathogenicity</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>To what is the entity pathogenic</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../lib_reads_seqd/">lib_reads_seqd</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of clones sequenced from the library</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../samp_collect_device/">samp_collect_device</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The device used to collect an environmental sample</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../number_contig/">number_contig</a></td>
<td>1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a giv...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../biotic_relationship/">biotic_relationship</a></td>
<td>0..1 <br/> <a href="../BioticRelationshipEnum/">BioticRelationshipEnum</a></td>
<td>Description of relationship(s) between the subject organism and other organis...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../trna_ext_software/">trna_ext_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for tRNA identification</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../lib_layout/">lib_layout</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../LibLayoutEnum/">LibLayoutEnum</a></td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../assembly_qual/">assembly_qual</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The assembly quality category is based on sets of criteria outlined for each ...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../ref_biomaterial/">ref_biomaterial</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Primary publication if isolated before genome publication; otherwise, primary...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../project_name/">project_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Name of the project within which the sequencing was organized</td>
<td><a href="../Miuvig/">Miuvig</a>, <a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../lib_vector/">lib_vector</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../host_spec_range/">host_spec_range</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>The range and diversity of host species that an organism is capable of infect...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../neg_cont_type/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../NegContTypeEnum/">NegContTypeEnum</a></td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../virus_enrich_appr/">virus_enrich_appr</a></td>
<td>1 <br/> <a href="../VirusEnrichApprEnum/">VirusEnrichApprEnum</a></td>
<td>List of approaches used to enrich the sample for viruses, if any</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../adapters/">adapters</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Adapters provide priming sequences for both amplification and sequencing of t...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../assembly_software/">assembly_software</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../tax_ident/">tax_ident</a></td>
<td>0..1 <br/> <a href="../TaxIdentEnum/">TaxIdentEnum</a></td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../annot/">annot</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tool used for annotation, or for cases where annotation was provided by a com...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../pos_cont_type/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The substance, mixture, product, or apparatus used to verify that a process w...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../feat_pred/">feat_pred</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../compl_software/">compl_software</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Tools used for completion estimate, i</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the entity or entities which are in the sample or specimen s local vic...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../samp_mat_process/">samp_mat_process</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A brief description of any processing applied to the sample during or after r...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../sim_search_meth/">sim_search_meth</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../host_disease_stat/">host_disease_stat</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>List of diseases with which the host has been diagnosed; can include multiple...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../depth/">depth</a></td>
<td>1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The vertical distance below local surface</td>
<td><a href="../Miuvig/">Miuvig</a>, <a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../samp_collect_method/">samp_collect_method</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>The method employed for collecting the sample</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../compl_appr/">compl_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../ComplApprEnum/">ComplApprEnum</a></td>
<td>The approach used to determine the completeness of a given genomic assembly, ...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../specific_host/">specific_host</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../env_medium/">env_medium</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../samp_taxon_id/">samp_taxon_id</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>NCBI taxon id of the sample</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by the country or sea name f...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../collection_date/">collection_date</a></td>
<td>1 <br/> <a href="../Datetime/">Datetime</a></td>
<td>The time of sampling, either as an instance (single point in time) or interva...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../seq_meth/">seq_meth</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Sequencing machine used</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The geographical origin of the sample as defined by latitude and longitude</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../elev/">elev</a></td>
<td>1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Elevation of the sampling site is its height above a fixed reference point, m...</td>
<td><a href="../Miuvig/">Miuvig</a>, <a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../tax_class/">tax_class</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Method used for taxonomic classification, along with reference database used,...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../experimental_factor/">experimental_factor</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Variable aspects of an experiment design that can be used to describe an expe...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../sort_tech/">sort_tech</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../SortTechEnum/">SortTechEnum</a></td>
<td>Method used to sort/isolate cells or particles of interest</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../sc_lysis_approach/">sc_lysis_approach</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../ScLysisApproachEnum/">ScLysisApproachEnum</a></td>
<td>Method used to free DNA from interior of the cell(s) or particle(s)</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../sc_lysis_method/">sc_lysis_method</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Name of the kit or standard protocol used for cell(s) or particle(s) lysis</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../wga_amp_appr/">wga_amp_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../WgaAmpApprEnum/">WgaAmpApprEnum</a></td>
<td>Method used to amplify genomic DNA in preparation for sequencing</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../wga_amp_kit/">wga_amp_kit</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Kit used to amplify genomic DNA in preparation for sequencing</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../bin_param/">bin_param</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../BinParamEnum/">BinParamEnum</a></td>
<td>The parameters that have been applied during the extraction of genomes from m...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../bin_software/">bin_software</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for the extraction of genomes from metagenomic datasets, where p...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../reassembly_bin/">reassembly_bin</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../Boolean/">Boolean</a></td>
<td>Has an assembly been performed on a genome bin extracted from a metagenomic a...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../mag_cov_software/">mag_cov_software</a></td>
<td>0..1 <br/> <a href="../MagCovSoftwareEnum/">MagCovSoftwareEnum</a></td>
<td>Tool(s) used to determine the genome coverage if coverage is used as a binnin...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../vir_ident_software/">vir_ident_software</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Tool(s) used for the identification of UViG as a viral genome, software or pr...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../pred_genome_type/">pred_genome_type</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Type of genome predicted for the UViG</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../pred_genome_struc/">pred_genome_struc</a></td>
<td>1 <br/> <a href="../PredGenomeStrucEnum/">PredGenomeStrucEnum</a></td>
<td>Expected structure of the viral genome</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../detec_type/">detec_type</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Type of UViG detection</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../otu_class_appr/">otu_class_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Cutoffs and approach used when clustering  species-level  OTUs</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../otu_seq_comp_appr/">otu_seq_comp_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Tool and thresholds used to compare sequences when computing "species-level" ...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../otu_db/">otu_db</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Reference database (i</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../host_pred_appr/">host_pred_appr</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../HostPredApprEnum/">HostPredApprEnum</a></td>
<td>Tool or approach used for host prediction</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../host_pred_est_acc/">host_pred_est_acc</a></td>
<td>0..1 <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>For each tool or approach used for host prediction, estimated false discovery...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../associated_resource/">associated_resource</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>A related resource that is referenced, cited, or otherwise associated to the ...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../sop/">sop</a></td>
<td>* <em>recommended</em> <br/> <a href="../String/">String</a></td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, ...</td>
<td><a href="../Miuvig/">Miuvig</a></td>
</tr>
<tr>
<td><a href="../cur_land_use/">cur_land_use</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Present state of sample site</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../cur_vegetation/">cur_vegetation</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Vegetation classification from one or more standard classification systems, o...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../cur_vegetation_meth/">cur_vegetation_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in vegetation classification</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../previous_land_use/">previous_land_use</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Previous land use and dates</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../prev_land_use_meth/">prev_land_use_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining previous land use and dates</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../crop_rotation/">crop_rotation</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Whether or not crop is rotated, and if yes, rotation schedule</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../agrochem_addition/">agrochem_addition</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Addition of fertilizers, pesticides, etc</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../tillage/">tillage</a></td>
<td>* <br/> <a href="../TillageEnum/">TillageEnum</a></td>
<td>Note method(s) used for tilling</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../fire/">fire</a></td>
<td>0..1 <br/> <a href="../Datetime/">Datetime</a></td>
<td>Historical and/or physical evidence of fire</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../flooding/">flooding</a></td>
<td>0..1 <br/> <a href="../Datetime/">Datetime</a></td>
<td>Historical and/or physical evidence of flooding</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../extreme_event/">extreme_event</a></td>
<td>0..1 <br/> <a href="../Datetime/">Datetime</a></td>
<td>Unusual physical events that may have affected microbial populations</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../soil_horizon/">soil_horizon</a></td>
<td>0..1 <br/> <a href="../SoilHorizonEnum/">SoilHorizonEnum</a></td>
<td>Specific layer in the land area which measures parallel to the soil surface a...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../horizon_meth/">horizon_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining the horizon</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../sieving/">sieving</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Collection design of pooled samples and/or sieve size and amount of sample si...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../water_content/">water_content</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Water content measurement</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../water_cont_soil_meth/">water_cont_soil_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining the water content of soil</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../pool_dna_extracts/">pool_dna_extracts</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Indicate whether multiple DNA extractions were mixed</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../store_cond/">store_cond</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Explain how and for how long the soil sample was stored before DNA extraction...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../link_climate_info/">link_climate_info</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Link to climate resource</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../annual_temp/">annual_temp</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Mean annual temperature</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../season_temp/">season_temp</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Mean seasonal temperature</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../annual_precpt/">annual_precpt</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The average of all annual precipitation values known, or an estimated equival...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../season_precpt/">season_precpt</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The average of all seasonal precipitation values known, or an estimated equiv...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../link_class_info/">link_class_info</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Link to digitized soil maps or other soil classification information</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../fao_class/">fao_class</a></td>
<td>0..1 <br/> <a href="../FaoClassEnum/">FaoClassEnum</a></td>
<td>Soil classification from the FAO World Reference Database for Soil Resources</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../local_class/">local_class</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Soil classification based on local soil classification system</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../local_class_meth/">local_class_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining the local soil classification</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../org_nitro/">org_nitro</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Concentration of organic nitrogen</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../soil_type/">soil_type</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Description of the soil type or classification</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../soil_type_meth/">soil_type_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining soil series name or other lower-level...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../slope_gradient/">slope_gradient</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Commonly called 'slope'</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../slope_aspect/">slope_aspect</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The direction a slope faces</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../profile_position/">profile_position</a></td>
<td>0..1 <br/> <a href="../ProfilePositionEnum/">ProfilePositionEnum</a></td>
<td>Cross-sectional position in the hillslope where sample was collected</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../drainage_class/">drainage_class</a></td>
<td>0..1 <br/> <a href="../DrainageClassEnum/">DrainageClassEnum</a></td>
<td>Drainage classification from a standard system such as the USDA system</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../soil_texture/">soil_texture</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The relative proportion of different grain sizes of mineral particles in a so...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../soil_texture_meth/">soil_texture_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining soil texture</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../ph/">ph</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Ph measurement of the sample, or liquid portion of sample, or aqueous phase o...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../ph_meth/">ph_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining pH</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../org_matter/">org_matter</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Concentration of organic matter</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../tot_org_carb/">tot_org_carb</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Total organic carbon content</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../tot_org_c_meth/">tot_org_c_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining total organic carbon</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../tot_nitro_content/">tot_nitro_content</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Total nitrogen content of the sample</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../tot_nitro_cont_meth/">tot_nitro_cont_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining the total nitrogen</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../microbial_biomass/">microbial_biomass</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The part of the organic matter in the soil that constitutes living microorgan...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../micro_biomass_meth/">micro_biomass_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining microbial biomass</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../link_addit_analys/">link_addit_analys</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Link to additional analysis results performed on the sample</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../heavy_metals/">heavy_metals</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Heavy metals present in the sequenced sample and their concentrations</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../heavy_metals_meth/">heavy_metals_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining heavy metals</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../al_sat/">al_sat</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Aluminum saturation (esp</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../al_sat_meth/">al_sat_meth</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Reference or method used in determining Al saturation</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
<tr>
<td><a href="../misc_param/">misc_param</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>Any other measurement performed or parameter collected, that is not listed he...</td>
<td><a href="../Soil/">Soil</a></td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<table>
<thead>
<tr>
<th>used by</th>
<th>used in</th>
<th>type</th>
<th>used</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../MixsCompliantData/">MixsCompliantData</a></td>
<td><a href="../miuvig_soil_data/">miuvig_soil_data</a></td>
<td>range</td>
<td><a href="./">MiuvigSoil</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: <a href="https://w3id.org/mixs">https://w3id.org/mixs</a></li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>MIXS:0010012_0016012</td>
</tr>
<tr>
<td>native</td>
<td>MIXS:MiuvigSoil</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: MiuvigSoil
description: MIxS Data that comply with the Miuvig checklist and the Soil Extension
title: Miuvig combined with Soil
in_subset:
- combination_classes
from_schema: https://w3id.org/mixs
is_a: Soil
mixins:
- Miuvig
class_uri: MIXS:0010012_0016012

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: MiuvigSoil
description: MIxS Data that comply with the Miuvig checklist and the Soil Extension
title: Miuvig combined with Soil
in_subset:
- combination_classes
from_schema: https://w3id.org/mixs
is_a: Soil
mixins:
- Miuvig
attributes:
  samp_name:
    name: samp_name
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: ''
    description: A local identifier or name that for the material sample used for
      extracting nucleic acids, and subsequent sequencing. It can refer either to
      the original material collected or to any derived sub-samples. It can have any
      format, but we suggest that you make it concise, unique and consistent within
      your lab, and as informative as possible. INSDC requires every sample name from
      a single Submitter to be unique. Use of a globally unique identifier for the
      field source_mat_id is recommended in addition to sample_name
    title: sample name
    examples:
    - value: ISDsoil1
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - sample
    slot_uri: MIXS:0001107
    alias: samp_name
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
  size_frac:
    name: size_frac
    description: Filtering pore size used in sample preparation
    title: size fraction selected
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{float}-{float} {unit}'
    slot_uri: MIXS:0000017
    alias: size_frac
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  lib_screen:
    name: lib_screen
    description: Specific enrichment or screening methods applied before and/or after
      creating libraries
    title: library screening strategy
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000043
    alias: lib_screen
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  source_uvig:
    name: source_uvig
    description: Type of dataset from which the UViG was obtained
    title: source of UViGs
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '[metagenome (not viral targeted)|viral fraction metagenome
      (virome)|sequence-targeted metagenome|metatranscriptome (not viral targeted)|viral
      fraction RNA metagenome (RNA virome)|sequence-targeted RNA metagenome|microbial
      single amplified genome (SAG)|viral single amplified genome (vSAG)|isolate microbial
      genome|other]'
    slot_uri: MIXS:0000035
    alias: source_uvig
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: string
    required: true
  ref_db:
    name: ref_db
    description: List of database(s) used for ORF annotation, along with version number
      and reference to website or publication
    title: reference database(s)
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{database};{version};{reference}'
    slot_uri: MIXS:0000062
    alias: ref_db
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  nucl_acid_amp:
    name: nucl_acid_amp
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the enzymatic amplification (PCR,
      TMA, NASBA) of specific nucleic acids
    title: nucleic acid amplification
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000038
    alias: nucl_acid_amp
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  lib_size:
    name: lib_size
    description: Total number of clones in the library prepared for the project
    title: library size
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000039
    alias: lib_size
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: integer
    recommended: true
  mid:
    name: mid
    description: Molecular barcodes, called Multiplex Identifiers (MIDs), that are
      used to specifically tag unique samples in a sequencing run. Sequence should
      be reported in uppercase letters
    title: multiplex identifiers
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000047
    alias: mid
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
    pattern: ^[ACGTRKSYMWBHDVN]+$
    structured_pattern:
      syntax: ^{ambiguous_nucleotides}$
      interpolated: true
      partial_match: true
  assembly_name:
    name: assembly_name
    description: Name/version of the assembly provided by the submitter that is used
      in the genome browsers and in the community
    title: assembly name
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text} {text}'
    slot_uri: MIXS:0000057
    alias: assembly_name
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  temp:
    name: temp
    description: Temperature of the sample at the time of sampling
    title: temperature
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000113
    alias: temp
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  compl_score:
    name: compl_score
    description: 'Completeness score is typically based on either the fraction of
      markers found as compared to a database or the percent of a genome found as
      compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium
      Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated
      completeness scores'
    title: completeness score
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '[high|med|low];{percentage}'
    slot_uri: MIXS:0000069
    alias: compl_score
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    range: string
    recommended: true
  trnas:
    name: trnas
    annotations:
      Expected_value:
        tag: Expected_value
        value: value from 0-21
    description: The total number of tRNAs identified from the SAG or MAG
    title: number of standard tRNAs extracted
    examples:
    - value: '18'
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - number
    string_serialization: '{integer}'
    slot_uri: MIXS:0000067
    alias: trnas
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    range: string
  nucl_acid_ext:
    name: nucl_acid_ext
    description: A link to a literature reference, electronic resource or a standard
      operating procedure (SOP), that describes the material separation to recover
      the nucleic acid fraction from a sample
    title: nucleic acid extraction
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000037
    alias: nucl_acid_ext
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  samp_size:
    name: samp_size
    description: The total amount or size (volume (ml), mass (g) or area (m2) ) of
      sample collected
    title: amount or size of sample collected
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000001
    alias: samp_size
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  alt:
    name: alt
    description: Heights of objects such as airplanes, space shuttles, rockets, atmospheric
      balloons and heights of places such as atmospheric layers and clouds. It is
      used to measure the height of an object which is above the earth's surface.
      In this context, the altitude measurement is the vertical distance between the
      earth's surface above sea level and the sampled position in the air
    title: altitude
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000094
    alias: alt
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - HostAssociated
    - MiscellaneousNaturalOrArtificialEnvironment
    - SymbiontAssociated
    range: string
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  estimated_size:
    name: estimated_size
    annotations:
      Expected_value:
        tag: Expected_value
        value: number of base pairs
    description: The estimated size of the genome prior to sequencing. Of particular
      importance in the sequencing of (eukaryotic) genome which could remain in draft
      form for a long or unspecified period
    title: estimated size
    examples:
    - value: 300000 bp
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - size
    string_serialization: '{integer} bp'
    slot_uri: MIXS:0000024
    alias: estimated_size
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Miuvig
    range: string
  source_mat_id:
    name: source_mat_id
    description: A unique identifier assigned to a material sample (as defined by
      http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular
      digital record of a material sample) used for extracting nucleic acids, and
      subsequent sequencing. The identifier can refer either to the original material
      collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher,
      /bio_material, or /culture_collection may or may not share the same value as
      the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id
      may both contain 'UAM:Herps:14' , referring to both the specimen voucher and
      sampled tissue with the same identifier. However, the /culture_collection qualifier
      may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id
      would refer to an identifier from some derived culture from which the nucleic
      acids were extracted (e.g. xatc123 or ark:/2154/R2)
    title: source material identifiers
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000026
    multivalued: true
    alias: source_mat_id
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - SymbiontAssociated
    range: string
    recommended: true
  samp_vol_we_dna_ext:
    name: samp_vol_we_dna_ext
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: milliliter, gram, milligram, square centimeter
    description: 'Volume (ml) or mass (g) of total collected sample processed for
      DNA extraction. Note: total sample collected should be entered under the term
      Sample Size (MIXS:0000001)'
    title: sample volume or weight for DNA extraction
    examples:
    - value: 1500 milliliter
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - dna
    - sample
    - volume
    - weight
    slot_uri: MIXS:0000111
    alias: samp_vol_we_dna_ext
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  pathogenicity:
    name: pathogenicity
    annotations:
      Expected_value:
        tag: Expected_value
        value: names of organisms that the entity is pathogenic to
    description: To what is the entity pathogenic
    title: known pathogenicity
    examples:
    - value: human, animal, plant, fungi, bacteria
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000027
    alias: pathogenicity
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    range: string
  lib_reads_seqd:
    name: lib_reads_seqd
    description: Total number of clones sequenced from the library
    title: library reads sequenced
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000040
    alias: lib_reads_seqd
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: integer
    recommended: true
  samp_collect_device:
    name: samp_collect_device
    description: The device used to collect an environmental sample. This field accepts
      terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO).
      This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)
    title: sample collection device
    examples:
    - value: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000002
    alias: samp_collect_device
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
  number_contig:
    name: number_contig
    description: Total number of contigs in the cleaned/submitted assembly that makes
      up a given genome, SAG, MAG, or UViG
    title: number of contigs
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000060
    alias: number_contig
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: integer
    required: true
  biotic_relationship:
    name: biotic_relationship
    description: Description of relationship(s) between the subject organism and other
      organism(s) it is associated with. E.g., parasite on species X; mutualist with
      species Y. The target organism is the subject of the relationship, and the other
      organism(s) is the object
    title: observed biotic relationship
    examples:
    - value: free living
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - observed
    - relationship
    slot_uri: MIXS:0000028
    alias: biotic_relationship
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - MimarksC
    - Miuvig
    - Agriculture
    range: BioticRelationshipEnum
  trna_ext_software:
    name: trna_ext_software
    description: Tools used for tRNA identification
    title: tRNA extraction software
    examples:
    - value: infernal;v2;default parameters
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - software
    slot_uri: MIXS:0000068
    alias: trna_ext_software
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    range: string
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  lib_layout:
    name: lib_layout
    description: Specify whether to expect single, paired, or other configuration
      of reads
    title: library layout
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000041
    alias: lib_layout
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: LibLayoutEnum
    recommended: true
  assembly_qual:
    name: assembly_qual
    description: 'The assembly quality category is based on sets of criteria outlined
      for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated,
      contiguous sequence per replicon without gaps or ambiguities with a consensus
      error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments
      where gaps span repetitive regions. Presence of the large subunit (LSU) RNA,
      small subunit (SSU) and the presence of 5.8S rRNA or 5S rRNA depending on whether
      it is a eukaryotic or prokaryotic genome, respectively. Medium Quality Draft:Many
      fragments with little to no review of assembly other than reporting of standard
      assembly statistics. Low Quality Draft:Many fragments with little to no review
      of assembly other than reporting of standard assembly statistics. Assembly statistics
      include, but are not limited to total assembly size, number of contigs, contig
      N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated,
      contiguous sequence per replicon without gaps or ambiguities, with extensive
      manual review and editing to annotate putative gene functions and transcriptional
      units. High-quality draft genome: One or multiple fragments, totaling   90%
      of the expected genome or replicon sequence or predicted complete. Genome fragment(s):
      One or multiple fragments, totalling &lt; 90% of the expected genome or replicon
      sequence, or for which no genome size could be estimated'
    title: assembly quality
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '[Finished genome|High-quality draft genome|Medium-quality
      draft genome|Low-quality draft genome|Genome fragment(s)]'
    slot_uri: MIXS:0000056
    alias: assembly_qual
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    required: true
  ref_biomaterial:
    name: ref_biomaterial
    description: Primary publication if isolated before genome publication; otherwise,
      primary genome report
    title: reference for biomaterial
    examples:
    - value: doi:10.1016/j.syapm.2018.01.009
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000025
    alias: ref_biomaterial
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  project_name:
    name: project_name
    description: Name of the project within which the sequencing was organized
    title: project name
    examples:
    - value: Forest soil metagenome
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - project
    slot_uri: MIXS:0000092
    alias: project_name
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Air
    - BuiltEnvironment
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
  lib_vector:
    name: lib_vector
    description: Cloning vector type(s) used in construction of libraries
    title: library vector
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000042
    alias: lib_vector
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  host_spec_range:
    name: host_spec_range
    annotations:
      Expected_value:
        tag: Expected_value
        value: NCBI taxid
    description: The range and diversity of host species that an organism is capable
      of infecting, defined by NCBI taxonomy identifier
    title: host specificity or range
    examples:
    - value: '9606'
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    - range
    string_serialization: '{integer}'
    slot_uri: MIXS:0000030
    multivalued: true
    alias: host_spec_range
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsPl
    - MigsVi
    - Miuvig
    - Agriculture
    range: string
  neg_cont_type:
    name: neg_cont_type
    annotations:
      Expected_value:
        tag: Expected_value
        value: enumeration or text
    description: The substance or equipment used as a negative control in an investigation
    title: negative control type
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - type
    slot_uri: MIXS:0001321
    alias: neg_cont_type
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: NegContTypeEnum
    recommended: true
  virus_enrich_appr:
    name: virus_enrich_appr
    description: List of approaches used to enrich the sample for viruses, if any
    title: virus enrichment approach
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000036
    alias: virus_enrich_appr
    owner: MiuvigSoil
    domain_of:
    - MigsVi
    - Miuvig
    range: VirusEnrichApprEnum
    required: true
  adapters:
    name: adapters
    description: Adapters provide priming sequences for both amplification and sequencing
      of the sample-library fragments. Both adapters should be reported; in uppercase
      letters
    title: adapters
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{dna};{dna}'
    slot_uri: MIXS:0000048
    alias: adapters
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  assembly_software:
    name: assembly_software
    description: Tool(s) used for assembly, including version number and parameters
    title: assembly software
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000058
    alias: assembly_software
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  tax_ident:
    name: tax_ident
    description: The phylogenetic marker(s) used to assign an organism name to the
      SAG or MAG
    title: taxonomic identity marker
    examples:
    - value: other
      description: was other &lt;colon&gt; rpoB gene
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - identifier
    - marker
    - taxon
    slot_uri: MIXS:0000053
    alias: tax_ident
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    range: TaxIdentEnum
  annot:
    name: annot
    annotations:
      Expected_value:
        tag: Expected_value
        value: name of tool or pipeline used, or annotation source description
    description: Tool used for annotation, or for cases where annotation was provided
      by a community jamboree or model organism database rather than by a specific
      submitter
    title: annotation
    examples:
    - value: prokka
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000059
    alias: annot
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
  pos_cont_type:
    name: pos_cont_type
    description: The substance, mixture, product, or apparatus used to verify that
      a process which is part of an investigation delivers a true positive
    title: positive control type
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - type
    string_serialization: '{term} or {text}'
    slot_uri: MIXS:0001322
    alias: pos_cont_type
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  feat_pred:
    name: feat_pred
    description: Method used to predict UViGs features such as ORFs, integration site,
      etc
    title: feature prediction
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000061
    alias: feat_pred
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  compl_software:
    name: compl_software
    annotations:
      Expected_value:
        tag: Expected_value
        value: names and versions of software(s) used
    description: Tools used for completion estimate, i.e. checkm, anvi'o, busco
    title: completeness software
    examples:
    - value: checkm
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - software
    string_serialization: '{software};{version}'
    slot_uri: MIXS:0000070
    alias: compl_software
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Misag
    - Miuvig
    range: string
  env_local_scale:
    name: env_local_scale
    annotations:
      Expected_value:
        tag: Expected_value
        value: Environmental entities having causal influences upon the entity at
          time of sampling
    description: 'Report the entity or entities which are in the sample or specimen
      s local vicinity and which you believe have significant causal influences on
      your sample or specimen. We recommend using EnvO terms which are of smaller
      spatial grain than your entry for env_broad_scale. Terms, such as anatomical
      sites, from other OBO Library ontologies which interoperate with EnvO (e.g.
      UBERON) are accepted in this field. EnvO documentation about how to use the
      field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: local environmental context
    examples:
    - value: hillside [ENVO:01000333]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - context
    - environmental
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000013
    alias: env_local_scale
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
  samp_mat_process:
    name: samp_mat_process
    description: A brief description of any processing applied to the sample during
      or after retrieving the sample from environment, or a link to the relevant protocol(s)
      performed
    title: sample material processing
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000016
    alias: samp_mat_process
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  sim_search_meth:
    name: sim_search_meth
    description: Tool used to compare ORFs with database, along with version and cutoffs
      used
    title: similarity search method
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000063
    alias: sim_search_meth
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+);([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{software};{version};{parameters}$
      interpolated: true
      partial_match: true
  host_disease_stat:
    name: host_disease_stat
    description: List of diseases with which the host has been diagnosed; can include
      multiple diagnoses. The value of the field depends on host; for humans the terms
      should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org,
      non-human host diseases are free text
    title: host disease status
    examples:
    - value: rabies [DOID:11260]
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000031
    alias: host_disease_stat
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsVi
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - PlantAssociated
    range: string
    recommended: true
  depth:
    name: depth
    description: The vertical distance below local surface. For sediment or soil samples
      depth is measured from sediment or soil surface, respectively. Depth can be
      reported as an interval for subsurface samples
    title: depth
    examples:
    - value: 10 meter
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000018
    alias: depth
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodFarmEnvironment
    - HostAssociated
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
    required: true
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  samp_collect_method:
    name: samp_collect_method
    description: The method employed for collecting the sample
    title: sample collection method
    examples:
    - value: swabbing
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0001225
    alias: samp_collect_method
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$|([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}|{text}$
      interpolated: true
      partial_match: true
  compl_appr:
    name: compl_appr
    description: The approach used to determine the completeness of a given genomic
      assembly, which would typically make use of a set of conserved marker genes
      or a closely related reference genome. For UViG completeness, include reference
      genome or group used, and contig feature suggesting a complete genome
    title: completeness approach
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000071
    alias: compl_appr
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - Misag
    - Miuvig
    range: ComplApprEnum
    recommended: true
  specific_host:
    name: specific_host
    annotations:
      Expected_value:
        tag: Expected_value
        value: host scientific name, taxonomy ID
    description: Report the host's taxonomic name and/or NCBI taxonomy ID
    title: host scientific name
    examples:
    - value: Homo sapiens and/or 9606
    in_subset:
    - nucleic acid sequence source
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - host
    - host.
    string_serialization: '{text}|{NCBI taxid}'
    slot_uri: MIXS:0000029
    alias: specific_host
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsPl
    - MigsVi
    - Miuvig
    - Agriculture
    range: string
  env_medium:
    name: env_medium
    description: 'Report the environmental material(s) immediately surrounding the
      sample or specimen at the time of sampling. We recommend using subclasses of
      ''environmental material'' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO
      documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
      . Terms from other OBO ontologies are permissible as long as they reference
      mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
      (e.g. a tree, a leaf, a table top)'
    title: environmental medium
    examples:
    - value: bluegrass field soil [ENVO:00005789]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - environmental
    slot_uri: MIXS:0000014
    alias: env_medium
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
    structured_pattern:
      syntax: ^{termLabel} \[{termID}\]$
      interpolated: true
      partial_match: true
  samp_taxon_id:
    name: samp_taxon_id
    description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa
      sample. Use 'synthetic metagenome  for mock community/positive controls, or
      'blank sample' for negative controls
    title: taxonomy ID of DNA sample
    examples:
    - value: Gut Metagenome [NCBITaxon:749906]
    in_subset:
    - investigation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - dna
    - identifier
    - sample
    - taxon
    slot_uri: MIXS:0001320
    alias: samp_taxon_id
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[NCBITaxon:\d+\]$
    structured_pattern:
      syntax: ^{text} \[{NCBItaxon_id}\]$
      interpolated: true
      partial_match: true
  geo_loc_name:
    name: geo_loc_name
    description: The geographical origin of the sample as defined by the country or
      sea name followed by specific region name. Country or sea names should be chosen
      from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology
      (http://purl.bioontology.org/ontology/GAZ)
    title: geographic location (country and/or sea,region)
    examples:
    - value: 'USA: Maryland, Bethesda'
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - geographic
    - location
    slot_uri: MIXS:0000010
    alias: geo_loc_name
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: string
    required: true
    pattern: '^([^\s-]{1,2}|[^\s-]+.+[^\s-]+): ([^\s-]{1,2}|[^\s-]+.+[^\s-]+), ([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$'
    structured_pattern:
      syntax: '^{text}: {text}, {text}$'
      interpolated: true
      partial_match: true
  collection_date:
    name: collection_date
    description: 'The time of sampling, either as an instance (single point in time)
      or interval. In case no exact time is available, the date/time can be right
      truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10;
      2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant'
    title: collection date
    examples:
    - value: '2013-03-25T12:42:31+01:00'
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - date
    slot_uri: MIXS:0000011
    alias: collection_date
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: datetime
    required: true
  seq_meth:
    name: seq_meth
    description: Sequencing machine used. Where possible the term should be taken
      from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)
    title: sequencing method
    examples:
    - value: 454 Genome Sequencer FLX [OBI:0000702]
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - method
    slot_uri: MIXS:0000050
    alias: seq_meth
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+)|(([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\])$
    structured_pattern:
      syntax: ^{text}|({termLabel} \[{termID}\])$
      interpolated: true
      partial_match: true
  lat_lon:
    name: lat_lon
    annotations:
      Expected_value:
        tag: Expected_value
        value: decimal degrees,  limit to 8 decimal points
    description: The geographical origin of the sample as defined by latitude and
      longitude. The values should be reported in decimal degrees and in WGS84 system
    title: geographic location (latitude and longitude)
    examples:
    - value: 50.586825 6.408977
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - geographic
    - location
    string_serialization: '{float} {float}'
    slot_uri: MIXS:0000009
    alias: lat_lon
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - SymbiontAssociated
    range: string
    required: true
    pattern: ^(-?((?:[0-8]?[0-9](?:\.\d{0,8})?)|90)) -?[0-9]+(?:\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$
    structured_pattern:
      syntax: ^{lat} {lon}$
      interpolated: true
      partial_match: true
  elev:
    name: elev
    description: Elevation of the sampling site is its height above a fixed reference
      point, most commonly the mean sea level. Elevation is mainly used when referring
      to points on the earth's surface, while altitude is used for points above the
      surface, such as an aircraft in flight or a spacecraft in orbit
    title: elevation
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000093
    alias: elev
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    - Air
    - HostAssociated
    - HydrocarbonResourcesCores
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - Water
    range: string
    required: true
    recommended: true
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  env_broad_scale:
    name: env_broad_scale
    description: 'Report the major environmental system the sample or specimen came
      from. The system(s) identified should have a coarse spatial grain, to provide
      the general environmental context of where the sampling was done (e.g. in the
      desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428.
      EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS'
    title: broad-scale environmental context
    examples:
    - value: rangeland biome [ENVO:01000247]
    in_subset:
    - environment
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - context
    - environmental
    slot_uri: MIXS:0000012
    alias: env_broad_scale
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    range: string
    required: true
    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
    structured_pattern:
      syntax: ^{termLabel} \[{termID}\]$
      interpolated: true
      partial_match: true
  tax_class:
    name: tax_class
    description: Method used for taxonomic classification, along with reference database
      used, classification rank, and thresholds used to classify new genomes
    title: taxonomic classification
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000064
    alias: tax_class
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - Mims
    - Misag
    - Miuvig
    range: string
    recommended: true
  experimental_factor:
    name: experimental_factor
    description: Variable aspects of an experiment design that can be used to describe
      an experiment, or set of experiments, in an increasingly detailed manner. This
      field accepts ontology terms from Experimental Factor Ontology (EFO) and/or
      Ontology for Biomedical Investigations (OBI)
    title: experimental factor
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]|{text}'
    slot_uri: MIXS:0000008
    multivalued: true
    alias: experimental_factor
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    range: string
    recommended: true
    pattern: ^\S+.*\S+ \[[a-zA-Z]{2,}:\d+\]$
  sort_tech:
    name: sort_tech
    description: Method used to sort/isolate cells or particles of interest
    title: sorting technology
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000075
    alias: sort_tech
    owner: MiuvigSoil
    domain_of:
    - Misag
    - Miuvig
    range: SortTechEnum
    recommended: true
  sc_lysis_approach:
    name: sc_lysis_approach
    description: Method used to free DNA from interior of the cell(s) or particle(s)
    title: single cell or viral particle lysis approach
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000076
    alias: sc_lysis_approach
    owner: MiuvigSoil
    domain_of:
    - Misag
    - Miuvig
    range: ScLysisApproachEnum
    recommended: true
  sc_lysis_method:
    name: sc_lysis_method
    description: Name of the kit or standard protocol used for cell(s) or particle(s)
      lysis
    title: single cell or viral particle lysis kit protocol
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000054
    alias: sc_lysis_method
    owner: MiuvigSoil
    domain_of:
    - Misag
    - Miuvig
    range: string
    recommended: true
  wga_amp_appr:
    name: wga_amp_appr
    description: Method used to amplify genomic DNA in preparation for sequencing
    title: WGA amplification approach
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000055
    alias: wga_amp_appr
    owner: MiuvigSoil
    domain_of:
    - Misag
    - Miuvig
    range: WgaAmpApprEnum
    recommended: true
  wga_amp_kit:
    name: wga_amp_kit
    description: Kit used to amplify genomic DNA in preparation for sequencing
    title: WGA amplification kit
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text}'
    slot_uri: MIXS:0000006
    alias: wga_amp_kit
    owner: MiuvigSoil
    domain_of:
    - Misag
    - Miuvig
    range: string
    recommended: true
  bin_param:
    name: bin_param
    description: The parameters that have been applied during the extraction of genomes
      from metagenomic datasets
    title: binning parameters
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000077
    alias: bin_param
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - Miuvig
    range: BinParamEnum
    recommended: true
  bin_software:
    name: bin_software
    description: Tool(s) used for the extraction of genomes from metagenomic datasets,
      where possible include a product ID (PID) of the tool(s) used
    title: binning software
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{software};{version}{PID}'
    slot_uri: MIXS:0000078
    alias: bin_software
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - Miuvig
    range: string
    recommended: true
  reassembly_bin:
    name: reassembly_bin
    description: Has an assembly been performed on a genome bin extracted from a metagenomic
      assembly?
    title: reassembly post binning
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000079
    alias: reassembly_bin
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - Miuvig
    range: boolean
    recommended: true
  mag_cov_software:
    name: mag_cov_software
    description: Tool(s) used to determine the genome coverage if coverage is used
      as a binning parameter in the extraction of genomes from metagenomic datasets
    title: MAG coverage software
    examples:
    - value: bbmap
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - software
    slot_uri: MIXS:0000080
    alias: mag_cov_software
    owner: MiuvigSoil
    domain_of:
    - Mimag
    - Miuvig
    range: MagCovSoftwareEnum
  vir_ident_software:
    name: vir_ident_software
    description: Tool(s) used for the identification of UViG as a viral genome, software
      or protocol name including version number, parameters, and cutoffs used
    title: viral identification software
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000081
    alias: vir_ident_software
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: string
    required: true
  pred_genome_type:
    name: pred_genome_type
    description: Type of genome predicted for the UViG
    title: predicted genome type
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '[DNA|dsDNA|ssDNA|RNA|dsRNA|ssRNA|ssRNA (+)|ssRNA (-)|mixed|uncharacterized]'
    slot_uri: MIXS:0000082
    alias: pred_genome_type
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: string
    required: true
  pred_genome_struc:
    name: pred_genome_struc
    description: Expected structure of the viral genome
    title: predicted genome structure
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000083
    alias: pred_genome_struc
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: PredGenomeStrucEnum
    required: true
  detec_type:
    name: detec_type
    description: Type of UViG detection
    title: detection type
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '[independent sequence (UViG)|provirus (UpViG)]'
    slot_uri: MIXS:0000084
    alias: detec_type
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: string
    required: true
  otu_class_appr:
    name: otu_class_appr
    description: Cutoffs and approach used when clustering  species-level  OTUs. Note
      that results from standard 95% ANI / 85% AF clustering should be provided alongside
      OTUS defined from another set of thresholds, even if the latter are the ones
      primarily used during the analysis
    title: OTU classification approach
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{ANI cutoff};{AF cutoff};{clustering method}'
    slot_uri: MIXS:0000085
    alias: otu_class_appr
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: string
    recommended: true
  otu_seq_comp_appr:
    name: otu_seq_comp_appr
    description: Tool and thresholds used to compare sequences when computing "species-level"
      OTUs
    title: OTU sequence comparison approach
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{software};{version};{parameters}'
    slot_uri: MIXS:0000086
    alias: otu_seq_comp_appr
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: string
    recommended: true
  otu_db:
    name: otu_db
    description: Reference database (i.e. sequences not generated as part of the current
      study) used to cluster new genomes in "species-level" OTUs, if any
    title: OTU database
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{database};{version}'
    slot_uri: MIXS:0000087
    alias: otu_db
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: string
    recommended: true
  host_pred_appr:
    name: host_pred_appr
    description: Tool or approach used for host prediction
    title: host prediction approach
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000088
    alias: host_pred_appr
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: HostPredApprEnum
    recommended: true
  host_pred_est_acc:
    name: host_pred_est_acc
    description: For each tool or approach used for host prediction, estimated false
      discovery rates should be included, either computed de novo or from the literature
    title: host prediction estimated accuracy
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0000089
    alias: host_pred_est_acc
    owner: MiuvigSoil
    domain_of:
    - Miuvig
    range: string
    recommended: true
  associated_resource:
    name: associated_resource
    annotations:
      Expected_value:
        tag: Expected_value
        value: reference to resource
    description: A related resource that is referenced, cited, or otherwise associated
      to the sequence
    title: relevant electronic resources
    examples:
    - value: http://www.earthmicrobiome.org/
    in_subset:
    - sequencing
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - resource
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000091
    multivalued: true
    alias: associated_resource
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  sop:
    name: sop
    description: Standard operating procedures used in assembly and/or annotation
      of genomes, metagenomes or environmental sequences
    title: relevant standard operating procedures
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000090
    multivalued: true
    alias: sop
    owner: MiuvigSoil
    domain_of:
    - MigsBa
    - MigsEu
    - MigsOrg
    - MigsPl
    - MigsVi
    - Mimag
    - MimarksC
    - MimarksS
    - Mims
    - Misag
    - Miuvig
    - Agriculture
    range: string
    recommended: true
  cur_land_use:
    name: cur_land_use
    annotations:
      Expected_value:
        tag: Expected_value
        value: enumeration
    description: Present state of sample site
    title: current land use
    examples:
    - value: conifers
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - land
    - use
    string_serialization: '[cities|farmstead|industrial areas|roads/railroads|rock|sand|gravel|mudflats|salt
      flats|badlands|permanent snow or ice|saline seeps|mines/quarries|oil waste areas|small
      grains|row crops|vegetable crops|horticultural plants (e.g. tulips)|marshlands
      (grass,sedges,rushes)|tundra (mosses,lichens)|rangeland|pastureland (grasslands
      used for livestock grazing)|hayland|meadows (grasses,alfalfa,fescue,bromegrass,timothy)|shrub
      land (e.g. mesquite,sage-brush,creosote bush,shrub oak,eucalyptus)|successional
      shrub land (tree saplings,hazels,sumacs,chokecherry,shrub dogwoods,blackberries)|shrub
      crops (blueberries,nursery ornamentals,filberts)|vine crops (grapes)|conifers
      (e.g. pine,spruce,fir,cypress)|hardwoods (e.g. oak,hickory,elm,aspen)|intermixed
      hardwood and conifers|tropical (e.g. mangrove,palms)|rainforest (evergreen forest
      receiving &gt;406 cm annual rainfall)|swamp (permanent or semi-permanent water
      body dominated by woody plants)|crop trees (nuts,fruit,christmas trees,nursery
      trees)]'
    slot_uri: MIXS:0001080
    alias: cur_land_use
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
  cur_vegetation:
    name: cur_vegetation
    annotations:
      Expected_value:
        tag: Expected_value
        value: current vegetation type
    description: Vegetation classification from one or more standard classification
      systems, or agricultural crop
    title: current vegetation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - vegetation
    string_serialization: '{text}'
    slot_uri: MIXS:0000312
    alias: cur_vegetation
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
  cur_vegetation_meth:
    name: cur_vegetation_meth
    description: Reference or method used in vegetation classification
    title: current vegetation method
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - method
    - vegetation
    slot_uri: MIXS:0000314
    alias: cur_vegetation_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  previous_land_use:
    name: previous_land_use
    annotations:
      Expected_value:
        tag: Expected_value
        value: land use name;date
    description: Previous land use and dates
    title: history/previous land use
    examples:
    - value: fallow; 2018-05-11:T14:30Z
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - history
    - land
    - use
    string_serialization: '{text};{timestamp}'
    slot_uri: MIXS:0000315
    alias: previous_land_use
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
  prev_land_use_meth:
    name: prev_land_use_meth
    description: Reference or method used in determining previous land use and dates
    title: history/previous land use method
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - history
    - land
    - method
    - use
    slot_uri: MIXS:0000316
    alias: prev_land_use_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  crop_rotation:
    name: crop_rotation
    description: Whether or not crop is rotated, and if yes, rotation schedule
    title: history/crop rotation
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{boolean};{Rn/start_time/end_time/duration}'
    slot_uri: MIXS:0000318
    alias: crop_rotation
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
  agrochem_addition:
    name: agrochem_addition
    annotations:
      Expected_value:
        tag: Expected_value
        value: agrochemical name;agrochemical amount;timestamp
      Preferred_unit:
        tag: Preferred_unit
        value: gram, mole per liter, milligram per liter
    description: Addition of fertilizers, pesticides, etc. - amount and time of applications
    title: history/agrochemical additions
    examples:
    - value: roundup;5 milligram per liter;2018-06-21
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - history
    string_serialization: '{text};{float} {unit};{timestamp}'
    slot_uri: MIXS:0000639
    multivalued: true
    alias: agrochem_addition
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
  tillage:
    name: tillage
    description: Note method(s) used for tilling
    title: history/tillage
    examples:
    - value: chisel
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - history
    slot_uri: MIXS:0001081
    multivalued: true
    alias: tillage
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: TillageEnum
  fire:
    name: fire
    description: Historical and/or physical evidence of fire
    title: history/fire
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - history
    slot_uri: MIXS:0001086
    alias: fire
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: datetime
  flooding:
    name: flooding
    description: Historical and/or physical evidence of flooding
    title: history/flooding
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - history
    slot_uri: MIXS:0000319
    alias: flooding
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: datetime
  extreme_event:
    name: extreme_event
    description: Unusual physical events that may have affected microbial populations
    title: history/extreme events
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - event
    - history
    slot_uri: MIXS:0000320
    alias: extreme_event
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: datetime
  soil_horizon:
    name: soil_horizon
    description: Specific layer in the land area which measures parallel to the soil
      surface and possesses physical characteristics which differ from the layers
      above and beneath
    title: soil horizon
    examples:
    - value: A horizon
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - horizon
    - soil
    slot_uri: MIXS:0001082
    alias: soil_horizon
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: SoilHorizonEnum
  horizon_meth:
    name: horizon_meth
    description: Reference or method used in determining the horizon
    title: horizon method
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - horizon
    - method
    slot_uri: MIXS:0000321
    alias: horizon_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  sieving:
    name: sieving
    annotations:
      Expected_value:
        tag: Expected_value
        value: design name and/or size;amount
    description: Collection design of pooled samples and/or sieve size and amount
      of sample sieved
    title: sieving
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{text};{float} {unit}'
    slot_uri: MIXS:0000322
    alias: sieving
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
  water_content:
    name: water_content
    description: Water content measurement
    title: water content
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{float}'
    slot_uri: MIXS:0000185
    alias: water_content
    owner: MiuvigSoil
    domain_of:
    - MicrobialMatBiofilm
    - Sediment
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  water_cont_soil_meth:
    name: water_cont_soil_meth
    description: Reference or method used in determining the water content of soil
    title: water content method
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - content
    - method
    - water
    slot_uri: MIXS:0000323
    alias: water_cont_soil_meth
    owner: MiuvigSoil
    domain_of:
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  pool_dna_extracts:
    name: pool_dna_extracts
    description: Indicate whether multiple DNA extractions were mixed. If the answer
      yes, the number of extracts that were pooled should be given
    title: pooling of DNA extracts (if done)
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{boolean};{integer}'
    slot_uri: MIXS:0000325
    alias: pool_dna_extracts
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodAnimalAndAnimalFeed
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - Soil
    range: string
  store_cond:
    name: store_cond
    annotations:
      Expected_value:
        tag: Expected_value
        value: storage condition type;duration
    description: Explain how and for how long the soil sample was stored before DNA
      extraction (fresh/frozen/other)
    title: storage conditions
    examples:
    - value: -20 degree Celsius freezer;P2Y10D
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - condition
    - storage
    string_serialization: '{text};{period}'
    slot_uri: MIXS:0000327
    alias: store_cond
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
  link_climate_info:
    name: link_climate_info
    description: Link to climate resource
    title: link to climate information
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - information
    - link
    slot_uri: MIXS:0000328
    alias: link_climate_info
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  annual_temp:
    name: annual_temp
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: degree Celsius
    description: Mean annual temperature
    title: mean annual temperature
    examples:
    - value: 12.5 degree Celsius
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - mean
    - temperature
    slot_uri: MIXS:0000642
    alias: annual_temp
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  season_temp:
    name: season_temp
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: degree Celsius
    description: Mean seasonal temperature
    title: mean seasonal temperature
    examples:
    - value: 18 degree Celsius
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - mean
    - season
    - temperature
    slot_uri: MIXS:0000643
    alias: season_temp
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  annual_precpt:
    name: annual_precpt
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: millimeter
    description: The average of all annual precipitation values known, or an estimated
      equivalent value derived by such methods as regional indexes or Isohyetal maps
    title: mean annual precipitation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - mean
    slot_uri: MIXS:0000644
    alias: annual_precpt
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  season_precpt:
    name: season_precpt
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: millimeter
    description: The average of all seasonal precipitation values known, or an estimated
      equivalent value derived by such methods as regional indexes or Isohyetal maps
    title: mean seasonal precipitation
    examples:
    - value: 75 millimeters
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - mean
    - season
    slot_uri: MIXS:0000645
    alias: season_precpt
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  link_class_info:
    name: link_class_info
    annotations:
      Expected_value:
        tag: Expected_value
        value: PMID,DOI or url
    description: Link to digitized soil maps or other soil classification information
    title: link to classification information
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - classification
    - information
    - link
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000329
    alias: link_class_info
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
  fao_class:
    name: fao_class
    description: Soil classification from the FAO World Reference Database for Soil
      Resources. The list can be found at http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups
    title: soil_taxonomic/FAO classification
    examples:
    - value: Luvisols
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - classification
    slot_uri: MIXS:0001083
    alias: fao_class
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: FaoClassEnum
  local_class:
    name: local_class
    annotations:
      Expected_value:
        tag: Expected_value
        value: local classification name
    description: Soil classification based on local soil classification system
    title: soil_taxonomic/local classification
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - classification
    string_serialization: '{text}'
    slot_uri: MIXS:0000330
    alias: local_class
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
  local_class_meth:
    name: local_class_meth
    description: Reference or method used in determining the local soil classification
    title: soil_taxonomic/local classification method
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - classification
    - method
    slot_uri: MIXS:0000331
    alias: local_class_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  org_nitro:
    name: org_nitro
    description: Concentration of organic nitrogen
    title: organic nitrogen
    examples:
    - value: 4 micromole per liter
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - nitrogen
    - organic
    slot_uri: MIXS:0000205
    alias: org_nitro
    owner: MiuvigSoil
    domain_of:
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - Sediment
    - Soil
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  soil_type:
    name: soil_type
    description: Description of the soil type or classification. This field accepts
      terms under soil (http://purl.obolibrary.org/obo/ENVO_00001998).  Multiple terms
      can be separated by pipes
    title: soil type
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{termLabel} [{termID}]'
    slot_uri: MIXS:0000332
    alias: soil_type
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
  soil_type_meth:
    name: soil_type_meth
    description: Reference or method used in determining soil series name or other
      lower-level classification
    title: soil type method
    examples:
    - value: https://www.lrh.usace.army.mil/Portals/38/docs/PR/BluestoneSFEIS/Appendix%20K-Soil%20Descriptions.pdf
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - method
    - soil
    - type
    slot_uri: MIXS:0000334
    alias: soil_type_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  slope_gradient:
    name: slope_gradient
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: percentage
    description: Commonly called 'slope'. The angle between ground surface and a horizontal
      line (in percent). This is the direction that overland water would flow. This
      measure is usually taken with a hand level meter or clinometer
    title: slope gradient
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - slope
    slot_uri: MIXS:0000646
    alias: slope_gradient
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  slope_aspect:
    name: slope_aspect
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: degree
    description: The direction a slope faces. While looking down a slope use a compass
      to record the direction you are facing (direction or degrees); e.g., nw or 315
      degrees. This measure provides an indication of sun and wind exposure that will
      influence soil temperature and evapotranspiration
    title: slope aspect
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - slope
    slot_uri: MIXS:0000647
    alias: slope_aspect
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  profile_position:
    name: profile_position
    description: Cross-sectional position in the hillslope where sample was collected.sample
      area position in relation to surrounding areas
    title: profile position
    examples:
    - value: summit
    from_schema: https://w3id.org/mixs
    rank: 1000
    slot_uri: MIXS:0001084
    alias: profile_position
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: ProfilePositionEnum
  drainage_class:
    name: drainage_class
    description: Drainage classification from a standard system such as the USDA system
    title: drainage classification
    examples:
    - value: well
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - classification
    slot_uri: MIXS:0001085
    alias: drainage_class
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: DrainageClassEnum
  soil_texture:
    name: soil_texture
    description: The relative proportion of different grain sizes of mineral particles
      in a soil, as described using a standard system; express as % sand (50 um to
      2 mm), silt (2 um to 50 um), and clay (&lt;2 um) with textural name (e.g., silty
      clay loam) optional
    title: soil texture
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - soil
    - texture
    slot_uri: MIXS:0000335
    alias: soil_texture
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  soil_texture_meth:
    name: soil_texture_meth
    description: Reference or method used in determining soil texture
    title: soil texture method
    examples:
    - value: https://uwlab.soils.wisc.edu/wp-content/uploads/sites/17/2015/09/particle_size.pdf
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - method
    - soil
    - texture
    slot_uri: MIXS:0000336
    alias: soil_texture_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  ph:
    name: ph
    description: Ph measurement of the sample, or liquid portion of sample, or aqueous
      phase of the fluid
    title: pH
    examples:
    - value: '7.2'
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - ph
    slot_uri: MIXS:0001001
    alias: ph
    owner: MiuvigSoil
    domain_of:
    - FoodFarmEnvironment
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - Sediment
    - Soil
    - WastewaterSludge
    - Water
    range: float
  ph_meth:
    name: ph_meth
    description: Reference or method used in determining pH
    title: pH method
    examples:
    - value: https://www.epa.gov/sites/production/files/2015-12/documents/9040c.pdf
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - method
    - ph
    slot_uri: MIXS:0001106
    alias: ph_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  org_matter:
    name: org_matter
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: microgram per liter
    description: Concentration of organic matter
    title: organic matter
    examples:
    - value: 1.75 milligram per cubic meter
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - organic
    slot_uri: MIXS:0000204
    alias: org_matter
    owner: MiuvigSoil
    domain_of:
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - Sediment
    - Soil
    - Water
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  tot_org_carb:
    name: tot_org_carb
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: gram Carbon per kilogram sample material
    description: Total organic carbon content
    title: total organic carbon
    examples:
    - value: '0.02'
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - carbon
    - organic
    - total
    slot_uri: MIXS:0000533
    alias: tot_org_carb
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - MicrobialMatBiofilm
    - Sediment
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  tot_org_c_meth:
    name: tot_org_c_meth
    description: Reference or method used in determining total organic carbon
    title: total organic carbon method
    examples:
    - value: https://www.epa.gov/sites/production/files/2015-12/documents/9060a.pdf
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - carbon
    - method
    - organic
    - total
    slot_uri: MIXS:0000337
    alias: tot_org_c_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  tot_nitro_content:
    name: tot_nitro_content
    description: Total nitrogen content of the sample
    title: total nitrogen content
    examples:
    - value: 35 milligrams Nitrogen per kilogram of soil
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - content
    - nitrogen
    - total
    slot_uri: MIXS:0000530
    alias: tot_nitro_content
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - MicrobialMatBiofilm
    - Sediment
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  tot_nitro_cont_meth:
    name: tot_nitro_cont_meth
    description: Reference or method used in determining the total nitrogen
    title: total nitrogen content method
    examples:
    - value: https://currentprotocols.onlinelibrary.wiley.com/doi/abs/10.1002/0471142913.fab0102s00
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - content
    - method
    - nitrogen
    - total
    slot_uri: MIXS:0000338
    alias: tot_nitro_cont_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - FoodFarmEnvironment
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  microbial_biomass:
    name: microbial_biomass
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: ton, kilogram, gram per kilogram soil
    description: The part of the organic matter in the soil that constitutes living
      microorganisms smaller than 5-10 micrometer. If you keep this, you would need
      to have correction factors used for conversion to the final units
    title: microbial biomass
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - biomass
    - microbial
    slot_uri: MIXS:0000650
    alias: microbial_biomass
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  micro_biomass_meth:
    name: micro_biomass_meth
    description: Reference or method used in determining microbial biomass
    title: microbial biomass method
    comments:
    - slot name/scn was microbial_biomass_meth
    examples:
    - value: http://dx.doi.org/10.1016/j.soilbio.2005.01.021
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - biomass
    - method
    - microbial
    slot_uri: MIXS:0000339
    alias: micro_biomass_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  link_addit_analys:
    name: link_addit_analys
    description: Link to additional analysis results performed on the sample
    title: links to additional analysis
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - link
    slot_uri: MIXS:0000340
    alias: link_addit_analys
    owner: MiuvigSoil
    domain_of:
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  heavy_metals:
    name: heavy_metals
    annotations:
      Expected_value:
        tag: Expected_value
        value: heavy metal name;measurement value unit
      Preferred_unit:
        tag: Preferred_unit
        value: microgram per gram
    description: Heavy metals present in the sequenced sample and their concentrations.
      For multiple heavy metals and concentrations, add multiple copies of this field
    title: extreme_unusual_properties/heavy metals
    examples:
    - value: mercury;0.09 micrograms per gram
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - extreme
    - properties
    - unusual
    string_serialization: '{text};{float} {unit}'
    slot_uri: MIXS:0000652
    multivalued: true
    alias: heavy_metals
    owner: MiuvigSoil
    domain_of:
    - Soil
    range: string
  heavy_metals_meth:
    name: heavy_metals_meth
    description: Reference or method used in determining heavy metals
    title: extreme_unusual_properties/heavy metals method
    from_schema: https://w3id.org/mixs
    rank: 1000
    string_serialization: '{PMID}|{DOI}|{URL}'
    slot_uri: MIXS:0000343
    alias: heavy_metals_meth
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  al_sat:
    name: al_sat
    annotations:
      Preferred_unit:
        tag: Preferred_unit
        value: percentage
    description: Aluminum saturation (esp. For tropical soils)
    title: extreme_unusual_properties/Al saturation
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - extreme
    - properties
    - saturation
    - unusual
    slot_uri: MIXS:0000607
    alias: al_sat
    owner: MiuvigSoil
    domain_of:
    - Soil
    range: string
    pattern: ^[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?)?
      *([^\s-]{1,2}|[^\s-]+.+[^\s-]+)$
    structured_pattern:
      syntax: ^{scientific_float}( *- *{scientific_float})? *{text}$
      interpolated: true
      partial_match: true
  al_sat_meth:
    name: al_sat_meth
    description: Reference or method used in determining Al saturation
    title: extreme_unusual_properties/Al saturation method
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - extreme
    - method
    - properties
    - saturation
    - unusual
    slot_uri: MIXS:0000324
    alias: al_sat_meth
    owner: MiuvigSoil
    domain_of:
    - Soil
    range: string
    pattern: ^^PMID:\d+$|^doi:10.\d{2,9}/.*$|^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&amp;\/=]*)$$
    structured_pattern:
      syntax: ^{PMID}|{DOI}|{URL}$
      interpolated: true
      partial_match: true
  misc_param:
    name: misc_param
    annotations:
      Expected_value:
        tag: Expected_value
        value: parameter name;measurement value
    description: Any other measurement performed or parameter collected, that is not
      listed here
    title: miscellaneous parameter
    examples:
    - value: Bicarbonate ion concentration;2075 micromole per kilogram
    from_schema: https://w3id.org/mixs
    rank: 1000
    keywords:
    - parameter
    string_serialization: '{text};{float} {unit}'
    slot_uri: MIXS:0000752
    multivalued: true
    alias: misc_param
    owner: MiuvigSoil
    domain_of:
    - Agriculture
    - Air
    - FoodAnimalAndAnimalFeed
    - FoodFarmEnvironment
    - FoodFoodProductionFacility
    - FoodHumanFoods
    - HostAssociated
    - HumanAssociated
    - HumanGut
    - HumanOral
    - HumanSkin
    - HumanVaginal
    - HydrocarbonResourcesCores
    - HydrocarbonResourcesFluidsSwabs
    - MicrobialMatBiofilm
    - MiscellaneousNaturalOrArtificialEnvironment
    - PlantAssociated
    - Sediment
    - Soil
    - SymbiontAssociated
    - WastewaterSludge
    - Water
    range: string
class_uri: MIXS:0010012_0016012

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link", "search.suggest"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
<script src="../assets/javascripts/bundle.ad660dcc.min.js"></script>
<script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
<script src="../javascripts/tablesort.js"></script>
<script src="https://unpkg.com/mermaid@8.8.0/dist/mermaid.min.js"></script><script>mermaid.initialize({});</script></body>
</html>