<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Food-farmEnvironmentMIUVIG/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>Food-farmEnvironmentMIUVIG - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">About</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background.md" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation.md" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents.md" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Food-farmEnvironmentMIUVIG.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#food-farmenvironmentmiuvig" class="nav-link">Food-farmEnvironmentMIUVIG</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#inheritance" class="nav-link">Inheritance</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#slots" class="nav-link">Slots</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#usages" class="nav-link">Usages</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#identifier-and-mapping-information" class="nav-link">Identifier and Mapping Information</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="food-farmenvironmentmiuvig">Food-farmEnvironmentMIUVIG</h1>
<p>Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package food-farm environment</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Food-farmEnvironmentMIUVIG">mixs.vocab:Food-farmEnvironmentMIUVIG</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Food-farmEnvironment/">Food-farmEnvironment</a><ul>
<li><strong>Food-farmEnvironmentMIUVIG</strong> [ MIUVIG]</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
<th>Info</th>
</tr>
</thead>
<tbody>
<tr>
<td>MIXS:0001107</td>
<td><a href="../samp_name/">samp_name</a></td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001320</td>
<td><a href="../samp_taxon_id/">samp_taxon_id</a></td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>{text} [NCBI:txid]</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000092</td>
<td><a href="../project_name/">project_name</a></td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000008</td>
<td><a href="../experimental_factor/">experimental_factor</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000009</td>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000010</td>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000011</td>
<td><a href="../collection_date/">collection_date</a></td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001321</td>
<td><a href="../neg_cont_type/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001322</td>
<td><a href="../pos_cont_type/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>{term} or {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000012</td>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000013</td>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000014</td>
<td><a href="../env_medium/">env_medium</a></td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000024</td>
<td><a href="../estimated_size/">estimated_size</a></td>
<td>0..1</td>
<td>The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</td>
<td>{integer} bp</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000025</td>
<td><a href="../ref_biomaterial/">ref_biomaterial</a></td>
<td>0..1</td>
<td>Primary publication if isolated before genome publication; otherwise, primary genome report.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000026</td>
<td><a href="../source_mat_id/">source_mat_id</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000027</td>
<td><a href="../pathogenicity/">pathogenicity</a></td>
<td>0..1</td>
<td>To what is the entity pathogenic</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000028</td>
<td><a href="../biotic_relationship/">biotic_relationship</a></td>
<td>0..1</td>
<td>Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000029</td>
<td><a href="../specific_host/">specific_host</a></td>
<td>0..1</td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID.</td>
<td>{text}</td>
<td>{NCBI taxid}</td>
</tr>
<tr>
<td>MIXS:0000030</td>
<td><a href="../host_spec_range/">host_spec_range</a></td>
<td>0..1</td>
<td>The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000031</td>
<td><a href="../host_disease_stat/">host_disease_stat</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000002</td>
<td><a href="../samp_collec_device/">samp_collec_device</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0001225</td>
<td><a href="../samp_collec_method/">samp_collec_method</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000016</td>
<td><a href="../samp_mat_process/">samp_mat_process</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000017</td>
<td><a href="../size_frac/">size_frac</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Filtering pore size used in sample preparation</td>
<td>{float}-{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000001</td>
<td><a href="../samp_size/">samp_size</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000111</td>
<td><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000035</td>
<td><a href="../source_uvig/">source_uvig</a></td>
<td>1..1</td>
<td>Type of dataset from which the UViG was obtained</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000036</td>
<td><a href="../virus_enrich_appr/">virus_enrich_appr</a></td>
<td>1..1</td>
<td>List of approaches used to enrich the sample for viruses, if any</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000037</td>
<td><a href="../nucl_acid_ext/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000038</td>
<td><a href="../nucl_acid_amp/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000039</td>
<td><a href="../lib_size/">lib_size</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Total number of clones in the library prepared for the project</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000040</td>
<td><a href="../lib_reads_seqd/">lib_reads_seqd</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Total number of clones sequenced from the library</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000041</td>
<td><a href="../lib_layout/">lib_layout</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000042</td>
<td><a href="../lib_vector/">lib_vector</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000043</td>
<td><a href="../lib_screen/">lib_screen</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Specific enrichment or screening methods applied before and/or after creating libraries</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000047</td>
<td><a href="../mid/">mid</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters</td>
<td>{dna}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000048</td>
<td><a href="../adapters/">adapters</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</td>
<td>{dna};{dna}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000050</td>
<td><a href="../seq_meth/">seq_meth</a></td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000053</td>
<td><a href="../tax_ident/">tax_ident</a></td>
<td>0..1</td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000056</td>
<td><a href="../assembly_qual/">assembly_qual</a></td>
<td>1..1</td>
<td>The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000057</td>
<td><a href="../assembly_name/">assembly_name</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</td>
<td>{text} {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000058</td>
<td><a href="../assembly_software/">assembly_software</a></td>
<td>1..1</td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000059</td>
<td><a href="../annot/">annot</a></td>
<td>0..1</td>
<td>Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000060</td>
<td><a href="../number_contig/">number_contig</a></td>
<td>1..1</td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000061</td>
<td><a href="../feat_pred/">feat_pred</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc.</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000062</td>
<td><a href="../ref_db/">ref_db</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>List of database(s) used for ORF annotation, along with version number and reference to website or publication</td>
<td>{database};{version};{reference}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000063</td>
<td><a href="../sim_search_meth/">sim_search_meth</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000064</td>
<td><a href="../tax_class/">tax_class</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000067</td>
<td><a href="../trnas/">trnas</a></td>
<td>0..1</td>
<td>The total number of tRNAs identified from the SAG or MAG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000068</td>
<td><a href="../trna_ext_software/">trna_ext_software</a></td>
<td>0..1</td>
<td>Tools used for tRNA identification</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000069</td>
<td><a href="../compl_score/">compl_score</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000070</td>
<td><a href="../compl_software/">compl_software</a></td>
<td>0..1</td>
<td>Tools used for completion estimate, i.e. checkm, anvi'o, busco</td>
<td>{software};{version}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000071</td>
<td><a href="../compl_appr/">compl_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The approach used to determine the completeness of a given genomic assembly, which would typically make use of a set of conserved marker genes or a closely related reference genome. For UViG completeness, include reference genome or group used, and contig feature suggesting a complete genome</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000075</td>
<td><a href="../sort_tech/">sort_tech</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to sort/isolate cells or particles of interest</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000076</td>
<td><a href="../single_cell_lysis_appr/">single_cell_lysis_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to free DNA from interior of the cell(s) or particle(s)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000054</td>
<td><a href="../single_cell_lysis_prot/">single_cell_lysis_prot</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name of the kit or standard protocol used for cell(s) or particle(s) lysis</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000055</td>
<td><a href="../wga_amp_appr/">wga_amp_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to amplify genomic DNA in preparation for sequencing</td>
<td>[pcr based</td>
<td>mda based]</td>
</tr>
<tr>
<td>MIXS:0000006</td>
<td><a href="../wga_amp_kit/">wga_amp_kit</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Kit used to amplify genomic DNA in preparation for sequencing</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000077</td>
<td><a href="../bin_param/">bin_param</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The parameters that have been applied during the extraction of genomes from metagenomic datasets</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000078</td>
<td><a href="../bin_software/">bin_software</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool(s) used for the extraction of genomes from metagenomic datasets, where possible include a product ID (PID) of the tool(s) used.</td>
<td>{software};{version}{PID}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000079</td>
<td><a href="../reassembly_bin/">reassembly_bin</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Has an assembly been performed on a genome bin extracted from a metagenomic assembly?</td>
<td>{boolean}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000080</td>
<td><a href="../mag_cov_software/">mag_cov_software</a></td>
<td>0..1</td>
<td>Tool(s) used to determine the genome coverage if coverage is used as a binning parameter in the extraction of genomes from metagenomic datasets</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000081</td>
<td><a href="../vir_ident_software/">vir_ident_software</a></td>
<td>1..1</td>
<td>Tool(s) used for the identification of UViG as a viral genome, software or protocol name including version number, parameters, and cutoffs used</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000082</td>
<td><a href="../pred_genome_type/">pred_genome_type</a></td>
<td>1..1</td>
<td>Type of genome predicted for the UViG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000083</td>
<td><a href="../pred_genome_struc/">pred_genome_struc</a></td>
<td>1..1</td>
<td>Expected structure of the viral genome</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000084</td>
<td><a href="../detec_type/">detec_type</a></td>
<td>1..1</td>
<td>Type of UViG detection</td>
<td>[independent sequence (UViG)</td>
<td>provirus (UpViG)]</td>
</tr>
<tr>
<td>MIXS:0000085</td>
<td><a href="../otu_class_appr/">otu_class_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Cutoffs and approach used when clustering “species-level” OTUs. Note that results from standard 95% ANI / 85% AF clustering should be provided alongside OTUS defined from another set of thresholds, even if the latter are the ones primarily used during the analysis</td>
<td>{ANI cutoff};{AF cutoff};{clustering method}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000086</td>
<td><a href="../otu_seq_comp_appr/">otu_seq_comp_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool and thresholds used to compare sequences when computing "species-level" OTUs</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000087</td>
<td><a href="../otu_db/">otu_db</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Reference database (i.e. sequences not generated as part of the current study) used to cluster new genomes in "species-level" OTUs, if any</td>
<td>{database};{version}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000088</td>
<td><a href="../host_pred_appr/">host_pred_appr</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool or approach used for host prediction</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000089</td>
<td><a href="../host_pred_est_acc/">host_pred_est_acc</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>For each tool or approach used for host prediction, estimated false discovery rates should be included, either computed de novo or from the literature</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000091</td>
<td><a href="../associated_resource/">associated_resource</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000090</td>
<td><a href="../sop/">sop</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000018</td>
<td><a href="../depth/">depth</a></td>
<td>1..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000094</td>
<td><a href="../alt/">alt</a></td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000093</td>
<td><a href="../elev/">elev</a></td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000113</td>
<td><a href="../temp/">temp</a></td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000002</td>
<td><a href="../samp_collect_device/">samp_collect_device</a></td>
<td>0..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000100</td>
<td><a href="../humidity/">humidity</a></td>
<td>0..1</td>
<td>Amount of water vapour in the air, at the time of sampling</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000103</td>
<td><a href="../organism_count/">organism_count</a></td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001335</td>
<td><a href="../spikein_count/">spikein_count</a></td>
<td>0..1</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) should also be provided (example: total prokaryotes; 3.5e7 cells per ml; qPCR).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000110</td>
<td><a href="../samp_stor_temp/">samp_stor_temp</a></td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000112</td>
<td><a href="../solar_irradiance/">solar_irradiance</a></td>
<td>0..1</td>
<td>The amount of solar energy that arrives at a specific area of a surface during a specific time interval</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000114</td>
<td><a href="../ventilation_rate/">ventilation_rate</a></td>
<td>0..1</td>
<td>Ventilation rate of the system in the sampled premises</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000116</td>
<td><a href="../samp_stor_dur/">samp_stor_dur</a></td>
<td>0..1</td>
<td>Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format.</td>
<td>{duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000118</td>
<td><a href="../wind_speed/">wind_speed</a></td>
<td>0..1</td>
<td>Speed of wind measured at the time of sampling</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000183</td>
<td><a href="../salinity/">salinity</a></td>
<td>0..1</td>
<td>The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000315</td>
<td><a href="../previous_land_use/">previous_land_use</a></td>
<td>0..1</td>
<td>Previous land use and dates</td>
<td>{text};{timestamp}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000318</td>
<td><a href="../crop_rotation/">crop_rotation</a></td>
<td>0..1</td>
<td>Whether or not crop is rotated, and if yes, rotation schedule</td>
<td>{boolean};{Rn/start_time/end_time/duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000334</td>
<td><a href="../soil_type_meth/">soil_type_meth</a></td>
<td>0..1</td>
<td>Reference or method used in determining soil series name or other lower-level classification</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000337</td>
<td><a href="../tot_org_c_meth/">tot_org_c_meth</a></td>
<td>0..1</td>
<td>Reference or method used in determining total organic carbon</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000338</td>
<td><a href="../tot_nitro_cont_meth/">tot_nitro_cont_meth</a></td>
<td>0..1</td>
<td>Reference or method used in determining the total nitrogen</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000341</td>
<td><a href="../salinity_meth/">salinity_meth</a></td>
<td>0..1</td>
<td>Reference or method used in determining salinity</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000255</td>
<td><a href="../host_age/">host_age</a></td>
<td>0..1</td>
<td>Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000257</td>
<td><a href="../host_dry_mass/">host_dry_mass</a></td>
<td>0..1</td>
<td>Measurement of dry mass</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000264</td>
<td><a href="../host_height/">host_height</a></td>
<td>0..1</td>
<td>The height of subject</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000256</td>
<td><a href="../host_length/">host_length</a></td>
<td>0..1</td>
<td>The length of subject</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000263</td>
<td><a href="../host_tot_mass/">host_tot_mass</a></td>
<td>0..1</td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000577</td>
<td><a href="../root_med_carbon/">root_med_carbon</a></td>
<td>0..1</td>
<td>Source of organic carbon in the culture rooting medium; e.g. sucrose.</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000578</td>
<td><a href="../root_med_macronutr/">root_med_macronutr</a></td>
<td>0..1</td>
<td>Measurement of the culture rooting medium macronutrients (N,P, K, Ca, Mg, S); e.g. KH2PO4 (170¬†mg/L).</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000579</td>
<td><a href="../root_med_micronutr/">root_med_micronutr</a></td>
<td>0..1</td>
<td>Measurement of the culture rooting medium micronutrients (Fe, Mn, Zn, B, Cu, Mo); e.g. H3BO3 (6.2¬†mg/L).</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000643</td>
<td><a href="../season_temp/">season_temp</a></td>
<td>0..1</td>
<td>Mean seasonal temperature</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000645</td>
<td><a href="../season_precpt/">season_precpt</a></td>
<td>0..1</td>
<td>The average of all seasonal precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000533</td>
<td><a href="../tot_org_carb/">tot_org_carb</a></td>
<td>0..1</td>
<td>Definition for soil: total organic carbon content of the soil, definition otherwise: total organic carbon content</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000530</td>
<td><a href="../tot_nitro_content/">tot_nitro_content</a></td>
<td>0..1</td>
<td>Total nitrogen content of the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000692</td>
<td><a href="../conduc/">conduc</a></td>
<td>0..1</td>
<td>Electrical conductivity of water</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000191</td>
<td><a href="../turbidity/">turbidity</a></td>
<td>0..1</td>
<td>Measure of the amount of cloudiness or haziness in water caused by individual particles</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000735</td>
<td><a href="../size_frac_low/">size_frac_low</a></td>
<td>0..1</td>
<td>Refers to the mesh/pore size used to pre-filter/pre-sort the sample. Materials larger than the size threshold are excluded from the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000736</td>
<td><a href="../size_frac_up/">size_frac_up</a></td>
<td>0..1</td>
<td>Refers to the mesh/pore size used to retain the sample. Materials smaller than the size threshold are excluded from the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000756</td>
<td><a href="../ventilation_type/">ventilation_type</a></td>
<td>0..1</td>
<td>Ventilation system used in the sampled premises</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000757</td>
<td><a href="../wind_direction/">wind_direction</a></td>
<td>0..1</td>
<td>Wind direction is the direction from which a wind originates</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000859</td>
<td><a href="../genetic_mod/">genetic_mod</a></td>
<td>0..1</td>
<td>Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000874</td>
<td><a href="../host_phenotype/">host_phenotype</a></td>
<td>0..1</td>
<td>Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001001</td>
<td><a href="../ph/">ph</a></td>
<td>0..1</td>
<td>Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000247</td>
<td><a href="../ances_data/">ances_data</a></td>
<td>0..1</td>
<td>Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001038</td>
<td><a href="../biotic_regm/">biotic_regm</a></td>
<td>1..1</td>
<td>Information about treatment(s) involving use of biotic factors, such as bacteria, viruses or fungi.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000751</td>
<td><a href="../chem_administration/">chem_administration</a></td>
<td>1..*</td>
<td>List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</td>
<td>{termLabel} {[termID]};{timestamp}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001044</td>
<td><a href="../growth_habit/">growth_habit</a></td>
<td>0..1</td>
<td>Characteristic shape, appearance or growth form of a plant species</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000365</td>
<td><a href="../host_genotype/">host_genotype</a></td>
<td>0..1</td>
<td>Observed genotype</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000250</td>
<td><a href="../host_taxid/">host_taxid</a></td>
<td>0..1</td>
<td>NCBI taxon id of the host, e.g. 9606</td>
<td>{NCBI taxid}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001052</td>
<td><a href="../mechanical_damage/">mechanical_damage</a></td>
<td>0..*</td>
<td>Information about any mechanical damage exerted on the plant; can include multiple damages and sites</td>
<td>{text};{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000754</td>
<td><a href="../perturbation/">perturbation</a></td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001061</td>
<td><a href="../root_cond/">root_cond</a></td>
<td>0..1</td>
<td>Relevant rooting conditions such as field plot size, sowing density, container dimensions, number of plants per container.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0001062</td>
<td><a href="../root_med_ph/">root_med_ph</a></td>
<td>0..1</td>
<td>pH measurement of the culture rooting medium; e.g. 5.5.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001081</td>
<td><a href="../tillage/">tillage</a></td>
<td>0..*</td>
<td>Note method(s) used for tilling</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001106</td>
<td><a href="../ph_meth/">ph_meth</a></td>
<td>0..1</td>
<td>Reference or method used in determining ph</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0001108</td>
<td><a href="../growth_medium/">growth_medium</a></td>
<td>0..1</td>
<td>A liquid or gel containing nutrients, salts, and other factors formulated to support the growth of microorganisms, cells, or plants (National Cancer Institute Thesaurus).  The name of the medium used to grow the microorganism.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000829</td>
<td><a href="../season/">season</a></td>
<td>0..1</td>
<td>The season when sampling occurred. Any of the four periods into which the year is divided by the equinoxes and solstices. This field accepts terms listed under season (http://purl.obolibrary.org/obo/NCIT_C94729).</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001184</td>
<td><a href="../food_product_type/">food_product_type</a></td>
<td>1..*</td>
<td>A food product type is a class of food products that is differentiated by its food composition (e.g., single- or multi-ingredient), processing and/or consumption characteristics. This does not include brand name products but it may include generic food dish categories. This field accepts terms under food product type (http://purl.obolibrary.org/obo/FOODON:03400361). For terms related to food product for an animal, consult food product for animal (http://purl.obolibrary.org/obo/FOODON_03309997). If the proper descriptor is not listed please use text to describe the food type. Multiple terms can be separated by one or more pipes.</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0000998</td>
<td><a href="../samp_type/">samp_type</a></td>
<td>1..1</td>
<td>The type of material from which the sample was obtained. For the Hydrocarbon package, samples include types like core, rock trimmings, drill cuttings, piping section, coupon, pigging debris, solid deposit, produced fluid, produced water, injected water, swabs, etc. For the Food Package, samples are usually categorized as food, body products or tissues, or environmental material. This field accepts terms listed under environmental specimen (http://purl.obolibrary.org/obo/GENEPIO_0001246).</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001110</td>
<td><a href="../farm_water_source/">farm_water_source</a></td>
<td>0..*</td>
<td>Source of water used on the farm for irrigation of crops or watering of livestock.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001111</td>
<td><a href="../plant_water_method/">plant_water_method</a></td>
<td>0..1</td>
<td>Description of the equipment or method used to distribute water to crops. This field accepts termed listed under irrigation process (http://purl.obolibrary.org/obo/AGRO_00000006). Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000108</td>
<td><a href="../air_PM_concen/">air_PM_concen</a></td>
<td>0..*</td>
<td>Concentration of substances that remain suspended in the air, and comprise mixtures of organic and inorganic substances (PM10 and PM2.5); can report multiple PM's by entering numeric values preceded by name of PM</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001113</td>
<td><a href="../animal_feed_equip/">animal_feed_equip</a></td>
<td>0..*</td>
<td>Description of the feeding equipment used for livestock. This field accepts terms listed under feed delivery (http://opendata.inra.fr/EOL/EOL_0001757). Multiple terms can be separated by one or more pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001114</td>
<td><a href="../animal_intrusion/">animal_intrusion</a></td>
<td>0..*</td>
<td>Identification of animals intruding on the sample or sample site including invertebrates (such as pests or pollinators) and vertebrates (such as wildlife or domesticated animals). This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy. Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>{integer}</td>
</tr>
<tr>
<td>MIXS:0001115</td>
<td><a href="../anim_water_method/">anim_water_method</a></td>
<td>0..*</td>
<td>Description of the equipment or method used to distribute water to livestock. This field accepts termed listed under water delivery equipment (http://opendata.inra.fr/EOL/EOL_0001653). Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001116</td>
<td><a href="../crop_yield/">crop_yield</a></td>
<td>0..1</td>
<td>Amount of crop produced per unit or area of land.</td>
<td>{float}{unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001117</td>
<td><a href="../cult_result/">cult_result</a></td>
<td>0..1</td>
<td>Any result of a bacterial culture experiment reported as a binary assessment such as positive/negative, active/inactive.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001118</td>
<td><a href="../cult_result_org/">cult_result_org</a></td>
<td>0..*</td>
<td>Taxonomic information about the cultured organism(s).</td>
<td>{termLabel} {[termID]}</td>
<td>{integer}</td>
</tr>
<tr>
<td>MIXS:0001119</td>
<td><a href="../cult_target/">cult_target</a></td>
<td>0..*</td>
<td>The target microbial analyte in terms of investigation scope. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy.</td>
<td>{termLabel} {[termID]}</td>
<td>{integer}</td>
</tr>
<tr>
<td>MIXS:0001120</td>
<td><a href="../plant_part_maturity/">plant_part_maturity</a></td>
<td>0..1</td>
<td>A description of the stage of development of a plant or plant part based on maturity or ripeness. This field accepts terms listed under degree of plant maturity (http://purl.obolibrary.org/obo/FOODON_03530050).</td>
<td>{term label}{term ID}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001121</td>
<td><a href="../adjacent_environment/">adjacent_environment</a></td>
<td>0..*</td>
<td>Description of the environmental system or features that are adjacent to the sampling site. This field accepts terms under ecosystem (http://purl.obolibrary.org/obo/ENVO_01001110) and human construction (http://purl.obolibrary.org/obo/ENVO_00000070). Multiple terms can be separated by pipes.</td>
<td>{termLabel}{[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001122</td>
<td><a href="../water_source_adjac/">water_source_adjac</a></td>
<td>0..*</td>
<td>Description of the environmental features that are adjacent to the farm water source. This field accepts terms under ecosystem (http://purl.obolibrary.org/obo/ENVO_01001110) and human construction (http://purl.obolibrary.org/obo/ENVO_00000070). Multiple terms can be separated by pipes.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001123</td>
<td><a href="../farm_equip_shared/">farm_equip_shared</a></td>
<td>0..*</td>
<td>List of planting, growing or harvesting equipment shared with other farms. This field accepts terms listed under agricultural implement (http://purl.obolibrary.org/obo/AGRO_00000416). Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001124</td>
<td><a href="../farm_equip_san/">farm_equip_san</a></td>
<td>0..*</td>
<td>Method used to sanitize growing and harvesting equipment. This can including type and concentration of sanitizing solution.  Multiple terms can be separated by one or more pipes.</td>
<td>{text} {float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001125</td>
<td><a href="../farm_equip_san_freq/">farm_equip_san_freq</a></td>
<td>0..1</td>
<td>The number of times farm equipment is cleaned. Frequency of cleaning might be on a Daily basis, Weekly, Monthly, Quarterly or Annually.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001126</td>
<td><a href="../farm_equip/">farm_equip</a></td>
<td>0..*</td>
<td>List of equipment used for planting, fertilization, harvesting, irrigation, land levelling, residue management, weeding or transplanting during the growing season.  This field accepts terms listed under agricultural implement (http://purl.obolibrary.org/obo/AGRO_00000416). Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001127</td>
<td><a href="../fertilizer_admin/">fertilizer_admin</a></td>
<td>0..1</td>
<td>Type of fertilizer or amendment added to the soil or water for the purpose of improving substrate health and quality for plant growth. This field accepts terms listed under agronomic fertilizer (http://purl.obolibrary.org/obo/AGRO_00002062). Multiple terms may apply and can be separated by pipes, listing in reverse chronological order.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001128</td>
<td><a href="../fertilizer_date/">fertilizer_date</a></td>
<td>0..1</td>
<td>Date of administration of soil amendment or fertilizer. Multiple terms may apply and can be separated by pipes, listing in reverse chronological order.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001129</td>
<td><a href="../animal_group_size/">animal_group_size</a></td>
<td>0..1</td>
<td>The number of food animals of the same species that are maintained together as a unit, i.e. a herd or flock.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001130</td>
<td><a href="../animal_diet/">animal_diet</a></td>
<td>0..*</td>
<td>If the isolate is from a food animal, the type of diet eaten by the food animal.  Please list the main food staple and the setting, if appropriate.  For a list of acceptable animal feed terms or categories, please see http://www.feedipedia.org.  Multiple terms may apply and can be separated by pipes</td>
<td>Food product for animal covers foods intended for consumption by domesticated animals. Consult http://purl.obolibrary.org/obo/FOODON_03309997. If the proper descriptor is not listed please use text to describe the food type. Multiple terms can be separated by one or more pipes. If the proper descriptor is not listed please use text to describe the food product type.</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0001131</td>
<td><a href="../food_contact_surf/">food_contact_surf</a></td>
<td>0..*</td>
<td>The specific container or coating materials in direct contact with the food. Multiple values can be assigned.  This field accepts terms listed under food contact surface (http://purl.obolibrary.org/obo/FOODON_03500010).</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001132</td>
<td><a href="../food_contain_wrap/">food_contain_wrap</a></td>
<td>0..1</td>
<td>Type of container or wrapping defined by the main container material, the container form, and the material of the liner lids or ends. Also type of container or wrapping by form; prefer description by material first, then by form. This field accepts terms listed under food container or wrapping (http://purl.obolibrary.org/obo/FOODON_03490100).</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001133</td>
<td><a href="../food_harvest_proc/">food_harvest_proc</a></td>
<td>0..*</td>
<td>A harvesting process is a process which takes in some food material from an individual or community of plant or animal organisms in a given context and time, and outputs a precursor or consumable food product. This may include a part of an organism or the whole, and may involve killing the organism.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001134</td>
<td><a href="../food_pack_medium/">food_pack_medium</a></td>
<td>0..*</td>
<td>The medium in which the food is packed for preservation and handling or the medium surrounding homemade foods, e.g., peaches cooked in sugar syrup. The packing medium may provide a controlled environment for the food. It may also serve to improve palatability and consumer appeal.  This includes edible packing media (e.g. fruit juice), gas other than air (e.g. carbon dioxide), vacuum packed, or packed with aerosol propellant. This field accepts terms under food packing medium (http://purl.obolibrary.org/obo/FOODON_03480020). Multiple terms may apply and can be separated by pipes.</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001135</td>
<td><a href="../food_preserv_proc/">food_preserv_proc</a></td>
<td>0..*</td>
<td>The methods contributing to the prevention or retardation of microbial, enzymatic or oxidative spoilage and thus to the extension of shelf life. This field accepts terms listed under food preservation process (http://purl.obolibrary.org/obo/FOODON_03470107).</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001136</td>
<td><a href="../food_prod_char/">food_prod_char</a></td>
<td>0..*</td>
<td>Descriptors of the food production system such as wild caught, free-range, organic, free-range, industrial, dairy, beef.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>None</td>
<td><a href="../prod_label_claims/">prod_label_claims</a></td>
<td>0..*</td>
<td>Labeling claims containing descriptors such as wild caught, free-range, organic, free-range, industrial, hormone-free, antibiotic free, cage free. Can include more than one term, separated by ";"</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001137</td>
<td><a href="../food_trav_mode/">food_trav_mode</a></td>
<td>0..*</td>
<td>A descriptor for the method of movement of food commodity along the food distribution system.  This field accepts terms listed under travel mode (http://purl.obolibrary.org/obo/GENEPIO_0001064). If the proper descrptor is not listed please use text to describe the mode of travel. Multiple terms can be separated by one or more pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001138</td>
<td><a href="../food_trav_vehic/">food_trav_vehic</a></td>
<td>0..*</td>
<td>A descriptor for the mobile machine which is used to transport food commodities along the food distribution system.  This field accepts terms listed under vehicle (http://purl.obolibrary.org/obo/ENVO_01000604). If the proper descrptor is not listed please use text to describe the mode of travel. Multiple terms can be separated by one or more pipes.</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001139</td>
<td><a href="../food_source/">food_source</a></td>
<td>0..1</td>
<td>Type of plant or animal from which the food product or its major ingredient is derived or a chemical food source [FDA CFSAN 1995].</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001140</td>
<td><a href="../food_treat_proc/">food_treat_proc</a></td>
<td>0..*</td>
<td>Used to specifically characterize a food product based on the treatment or processes applied to the product or any indexed ingredient. The processes include adding, substituting or removing components or modifying the food or component, e.g., through fermentation. Multiple values can be assigned. This fields accepts terms listed under food treatment process (http://purl.obolibrary.org/obo/FOODON_03460111).</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001141</td>
<td><a href="../extr_weather_event/">extr_weather_event</a></td>
<td>0..*</td>
<td>Unusual weather events that may have affected microbial populations. Multiple terms can be separated by pipes, listed in reverse chronological order.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001142</td>
<td><a href="../date_extr_weath/">date_extr_weath</a></td>
<td>0..*</td>
<td>Date of unusual weather events that may have affected microbial populations. Multiple terms can be separated by pipes, listed in reverse chronological order.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001318</td>
<td><a href="../host_subspecf_genlin/">host_subspecf_genlin</a></td>
<td>0..*</td>
<td>Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>{rank name}:{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001144</td>
<td><a href="../intended_consumer/">intended_consumer</a></td>
<td>0..*</td>
<td>Food consumer type, human or animal, for which the food product is produced and marketed. This field accepts terms listed under food consumer group (http://purl.obolibrary.org/obo/FOODON_03510136) or NCBI taxid.</td>
<td>{integer}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001145</td>
<td><a href="../library_prep_kit/">library_prep_kit</a></td>
<td>0..1</td>
<td>Packaged kits (containing adapters, indexes, enzymes, buffers etc.), tailored for specific sequencing workflows, which allow the simplified preparation of sequencing-ready libraries for small genomes, amplicons, and plasmids.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001146</td>
<td><a href="../air_flow_impede/">air_flow_impede</a></td>
<td>0..*</td>
<td>Presence of objects in the area that would influence or impede air flow through the air filter.</td>
<td>[obstructed</td>
<td>unobstructed]; {text}; {measurement value}</td>
</tr>
<tr>
<td>MIXS:0001147</td>
<td><a href="../lot_number/">lot_number</a></td>
<td>0..*</td>
<td>A distinctive alpha-numeric identification code assigned by the manufacturer or distributor to a specific quantity of manufactured material or product within a batch. Synonym: Batch Number.  The submitter should provide lot number of the item followed by the item name for which the lot number was provided.</td>
<td>{integer}, {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001148</td>
<td><a href="../season_humidity/">season_humidity</a></td>
<td>0..1</td>
<td>Average humidity of the region throughout the growing season.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001149</td>
<td><a href="../part_plant_animal/">part_plant_animal</a></td>
<td>0..*</td>
<td>The anatomical part of the organism being involved in food production or consumption; e.g., a carrot is the root of the plant (root vegetable). This field accepts terms listed under part of plant or animal (http://purl.obolibrary.org/obo/FOODON_03420116).</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001057</td>
<td><a href="../plant_growth_med/">plant_growth_med</a></td>
<td>0..1</td>
<td>Specification of the media for growing the plants or tissue cultured samples, e.g. soil, aeroponic, hydroponic, in vitro solid culture medium, in vitro liquid culture medium. Recommended value is a specific value from EO:plant growth medium (follow this link for terms http://purl.obolibrary.org/obo/EO_0007147) or other controlled vocabulary</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001150</td>
<td><a href="../plant_reprod_crop/">plant_reprod_crop</a></td>
<td>0..*</td>
<td>Plant reproductive part used in the field during planting to start the crop.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001151</td>
<td><a href="../samp_purpose/">samp_purpose</a></td>
<td>0..1</td>
<td>The reason that the sample was collected.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001152</td>
<td><a href="../repository_name/">repository_name</a></td>
<td>0..*</td>
<td>The name of the institution where the sample or DNA extract is held or "sample not available" if the sample was used in its entirety for analysis or otherwise not retained.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001153</td>
<td><a href="../samp_pooling/">samp_pooling</a></td>
<td>0..*</td>
<td>Physical combination of several instances of like material, e.g. RNA extracted from samples or dishes of cell cultures into one big aliquot of cells. Please provide a short description of the samples that were pooled.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001154</td>
<td><a href="../samp_source_mat_cat/">samp_source_mat_cat</a></td>
<td>0..1</td>
<td>This is the scientific role or category that the subject organism or material has with respect to an investigation.  This field accepts terms listed under specimen source material category (http://purl.obolibrary.org/obo/GENEPIO_0001237 or http://purl.obolibrary.org/obo/OBI_0100051).</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001155</td>
<td><a href="../sequencing_kit/">sequencing_kit</a></td>
<td>0..1</td>
<td>Pre-filled, ready-to-use reagent cartridges. Used to produce improved chemistry, cluster density and read length as well as improve quality (Q) scores. Reagent components are encoded to interact with the sequencing system to validate compatibility with user-defined applications. Indicate name of the sequencing kit.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001156</td>
<td><a href="../sequencing_location/">sequencing_location</a></td>
<td>0..1</td>
<td>The location the sequencing run was performed. Indicate the name of the lab or core facility where samples were sequenced.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001157</td>
<td><a href="../serovar_or_serotype/">serovar_or_serotype</a></td>
<td>0..*</td>
<td>A characterization of a cell or microorganism based on the antigenic properties of the molecules on its surface. Indicate the name of a serovar or serotype of interest. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy.</td>
<td>{termLabel} {[termID]}</td>
<td>{integer}</td>
</tr>
<tr>
<td>MIXS:0001158</td>
<td><a href="../soil_conductivity/">soil_conductivity</a></td>
<td>0..1</td>
<td>Conductivity of soil at time of sampling.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001159</td>
<td><a href="../soil_cover/">soil_cover</a></td>
<td>0..1</td>
<td>Description of the material covering the sampled soil. This field accepts terms under ENVO:00010483, environmental material.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001160</td>
<td><a href="../soil_pH/">soil_pH</a></td>
<td>0..1</td>
<td>The pH of soil at time of sampling.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001161</td>
<td><a href="../rel_location/">rel_location</a></td>
<td>0..1</td>
<td>Location of sampled soil to other parts of the farm e.g. under crop plant, near irrigation ditch, from the dirt road.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001162</td>
<td><a href="../soil_porosity/">soil_porosity</a></td>
<td>0..1</td>
<td>Porosity of soil or deposited sediment is volume of voids divided by the total volume of sample.</td>
<td>{percentage}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001163</td>
<td><a href="../soil_temp/">soil_temp</a></td>
<td>0..*</td>
<td>Temperature of soil at the time of sampling.</td>
<td>{float}{unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001164</td>
<td><a href="../soil_texture_class/">soil_texture_class</a></td>
<td>0..1</td>
<td>One of the 12 soil texture classes use to describe soil texture based on the relative proportion of different grain sizes  of mineral particles [sand (50 um to 2 mm), silt (2 um to 50 um), and clay (&lt;2 um)] in a soil.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000336</td>
<td><a href="../soil_texture_meth/">soil_texture_meth</a></td>
<td>0..1</td>
<td>Reference or method used in determining soil texture</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000332</td>
<td><a href="../soil_type/">soil_type</a></td>
<td>0..1</td>
<td>Description of the soil type or classification. This field accepts terms under soil (http://purl.obolibrary.org/obo/ENVO_00001998).  Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001167</td>
<td><a href="../spikein_org/">spikein_org</a></td>
<td>0..*</td>
<td>Taxonomic information about the spike-in organism(s). This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy. Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>{integer}</td>
</tr>
<tr>
<td>MIXS:0001168</td>
<td><a href="../spikein_serovar/">spikein_serovar</a></td>
<td>0..*</td>
<td>Taxonomic information about the spike-in organism(s) at the serovar or serotype level. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy. Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>{integer}</td>
</tr>
<tr>
<td>MIXS:0001169</td>
<td><a href="../spikein_growth_med/">spikein_growth_med</a></td>
<td>0..*</td>
<td>A liquid or gel containing nutrients, salts, and other factors formulated to support the growth of microorganisms, cells, or plants (National Cancer Institute Thesaurus).  A growth medium is a culture medium which has the disposition to encourage growth of particular bacteria to the exclusion of others in the same growth environment.  In this case, list the culture medium used to propagate the spike-in bacteria during preparation of spike-in inoculum. This field accepts terms listed under microbiological culture medium (http://purl.obolibrary.org/obo/MICRO_0000067). If the proper descriptor is not listed please use text to describe the spike in growth media.</td>
<td>{text}</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td>MIXS:0001170</td>
<td><a href="../spikein_strain/">spikein_strain</a></td>
<td>0..*</td>
<td>Taxonomic information about the spike-in organism(s) at the strain level. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy. Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
<td>{integer}</td>
</tr>
<tr>
<td>MIXS:0001171</td>
<td><a href="../spikein_antibiotic/">spikein_antibiotic</a></td>
<td>0..*</td>
<td>Antimicrobials used in research study to assess effects of exposure on microbiome of a specific site.  Please list antimicrobial, common name and/or class and concentration used for spike-in.</td>
<td>{text} {integer}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001172</td>
<td><a href="../spikein_metal/">spikein_metal</a></td>
<td>0..*</td>
<td>Heavy metals used in research study to assess effects of exposure on microbiome of a specific site.  Please list heavy metals and concentration used for spike-in.</td>
<td>{text} {integer}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001173</td>
<td><a href="../timepoint/">timepoint</a></td>
<td>0..1</td>
<td>Time point at which a sample or observation is made or taken from a biomaterial as measured from some reference point. Indicate the timepoint written in ISO 8601 format.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001174</td>
<td><a href="../water_frequency/">water_frequency</a></td>
<td>0..1</td>
<td>Number of water delivery events within a given period of time.</td>
<td>{float}{unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001175</td>
<td><a href="../water_pH/">water_pH</a></td>
<td>0..1</td>
<td>pH measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001176</td>
<td><a href="../water_source_shared/">water_source_shared</a></td>
<td>0..*</td>
<td>Other users sharing access to the same water source. Multiple terms can be separated by one or more pipes.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001177</td>
<td><a href="../enrichment_protocol/">enrichment_protocol</a></td>
<td>0..1</td>
<td>The microbiological workflow or protocol followed to test for the presence or enumeration of the target microbial analyte(s). Please provide a PubMed or DOI reference for published protocols.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0001178</td>
<td><a href="../food_quality_date/">food_quality_date</a></td>
<td>0..1</td>
<td>The date recommended for the use of the product while at peak quality, this date is not a reflection of safety unless used on infant formula this date is not a reflection of safety and is typically labeled on a food product as "best if used by," best by," "use by," or "freeze by."</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001179</td>
<td><a href="../IFSAC_category/">IFSAC_category</a></td>
<td>1..*</td>
<td>The IFSAC food categorization scheme has five distinct levels to which foods can be assigned, depending upon the type of food. First, foods are assigned to one of four food groups (aquatic animals, land animals, plants, and other). Food groups include increasingly specific food categories; dairy, eggs, meat and poultry, and game are in the land animal food group, and the category meat and poultry is further subdivided into more specific categories of meat (beef, pork, other meat) and poultry (chicken, turkey, other poultry). Finally, foods are differentiated by differences in food processing (such as pasteurized fluid dairy products, unpasteurized fluid dairy products, pasteurized solid and semi-solid dairy products, and unpasteurized solid and semi-solid dairy products. An IFSAC food category chart is available from https://www.cdc.gov/foodsafety/ifsac/projects/food-categorization-scheme.html PMID: 28926300.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001180</td>
<td><a href="../animal_housing/">animal_housing</a></td>
<td>0..*</td>
<td>Description of the housing system of the livestock. This field accepts terms listed under terrestrial management housing system (http://opendata.inra.fr/EOL/EOL_0001605).</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001181</td>
<td><a href="../cult_isol_date/">cult_isol_date</a></td>
<td>0..1</td>
<td>A culture isolation date is a date-time entity marking the end of a process in which a sample yields a positive result for the target microbial analyte(s) in the form of an isolated colony or colonies.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001182</td>
<td><a href="../food_clean_proc/">food_clean_proc</a></td>
<td>0..*</td>
<td>The process of cleaning food to separate other environmental materials from the food source. Multiple terms can be separated by pipes.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000752</td>
<td><a href="../misc_param/">misc_param</a></td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
